/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        for (int __cfwr_i25 = 0; __cfwr_i25 < 1; __cfwr_i25++) {
            try {
            if ((('F' << null) + -512) || true) {
            try {
            float __cfwr_var61 = -18.79f;
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      private static boolean __cfwr_process841(long __cfwr_p0, Double __cfwr_p1, short __cfwr_p2) {
        try {
            return null;
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        return true;
    }
}
