/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        int __cfwr_val42 = 89;

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected Float __cfwr_aux295(Boolean __cfwr_p0, Long __cfwr_p1, Object __cfwr_p2) {
        if (true || false) {
            int __cfwr_temp42 = -587;
        }
        char __cfwr_result86 = '4';
        if (true || true) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 9; __cfwr_i37++) {
            while ((null / null)) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
}
