/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Issue3224_slice {
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        try {
            while (false) {
            try {
            if (true || true) {
            while (false) {
            if (((null >> null) - (386 - 96.36f)) 
        Float __cfwr_temp39 = null;
&& true) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 5; __cfwr_i8++) {
            try {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 5; __cfwr_i93++) {
            if (true && true) {
            long __cfwr_result18 = (('c' / 524L) - 'c');
        }
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.length);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
  }

    private Boolean __cfwr_calc252(short __cfwr_p0) {
        if (true || true) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 10; __cfwr_i15++) {
            while (true) {
            return true;
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
    public int __cfwr_func751(double __cfwr_p0) {
        try {
            byte __cfwr_result53 = null;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        try {
            try {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 1; __cfwr_i58++) {
            if (false && false) {
            if (false || false) {
            long __cfwr_var79 = ((-32.70 << true) * -751L);
        }
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        return -112;
    }
    private static char __cfwr_proc398(Boolean __cfwr_p0, double __cfwr_p1, Double __cfwr_p2) {
        if (true && false) {
            if (false && true) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 6; __cfwr_i36++) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 10; __cfwr_i66++) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 7; __cfwr_i89++) {
            if ((-510 ^ (-47.83f | false)) || (50.92 << -200L)) {
            while (true) {
            try {
            try {
            return "result24";
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
        }
        return "world50";
        Object __cfwr_temp9 = null;
        if (true || (null & -74.82)) {
            while ((-49.52 << null)) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 4; __cfwr_i74++) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 6; __cfwr_i64++) {
            try {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 3; __cfwr_i64++) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 2; __cfwr_i30++) {
            if ((-41.35f << 238L) || (true + ('R' ^ '5'))) {
            short __cfwr_elem13 = null;
        }
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        return (-705 >> -69.86f);
    }
}