/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        while ((null ^ 51.5
        try {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 9; __cfwr_i82++) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 9; __cfwr_i95++) {
            try {
            if (true || false) {
            if (true && false) {
            while (true) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 4; __cfwr_i40++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
1f)) {
            try {
            try {
            try {
            while (false) {
            try {
            char __cfwr_temp60 = 'Q';
        } catch (Exception __cfwr_e38) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      protected static String __cfwr_aux738(float __cfwr_p0) {
        for (int __cfwr_i31 = 0; __cfwr_i31 < 1; __cfwr_i31++) {
            try {
            while (false) {
            return (-49.74 >> (35.08f - true));
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }
        return "hello39";
    }
}
