/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Polymorphic3_slice {
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        for (int __cfwr_i77 = 0; __cfwr_i77 < 5; __cfwr_i77++) {
            double __cfwr_var84 = (-63.84f << null);
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLe
        return true;
ngthOf({"a", "b", "c"}) int abl2 = identity(abl);
  }

    public double __cfwr_helper937(boolean __cfwr_p0, Boolean __cfwr_p1) {
        try {
            if (false && (null * -820L)) {
            Object __cfwr_val36 = null;
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        Character __cfwr_var55 = null;
        Boolean __cfwr_elem44 = null;
        short __cfwr_val81 = (null >> (true >> 's'));
        return (753 % 'z');
    }
}