/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        try {
            if (true && false) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 5; __cfwr_i97++) {
            long __cfwr_var33 = -248L;
        }
        }
        } catch (Exception __cfwr_e37) {
  
        return null;
          // ignore
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      static boolean __cfwr_util942(long __cfwr_p0) {
        try {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 8; __cfwr_i30++) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 7; __cfwr_i30++) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 10; __cfwr_i90++) {
            return null;
        }
        }
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        Boolean __cfwr_node45 = null;
        return false;
    }
}
