/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testAppend(Appendable app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

  void testAppend(StringWriter app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
      public Float __cfwr_compute399(long __cfwr_p0, Double __cfwr_p1, char __cfwr_p2) {
        return (-5.62f | -38.22);

        byte __cfwr_result3 = null;
        try {
          
        Integer __cfwr_data29 = null;
  try {
            while (false) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 4; __cfwr_i38++) {
            float __cfwr_node26 = -67.78f;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        return null;
        return null;
    }
}
