/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        for (int __cfwr_i23 = 0; __cfwr_i23 < 4; __cfwr_i23++) {
            return null;
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      private Character __cfwr_func938(Object __cfwr_p0, Object __cfwr_p1, int __cfwr_p2) {
        return (('4' % null) * -535L);
        if (('j' << '7') || true) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 4; __cfwr_i67++) {
            return null;
        }
        }
        return null;
    }
    protected static boolean __cfwr_process650(Double __cfwr_p0, int __cfwr_p1, float __cfwr_p2) {
        while (false) {
            return 's';
            break; // Prevent infinite loops
        }
        return true;
    }
}
