/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        String __cfwr_obj9 = "hell
        if ((false / null) || false) {
            while (true) {
            while (('h' ^ -25.08f)) {
            return (-45.14f << 701L);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
o86";

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      public static Float __cfwr_temp726(short __cfwr_p0, Character __cfwr_p1, Object __cfwr_p2) {
        while (false) {
            int __cfwr_node66 = -513;
            break; // Prevent infinite loops
        }
        return null;
    }
    protected static double __cfwr_handle545() {
        return true;
        if ((730L / (null / 725)) || (-884 ^ (990 - 75.05))) {
            byte __cfwr_entry99 = null;
        }
        if ((false % ('l' | 1.06)) && true) {
            while (false) {
            if (false || false) {
            if (false && ((982L - false) << (-35.19f / null))) {
            if (true || (null << -61.93f)) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        if (false && (-12.75f % -99.71f)) {
            return null;
        }
        return ('G' % false);
    }
}
