/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IndexForVarargs_slice {
  void m() {
        if ((635 & 'f') && true) {
            try {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 7; __cfwr_i55++) {
            while ((-919 - 'e')) {
            if (true || ((612L * null) << 'w')) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 5; __cfwr_i66++) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 7; __cfwr_i25++) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 3; __cfwr_i31++) {
            if (true || true) {
            return (22.93 - (57.51f >> 76));
 
        for (int __cfwr_i1 = 0; __cfwr_i1 < 1; __cfwr_i1++) {
            long __cfwr_node67 = (null & 36.78f);
        }
       }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
  }

    private float __cfwr_compute84() {
        try {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            while ((44.67 << true)) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 7; __cfwr_i78++) {
            while (true) {
            if ((68.97 ^ (-579L / null)) || false) {
            Integer __cfwr_temp74 = null;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        int __cfwr_entry26 = -572;
        try {
            return (-88.25 | null);
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        return (1000 ^ -40.20f);
    }
    public static char __cfwr_compute194(Character __cfwr_p0, Float __cfwr_p1, Long __cfwr_p2) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 6; __cfwr_i55++) {
            int __cfwr_result82 = -162;
        }
        if (false && true) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 9; __cfwr_i49++) {
            return "test63";
        }
        }
        Long __cfwr_node28 = null;
        return 'L';
    }
}