/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        for (int __cfwr_i87 = 0; __cfwr_i87 < 4; __cfwr_i87++) {
            return (false & null);
        }
}

  void badParam2(
      // :: error: (different.length.sequences
        while (true) {
            while (true) {
            while ((('X' >> false) ^ 55.58f)) {
            while (((-242 ^ -16.28f) | 97.89)) {
            if (false || false) {
            if (true || (null - 16.91)) {
            try {
            try {
            Boolean __cfwr_obj11 = null;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    public static Boolean __cfwr_handle860() {
        try {
            float __cfwr_val88 = 9.87f;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        return null;
    }
    int __cfwr_calc12() {
        Character __cfwr_data27 = null;
        try {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 9; __cfwr_i80++) {
            double __cfwr_node89 = (null / null);
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        return -297;
    }
    private Double __cfwr_func900(Float __cfwr_p0, short __cfwr_p1, byte __cfwr_p2) {
        if (true || true) {
            float __cfwr_var70 = 73.11f;
        }
        return null;
    }
}
