/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        for (int __cfwr_i87 = 0; __cfwr_i87 < 6; __cfwr_i87++) {
            while (false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 3; __cfwr_i73++) {
            char __cfwr_entry95 = 'B';
        }
            break; // Prevent infinite loops
        }
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      private static int __cfwr_calc295(Object __cfwr_p0) {
        if (true && (37 * 94.46f)) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 2; __cfwr_i9++) {
            try {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 10; __cfwr_i81++) {
            short __cfwr_temp79 = null;
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        }
        return -265;
    }
    protected static Long __cfwr_handle349(long __cfwr_p0) {
        while ((null + null)) {
            return -123;
            break; // Prevent infinite loops
        }
        try {
            double __cfwr_node50 = -4.50;
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        return null;
        return null;
    }
    public static byte __cfwr_handle869(Object __cfwr_p0, Long __cfwr_p1) {
        int __cfwr_var63 = 553;
        if (true && (-223 & -734L)) {
            while (((-55.74 + -80.95) - (-905 + null))) {
            Float __cfwr_node33 = null;
            break; // Prevent infinite loops
        }
        }
        return (-550L - null);
    }
}
