/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        for (int __cfwr_i23 = 0; __cfwr_i23 < 6; __cfwr_i23++) {
            if (true && false) {
            char __cfwr_var65 = (-49.91f / -82.95);
        }
        }

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      public static byte __cfwr_process757(String __cfwr_p0, double __cfwr_p1) {
        if (true && false) {
            while (true) {
            return (null / 195L);
            break; // Prevent infinite loops
        }
        }
        return (null | (-717 >> 'A'));
    }
    public static byte __cfwr_calc720(char __cfwr_p0, Boolean __cfwr_p1, char __cfwr_p2) {
        return null;
        for (int __cfwr_i52 = 0; __cfwr_i52 < 9; __cfwr_i52++) {
            while (true) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 10; __cfwr_i2++) {
            return -11.11;
        }
            break; // Prevent infinite loops
        }
        }
        return null;
        return null;
    }
    static int __cfwr_aux195() {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 1; __cfwr_i21++) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 5; __cfwr_i74++) {
            if (true || true) {
            return 27.23f;
        }
        }
        }
        while (true) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 8; __cfwr_i95++) {
            return -61.39;
        }
        }
            break; // Prevent infinite loops
        }
        Double __cfwr_val64 = null;
        return -504;
    }
}
