/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      public int __cfwr_proc515(Double __cfwr_p0, long __cfwr_p1, Long __cfwr_p2) {
        if (true && true) {
 
        try {
            try {
            if ((null * null) || false) {
            if (true || (-972L % ('u' >> -864))) {
            double __cfwr_val84 = 17.92;
        }
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
           return ((null * 99.40f) + (null % 917));
        }

        if (false && false) {
            boolean __cfwr_entry13 = false;
        }
        return null;
        return (-4.80 << -499L);
        return (-88.09 << -5.04);
    }
}
