/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i73 = 0; __cfwr_i73 < 2; __cfwr_i73++) {
            if (true || false) {
            try {
            if ((false << (63.31 + 75.69f)) || false) {
            return 382L;
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
        }

    // The reason f
        try {
            while ((710L - -86.53f)) {
            try {
            float __cfwr_obj61 = -77.49f;
        } catch (Exception __cfwr_e6) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
or the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private static Integer __cfwr_func641(char __cfwr_p0, String __cfwr_p1) {
        if (true && ((-85.09f + 'N') << 407L)) {
            Long __cfwr_item19 = null;
        }
        return null;
    }
    public Integer __cfwr_helper852(int __cfwr_p0, Character __cfwr_p1, double __cfwr_p2) {
        while (false) {
            while (true) {
            return -980L;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        while (((-636L & -436L) & (true - true))) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 2; __cfwr_i29++) {
            while (((41.87 & 86.31) * (124L << 74.62))) {
            while (true) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 3; __cfwr_i82++) {
            while ((null / 'A')) {
            try {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 7; __cfwr_i42++) {
            try {
            return "data11";
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
