/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Issue2420_slice {
  static void str(String argStr) {
        while (true) {
    
        try {
            if (((9.81 / null) >> false) || true) {
            return -73.15f;
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        if (false || true) {
            try {
            Boolean __cfwr_temp46 = null;
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
  }

    public Character __cfwr_proc742() {
        try {
            if (true && false) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 3; __cfwr_i47++) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 5; __cfwr_i99++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 5; __cfwr_i7++) {
            return null;
        }
        }
        }
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        if ((null & '7') || false) {
            while (true) {
            while ((false % true)) {
            if (('v' - (391 * -50L)) && false) {
            try {
            try {
            try {
            try {
            if (true || false) {
            while ((null & (null ^ true))) {
            while (false) {
            short __cfwr_elem42 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
        return null;
    }
    private int __cfwr_calc955() {
        Double __cfwr_data51 = null;
        return ('w' / 464L);
        return ('d' / 't');
    }
}