/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        boolean __cfwr_temp89 = ((null + true) / (869 - null));

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int
        for (int __cfwr_i65 = 0; __cfwr_i65 < 5; __cfwr_i65++) {
            try {
            if (true || ('k' << 'X')) {
            char __cfwr_result32 = (54.47f / -235L);
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }
 a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      int __cfwr_calc919(long __cfwr_p0) {
        while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e4) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        boolean __cfwr_obj81 = true;
        while (true) {
            try {
            while (false) {
            try {
            if ((null ^ (-184 >> 'R')) && true) {
            while (false) {
            if ((null | -99.82) && true) {
            try {
            try {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 6; __cfwr_i59++) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            while ((393 / 1.55)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        short __cfwr_result44 = null;
        return -18;
    }
    boolean __cfwr_proc95(Boolean __cfwr_p0) {
        while (false) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 2; __cfwr_i18++) {
            boolean __cfwr_val69 = false;
        }
            break; // Prevent infinite loops
        }
        if ((null | (-391 & 617L)) || true) {
            if (false && false) {
            while (false) {
            while (false) {
            if (false && true) {
            return 20.24f;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        if (false || true) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 8; __cfwr_i86++) {
            if (false || ('L' / 'P')) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 1; __cfwr_i78++) {
            if (true && false) {
            try {
            while (false) {
            return -14.31;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }
        }
        }
        }
        }
        if (false && false) {
            try {
            double __cfwr_val98 = ((51.41 << null) / (892L / 62.78));
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        return true;
    }
}
