/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while (true) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 10; __cfwr_i19++) {
            try {
            if (true && (null % null)) {
            Float __cfwr_entry53 = null;
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      private static String __cfwr_aux381() {
        if (false && (53.78 ^ 939)) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 8; __cfwr_i13++) {
            while ((null + (true / null))) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        return "hello82";
    }
    public Double __cfwr_handle868(Float __cfwr_p0) {
        while (false) {
            try {
            while (false) {
            try {
            Character __cfwr_obj55 = null;
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            while ((false << 'a')) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 4; __cfwr_i70++) {
            if (true || false) {
            while (true) {
            while (true) {
            return 94.48f;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        return null;
    }
}
