/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        try {
         
        if (true || true) {
            while (false) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 5; __cfwr_i75++) {
            while (true) {
            return (44.20 << true);
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
   if (false && true) {
            return "hello9";
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      long __cfwr_helper414(Double __cfwr_p0) {
        if (true || false) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 1; __cfwr_i58++) {
            try {
            if (((99.57 * 488) >> '8') && false) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 4; __cfwr_i1++) {
            if (((null % null) + (-169L << -953L)) && true) {
            if ((16.39f ^ null) || true) {
            while ((false & -472L)) {
            while ((-490 + true)) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        }
        }
        double __cfwr_temp12 = 11.06;
        return -470L;
    }
}
