/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        try {
            return 802;
        } catch (Exception __cfwr_e99) {
            // ignore
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = a
        for (int __cfwr_i55 = 0; __cfwr_i55 < 2; __cfwr_i55++) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 8; __cfwr_i72++) {
            try {
            return null;
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        }
        }
rr[nn];
    int d = arr[p];
      protected static Boolean __cfwr_util245(boolean __cfwr_p0, String __cfwr_p1) {
        double __cfwr_temp52 = ((-993 >> -55.09f) & null);
        try {
            if (false || false) {
            if (false || (null >> (93.48f | -719))) {
            return "hello9";
        }
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        return null;
    }
    protected int __cfwr_compute431(Double __cfwr_p0) {
        try {
            if (true || (null ^ 35.21f)) {
            return null;
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        return null;
        return 238;
    }
    char __cfwr_handle511(char __cfwr_p0, boolean __cfwr_p1, Double __cfwr_p2) {
        for (int __cfwr_i31 = 0; __cfwr_i31 < 10; __cfwr_i31++) {
            if (true || (null | -56.99f)) {
            return (null & (null & -197));
        }
        }
        while (true) {
            Double __cfwr_var48 = null;
            break; // Prevent infinite loops
        }
        try {
            try {
            Object __cfwr_entry91 = null;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        return null;
        return '5';
    }
}
