/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        try {
         
        for (int __cfwr_i72 = 0; __cfwr_i72 < 5; __cfwr_i72++) {
            try {
            if (false || true) {
            try {
            while (('v' | ('M' - -34.68))) {
            try {
            while (false) {
            long __cfwr_var33 = 815L;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
   return null;
        } catch (Exception __cfwr_e52) {
            // ignore
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      static Long __cfwr_util956(Long __cfwr_p0) {
        while (false) {
            if (((null - -29.25f) - 45.72f) || (55.83 & 84.66f)) {
            while (false) {
            byte __cfwr_obj9 = (-948L / -636L);
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
