/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class MinMaxIndex_slice {
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        if (true && false) {
            try {
            char __cfwr_var62 = ('e' ^ 'U');
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(M
        try {
            while (true) {
            while (true) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 4; __cfwr_i67++) {
            if (false && false) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 4; __cfwr_i36++) {
            while (false) {
            return -57.57f;
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
ath.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
  }

    public char __cfwr_func467(byte __cfwr_p0, int __cfwr_p1, Integer __cfwr_p2) {
        for (int __cfwr_i52 = 0; __cfwr_i52 < 10; __cfwr_i52++) {
            return null;
        }
        return null;
        try {
            String __cfwr_result4 = "test57";
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        try {
            if (false && (52.70f * ('i' | -53.89))) {
            while (false) {
            try {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 5; __cfwr_i9++) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 4; __cfwr_i23++) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 5; __cfwr_i38++) {
            try {
            try {
            try {
            char __cfwr_result71 = 'S';
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        return ('b' - (null >> 0.55));
    }
}