/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class CharSequenceTest_slice {
  void testAppend(Appendable app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

  void testAppend(StringWriter app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

    protected char __cfwr_handle950(double __cfwr_p0) {
        Character __cfwr_temp17 = null;

        if (false && false) {
            while (true) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            if ((486 >> null) && (null % 952)) {
            while (true) {
            Boolean __cfwr_val2 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        try {
            try {
            if (('4' << (null - -55.40f)) && false) {
            while (false) {
            try {
            if ((-10.99 % -27L) && false) {
            return 782L;
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        return 'c';
    }
    private Float __cfwr_helper482(short __cfwr_p0, Double __cfwr_p1) {
        try {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 8; __cfwr_i66++) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 1; __cfwr_i2++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        for (int __cfwr_i74 = 0; __cfwr_i74 < 8; __cfwr_i74++) {
            try {
            try {
            while ((null / (-910L + true))) {
            return (-53.76 >> null);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        for (int __cfwr_i6 = 0; __cfwr_i6 < 1; __cfwr_i6++) {
            return '7';
        }
        try {
            String __cfwr_item77 = "world87";
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        return null;
    }
}