/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class NotEnoughOffsets_slice {
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        try {
            byte __cfwr_var80 = null;
        } catch (Exception __cfwr_e53) {
            // ignore
        }
}

  void badParam2(
      // :: error: (different.l
        return 627L;
ength.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {}

    public byte __cfwr_process498() {
        for (int __cfwr_i51 = 0; __cfwr_i51 < 4; __cfwr_i51++) {
            while ((-493L / 'a')) {
            float __cfwr_result22 = 79.33f;
            break; // Prevent infinite loops
        }
        }
        Object __cfwr_obj25 = null;
        return null;
        while ((64.75 >> true)) {
            if (true || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    public String __cfwr_helper791() {
        try {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 3; __cfwr_i2++) {
            byte __cfwr_var7 = null;
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        return "item76";
    }
}