/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  ArrayAssignmentSameLen(int[] array, @IndexFor("#1") int index) {
        Integer __cfwr_node54 = null;

    i_array = array;
    i_index = index;
  }

  void test1(int[] a, in
        if (true || (19.95 % 154L)) {
            try {
            return null;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
t[] b, @LTEqLengthOf("#1") int index) {
    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {
    int[] c1 = a;
    // See useTest3 for an example of why this assignment should fail.
    @LTLengthOf(
        value = {"c1", "c1"},
        offset = {"0", "x"})
    // :: error: (assignment)
    int z = i;
      static double __cfwr_calc898(boolean __cfwr_p0, Double __cfwr_p1, long __cfwr_p2) {
        if (((null - -43.42) ^ (null - 123)) && false) {
            while (true) {
            Float __cfwr_elem16 = null;
            break; // Prevent infinite loops
        }
        }
        return null;
        return 18.97;
    }
    protected byte __cfwr_process808(Long __cfwr_p0) {
        char __cfwr_item96 = '7';
        try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 8; __cfwr_i98++) {
            try {
            while (true) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 6; __cfwr_i51++) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 5; __cfwr_i41++) {
            if (true || (true ^ -17.36f)) {
            while (true) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 8; __cfwr_i69++) {
            byte __cfwr_entry42 = null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        if (true && (228L * -97.92f)) {
            try {
            while (true) {
            while (true) {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 6; __cfwr_i68++) {
            long __cfwr_val35 = -566L;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        }
        return null;
    }
    protected short __cfwr_calc269(Boolean __cfwr_p0, long __cfwr_p1) {
        try {
            long __cfwr_entry42 = -672L;
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        if (false && true) {
            return false;
        }
        return (950L >> 78.11);
    }
}
