/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            while ((-26.70 - null)) {
            if (false && false) {
            try {
            while ((826L << (941 >> null))) {
            float __cfwr_obj83 = -0.08f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e55) {
            // ignore
  
        while (true) {
            Double __cfwr_obj89 = null;
            break; // Prevent infinite loops
        }
      }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private Integer __cfwr_handle229(double __cfwr_p0, long __cfwr_p1) {
        Double __cfwr_item75 = null;
        try {
            while ((3.72 << (582L << null))) {
            while (false) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 8; __cfwr_i87++) {
            try {
            try {
            return "item37";
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        try {
            try {
            Integer __cfwr_item18 = null;
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        return null;
    }
    Float __cfwr_func530() {
        Float __cfwr_data64 = null;
        return null;
    }
}
