/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        return 71.04;

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static v
        try {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 8; __cfwr_i15++) {
            Long __cfwr_item27 = null;
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
oid m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      protected static int __cfwr_handle341(String __cfwr_p0, Character __cfwr_p1) {
        if (false && ((82.13f & null) * (-94.38 & 94.42f))) {
            if (false && true) {
            long __cfwr_result98 = 242L;
        }
        }
        return 404;
    }
    Long __cfwr_aux710(Long __cfwr_p0, char __cfwr_p1) {
        return null;
        byte __cfwr_val36 = null;
        return null;
    }
}
