/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i80 = 0; __cfwr_i80 < 10; __cfwr_i80++) {
            return -27.07;
        }

    // :: error:
        return "item60";
 (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      Float __cfwr_helper84() {
        try {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 10; __cfwr_i25++) {
            return -135L;
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        return null;
    }
    public Character __cfwr_func107(Double __cfwr_p0, int __cfwr_p1) {
        if (true && false) {
            return null;
        }
        return null;
    }
}
