/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineGT_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i89 = 0; __cfwr_i89 < 9; __cfwr_i89++) {
            try {
            double __cfwr_temp40 = 35.48;
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans ca
        if ((null << ('h' >> null)) || false) {
            while ((-745L / 34.67)) {
            String __cfwr_obj57 = "result9";
            break; // Prevent infinite loops
        }
        }
n.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

    public Boolean __cfwr_aux540(char __cfwr_p0) {
        try {
            try {
            try {
            while (true) {
            Long __cfwr_entry30 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        try {
            while (true) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 3; __cfwr_i47++) {
            return (false + null);
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        String __cfwr_entry80 = "item19";
        return null;
    }
    public static char __cfwr_temp653(long __cfwr_p0) {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 6; __cfwr_i36++) {
            char __cfwr_var50 = 'z';
        }
        while (false) {
            while (true) {
            while (true) {
            try {
            try {
            try {
            if ((('d' - -43) ^ 78.30f) || false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if (false && true) {
            if (false && false) {
            if (false && true) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        if (false || (null | 102L)) {
            try {
            while (true) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 1; __cfwr_i46++) {
            if (true && true) {
            return 't';
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        return 'M';
    }
}