/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        for (int __cfwr_i12 = 0; __cfwr_i12 < 3; __cfwr_i12++) {
            if (true || (true * ('m' ^ 299))) {
            byte __cfwr_result74 = ((null >> -40.48f) - 'j');
        }
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLeng
        return (true | 'A');
thOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      public static boolean __cfwr_func704(Object __cfwr_p0) {
        return 'M';
        while (true) {
            return "result46";
            break; // Prevent infinite loops
        }
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        return (null >> -987);
    }
}
