/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class PreAndPostDec_slice {
  void pre1(int[] args) {
        try {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 10; __cfwr_i74++) {
            try {
            while (false) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 1; __cfwr_i10++) {
            return "result60";
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
  }

    double __cfwr_aux174() {
        try {
            try {
            if (false && ('m' ^ (false >> true))) {
            return 47L;
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        try {
            Double __cfwr_temp33 = null;
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        return ((-14.14 / false) ^ (-497L / -269));
        try {
            float __cfwr_var77 = (null - 70.42);
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        return -30.94;
    }
}