/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ArrayAssignmentSameLen_slice {
// Source-based slice around line 19
// Method: ArrayAssignmentSameLen#test1(int[],int[],int)

    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
        long __cfwr_val94 = (-25L - null);

    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(
        if (true && true) {
            return null;
        }
int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {

    protected static byte __cfwr_calc600() {
        if ((93.38f & (false % false)) && true) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 7; __cfwr_i18++) {
            try {
            if (true || false) {
            try {
            if (false && ('B' ^ '9')) {
            while (true) {
            while ((false * (false << 372))) {
            Boolean __cfwr_node59 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
        }
        for (int __cfwr_i84 = 0; __cfwr_i84 < 2; __cfwr_i84++) {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 2; __cfwr_i77++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            return null;
        }
        }
        }
        long __cfwr_obj67 = (27.93 ^ (-589 % null));
        return ((null - -887) >> -386L);
    }
    char __cfwr_aux862(Boolean __cfwr_p0, float __cfwr_p1) {
        if (true && true) {
            if (true || false) {
            return -1.41f;
        }
        }
        return "world60";
        while (true) {
            Object __cfwr_elem16 = null;
            break; // Prevent infinite loops
        }
        Object __cfwr_val83 = null;
        return 'C';
    }
    private boolean __cfwr_compute904(String __cfwr_p0, int __cfwr_p1, char __cfwr_p2) {
        try {
            try {
            if (false && ('2' * 'r')) {
            return null;
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        while (false) {
            try {
            return '7';
        } catch (Exception __cfwr_e65) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "temp76";
        return false;
    }
}