/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        byte __cfwr_item43 = null;

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      static String __cfwr_aux642(Boolean __cfwr_p0) {
        if (true && false) {
            while (true) {
            while (true) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 6; __cfwr_i85++) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return "value86";
    }
    private byte __cfwr_calc441() {
        for (int __cfwr_i88 = 0; __cfwr_i88 < 2; __cfwr_i88++) {
            return null;
        }
        return null;
    }
    public static long __cfwr_aux403(Float __cfwr_p0) {
        if (false && true) {
            while (true) {
            try {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 7; __cfwr_i53++) {
            while (false) {
            if (('s' << 41.64) && false) {
            float __cfwr_node95 = (-77.88f ^ -82.71);
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        if (false || false) {
            while ((80.32 << 'j')) {
            try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 10; __cfwr_i13++) {
            return 10L;
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return (null * null);
    }
}
