/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        if (((null + 13.94) - true) || true) {
            if (true && true) {
            if (false || false) {
            if (false && true) {
            while (false) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 5; __cfwr_i84++) {
            try {
            while (true) {
            return (('N' + 'E') << 520);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
      private long __cfwr_util538(short __cfwr_p0, short __cfwr_p1) {
        for (int __cfwr_i24 = 0; __cfwr_i24 < 3; __cfwr_i24++) {
            return (314 << (56L & 973));
        }
        for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 1; __cfwr_i38++) {
            while (((null + '9') | 94.06)) {
            if (false && (99.25f ^ 101)) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 10; __cfwr_i13++) {
            try {
            Boolean __cfwr_data29 = null;
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        return -350L;
    }
    protected static short __cfwr_compute475(Double __cfwr_p0, Integer __cfwr_p1, Double __cfwr_p2) {
        Double __cfwr_val56 = null;
        Object __cfwr_data81 = null;
        for (int __cfwr_i77 = 0; __cfwr_i77 < 10; __cfwr_i77++) {
            if (true && (52.55f * null)) {
            if (true && true) {
            if (('F' ^ (null << null)) && true) {
            return null;
        }
        }
        }
        }
        return null;
    }
    protected static Long __cfwr_helper67(float __cfwr_p0) {
        for (int __cfwr_i80 = 0; __cfwr_i80 < 8; __cfwr_i80++) {
            return (true << true);
        }
        for (int __cfwr_i22 = 0; __cfwr_i22 < 7; __cfwr_i22++) {
            while (('A' >> -119)) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 5; __cfwr_i5++) {
            while (((739L >> 18.37) ^ null)) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 3; __cfwr_i31++) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 10; __cfwr_i69++) {
            try {
            if (((-820 << -27.60f) ^ 770L) && false) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 7; __cfwr_i15++) {
            while (false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 3; __cfwr_i73++) {
            while (false) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 10; __cfwr_i21++) {
            if (false && (-496L << ('t' + '0'))) {
            while (false) {
            while (true) {
            try {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 10; __cfwr_i70++) {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 6; __cfwr_i96++) {
            try {
            while (false) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 5; __cfwr_i15++) {
            try {
            Boolean __cfwr_result89 = null;
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        try {
            try {
            if (('2' | 366) || true) {
            return null;
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        try {
            try {
            while (false) {
            while (false) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 5; __cfwr_i86++) {
            if ((28.44 ^ -549L) || (null >> (8.61 << null))) {
            if (false || true) {
            if ((('9' - 32.16) | -537L) || ('I' << (-746L * -747))) {
            if (true && false) {
            Object __cfwr_val12 = null;
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        return null;
    }
}
