/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        while ((32.26 << true)) {
            Lo
        try {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 5; __cfwr_i34++) {
            double __cfwr_result38 = 22.48;
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
ng __cfwr_elem86 = null;
            break; // Prevent infinite loops
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      public static float __cfwr_calc42(short __cfwr_p0, byte __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i86 = 0; __cfwr_i86 < 3; __cfwr_i86++) {
            while (('A' & -52.61f)) {
            if ((-34.46 % (-423 << null)) && true) {
            try {
            try {
            return -8.50f;
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        return (-26.26 >> -254L);
    }
    protected static Object __cfwr_temp981(float __cfwr_p0) {
        return -87.01f;
        if (true && false) {
            while ((-80.55 | 143)) {
            Long __cfwr_entry41 = null;
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i78 = 0; __cfwr_i78 < 5; __cfwr_i78++) {
            try {
            try {
            while (true) {
            while (true) {
            try {
            try {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 4; __cfwr_i54++) {
            if ((true % (true ^ -91.88)) && true) {
            if (false && (null / -27.81f)) {
            if ((-0.25 | -22.67f) && true) {
            try {
            try {
            byte __cfwr_data94 = (null * null);
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e96) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        Double __cfwr_temp90 = null;
        return null;
    }
    static byte __cfwr_calc10(short __cfwr_p0, Character __cfwr_p1, Boolean __cfwr_p2) {
        if ((-814 | -709L) || false) {
            while (false) {
            char __cfwr_result1 = 'a';
            break; // Prevent infinite loops
        }
        }
        if (false || false) {
            return null;
        }
        while ((null >> null)) {
            try {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 6; __cfwr_i46++) {
            try {
            return 541L;
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            return null;
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        return (false - (-58.99f - -36.39));
    }
}
