/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        while ((-78L / null)) {
            while ((true ^ 460L)) {
            char __cfwr_val29 = 'a';
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      protected static Double __cfwr_temp182(int __cfwr_p0) {
        while (false) {
            short __cfwr_item13 = null;
            break; // Prevent infinite loops
        }
        return null;
    }
    private static Object __cfwr_proc972(Object __cfwr_p0, double __cfwr_p1, Float __cfwr_p2) {
        return null;
        while (((true - null) | (34.77 / -478L))) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 2; __cfwr_i64++) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 3; __cfwr_i42++) {
            byte __cfwr_var48 = null;
        }
        }
            break; // Prevent infinite loops
        }
        if (false || (-13 ^ null)) {
            while (true) {
            try {
            Integer __cfwr_result13 = null;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        try {
            while (((false + -139L) & 53.00)) {
            if (false || (19.28f << 'R')) {
            while (false) {
            char __cfwr_val86 = '1';
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        return null;
    }
}
