/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 9; __cfwr_i21++) {
            try {
            return '8';
        } cat
        try {
            if (false || true) {
            if (true || false) {
            try {
            while (false) {
            if (true || false) {
            if (false && true) {
            while ((82.66f & -99.87f)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
ch (Exception __cfwr_e7) {
            // ignore
        }
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      private char __cfwr_util836() {
        return (-558L + null);
        if (true || ((-2.12f << 'H') - true)) {
            if ((null - 'Z') || ((510 % 979) << -592)) {
            if (false || true) {
            while ((-7.63f | -279L)) {
            while (('9' / (93.74f % 273))) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        if (false || false) {
            return (-649 ^ -797L);
        }
        char __cfwr_var79 = 'F';
        return '4';
    }
    public static Double __cfwr_calc582() {
        while (((null % null) | -8.65f)) {
            Integer __cfwr_elem50 = null;
            break; // Prevent infinite loops
        }
        while (true) {
            double __cfwr_item87 = -95.63;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i26 = 0; __cfwr_i26 < 2; __cfwr_i26++) {
            try {
            long __cfwr_val79 = -671L;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        return null;
    }
    public static Object __cfwr_aux410(Object __cfwr_p0, Boolean __cfwr_p1) {
        if (false && false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 6; __cfwr_i73++) {
            while (false) {
            String __cfwr_var66 = "temp58";
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
}
