/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static String nocheck(String l, String s) {
        if (false && false) {
            try {
            try {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e66) {
            // ignore
    
        if (true || true) {
            if (true && true) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 4; __cfwr_i29++) {
            byte __cfwr_var2 = ((false >> 'M') ^ (-58.71f & null));
        }
        }
        }
    }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        }

    int i = l.indexOf(s);
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + s.length());
  }

  public static String remove(String l, String s, int from, boolean last) {
    int i = last ? l.lastIndexOf(s, from) : l.indexOf(s, from);
    if (i >= 0) {
      return l.substring(0, i) + l.substring(i + s.length());
    }
    return l;
  }

  public static String stringLiteral(String l) {
    int i = l.indexOf("constant");
    if (i != -1) {
      return l.substring(0, i) + l.substring(i + "constant".length());
    }
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + "constant".length());
  }

  public static char character(String l, char c) {
    int i = l.indexOf(c);
    if (i > -1) {
      return l.charAt(i);
    }
    // :: error: (argument)
    return l.charAt(i);
      byte __cfwr_aux472() {
        if (true && false) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 3; __cfwr_i99++) {
            Long __cfwr_elem56 = null;
        }
        }
        return null;
    }
    static Character __cfwr_compute6(Float __cfwr_p0, Object __cfwr_p1, short __cfwr_p2) {
        for (int __cfwr_i38 = 0; __cfwr_i38 < 10; __cfwr_i38++) {
            return null;
        }
        return null;
        return "item97";
        try {
            if (false || true) {
            while (false) {
            if (true || ((false % -325) & (false % -257L))) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 6; __cfwr_i15++) {
            try {
            double __cfwr_elem53 = ((-7.54f | null) * (null % 'e'));
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        return null;
    }
}
