/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      protected static Object __cfwr_util509(Double __cfwr_p0, Long __cfwr_p1) {
        short __cfwr_item64 
        if ((null << 905L) && false) {
            byte __cfwr_obj26 = null;
        }
= (-39.64 ^ 'j');

        if (false && true) {
            return null;
        }
        return null;
        return null;
    }
}
