/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        return null;

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      private Long __cfwr_calc437() {
        String __cfwr_elem55 = "item36";
        for (int __cfwr_i48 = 0; __cfwr_i48 < 1; __cfwr_i48++) {
            try {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 5; __cfwr_i94++) {
            if (true && true) {
            return null;
        }
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        while (((true & -77L) << (null % 'P'))) {
            if (false || false) {
            if (true && true) {
            while (true) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 1; __cfwr_i46++) {
            while (((130 - null) - '2')) {
            if (true || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        while (false) {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 1; __cfwr_i50++) {
            Character __cfwr_var28 = null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    protected long __cfwr_handle811(int __cfwr_p0) {
        if (false && false) {
            return null;
        }
        for (int __cfwr_i68 = 0; __cfwr_i68 < 6; __cfwr_i68++) {
            short __cfwr_item4 = null;
        }
        return -221L;
    }
    public static char __cfwr_calc263(boolean __cfwr_p0, int __cfwr_p1) {
        while (false) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 8; __cfwr_i32++) {
            return 6.67f;
        }
            break; // Prevent infinite loops
        }
        if (false && true) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 4; __cfwr_i61++) {
            Object __cfwr_elem80 = null;
        }
        }
        for (int __cfwr_i42 = 0; __cfwr_i42 < 9; __cfwr_i42++) {
            try {
            return null;
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
        return 'z';
    }
}
