/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        return ('G' - null);

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    i
        if (true || (-44.21 | -609)) {
            if (false && false) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 7; __cfwr_i4++) {
            if (false || (null + (false << 29.78))) {
            return null;
        }
        }
        }
        }
nt b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      byte __cfwr_temp778() {
        try {
            try {
            try {
            while (true) {
            if (true || ((true / true) % true)) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 1; __cfwr_i51++) {
            int __cfwr_result15 = ('V' / 't');
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        while (true) {
            return false;
            break; // Prevent infinite loops
        }
        while (false) {
            if (false || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    Float __cfwr_compute754(Long __cfwr_p0, double __cfwr_p1, byte __cfwr_p2) {
        while (true) {
            if (true && true) {
            String __cfwr_node61 = "value47";
        }
            break; // Prevent infinite loops
        }
        String __cfwr_temp44 = "result66";
        Boolean __cfwr_obj36 = null;
        try {
            return 995;
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        return null;
    }
}
