/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        for (int __cfwr_i75 = 0; __cfwr_i75 < 2; __cfwr_i75++) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 2; __cfwr_i51++) {
            long __cfwr_elem17 = -45L;
        }
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      public static Double __cfwr_helper813(char __cfwr_p0) {
        return null;
        try {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 8; __cfwr_i14++) {
            if (true || (5.86 << -801)) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 1; __cfwr_i25++) {
            return true;
        }
        }
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return true;
        return null;
    }
    static Character __cfwr_process178(short __cfwr_p0, String __cfwr_p1) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        while ((87.90f - 80L)) {
            try {
            return -63.24;
        } catch (Exception __cfwr_e55) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (false) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 3; __cfwr_i97++) {
            if (true || false) {
            while (('x' ^ false)) {
            double __cfwr_elem2 = (true % -751L);
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i43 = 0; __cfwr_i43 < 2; __cfwr_i43++) {
            return (5.33 / 644L);
        }
        return null;
    }
}
