/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        for (int __cfwr_i9 = 0; __cfwr_i9 < 8; __cfwr_i9++) {
            if (((null - true) * -81.25) && (null - 478)) {
            while (((652L * -29.56f) >> ('c' / null))) {
            if (t
        while (((-19.65 << -494L) >> (true >> -1.03f))) {
            Boolean __cfwr_var8 = null;
            break; // Prevent infinite loops
        }
rue && true) {
            Boolean __cfwr_data36 = null;
        }
            break; // Prevent infinite loops
        }
        }
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      Object __cfwr_util770() {
        if (false || (604L - null)) {
            if (false && true) {
            while ((true - 'K')) {
            Double __cfwr_temp40 = null;
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
    private static Character __cfwr_util571(Integer __cfwr_p0, byte __cfwr_p1, double __cfwr_p2) {
        for (int __cfwr_i18 = 0; __cfwr_i18 < 7; __cfwr_i18++) {
            return -581;
        }
        return null;
    }
}
