/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        if ((845L % -398L) || true) {
            if ((-202L - null) || (null / (null + -82))) {
            for (int __cfwr_i19 = 0; __cfw
        for (int __cfwr_i53 = 0; __cfwr_i53 < 2; __cfwr_i53++) {
            if (true && false) {
            try {
            return (null ^ 'v');
        } catch (Exception __cfwr_e81) {
            // ignore
        }
        }
        }
r_i19 < 10; __cfwr_i19++) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 7; __cfwr_i20++) {
            byte __cfwr_elem67 = ((73.41f - 713L) + null);
        }
        }
        }
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      double __cfwr_temp654(Float __cfwr_p0, Character __cfwr_p1) {
        if (false || true) {
            try {
            return false;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        try {
            char __cfwr_result10 = 'x';
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        for (int __cfwr_i61 = 0; __cfwr_i61 < 2; __cfwr_i61++) {
            while (true) {
            if (false || true) {
            if (true && true) {
            if (((null - '2') << -447L) || true) {
            if (false && ((656L + 778) % true)) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 2; __cfwr_i18++) {
            if (false || false) {
            return 18.28f;
        }
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        try {
            try {
            try {
            try {
            char __cfwr_obj98 = ((null % 686) + 33.17);
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        return (-54.82f << true);
    }
    protected static Float __cfwr_proc581(char __cfwr_p0, boolean __cfwr_p1) {
        return null;
        return "world30";
        try {
            if ((('2' | 'Q') | null) || false) {
            while (true) {
            char __cfwr_obj77 = '0';
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        return null;
    }
    public double __cfwr_handle957(double __cfwr_p0, short __cfwr_p1) {
        if (false && true) {
            try {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 6; __cfwr_i85++) {
            return null;
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
        try {
            return (-0.49 << -59.32f);
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        try {
            while (true) {
            return "value29";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        while ((null | null)) {
            while (false) {
            String __cfwr_val68 = "test72";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return -4.89;
    }
}
