/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        return "hello88";

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      public static float __cfwr_helper66(double __cfwr_p0, Boolean __
        while (false) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 1; __cfwr_i41++) {
            try {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 1; __cfwr_i61++) {
            if ((856L - (null + -768L)) && false) {
            if (true && true) {
            if (false || false) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 1; __cfwr_i2++) {
            double __cfwr_temp38 = 56.23;
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
cfwr_p1, float __cfwr_p2) {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 5; __cfwr_i60++) {
            Integer __cfwr_data78 = null;
        }
        try {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 9; __cfwr_i4++) {
            int __cfwr_val97 = -155;
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        return -54.83f;
    }
    protected static Character __cfwr_calc245(char __cfwr_p0, String __cfwr_p1) {
        if (true && false) {
            while (true) {
            while (false) {
            char __cfwr_result75 = ('s' / (20 % 'j'));
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    Long __cfwr_temp404() {
        try {
            while (true) {
            while (true) {
            Character __cfwr_data41 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        double __cfwr_result21 = -76.80;
        for (int __cfwr_i90 = 0; __cfwr_i90 < 9; __cfwr_i90++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 5; __cfwr_i47++) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
}
