/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        while (((-77.27f / 74.98) * null)) {
            if (true && (73.06 >> 'r')) {
            if (false && (47.15f >> -77.17f)) {
            while (false) {
            Float __cfwr_val89 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
      
        return null;
  }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private Float __cfwr_temp366() {
        if (true && (-71.30f >> 61.38f)) {
            return (null + false);
        }
        int __cfwr_temp33 = -925;
        for (int __cfwr_i32 = 0; __cfwr_i32 < 9; __cfwr_i32++) {
            long __cfwr_node49 = -323L;
        }
        try {
            return -58.23;
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        return null;
    }
    static boolean __cfwr_aux790(byte __cfwr_p0) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 10; __cfwr_i55++) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 10; __cfwr_i35++) {
            return null;
        }
        }
        for (int __cfwr_i90 = 0; __cfwr_i90 < 10; __cfwr_i90++) {
            if (('n' % (317L ^ null)) && false) {
            if (true || (-75 * null)) {
            if (true || (-418 * (-74.60 & -35.83))) {
            try {
            if (false || false) {
            if ((98.02f & null) || true) {
            while (true) {
            while (false) {
            while (true) {
            if (true || false) {
            double __cfwr_item25 = -3.37;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        }
        }
        }
        }
        if ((true >> null) || ('t' * (null << null))) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 10; __cfwr_i73++) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 9; __cfwr_i16++) {
            try {
            if (true && true) {
            if (((947L << null) >> false) && true) {
            return '8';
        }
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
        }
        }
        for (int __cfwr_i18 = 0; __cfwr_i18 < 6; __cfwr_i18++) {
            int __cfwr_obj7 = 410;
        }
        return true;
    }
}
