/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i82 = 0; __cfwr_i82 < 7; __cfwr_i82++) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 6; __cfwr_i78++) {
            if (true && false) {
            for (int __cfwr_i83 = 0; __cfwr_i83 < 2; __cfwr_i83++) {
            float __cfwr_entry64 = 9.34f;
        }
        }
        }
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      public static short __cfwr_process458(int __cfwr_p0, boolean __cfwr_p1) {
        if (((82.55 & null) & (-9.11 % -82.56f)) && true) {
            try {
            return null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        return null;
        return 'u';
        for (int __cfwr_i73 = 0; __cfwr_i73 < 1; __cfwr_i73++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 7; __cfwr_i86++) {
            long __cfwr_val60 = -552L;
        }
        }
        return ((-94.08f * 49.13) | (null * 341L));
    }
    protected int __cfwr_compute754(Float __cfwr_p0, Integer __cfwr_p1, short __cfwr_p2) {
        for (int __cfwr_i49 = 0; __cfwr_i49 < 1; __cfwr_i49++) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 3; __cfwr_i84++) {
            if (false && false) {
            while (true) {
            double __cfwr_temp31 = (-70.22f ^ (null << -654L));
            break; // Prevent infinite loops
        }
        }
        }
        }
        if (true && true) {
            while (true) {
            try {
            while (false) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 7; __cfwr_i90++) {
            if (false && false) {
            while (false) {
            try {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 5; __cfwr_i80++) {
            while (true) {
            return 'O';
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        long __cfwr_var14 = -379L;
        return -11;
    }
    protected static boolean __cfwr_aux321(byte __cfwr_p0, Object __cfwr_p1) {
        for (int __cfwr_i93 = 0; __cfwr_i93 < 7; __cfwr_i93++) {
            return null;
        }
        return true;
    }
}
