/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      public static float __cfwr_temp145() {
        if ((('q' * null) >> true) || false) {
            return 21L;
        }

        return -67.01f;
        return (false * 712L);
        int __cfwr_elem71 = -778;
        try {
            Integer __cfwr_elem93 = null;
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        return -82.74f;
    }
    private int __cfwr_compute533(float __cfwr_p0) {
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        while (false) {
            return false;
            break; // Prevent infinite loops
        }
        return (null - (70.27 << 83.75f));
    }
}
