/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTLengthOf("array") int x) {
        Character __cfwr_val84 = null;

    // :: error: (unary.increment)
    x++;
    // :: error: (unary.increment)
    ++x;
    // :: error: (assignment)
    x = x + 1;
      publ
        Character __cfwr_obj40 = null;
ic static Character __cfwr_handle595() {
        return -7.77f;
        try {
            Object __cfwr_data32 = null;
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        return null;
    }
    Long __cfwr_aux448(float __cfwr_p0) {
        try {
            return null;
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        return null;
    }
}
