/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTLengthOf("array") int x) {
        return (21.40f & 'b');

    // :: error: (unary.increment)
    x++;
    // :: error: (unary.increment)
    ++x;
    // :: error: (assignment)
    x = x + 1;
      public int __cfwr_aux783(Object __cfwr_p0, int __cfwr_p1, double __cfwr_p2) {
        while (((-63.60f & -772L) << '9')) {
            while (true) {
            if (false && ((48.49f << 58.39f) | null)) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 1; __cfwr_i12++) {
            Float __cfwr_elem11 = null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return -771;
        return null;
        return null;
        return -435;
    }
}
