/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class InvalidSubsequence_slice {
  void assignB(int[] d) {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 2; __cfwr_i36++) {
            while (false) {
            if (true || (null >> nu
        try {
            String __cfwr_result38 = "temp20";
        } catch (Exception __cfwr_e62) {
            // ignore
        }
ll)) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
  }

    private static short __cfwr_process851(float __cfwr_p0) {
        while (false) {
            Integer __cfwr_item3 = null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i93 = 0; __cfwr_i93 < 10; __cfwr_i93++) {
            try {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 2; __cfwr_i8++) {
            return (null >> 248);
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        return null;
        return (false ^ (-5.13f - null));
    }
    long __cfwr_handle629() {
        while (false) {
            Long __cfwr_data27 = null;
            break; // Prevent infinite loops
        }
        while (true) {
            try {
            if (true || true) {
            return null;
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (false) {
            Long __cfwr_data92 = null;
            break; // Prevent infinite loops
        }
        return (null >> 31);
    }
    static float __cfwr_process802(Boolean __cfwr_p0, short __cfwr_p1, long __cfwr_p2) {
        if (false && (-32.35f | null)) {
            char __cfwr_var55 = 's';
        }
        for (int __cfwr_i57 = 0; __cfwr_i57 < 4; __cfwr_i57++) {
            Double __cfwr_node31 = null;
        }
        return ((97.01f * true) ^ (null % 86.22f));
    }
}