/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        byte __cfwr_node86 = (null - (true | null));

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a + 1;
    @LTEqLengthOf({"arr"}) int c1 = a + 1;
    @LTLengthOf({"arr"}) int d = a + 0;
    @LTLengthOf({"arr"}) int e = a + (-7);
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a + 7;
      protected Boolean __cfwr_util13
        if (false || false) {
            Object __cfwr_val36 = null;
        }
4(String __cfwr_p0, short __cfwr_p1, long __cfwr_p2) {
        return null;
        return null;
    }
    public short __cfwr_util316(Long __cfwr_p0) {
        Long __cfwr_entry66 = null;
        for (int __cfwr_i41 = 0; __cfwr_i41 < 3; __cfwr_i41++) {
            while ((-817 - null)) {
            String __cfwr_temp39 = "item56";
            break; // Prevent infinite loops
        }
        }
        if (false || false) {
            while (false) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 6; __cfwr_i57++) {
            while (false) {
            try {
            while (true) {
            String __cfwr_obj32 = "item33";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
        return null;
    }
}
