/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineSubtrahend_slice {
  void withConstant(int[] a, @NonNegative int l) {
        while (true) {
     
        for (int __cfwr_i61 = 0; __cfwr_i61 < 8; __cfwr_i61++) {
            try {
            return false;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
       try {
            int __cfwr_item27 = (970 + true);
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
  }

    Long __cfwr_util448(short __cfwr_p0, double __cfwr_p1, long __cfwr_p2) {
        return -80.86f;
        if ((-142L << false) || (true * 'd')) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 10; __cfwr_i2++) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 7; __cfwr_i85++) {
            try {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 5; __cfwr_i40++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 3; __cfwr_i7++) {
            while ((2.74 - 51.76)) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 5; __cfwr_i93++) {
            while (true) {
            if ((true * null) || false) {
            while ((null % -72.18f)) {
            char __cfwr_elem20 = 'V';
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        }
        }
        }
        return null;
    }
    static float __cfwr_handle609(char __cfwr_p0, float __cfwr_p1) {
        Character __cfwr_temp75 = null;
        if (false && false) {
            if (true && false) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 2; __cfwr_i20++) {
            while (true) {
            try {
            try {
            while (false) {
            while (true) {
            Boolean __cfwr_val53 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        while (false) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 4; __cfwr_i34++) {
            try {
            return null;
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i94 = 0; __cfwr_i94 < 7; __cfwr_i94++) {
            boolean __cfwr_elem85 = false;
        }
        return 31.74f;
    }
}