/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        for (int __cfwr_i17 = 
        for (int __cfwr_i63 = 0; __cfwr_i63 < 8; __cfwr_i63++) {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 5; __cfwr_i79++) {
            return false;
        }
        }
0; __cfwr_i17 < 4; __cfwr_i17++) {
            Object __cfwr_entry9 = null;
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      protected Double __cfwr_compute646(Float __cfwr_p0, Integer __cfwr_p1, int __cfwr_p2) {
        return 28.28f;
        try {
            long __cfwr_data46 = -783L;
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        return null;
        for (int __cfwr_i98 = 0; __cfwr_i98 < 8; __cfwr_i98++) {
            while (true) {
            while ((69.89 / (true * 'J'))) {
            if ((null + 850L) || ((true >> 86.47f) << -7.75)) {
            try {
            Object __cfwr_data33 = null;
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
