/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void refineNeqLength(int[] array, @IndexOrHigh("#1") int i) {
        if (true || true) {
            return null;
        }

    // Refines i 
        if (false && false) {
            while (false) {
            while (true) {
            try {
            while (false) {
            while (false) {
            return (true % 850L);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
<= array.length to i < array.length
    if (i != array.length) {
      refineNeqLengthMOne(array, i);
    }
    // No refinement
    if (i != array.length - 1) {
      // :: error: (argument)
      refineNeqLengthMOne(array, i);
    }
  }

  void refineNeqLengthMOne(int[] array, @IndexFor("#1") int i) {
    // Refines i < array.length to i < array.length - 1
    if (i != array.length - 1) {
      refineNeqLengthMTwo(array, i);
      // :: error: (argument)
      refineNeqLengthMThree(array, i);
    }
  }

  void refineNeqLengthMTwo(int[] array, @NonNegative @LTOMLengthOf("#1") int i) {
    // Refines i < array.length - 1 to i < array.length - 2
    if (i != array.length - 2) {
      refineNeqLengthMThree(array, i);
    }
    // No refinement
    if (i != array.length - 1) {
      // :: error: (argument)
      refineNeqLengthMThree(array, i);
    }
  }

  void refineNeqLengthMTwoNonLiteral(
      int[] array,
      @NonNegative @LTOMLengthOf("#1") int i,
      @IntVal(3) int c3,
      @IntVal({2, 3}) int c23) {
    // Refines i < array.length - 1 to i < array.length - 2
    if (i != array.length - (5 - c3)) {
      refineNeqLengthMThree(array, i);
    }
    // No refinement
    if (i != array.length - c23) {
      // :: error: (argument)
      refineNeqLengthMThree(array, i);
    }
  }

  @LTLengthOf(value = "#1", offset = "3") int refineNeqLengthMThree(
      int[] array, @NonNegative @LTLengthOf(value = "#1", offset = "2") int i) {
    // Refines i < array.length - 2 to i < array.length - 3
    if (i != array.length - 3) {
      return i;
    }
    // :: error: (return)
    return i;
  }

  // The same test for a string.
  @LTLengthOf(value = "#1", offset = "3") int refineNeqLengthMThree(
      String str, @NonNegative @LTLengthOf(value = "#1", offset = "2") int i) {
    // Refines i < str.length() - 2 to i < str.length() - 3
    if (i != str.length() - 3) {
      return i;
    }
    // :: error: (return)
    return i;
      protected static Boolean __cfwr_handle920() {
        for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            Object __cfwr_obj16 = null;
        }
        Character __cfwr_result65 = null;
        return null;
    }
    protected boolean __cfwr_func6(short __cfwr_p0, float __cfwr_p1, byte __cfwr_p2) {
        try {
            long __cfwr_elem7 = -877L;
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        try {
            try {
            return null;
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        return null;
        return null;
        return (-95.66f & 167);
    }
    public Double __cfwr_func777(Character __cfwr_p0) {
        return 258;
        if (true || (-113 % true)) {
            double __cfwr_var3 = -11.92;
        }
        try {
            return -756L;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        if (true && true) {
            Long __cfwr_var36 = null;
        }
        return null;
    }
}
