/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test1(@Positive int x, @Positive int y) {
        while ((null * (null >> true))) {
            Integer __cfwr_data38 = null;
            break; // Prevent infinite loops
        }

    int[] newArray = new in
        while (true) {
            Long __cfwr_val23 = null;
            break; // Prevent infinite loops
        }
t[x + y];
    @IndexFor("newArray") int i = x;
    @IndexFor("newArray") int j = y;
  }

  void test2(@NonNegative int x, @Positive int y) {
    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test3(@NonNegative int x, @NonNegative int y) {
    int[] newArray = new int[x + y];
    @IndexOrHigh("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test4(@GTENegativeOne int x, @NonNegative int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    @LTEqLengthOf("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test5(@GTENegativeOne int x, @GTENegativeOne int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexOrHigh("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test6(int x, int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexFor("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
      public Double __cfwr_helper416(Character __cfwr_p0, char __cfwr_p1) {
        try {
            while (true) {
            try {
            while (false) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 5; __cfwr_i89++) {
            try {
            if (true && ((-173 / 632) * (-56.39f & 'x'))) {
            return null;
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        try {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 3; __cfwr_i88++) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 3; __cfwr_i17++) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 9; __cfwr_i70++) {
            while (true) {
            while (false) {
            Float __cfwr_val70 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        for (int __cfwr_i15 = 0; __cfwr_i15 < 6; __cfwr_i15++) {
            try {
            try {
            try {
            Double __cfwr_item25 = null;
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        }
        return null;
    }
    public static double __cfwr_calc959() {
        while (false) {
            return -27.05;
            break; // Prevent infinite loops
        }
        try {
            Double __cfwr_entry31 = null;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        return 78.62;
    }
}
