/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        while (true) {
            try {
            try {
            return ((-221L % 'U') / null);
        } catch (Exception __cfwr_e52) {
            // ignore
        while (true) {
            try {
            if (true && false) {
            return null;
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
            break; // Prevent infinite loops
        }

        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      double __cfwr_calc893(Object __cfwr_p0) {
        for (int __cfwr_i95 = 0; __cfwr_i95 < 7; __cfwr_i95++) {
            try {
            int __cfwr_item97 = 889;
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
        try {
            try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 7; __cfwr_i62++) {
            return 'e';
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        return ('j' + (719L / 377L));
    }
    static String __cfwr_calc600(char __cfwr_p0, Integer __cfwr_p1) {
        if ((87.44f >> (null & 455)) || true) {
            if (('m' / true) && false) {
            return 'f';
        }
        }
        return "test99";
    }
    protected Double __cfwr_aux898(Double __cfwr_p0) {
        try {
            try {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 8; __cfwr_i36++) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 5; __cfwr_i1++) {
            if (('J' * false) && (null - -162)) {
            if ((('q' ^ 739) | -696) && false) {
            if ((81.23f >> (false ^ 60.62f)) || true) {
            if (false || false) {
            while (false) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 1; __cfwr_i12++) {
            if (false || (14.64f / -867)) {
            if (false && true) {
            while (false) {
            try {
            boolean __cfwr_obj56 = (-81.51 * 555L);
        } catch (Exception __cfwr_e56) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        return 27.24;
        if (false || false) {
            return -86.55f;
        }
        return null;
    }
}
