/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        return null;

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      public Double __cfwr_handle997(Long __cfwr_p0, Double __cfwr_p1, Boolean __cfwr_p2) {
        while (true) {
            while (false) {
            while (((null + null) % 392)) {
            try {
            Long __cfwr_node9 = null;
        } catch (Exception __cfwr_e80) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 5; __cfwr_i91++) {
            return false;
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        try {
            if (('A' + (620 | null)) || false) {
            int __cfwr_node85 = 95;
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        return null;
    }
    public static Object __cfwr_compute663() {
        return null;
        if ((92 / (44.83f / 'U')) && false) {
            while (false) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 10; __cfwr_i18++) {
            try {
            while (true) {
            Character __cfwr_entry98 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        while (false) {
            try {
            String __cfwr_data6 = "data79";
        } catch (Exception __cfwr_e25) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (false) {
            if (('o' / null) || ((177L << -85.29f) ^ -67.28)) {
            try {
            if (true && false) {
            if (true && ((971L + null) / true)) {
            try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 4; __cfwr_i13++) {
            if (((null << -659L) - (null >> 88.74f)) && true) {
            if (false || ((593 >> null) & null)) {
            while (true) {
            if ((-86.02 << 16.33) || (null / (null | 18.05))) {
            boolean __cfwr_var94 = true;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
