/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  ArrayAssignmentSameLen(int[] array, @IndexFor("#1") int index) {
        return null;

    i_array = array;
    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengt
        if (true && true) {
            if (((-33.08 % -921) * 'A') && false) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 5; __cfwr_i13++) {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e54) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
hOf("#1") int index) {
    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {
    int[] c1 = a;
    // See useTest3 for an example of why this assignment should fail.
    @LTLengthOf(
        value = {"c1", "c1"},
        offset = {"0", "x"})
    // :: error: (assignment)
    int z = i;
      public Character __cfwr_helper680(Integer __cfwr_p0, Long __cfwr_p1) {
        try {
            while (true) {
            if (false || (null << 'q')) {
            Long __cfwr_item39 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        return null;
    }
    private float __cfwr_process539(short __cfwr_p0, boolean __cfwr_p1) {
        return null;
        return -91.94f;
    }
}
