/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            while (('n' >> -44)) {
            while (true) {
            boolean __cfwr_entry61 = false;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      static Long __cfwr_proc808(boolean __cfwr_p0, Long __cfwr_p1) {
        for (int __cfwr_i91 = 0; __cfwr_i91 < 8; __cfwr_i91++) {
            while (false) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 10; __cfwr_i36++) {
            while ((-43.54f >> (789 | null))) {
            if (false && false) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 4; __cfwr_i47++) {
            while (true) {
            try {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 5; __cfwr_i84++) {
            return null;
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        if (((false & 'W') % null) && (null % ('s' * 404L))) {
            while (true) {
            if (true || (false >> false)) {
            if (true || (false ^ null)) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 6; __cfwr_i78++) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    private static Boolean __cfwr_aux744() {
        return null;
        return true;
        try {
            if (((82.85f >> null) * (false | -431)) || false) {
            double __cfwr_item71 = 0.93;
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        Long __cfwr_val18 = null;
        return null;
    }
    public static Character __cfwr_proc696(Long __cfwr_p0, char __cfwr_p1) {
        while (false) {
            while (false) {
            if (((null / -11.91f) / false) && ((-662 | null) & (null % null))) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 8; __cfwr_i28++) {
            Long __cfwr_temp61 = null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        Long __cfwr_item1 = null;
        return null;
    }
}
