/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return null;



        for (int __cfwr_i32 = 0; __cfwr_i32 < 5; __cfwr_i32++) {
            if (false || (276L | -55)) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 4; __cfwr_i82++) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 5; __cfwr_i69++) {
            while (true) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 1; __cfwr_i16++) {
            if (true && true) {
            String __cfwr_item63 = "result18";
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      protected static Float __cfwr_calc600(short __cfwr_p0) {
        Integer __cfwr_node69 = null;
        return null;
        Integer __cfwr_obj17 = null;
        return null;
    }
    private static short __cfwr_handle82() {
        try {
            if (false || ((-329 | 633) & (-876L << -387))) {
            if (true || (null & 83.00f)) {
            byte __cfwr_temp66 = null;
        }
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        try {
            if ((115 / 789) || ((null & null) % (-496L & 390L))) {
            try {
            boolean __cfwr_node31 = true;
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        return null;
    }
}
