/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        if (false && false) {
            return null;
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      protected 
        while (true) {
            if ((-31.73f & -820L) || ((15.18f / -188L) & (-81.58 >> true))) {
            if (false || true) {
            Double __cfwr_obj1 = null;
        }
        }
            break; // Prevent infinite loops
        }
static Long __cfwr_proc395(Character __cfwr_p0, Float __cfwr_p1) {
        while (true) {
            if (false || ('I' << (-813L - true))) {
            return (40.52 ^ (14.73f + false));
        }
            break; // Prevent infinite loops
        }
        while ((-37.68f >> (-21L | '3'))) {
            try {
            try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 5; __cfwr_i38++) {
            try {
            while (((null >> 'p') & false)) {
            return 64.63;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            return null;
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        if (true && false) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 2; __cfwr_i13++) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 5; __cfwr_i11++) {
            if (((-818 ^ 73.20f) << -52.38f) && true) {
            while (true) {
            if ((756L - 'm') && ((129 | -42.68f) * -57.16f)) {
            if ((-255 - (28.57 / -839L)) || false) {
            double __cfwr_data60 = ((null >> -265) - null);
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        return null;
    }
    public short __cfwr_util451(Boolean __cfwr_p0, Object __cfwr_p1) {
        while ((-83.69f + (-85.20 % 377L))) {
            while (false) {
            while (true) {
            try {
            return ('p' & true);
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        byte __cfwr_elem9 = null;
        return null;
    }
    Object __cfwr_process602(Float __cfwr_p0, char __cfwr_p1, boolean __cfwr_p2) {
        Character __cfwr_elem11 = null;
        String __cfwr_obj82 = "item83";
        return "result85";
        while (true) {
            if (true && true) {
            int __cfwr_obj34 = (14.54f - (false >> null));
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
