/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testAppend(Appendable app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

  void testAppend(StringWriter app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
      private static boolean __cfwr_helper989(int __cfwr_p0) {
        try {
            return null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }

        return null;
        while (true) {
            try {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 2; __cfwr_i26++) {
            return (234L * (null / null));
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
        try {
            short __cfwr_item54 = null;
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        return false;
    }
    protected static byte __cfwr_aux701(long __cfwr_p0, char __cfwr_p1, String __cfwr_p2) {
        if (true || true) {
            return -68.94f;
        }
        return (-34.00 & null);
    }
}
