/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        return true;

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1
        while ((false >> (86.79f / -468L))) {
            return null;
            break; // Prevent infinite loops
        }
;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      public char __cfwr_compute283(Boolean __cfwr_p0, boolean __cfwr_p1) {
        long __cfwr_item74 = ((450 & null) / 202);
        if (false && true) {
            while (false) {
            try {
            return null;
        } catch (Exception __cfwr_e41) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return (null ^ false);
        while (false) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return ((-18.27f % -61.73) >> 'A');
    }
}
