/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        try {
            String __cfwr_obj57 = "data28";
        } catch (Exception __cfwr_e9) {
            // ignore
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignmen
        if (true || false) {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 9; __cfwr_i52++) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 1; __cfwr_i85++) {
            if (true && false) {
            while (true) {
            while (true) {
            if (false || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
t)
    int @SameLen("f") [] e = a;
      protected byte __cfwr_helper912(Boolean __cfwr_p0, Integer __cfwr_p1, boolean __cfwr_p2) {
        if (((-89.10f / '7') << (null / null)) && true) {
            if (true || (-891L % '8')) {
            try {
            if (((false - 42.66f) | true) && false) {
            return null;
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        }
        }
        return null;
        return (-556 / true);
        return null;
        return ((null + 67.65) - 'U');
    }
}
