/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            double __cfwr_temp42 = -83.57;
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      public static double __cfwr_temp996(Character __cfwr_p0, int __cfwr_p1) {
        try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        while (true) {
            if (true && false) {
            if (false || ((-847 & null) + null)) {
            return false;
        }
        }
            break; // Prevent infinite loops
        }
        return -26.78;
    }
    static Integer __cfwr_calc254(long __cfwr_p0, boolean __cfwr_p1) {
        for (int __cfwr_i99 = 0; __cfwr_i99 < 6; __cfwr_i99++) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 7; __cfwr_i41++) {
            while ((null % 'V')) {
            try {
            try {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 4; __cfwr_i21++) {
            return false;
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        try {
            while ((53.10 + null)) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 4; __cfwr_i29++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        return false;
        return (null << null);
        return null;
    }
}
