/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testNewInstance(int i) {
        try {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 6; __cfwr_i63++) {
  
        while (true) {
            while (false) {
            while ((null * 'w')) {
            if (false && (-386L % false)) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 6; __cfwr_i47++) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
          while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }

    // :: error: (argument)
    Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
      Integer __cfwr_calc7() {
        try {
            Float __cfwr_data75 = null;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        try {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 7; __cfwr_i30++) {
            if ((-350L - 38.76) && true) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 2; __cfwr_i9++) {
            return null;
        }
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        return null;
    }
}
