/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        try {
            while ((null >> (-45.56 % 922L))) {
            return ((null * null) & -988);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      Object __cfwr_aux727(char __cfwr_p0, long __cfwr_p1) {
        while (false) {
            try {
            return (null + 32.60f);
        } catch (Exception __cfwr_e26) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        boolean __cfwr_item70 = (false | 'J');
        for (int __cfwr_i74 = 0; __cfwr_i74 < 3; __cfwr_i74++) {
            return (null >> 'l');
        }
        for (int __cfwr_i7 = 0; __cfwr_i7 < 2; __cfwr_i7++) {
            short __cfwr_entry48 = null;
        }
        return null;
    }
}
