/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        float __cfwr_var8 = ((-635L ^ true) << (null | true));

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      private static double __cfwr_calc816(Integer __cfwr_p0, float __cfwr_p1) {
        for (int __cfwr_i46 = 0; __cfwr_i46 < 8; __cfwr_i46++) {
            return null;
        }
        Long __cfwr_data1 = null;
        if (true && true) {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 8; __cfwr_i48++) {
            boolean __cfwr_entry42 = false;
        }
        }
        if (false || (446L * (null & false))) {
            while (false) {
            try {
            try {
            Long __cfwr_node82 = null;
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return 5.50;
    }
}
