/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testNewInstance(int i) {
        if ((-539L + -117) || true) {
            long __cfwr_val26 = 761L;
        }
        while (true) {
            try {
            while ((9.91 | ('B' - 'p'))) {
            if ((77.67f + null) && true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }


    // :: error: (argument)
    Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
      private static String __cfwr_util39() {
        for (int __cfwr_i78 = 0; __cfwr_i78 < 1; __cfwr_i78++) {
            if (false || ((true ^ 5.71f) >> -82.80f)) {
            float __cfwr_obj5 = 78.66f;
        }
        }
        for (int __cfwr_i53 = 0; __cfwr_i53 < 10; __cfwr_i53++) {
            try {
            Integer __cfwr_item88 = null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        return null;
        if (false || true) {
            Boolean __cfwr_var81 = null;
        }
        return "world17";
    }
    protected static float __cfwr_compute237() {
        short __cfwr_result33 = null;
        for (int __cfwr_i9 = 0; __cfwr_i9 < 3; __cfwr_i9++) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 9; __cfwr_i10++) {
            try {
            if (false || true) {
            try {
            while (false) {
            return (true | ('e' << 154));
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        }
        Long __cfwr_data54 = null;
        return -83.88f;
    }
}
