/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            while ((877 * (null | 597L))) {
            try {
            try {
            while (false) {
            while (false) {
            try {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 7; __cfwr_i39++) {
            while (true) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 8; __
        Long __cfwr_obj86 = null;
cfwr_i55++) {
            if (true && (('x' & -58.10f) << null)) {
            while (true) {
            Float __cfwr_val82 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      protected String __cfwr_compute897(Boolean __cfwr_p0) {
        return 95.17f;
        if ((true + (false ^ false)) && false) {
            if (false || false) {
            while (false) {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 9; __cfwr_i62++) {
            try {
            if (false || ((973 * null) / (null ^ -203))) {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 9; __cfwr_i88++) {
            long __cfwr_var11 = -47L;
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        return null;
        return "item27";
    }
}
