/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        while (true) {
            try {
            Character __cfwr_temp77 = null;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      public Integer __cfwr_proc418(Character __cfwr_p0, Integer __cfwr_p1, boolean __cfwr_p2) {
        char __cfwr_data76 = ((null << null) % (208L >> null));
        return null;
    }
    short __cfwr_handle751(byte __cfwr_p0, boolean __cfwr_p1) {
        while ((null << 'U')) {
            if (false || false) {
            Object __cfwr_entry16 = null;
        }
            break; // Prevent infinite loops
        }
        try {
            try {
            while (true) {
            try {
            Integer __cfwr_entry64 = null;
        } catch (Exception __cfwr_e66) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        return (null & -8.35f);
    }
}
