/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        try {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 8; __cfwr_i78++) {
            while (true) {
            if (true || ('n' ^ (null / -7
        while (true) {
            boolean __cfwr_var86 = true;
            break; // Prevent infinite loops
        }
0.02))) {
            try {
            while ((true >> false)) {
            try {
            while (false) {
            return (null % (false + -1000L));
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      public static boolean __cfwr_handle631() {
        for (int __cfwr_i67 = 0; __cfwr_i67 < 7; __cfwr_i67++) {
            if (true && ((747L / true) ^ 'D')) {
            byte __cfwr_item50 = null;
        }
        }
        while ((true & 736)) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 9; __cfwr_i58++) {
            while (false) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 1; __cfwr_i38++) {
            if (true && false) {
            try {
            Character __cfwr_var40 = null;
        } catch (Exception __cfwr_e66) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        while ((48.18 + false)) {
            return 224;
            break; // Prevent infinite loops
        }
        if (false && true) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 6; __cfwr_i31++) {
            double __cfwr_var10 = (null + 44.64);
        }
        }
        return true;
    }
    protected static Boolean __cfwr_calc20(boolean __cfwr_p0, int __cfwr_p1, Character __cfwr_p2) {
        return null;
        Double __cfwr_temp44 = null;
        char __cfwr_var39 = 'v';
        for (int __cfwr_i71 = 0; __cfwr_i71 < 10; __cfwr_i71++) {
            byte __cfwr_elem74 = ((-294 & false) - 230);
        }
        return null;
    }
}
