/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        for (int __cfwr_i15 
        try {
            return null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
= 0; __cfwr_i15 < 3; __cfwr_i15++) {
            while (true) {
            double __cfwr_val29 = (-709 << null);
            break; // Prevent infinite loops
        }
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      private static Integer __cfwr_proc408(Long __cfwr_p0, boolean __cfwr_p1, boolean __cfwr_p2) {
        for (int __cfwr_i29 = 0; __cfwr_i29 < 4; __cfwr_i29++) {
            try {
            return null;
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        double __cfwr_val21 = -32.11;
        return 'V';
        while (false) {
            Object __cfwr_var67 = null;
            break; // Prevent infinite loops
        }
        return null;
    }
    static boolean __cfwr_compute474(String __cfwr_p0, int __cfwr_p1) {
        for (int __cfwr_i46 = 0; __cfwr_i46 < 1; __cfwr_i46++) {
            try {
            return null;
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        }
        for (int __cfwr_i1 = 0; __cfwr_i1 < 4; __cfwr_i1++) {
            while (false) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 5; __cfwr_i37++) {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 10; __cfwr_i62++) {
            if (false && false) {
            try {
            try {
            try {
            boolean __cfwr_data13 = false;
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        if (true || false) {
            return null;
        }
        return false;
    }
    public Double __cfwr_aux62() {
        long __cfwr_obj89 = -776L;
        return null;
    }
}
