/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        if (true && (-973 - 56.98f)) {
            char __cfwr_var94 = 'p';
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      protected int __cfwr_compute466(Float __cfwr_p0, Character __cfwr_p1, byte __cfwr_p2) {
        try {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 8; __cfwr_i45++) {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 9; __cfwr_i52++) {
            try {
            return null;
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        return -66.74;
        return 800;
    }
    public static double __cfwr_func931() {
        if (true && ('O' ^ -71.96)) {
            try {
            while ((null | -508)) {
            while (false) {
            short __cfwr_entry12 = ((94.49f - 333L) - -869L);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        }
        while (true) {
            String __cfwr_temp92 = "world35";
            break; // Prevent infinite loops
        }
        Long __cfwr_val12 = null;
        if (false || (-211L * null)) {
            Long __cfwr_elem3 = null;
        }
        return -67.79;
    }
}
