/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        return null;

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      private static Fl
        byte __cfwr_temp60 = (85.96f - '6');
oat __cfwr_calc802() {
        while (false) {
            if (true && (null * (null % 'k'))) {
            while (true) {
            double __cfwr_data96 = -54.71;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
