/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    static Float __cfwr_func195(Float __cfwr_p0, char __cfwr_p1, Long __cfwr_p2) {
        return null;

 
        if (true || false) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 5; __cfwr_i87++) {
            char __cfwr_data25 = 'w';
        }
        }
       while (false) {
            byte __cfwr_data70 = ((true >> -220) << (null | null));
            break; // Prevent infinite loops
        }
        Integer __cfwr_node40 = null;
        while ((null / (null % -270))) {
            try {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 6; __cfwr_i91++) {
            byte __cfwr_node10 = (false >> ('v' % true));
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (false) {
            try {
            Float __cfwr_result85 = null;
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
