/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        while ((-56.99f / -96.34f)) {
            try {
            try {
            return 'o';
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        } catch (Exception __cfwr_e93) {
            /
        Float __cfwr_entry2 = null;
/ ignore
        }
            break; // Prevent infinite loops
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      public static boolean __cfwr_helper862(double __cfwr_p0) {
        char __cfwr_result81 = 'k';
        for (int __cfwr_i46 = 0; __cfwr_i46 < 1; __cfwr_i46++) {
            return null;
        }
        while (false) {
            try {
            try {
            while ((('D' << true) | null)) {
            if (false && false) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 1; __cfwr_i1++) {
            if (true || (-585 % null)) {
            while ((false ^ (null - null))) {
            long __cfwr_obj56 = 225L;
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            if ((-193 / -71.73f) || true) {
            while (((312 * -627) + null)) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 1; __cfwr_i67++) {
            while ((null * -43.34f)) {
            if (false || true) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 9; __cfwr_i97++) {
            while (false) {
            Object __cfwr_entry18 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        return ((67.40 + 170L) / 441L);
    }
}
