/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      long __cfwr_compute379() {
        byte __cfwr_var14 = (-17.29 / (null << -536L));

        for (int __cfwr_i45 = 0; __cfwr_i45 < 4; __cfwr_i45++) {
            try {
            long __cfwr_var3 = 927L;
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        }
        while (false) {
            if ((-51.26f - 287) || (null % (459L << 67.25))) {
            Character __cfwr_elem15 = null;
        }
            break; // Prevent infinite loops
        }
        return ('5' - -649);
    }
}
