/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        while (false) {
            if (false && true) {
            if ((-698 | -250) && true) {
            if (false || false) {
            String __cfwr_entry72 = "data7";
        }
        }
        }
            break; // Prevent infinite loops
        }

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      private static Long __cfwr_proc930(int __cfwr_p0, Character __cfwr_p1) {
        while ((-661L | (null | 'i'))) {
            if (false && (false - false)) {
            try {
            try {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 3; __cfwr_i75++) {
            if ((930 >> 282L) || false) {
            try {
            return null;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        while ((827L << (-59.50 + false))) {
            if (true || false) {
            try {
            return null;
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        Character __cfwr_val98 = null;
        return null;
    }
}
