/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        char __cfwr_data47 = '2';

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      protected Float __cfwr_util558(B
        try {
            if ((-453 % 64.90) || (963L % null)) {
            return -5.51;
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
oolean __cfwr_p0, Long __cfwr_p1) {
        if (((537 + 'V') - null) || (96.41 ^ 509)) {
            if ((false & (null + null)) && ('a' << '8')) {
            while (false) {
            if (false || false) {
            char __cfwr_item9 = ((null - true) & -31.83);
        }
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
    protected static Long __cfwr_aux884(String __cfwr_p0, boolean __cfwr_p1) {
        while ((null & true)) {
            while ((null + -38L)) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 7; __cfwr_i51++) {
            return 966;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
