/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        while (false) {
          
        if (((-86.76f << -212) + null) || true) {
            if (false || true) {
            return null;
        }
        }
  return null;
            break; // Prevent infinite loops
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      Object __cfwr_compute465(Boolean __cfwr_p0, Character __cfwr_p1, boolean __cfwr_p2) {
        return 43.53;
        if ((-9.20 | ('Z' << null)) && (null * -373)) {
            while ((30.58f >> 128)) {
            return 347L;
            break; // Prevent infinite loops
        }
        }
        while (('L' ^ null)) {
            while ((true >> (982 * 49.49))) {
            try {
            if ((968 + 68.18) || (41.95 ^ (-3.12f >> -102))) {
            while (false) {
            return (-60.67 >> null);
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    boolean __cfwr_util15(int __cfwr_p0, short __cfwr_p1) {
        try {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 7; __cfwr_i36++) {
            if (true && ((-21.88 | 798L) / 'f')) {
            if (true && (-0.09f >> (false ^ 'k'))) {
            return null;
        }
        }
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        return -200L;
        for (int __cfwr_i58 = 0; __cfwr_i58 < 9; __cfwr_i58++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 1; __cfwr_i47++) {
            while (false) {
            try {
            Float __cfwr_data46 = null;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        while ((null << (83.93f >> null))) {
            Boolean __cfwr_var53 = null;
            break; // Prevent infinite loops
        }
        return false;
    }
    Long __cfwr_handle516(Long __cfwr_p0, Long __cfwr_p1) {
        while ((17.95f & (79.97f >> 16.02f))) {
            return null;
            break; // Prevent infinite loops
        }
        double __cfwr_node61 = -59.85;
        try {
            if (false && ((14L * false) % 7)) {
            try {
            if (false || false) {
            Long __cfwr_entry50 = null;
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        for (int __cfwr_i82 = 0; __cfwr_i82 < 4; __cfwr_i82++) {
            Character __cfwr_node44 = null;
        }
        return null;
    }
}
