/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        if ((87.11f << (null + 444L)) || true) {
            Long __cfwr_val66 = null;
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (arr
        return null;
ay.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      protected static Double __cfwr_calc87() {
        while (((false >> null) % (null * null))) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 1; __cfwr_i35++) {
            if (true && false) {
            while (false) {
            try {
            if (false || false) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 10; __cfwr_i80++) {
            if (false || true) {
            try {
            char __cfwr_obj70 = (null ^ -478L);
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        while (false) {
            return 731L;
            break; // Prevent infinite loops
        }
        Integer __cfwr_elem44 = null;
        return null;
    }
    protected Double __cfwr_aux108(String __cfwr_p0) {
        return (59.23 % (true / null));
        if ((null | null) || (null + null)) {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
        return null;
    }
    protected static byte __cfwr_process838(String __cfwr_p0, long __cfwr_p1) {
        return null;
        short __cfwr_entry71 = null;
        try {
            if (false || true) {
            try {
            String __cfwr_result78 = "hello18";
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        return null;
    }
}
