/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void basic_test() {
        for (int __cfwr_i88 = 0; __cfwr_i88 < 3; __cfwr_i88++) {
            if (false && (null & null)) {
            while ((-2.60 % ('A' - null))) {
            while (false) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 4; __cfwr_i45++) {
            while (true) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 4; __cfwr_i11++) {
            try
        while (false) {
            Integer __cfwr_temp43 = null;
            break; // Prevent infinite loops
        }
 {
            int __cfwr_data43 = 807;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }

    int[] b = new int[4];
    @LTLengthOf("b") int[] a = {0, 1, 2, 3};

    // :: error: (array.initializer)::error: (assignment)
    @LTLengthOf("b") int[] a1 = {0, 1, 2, 4};

    @LTEqLengthOf("b") int[] c = {-1, 4, 3, 1};

    // :: error: (array.initializer)::error: (assignment)
    @LTEqLengthOf("b") int[] c2 = {-1, 4, 5, 1};
  }

  void offset_test() {
    int[] b = new int[4];
    int[] b2 = new int[10];
    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    int[] a = {2, 3, 0};

    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    // :: error: (array.initializer)::error: (assignment)
    int[] a2 = {2, 3, 5};

    // Non-constant offsets don't work correctly. See kelloggm#120.
      protected int __cfwr_calc777(float __cfwr_p0, short __cfwr_p1) {
        while (false) {
            if ((-75 & (null | -444L)) || ((62.03f / null) % (53.26f >> null))) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            Long __cfwr_temp81 = null;
        }
        }
            break; // Prevent infinite loops
        }
        while (true) {
            while (true) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 9; __cfwr_i32++) {
            Long __cfwr_item69 = null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return -786;
    }
    protected Object __cfwr_calc83(Boolean __cfwr_p0) {
        Integer __cfwr_val81 = null;
        while (true) {
            while (true) {
            if ((null - (-65 * null)) && ((-127L & -50.90) * (-760L % 38.91f))) {
            while ((null % null)) {
            while ((null & null)) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 7; __cfwr_i38++) {
            if (('X' << 951L) && true) {
            while (false) {
            while (true) {
            Long __cfwr_node59 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        double __cfwr_obj26 = 27.70;
        return null;
    }
}
