/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if ((-78.97f % (null << -13.60)) || false) {
            return ((true | 'i') << (true * -84L));
        }

    
        boolean __cfwr_node34 = true;
// :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      static Long __cfwr_helper835(int __cfwr_p0, Character __cfwr_p1, Character __cfwr_p2) {
        Object __cfwr_entry37 = null;
        try {
            try {
            while ((null & null)) {
            while ((82.39f - 251)) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        for (int __cfwr_i75 = 0; __cfwr_i75 < 6; __cfwr_i75++) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 4; __cfwr_i13++) {
            if (false && true) {
            try {
            while (((null | false) & (-485 & 'p'))) {
            while (true) {
            if ((null * null) || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        }
        }
        }
        return null;
    }
    private static double __cfwr_compute65(Long __cfwr_p0) {
        return null;
        return null;
        double __cfwr_item57 = 89.60;
        return 51.48;
    }
}
