/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        Double __cfwr_var51 = null;

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      public static int __cfwr_helper306(double __cfwr_p0) {
        try {
            try {
            return "result79";
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        try {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 8; __cfwr_i40++) {
            short __cfwr_obj54 = null;
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        if (false && false) {
            if (true && true) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 7; __cfwr_i71++) {
            try {
            return 826;
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
        }
        }
        return (-29.39 * 185L);
    }
    public static Long __cfwr_util623(long __cfwr_p0) {
        if ((-72.62 << 96.58f) && false) {
            if (false || ((158L << -678) + null)) {
            return false;
        }
        }
        return null;
    }
    public static Character __cfwr_proc657(Character __cfwr_p0) {
        Long __cfwr_result49 = null;
        for (int __cfwr_i21 = 0; __cfwr_i21 < 7; __cfwr_i21++) {
            short __cfwr_val98 = null;
        }
        return null;
    }
}
