/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        try {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 9; __cfwr_i78++) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 5; __cfwr_i19++) {
            try {
            Integer __cfwr_node49 = null;
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        while (false) {
            if (true || (-3.64f / ('L' & null))) {
            short __cfwr_entry66 = null;
        }
            break; // Prevent infinite loops
        }


    int[] banana;
    int @SameLen("a2") [] b = samelen_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      public int __cfwr_func226(Integer __cfwr_p0) {
        while (false) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 3; __cfwr_i19++) {
            try {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 3; __cfwr_i72++) {
            short __cfwr_temp2 = (911L & (532L << 'U'));
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return 664L;
        if (true || true) {
            while (false) {
            try {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 3; __cfwr_i4++) {
            while (false) {
            while (false) {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 6; __cfwr_i76++) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i20 = 0; __cfwr_i20 < 1; __cfwr_i20++) {
            return -40.64f;
        }
        return 319;
    }
    float __cfwr_util355(double __cfwr_p0) {
        for (int __cfwr_i44 = 0; __cfwr_i44 < 3; __cfwr_i44++) {
            Object __cfwr_obj21 = null;
        }
        return "result29";
        if (true || true) {
            while ((-47.84 - (38.80 / 'e'))) {
            try {
            return null;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return 494L;
        return 34.17f;
    }
    protected long __cfwr_func335(Boolean __cfwr_p0, short __cfwr_p1) {
        while ((85.70 ^ (-50.80f + null))) {
            boolean __cfwr_var81 = true;
            break; // Prevent infinite loops
        }
        if (true || false) {
            try {
            try {
            byte __cfwr_result41 = (502L - true);
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
        for (int __cfwr_i47 = 0; __cfwr_i47 < 7; __cfwr_i47++) {
            try {
            try {
            long __cfwr_elem2 = -303L;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
        return 330L;
    }
}
