/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class OffsetsAndConstants_slice {
  public static void main(String[] args) {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 5; __cfwr_i60++) {
            while (false) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 4; __cfwr_i19++) {
            while (false) {
            try {
            while (true) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 7; __cfwr_i20++) {
            try {
            try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 5; __cfwr_i62++) {
            if ((null + (-13.88 | null)) || false) {
            return '9';
        }
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
  }

    static char __cfwr_handle200(int __cfwr_p0, short __cfwr_p1) {
        if (true && false) {
            try {
            return null;
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        }
        return null;
        return 'o';
    }
    static short __cfwr_proc508(double __cfwr_p0, boolean __cfwr_p1, boolean __cfwr_p2) {
        for (int __cfwr_i87 = 0; __cfwr_i87 < 6; __cfwr_i87++) {
            if (true || true) {
            char __cfwr_elem80 = 'm';
        }
        }
        short __cfwr_result11 = null;
        for (int __cfwr_i46 = 0; __cfwr_i46 < 7; __cfwr_i46++) {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 7; __cfwr_i81++) {
            long __cfwr_result67 = -752L;
        }
        }
        return null;
    }
    private static String __cfwr_temp767() {
        while (false) {
            if (false || true) {
            if (false && ((null / null) | (null / 624))) {
            while ((-610L % 996)) {
            if (((-918L + 965L) << null) || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        return "data95";
    }
}