/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        if ((true >> null) || true) {
            try {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 8; __cfwr_i18++) {
            while (false) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 7; __cfwr_i14++) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 8; __cfwr_i31++) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 7; __cfwr_i53++) {
            while (false) {
            if (true || (('i' | '5') | -119L)) {
            try {
            while (true) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 4; __cfwr_i53++) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 4; __cfwr_i94++) {
            if (true && true) {
            if (true && true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 1; __cfwr_i10++) {
            double __cfwr_item6 = 29.39;
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      private short __cfwr_compute19(char __cfwr_p0, Character __cfwr_p1, String __cfwr_p2) {
        long __cfwr_temp7 = (false >> -536);
        try {
            while (true) {
            String __cfwr_node48 = "temp99";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        return (false & -10.58);
    }
    protected static Object __cfwr_util787(Character __cfwr_p0, float __cfwr_p1) {
        Boolean __cfwr_elem74 = null;
        Integer __cfwr_var79 = null;
        if ((16.46f - null) || false) {
            return -22;
        }
        return null;
    }
}
