/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IntroSub_slice {
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        while (('w' / null)) {
            while (true) {
            try {
            if (false || true) {
            Object __cfwr_temp81 = null;
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c =
        if (false && false) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 9; __cfwr_i87++) {
            return null;
        }
        }
 a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
  }

    short __cfwr_util67(Long __cfwr_p0, Integer __cfwr_p1) {
        if (false || false) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 2; __cfwr_i35++) {
            while ((true << 'J')) {
            if (((null & null) + (null / -2.58)) && (17 / -828L)) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 7; __cfwr_i17++) {
            float __cfwr_node36 = -81.66f;
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        if (('P' << null) || true) {
            try {
            if (false || false) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        while (true) {
            try {
            return ((-98.65f & null) * 404L);
        } catch (Exception __cfwr_e15) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (('i' << 685L)) {
            while (false) {
            try {
            try {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 10; __cfwr_i18++) {
            return null;
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return ((-93.76 + null) + 35.43f);
    }
    private Integer __cfwr_func457(String __cfwr_p0, long __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i14 = 0; __cfwr_i14 < 7; __cfwr_i14++) {
            return null;
        }
        for (int __cfwr_i72 = 0; __cfwr_i72 < 1; __cfwr_i72++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 5; __cfwr_i3++) {
            if (true || (89.36f << (42.40 * 909L))) {
            if (true || ((-25L / false) >> -628L)) {
            try {
            Boolean __cfwr_temp55 = null;
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        }
        }
        }
        }
        while (false) {
            while (true) {
            int __cfwr_entry95 = (879 & ('l' - 633L));
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    protected static Double __cfwr_util395() {
        float __cfwr_temp8 = -17.56f;
        return (-277L / (-153L >> true));
        Boolean __cfwr_data88 = null;
        while (false) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 7; __cfwr_i67++) {
            if (false && false) {
            while (((-59.67f - 936L) | (null / 945))) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 7; __cfwr_i42++) {
            while (true) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 7; __cfwr_i63++) {
            while ((-29.26 * -627)) {
            while ((null * (-51.78 + 80.10))) {
            return -93.51;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}