/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        while (true) {
            try {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 3; __cfwr_i14++) {
            while (('i' + -55.79f)) {
            if ((('G' + null) ^ null) && (-41L | null)) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 10; __cfwr_i74++) {
            if (false || true) {
            while (true) {
            try {
            Long __cfwr_temp34 = null;
        } catch (Exception __cfwr_e7) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Preve
        return null;
nt infinite loops
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      protected static Float __cfwr_proc762() {
        for (int __cfwr_i5 = 0; __cfwr_i5 < 1; __cfwr_i5++) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 2; __cfwr_i45++) {
            return true;
        }
        }
        }
        while (true) {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 9; __cfwr_i21++) {
            try {
            try {
            float __cfwr_var56 = -43.26f;
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "data20";
        for (int __cfwr_i77 = 0; __cfwr_i77 < 6; __cfwr_i77++) {
            while (false) {
            try {
            short __cfwr_obj64 = null;
        } catch (Exception __cfwr_e75) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    public Long __cfwr_process192(double __cfwr_p0) {
        Double __cfwr_entry33 = null;
        while (false) {
            try {
            try {
            return true;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (true) {
            while ((null ^ ('B' - false))) {
            if (((true % true) | null) || false) {
            try {
            try {
            try {
            try {
            while (true) {
            try {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 10; __cfwr_i68++) {
            if (true || true) {
            try {
            try {
            while (true) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 8; __cfwr_i35++) {
            Long __cfwr_entry74 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        } catch (Exception __cfwr_e81) {
            // ignore
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i80 = 0; __cfwr_i80 < 6; __cfwr_i80++) {
            return null;
        }
        return null;
    }
}
