/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            Integer __cfwr_node72 = null;
        } catch (Exception __cfwr_e8) {
            // ignore
  
        Long __cfwr_elem14 = null;
      }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      private Character __cfwr_proc944(short __cfwr_p0, byte __cfwr_p1) {
        Object __cfwr_item25 = null;
        for (int __cfwr_i55 = 0; __cfwr_i55 < 3; __cfwr_i55++) {
            while (false) {
            try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 7; __cfwr_i38++) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 6; __cfwr_i74++) {
            return -38.91f;
        }
        }
        } catch (Exception __cfwr_e50) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    public static Long __cfwr_helper922() {
        for (int __cfwr_i80 = 0; __cfwr_i80 < 1; __cfwr_i80++) {
            return null;
        }
        if ((-54.87 / -800) && false) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 5; __cfwr_i84++) {
            while ((770 | null)) {
            return 'X';
            break; // Prevent infinite loops
        }
        }
        }
        if (true && false) {
            char __cfwr_temp65 = 'R';
        }
        return null;
    }
    static float __cfwr_handle112() {
        while (false) {
            try {
            return 491;
        } catch (Exception __cfwr_e46) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return 80.45f;
    }
}
