/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        boolean __cfwr_val41 = (('E' - false) - 
        try {
            if ((16.44f % 60.32f) || false) {
            if (false || false) {
            if (true || false) {
            while (true) {
            Float __cfwr_temp19 = null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
(-73.34f / 183L));

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      protected static Double __cfwr_handle792(Character __cfwr_p0) {
        Character __cfwr_temp5 = null;
        try {
            if (false && true) {
            String __cfwr_elem83 = "data28";
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        if ((-42.21 / null) && false) {
            Object __cfwr_result24 = null;
        }
        return 'f';
        return null;
    }
}
