/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (true && (('L' | 53.68f) & false)) {
            short __cfwr_var64 = (null & (-41.67f % 121L));
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

 
        return 53.37f;
   // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      protected char __cfwr_calc826() {
        if (false || ((null >> -5.03f) % (73.53 & -488L))) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 7; __cfwr_i31++) {
            if (true && true) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 9; __cfwr_i61++) {
            if ((-850L * 22.24f) && true) {
            return null;
        }
        }
        }
        }
        }
        double __cfwr_data73 = -98.35;
        return (78.49 * false);
    }
}
