/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        while (false) {
      
        try {
            int __cfwr_item13 = -854;
        } catch (Exception __cfwr_e59) {
            // ignore
        }
      while (('i' % '2')) {
            while (false) {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      Boolean __cfwr_func813() {
        try {
            while (false) {
            while ((('E' - null) + 747)) {
            try {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 1; __cfwr_i48++) {
            try {
            return 803L;
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        for (int __cfwr_i82 = 0; __cfwr_i82 < 4; __cfwr_i82++) {
            while (true) {
            float __cfwr_data36 = 92.26f;
            break; // Prevent infinite loops
        }
        }
        if (true || false) {
            return 74.67;
        }
        for (int __cfwr_i52 = 0; __cfwr_i52 < 2; __cfwr_i52++) {
            try {
            try {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 5; __cfwr_i40++) {
            if (true || (87.69 / (null | 't'))) {
            Object __cfwr_obj21 = null;
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        }
        return null;
    }
    private boolean __cfwr_proc419(Boolean __cfwr_p0) {
        while (false) {
            return true;
            break; // Prevent infinite loops
        }
        return true;
    }
}
