/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        try {
            return null;
        } catch (Exception __cfwr_e54) {
            // ignore
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      public static int __cfwr_helper464(Character __cfwr_p0, short __cfwr_p1, char __cfwr_p2) {
        try {
            return null;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        if ((null * true) || (54.82f % 262)) {
            try {
            return null;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        return -334;
    }
    private float __cfwr_func597() {
        while (false) {
            if (true || (null ^ null)) {
            if (true && true) {
            while (('8' + -469L)) {
            try {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 6; __cfwr_i29++) {
            double __cfwr_obj25 = (false * null);
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return 20.19f;
    }
}
