/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        if (true || false) {
     
        try {
            return null;
        } catch (Exception __cfwr_e3) {
            // ignore
        }
       while (true) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 3; __cfwr_i46++) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 9; __cfwr_i95++) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 9; __cfwr_i35++) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 10; __cfwr_i67++) {
            try {
            Integer __cfwr_node8 = null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      protected static Float __cfwr_handle984() {
        for (int __cfwr_i18 = 0; __cfwr_i18 < 10; __cfwr_i18++) {
            try {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 6; __cfwr_i77++) {
            if (true && false) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 6; __cfwr_i44++) {
            return null;
        }
        }
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
        try {
            if (false || true) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 7; __cfwr_i99++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        for (int __cfwr_i66 = 0; __cfwr_i66 < 8; __cfwr_i66++) {
            try {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 10; __cfwr_i86++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 2; __cfwr_i7++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        for (int __cfwr_i80 = 0; __cfwr_i80 < 4; __cfwr_i80++) {
            return (-50.91 * 'q');
        }
        return null;
    }
    public byte __cfwr_compute353() {
        String __cfwr_result64 = "world67";
        Boolean __cfwr_elem3 = null;
        return null;
        try {
            Character __cfwr_data25 = null;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        return (20.26f / null);
    }
}
