/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        if (true && true) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 5; __cfwr_i82++) {
            return null;
        }
        }

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.length);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
      protected float __cfwr_handle901(Double __cfwr_p0, boolean __cfwr_p1, int __cfwr_p2) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        return (68.23f >> -465);
    }
}
