/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        try {
            if (false && ((null * null) << 'Y')) {
            if (false || false) {
            try {
            return "result20";
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      protected static String __cfwr_helper89(byte __cfwr_p0, Double __cfwr_p1) {
        Integer __cfwr_result59 = null;
        for (int __cfwr_i26 = 0; __cfwr_i26 < 2; __cfwr_i26++) {
            if (false && false) {
            return ((true >> 53.95) | (5.84 & null));
        }
        }
        return null;
        return "test97";
    }
    static String __cfwr_func133(Long __cfwr_p0) {
        if (false || false) {
            try {
            try {
            while (false) {
            try {
            while ((2.04 * null)) {
            return "temp94";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        }
        for (int __cfwr_i92 = 0; __cfwr_i92 < 10; __cfwr_i92++) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 9; __cfwr_i23++) {
            if (true && false) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 3; __cfwr_i4++) {
            long __cfwr_val20 = -238L;
        }
        }
        }
        }
        return null;
        if ((73.70 / ('H' + 'O')) || true) {
            return (null % null);
        }
        return "item89";
    }
    public Boolean __cfwr_compute737(Float __cfwr_p0, boolean __cfwr_p1) {
        if (true || (61.55f - (21.32f * 321L))) {
            Object __cfwr_entry12 = null;
        }
        return null;
    }
}
