/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        double __cfwr_data88 = 38.53;

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(
        for (int __cfwr_i34 = 0; __cfwr_i34 < 9; __cfwr_i34++) {
            try {
            float __cfwr_temp43 = -88.28f;
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      protected Integer __cfwr_process26(Integer __cfwr_p0, boolean __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            while ((-73L % (-18L + null))) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        long __cfwr_item91 = -155L;
        if ((-64.82f - true) && false) {
            Float __cfwr_var58 = null;
        }
        if (((null ^ -232L) >> -827) || false) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 4; __cfwr_i56++) {
            String __cfwr_result65 = "data72";
        }
        }
        return null;
    }
}
