/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        while (((-98.57f + 16.32f) | 840)) {
            if (true && false) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 6; __cfwr_i
        Double __cfwr_data96 = null;
72++) {
            try {
            while (((null % -88L) / 'P')) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      private Boolean __cfwr_proc203(Object __cfwr_p0, char __cfwr_p1, boolean __cfwr_p2) {
        float __cfwr_val42 = -26.24f;
        for (int __cfwr_i93 = 0; __cfwr_i93 < 7; __cfwr_i93++) {
            try {
            Float __cfwr_temp51 = null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        Object __cfwr_temp73 = null;
        return null;
    }
    private static Character __cfwr_util950() {
        if ((94.46f * -155L) || false) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
        try {
            byte __cfwr_elem21 = ((428 << -914) % (-35.41 / 34.05));
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        return null;
    }
}
