/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        if (false || (null & -10.62)) {
            if (true || true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 7; __cfwr_i10++) {
            return (false & (636 ^ 560));
        }
        }
        }

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array"
        boolean __cfwr_data59 = false;
) int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      private Float __cfwr_handle856(Boolean __cfwr_p0, Double __cfwr_p1) {
        short __cfwr_val36 = (false & (65.96f % 'E'));
        return null;
    }
    static float __cfwr_calc515(boolean __cfwr_p0, Long __cfwr_p1) {
        Character __cfwr_temp7 = null;
        byte __cfwr_elem3 = ('S' - null);
        return 47.55f;
    }
}
