/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class MinLenIndexFor_slice {
  void callTest(int x) {
        int __cfwr_elem72 = 66;

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@Index
        while (false) {
            Integer __cfwr_val99 = null;
            break; // Prevent infinite loops
        }
OrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
  }

    protected static String __cfwr_compute946(float __cfwr_p0, Double __cfwr_p1) {
        double __cfwr_val32 = 55.27;
        Object __cfwr_data66 = null;
        return "value26";
    }
    public long __cfwr_util671() {
        while ((-806L / 'S')) {
            try {
            if (true || (-978L >> null)) {
            return null;
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        Boolean __cfwr_item11 = null;
        return -897L;
    }
    static long __cfwr_helper629(long __cfwr_p0, Float __cfwr_p1) {
        return null;
        try {
            while (false) {
            String __cfwr_temp7 = "test11";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        try {
            Float __cfwr_temp6 = null;
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        return -302L;
        return (-30L >> 28.95);
    }
}