/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        try {
            wh
        for (int __cfwr_i73 = 0; __cfwr_i73 < 5; __cfwr_i73++) {
            if (((-27L << -126) * (-89.27 / -28.13f)) || true) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            Float __cfwr_elem28 = null;
        }
        }
        }
ile (((null | 132L) ^ (null ^ -34.83))) {
            while (true) {
            return false;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      protected double __cfwr_util804(int __cfwr_p0) {
        return -397;
        for (int __cfwr_i92 = 0; __cfwr_i92 < 8; __cfwr_i92++) {
            if (true || true) {
            try {
            Long __cfwr_data45 = null;
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        }
        while ((null ^ 890)) {
            if (true || true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 3; __cfwr_i10++) {
            if (true && false) {
            if (((-413 << null) << 'f') && true) {
            if ((333 >> 45.10f) || false) {
            if (false || false) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 8; __cfwr_i16++) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        if (false && true) {
            while (false) {
            while (true) {
            while (false) {
            try {
            Boolean __cfwr_item2 = null;
        } catch (Exception __cfwr_e24) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return 65.93;
    }
    Integer __cfwr_func504() {
        for (int __cfwr_i3 = 0; __cfwr_i3 < 3; __cfwr_i3++) {
            boolean __cfwr_node29 = false;
        }
        return null;
        return null;
    }
    private byte __cfwr_func868(short __cfwr_p0, Boolean __cfwr_p1) {
        for (int __cfwr_i61 = 0; __cfwr_i61 < 7; __cfwr_i61++) {
            if (false && false) {
            try {
            if (false && (false | 29.64f)) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 5; __cfwr_i66++) {
            long __cfwr_elem17 = ((null / 97.33f) & null);
        }
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        }
        }
        return null;
        if (false && false) {
            if (false && false) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 5; __cfwr_i34++) {
            try {
            while (((81.92 / 'p') % (true / 'l'))) {
            if (true || false) {
            while (((null ^ null) / -967)) {
            try {
            if (((-91.59f * 14.12f) % null) || false) {
            byte __cfwr_data68 = null;
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        }
        }
        return (-497 * 937);
    }
}
