/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (false) {
            while ((null & null)) {
            if (true && false) {
            while (false) {
            try {
            Long __cfwr_obj19 = null;
        } catch (Exception __cfwr_e24) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected long __cfwr_temp701(Long __cfwr_p0, int __cfwr_p1, boolean __cfwr_p2) {
        try {
            return ((null << -752L) + 69.05);
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        for (int __cfwr_i24 = 0; __cfwr_i24 < 4; __cfwr_i24++) {
            try {
            return null;
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
        boolean __cfwr_obj27 = ((null & 677) & 'H');
        try {
            Float __cfwr_val78 = null;
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        return -976L;
    }
    public float __cfwr_aux956() {
        while (false) {
            while (false) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 9; __cfwr_i44++) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 10; __cfwr_i1++) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 2; __cfwr_i74++) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 1; __cfwr_i43++) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 8; __cfwr_i80++) {
            if (false || false) {
            if (false || ((false / 'j') - (true | 80.85f))) {
            Float __cfwr_var58 = null;
        }
        }
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return (null << 'N');
    }
    private static long __cfwr_calc662(Boolean __cfwr_p0, boolean __cfwr_p1, Float __cfwr_p2) {
        Boolean __cfwr_entry31 = null;
        while (false) {
            if (true && false) {
            try {
            if (true || false) {
            return false;
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        while (false) {
            while ((null + (null << 'B'))) {
            return true;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return 942L;
    }
}
