/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        while ((289 ^ (null | null
        if (true && (46 ^ (45.22f + -49.02))) {
            if (true || false) {
            while (false) {
            while (true) {
            return false;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
))) {
            return null;
            break; // Prevent infinite loops
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      public static char __cfwr_util258() {
        try {
            try {
            return null;
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        while (false) {
            while (((-984L * -89.73f) / 21.52f)) {
            double __cfwr_obj33 = 36.14;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return "value56";
        char __cfwr_obj65 = 'q';
        return 'T';
    }
    public static Float __cfwr_process781(int __cfwr_p0, long __cfwr_p1, Integer __cfwr_p2) {
        for (int __cfwr_i48 = 0; __cfwr_i48 < 4; __cfwr_i48++) {
            return null;
        }
        return null;
        try {
            while (true) {
            Integer __cfwr_var68 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        return null;
    }
    private static Object __cfwr_calc730(double __cfwr_p0, int __cfwr_p1) {
        while (false) {
            try {
            if (true || ((null / null) / ('I' * -475))) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 4; __cfwr_i58++) {
            return "value22";
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
