/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        for (int __cfwr_i68 = 
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
0; __cfwr_i68 < 7; __cfwr_i68++) {
            try {
            if (true && true) {
            while (false) {
            while (false) {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 10; __cfwr_i68++) {
            try {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 10; __cfwr_i17++) {
            try {
            while (false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      private static byte __cfwr_process990(int __cfwr_p0, long __cfwr_p1, short __cfwr_p2) {
        for (int __cfwr_i63 = 0; __cfwr_i63 < 7; __cfwr_i63++) {
            return ((false << 668) & false);
        }
        if ((4.14 | 787L) && true) {
            while (false) {
            while (true) {
            return true;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        if (true || true) {
            return 'J';
        }
        return null;
    }
    private static Integer __cfwr_temp745() {
        while (false) {
            while (false) {
            if (true && (true >> -28.15f)) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            while (false) {
            if (false || (93.76f / 946)) {
            while (((null ^ 'B') / -915L)) {
            try {
            return null;
        } catch (Exception __cfwr_e51) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        return -76L;
        return null;
    }
    private static Double __cfwr_helper572(long __cfwr_p0, byte __cfwr_p1, Object __cfwr_p2) {
        while (false) {
            while ((323 % 44.53)) {
            char __cfwr_entry90 = 'z';
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        while (true) {
            if (true || (('y' ^ 997L) & 945L)) {
            if (false && false) {
            try {
            return (-38.10 * -87);
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        return (-49.38f * 751);
        while (true) {
            try {
            while ((null >> (-431 >> 11.51f))) {
            if (((true + 'l') * (-27.96 % null)) || false) {
            Float __cfwr_node9 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
