/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        try {
            return '1';
        } catch (Exception __cfwr_e61) {
            // ignore
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      protected boolean __cfwr_calc310(short __cfwr_p0, Long __cfwr_p1, Float __cfwr_p2) {
        return ((null & null) | -47.27f);
        try {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 1; __cfwr_i87++) {
            Float __cfwr_temp50 = null;
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return ((null / null) % null);
    }
    public Object __cfwr_handle757() {
        try {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 5; __cfwr_i63++) {
            try {
            if (false || true) {
            try {
            if ((-936L / 50.61) || true) {
            return null;
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        return null;
        return null;
    }
}
