/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        for (int __cfwr_i4 = 0; __
        return (949 >> 66.08f);
cfwr_i4 < 8; __cfwr_i4++) {
            while (true) {
            try {
            return -579L;
        } catch (Exception __cfwr_e2) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private byte __cfwr_process552(float __cfwr_p0, boolean __cfwr_p1, Boolean __cfwr_p2) {
        if (true || true) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        if (true && (-373 % 'H')) {
            if (false || true) {
            return (-367 ^ 'J');
        }
        }
        if (true && (null + (false % -7.99))) {
            byte __cfwr_item75 = (446L >> null);
        }
        long __cfwr_obj23 = (34.96 % false);
        return null;
    }
    private Long __cfwr_process50(Boolean __cfwr_p0, Long __cfwr_p1, Character __cfwr_p2) {
        try {
            while (((-944L | null) & (-75.81 + null))) {
            if (false && (null ^ null)) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        return null;
    }
    private byte __cfwr_aux689(int __cfwr_p0, byte __cfwr_p1, Float __cfwr_p2) {
        byte __cfwr_item28 = null;
        if (false || true) {
            Integer __cfwr_data55 = null;
        }
        return (-915L / (-104 + null));
    }
}
