/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if ((null | (92.10f + -793)) || true) {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 10; __cfwr_i52++) {
  
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
          if (((572 / -499L) + (false - 'Y')) && ((false / 45.77f) & (null | false))) {
            if (false && true) {
            if ((true & 884L) && true) {
            while ((-59.22f - null)) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 3; __cfwr_i17++) {
            Object __cfwr_result21 = null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected Object __cfwr_helper461(Boolean __cfwr_p0, Boolean __cfwr_p1, char __cfwr_p2) {
        long __cfwr_val59 = 390L;
        return -956;
        return null;
    }
    protected double __cfwr_util802(Object __cfwr_p0) {
        Boolean __cfwr_temp86 = null;
        return -428;
        int __cfwr_result71 = -10;
        Float __cfwr_result35 = null;
        return 53.92;
    }
    protected long __cfwr_proc577(Character __cfwr_p0, double __cfwr_p1, float __cfwr_p2) {
        Boolean __cfwr_elem2 = null;
        return (-541 ^ true);
    }
}
