/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            return "result35";
        } catch (Exception __cfwr_e17) {
            // ignore
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

 
        try {
            try {
            while (true) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 4; __cfwr_i95++) {
            if (false || true) {
            Float __cfwr_node90 = null;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
   // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      static float __cfwr_compute294(short __cfwr_p0, Boolean __cfwr_p1) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        if (('x' * true) && true) {
            return null;
        }
        if ((588L % -357L) && false) {
            if (true || (505 * null)) {
            if (true || false) {
            while (false) {
            while (false) {
            try {
            try {
            if (false || false) {
            return null;
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        for (int __cfwr_i49 = 0; __cfwr_i49 < 6; __cfwr_i49++) {
            while (true) {
            return (92.87 - (null >> -981L));
            break; // Prevent infinite loops
        }
        }
        return ((-9 / 1.80) % (64.50 / 348));
    }
    static Object __cfwr_calc893(long __cfwr_p0, byte __cfwr_p1, double __cfwr_p2) {
        try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        for (int __cfwr_i64 = 0; __cfwr_i64 < 6; __cfwr_i64++) {
            if (true || false) {
            if (true && true) {
            while (false) {
            Boolean __cfwr_temp59 = null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        return null;
    }
}
