/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class StringIndexOf_slice {
  public static String nocheck(String l, String s) {
        try {
            if (('a' / (true << -87.66f)) || false) {
            return (-99 + null);
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }

    int i
        for (int __cfwr_i75 = 0; __cfwr_i75 < 3; __cfwr_i75++) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 1; __cfwr_i32++) {
            if (true && true) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 4; __cfwr_i87++) {
            if ((-207L / true) || false) {
            if (true || false) {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 9; __cfwr_i76++) {
            try {
            if (true && false) {
            while ((95.44 + -249)) {
            return 'B';
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }
        }
        }
        }
        }
        }
        }
 = l.indexOf(s);
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + s.length());
  }

  public static String remove(String l, String s, int from, boolean last) {
    int i = last ? l.lastIndexOf(s, from) : l.indexOf(s, from);
    if (i >= 0) {
      return l.substring(0, i) + l.substring(i + s.length());
    }
    return l;
  }

  public static String stringLiteral(String l) {
    int i = l.indexOf("constant");
    if (i != -1) {
      return l.substring(0, i) + l.substring(i + "constant".length());
    }
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + "constant".length());
  }

  public static char character(String l, char c) {
    int i = l.indexOf(c);
    if (i > -1) {
      return l.charAt(i);
    }
    // :: error: (argument)
    return l.charAt(i);
  }

    protected static Long __cfwr_compute814() {
        for (int __cfwr_i49 = 0; __cfwr_i49 < 7; __cfwr_i49++) {
            try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 5; __cfwr_i62++) {
            if ((94.51 ^ -800) && ((-781L & null) * (null + 'C'))) {
            float __cfwr_var11 = ('R' / 'H');
        }
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        }
        if ((true ^ (null & -14.24f)) && false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 2; __cfwr_i73++) {
            if (true || true) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 8; __cfwr_i19++) {
            if (true && (null / 133L)) {
            return null;
        }
        }
        }
        }
        }
        return null;
    }
    Boolean __cfwr_compute630(Object __cfwr_p0) {
        while (true) {
            boolean __cfwr_entry94 = (-927L << 0.44);
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}