/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        if (false || (464 | (null << 40.13))) {
            String __cfwr_obj15 = "result38";
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      protected static double __cfwr_aux274() {
        try {
            int __cfwr_entry1 = 175;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        while (false) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 5; __cfwr_i14++) {
            try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 5; __cfwr_i13++) {
            try {
            return null;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        while (false) {
            try {
            Boolean __cfwr_item77 = null;
        } catch (Exception __cfwr_e93) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        long __cfwr_node33 = (-364L << null);
        return 42.70;
    }
    public static long __cfwr_compute951() {
        return null;
        for (int __cfwr_i1 = 0; __cfwr_i1 < 4; __cfwr_i1++) {
            int __cfwr_var86 = (717L & -528);
        }
        if (((null * -55.60) * null) || (-33.61 / (68.67f ^ 'W'))) {
            return null;
        }
        return (53.77f & null);
    }
}
