/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    String __cfwr_compute387(boolean __cfwr_p0, byte __cfwr_p1, int __cfwr_p2) {
        for (int __cfw
        for (int __cfwr_i86 = 0; __cfwr_i86 < 5; __cfwr_i86++) {
            if (true && (998L / true)) {
            try {
            try {
            return (null * true);
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        }
r_i75 = 0; __cfwr_i75 < 4; __cfwr_i75++) {
            if ((-485L | ('8' % 58.40f)) || true) {
            if (true && (('E' ^ null) * null)) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 6; __cfwr_i99++) {
            try {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 10; __cfwr_i56++) {
            while (true) {
            try {
            while (true) {
            while (true) {
            while ((-668 ^ '1')) {
            try {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 5; __cfwr_i80++) {
            long __cfwr_temp57 = (('j' - false) * (-67.56f / -622L));
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        }
        }
        }
        }

        while (true) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            double __cfwr_val71 = ((null ^ -31.33) + -76.67f);
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        if (true || false) {
            return null;
        }
        while (false) {
            try {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 7; __cfwr_i69++) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "item46";
    }
    protected static short __cfwr_process768(float __cfwr_p0, byte __cfwr_p1, Double __cfwr_p2) {
        for (int __cfwr_i8 = 0; __cfwr_i8 < 6; __cfwr_i8++) {
            try {
            Object __cfwr_item53 = null;
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
        while (('N' / null)) {
            try {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 7; __cfwr_i51++) {
            if (true || true) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 4; __cfwr_i33++) {
            if (true || true) {
            Double __cfwr_val11 = null;
        }
        }
        }
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        if (false && false) {
            return ((57.42 << 95.43f) ^ (false % -386L));
        }
        return null;
    }
