/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BinomialTest_slice {
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        short __cfwr_obj59 = (-58.50 / (13.92f % true));

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf(
        try {
            while (true) {
            if ((null / -74.69f) || (null & -275)) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 7; __cfwr_i42++) {
            while (((false / 'F') / true)) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 10; __cfwr_i17++) {
            float __cfwr_result43 = 36.26f;
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
"BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
  }

    public short __cfwr_compute605(short __cfwr_p0, boolean __cfwr_p1, Character __cfwr_p2) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 4; __cfwr_i55++) {
            return null;
        }
        return null;
        while (false) {
            Long __cfwr_obj20 = null;
            break; // Prevent infinite loops
        }
        try {
            byte __cfwr_val44 = (null + null);
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        return null;
    }
    public Object __cfwr_handle87() {
        return null;
        for (int __cfwr_i54 = 0; __cfwr_i54 < 9; __cfwr_i54++) {
            return null;
        }
        for (int __cfwr_i45 = 0; __cfwr_i45 < 9; __cfwr_i45++) {
            char __cfwr_temp50 = ((null / false) << null);
        }
        for (int __cfwr_i38 = 0; __cfwr_i38 < 5; __cfwr_i38++) {
            if (false || true) {
            String __cfwr_item87 = "data12";
        }
        }
        return null;
    }
}