/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        for (int __cfwr_i43 = 0; __cfwr_i43 < 4; __cfwr_i43++) {
            return 'Q';
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      public static Integer __cfwr_func780(Double __cfwr_p0, Boolean __cfwr_p1) {
        for (int __cfwr_i41 = 0; __cfwr_i41 < 4; __cfwr_i41++) {
            int __cfwr_node76 = 662;
        }
        if (((22.49f - null) << null) && ((68.93 >> null) * true)) {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 2; __cfwr_i50++) {
            try {
            while (true) {
            while (false) {
            if (false || true) {
            if (true || (613L >> 'P')) {
            try {
            try {
            while (false) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            double __cfwr_temp10 = 5.09;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        }
        }
        for (int __cfwr_i7 = 0; __cfwr_i7 < 5; __cfwr_i7++) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 9; __cfwr_i21++) {
            Boolean __cfwr_elem33 = null;
        }
        }
        return null;
    }
}
