/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i29 = 0; __cfwr_i29 < 7; __cfwr_i29++) {
            if (false || true) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 6; __cfwr_i49++) {
            if (false || false) {
            try {
            byte __cfwr_node81 = null;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        }
        }
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne 
        while (true) {
            try {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 8; __cfwr_i56++) {
            try {
            try {
            while (false) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 5; __cfwr_i95++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      protected static Object __cfwr_handle78(String __cfwr_p0, Double __cfwr_p1) {
        while ((119 << (-66.14 ^ -622L))) {
            return null;
            break; // Prevent infinite loops
        }
        try {
            return -888L;
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        return null;
    }
    public Float __cfwr_func648(Float __cfwr_p0, char __cfwr_p1, Object __cfwr_p2) {
        return null;
        return null;
    }
    int __cfwr_handle200() {
        if (true && true) {
            if ((83.17f / null) && false) {
            try {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 7; __cfwr_i1++) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 9; __cfwr_i46++) {
            if (('N' ^ 49.29f) || true) {
            long __cfwr_obj62 = 901L;
        }
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
        }
        return 686;
    }
}
