/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        if (true || true) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 1; __cfwr_i67++) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
        }

   
        try {
            while ((69.54 | (null | 'k'))) {
            if (false || false) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 8; __cfwr_i40++) {
            Double __cfwr_entry38 = null;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
 str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      protected static Integer __cfwr_proc215(char __cfwr_p0) {
        char __cfwr_elem72 = (-24.89 | true);
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        return -563L;
        Character __cfwr_data6 = null;
        return null;
    }
    private static Float __cfwr_compute965(float __cfwr_p0, Long __cfwr_p1) {
        while (true) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 1; __cfwr_i93++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        Integer __cfwr_temp3 = null;
        while (false) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 7; __cfwr_i74++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        try {
            if ((736L << (false + -84.15f)) && false) {
            long __cfwr_item91 = ('M' | null);
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        return null;
    }
}
