/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineLTE_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            while ((null - -8.27)) {
            while (true) {
            for (int __cfwr_i83 = 0; __cfwr_i83 < 2; __cfwr_i83++) {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 6; __cfwr_i65++) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 9; __cfwr_i56++) {
            Boolean __cfwr_elem87 =
        return (-52.66 >> false);
 null;
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

    static char __cfwr_aux753(float __cfwr_p0, int __cfwr_p1) {
        while (true) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 1; __cfwr_i61++) {
            while (false) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 10; __cfwr_i70++) {
            char __cfwr_node77 = '4';
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i6 = 0; __cfwr_i6 < 10; __cfwr_i6++) {
            if (true && false) {
            Boolean __cfwr_result22 = null;
        }
        }
        return ('U' << (-483L << 394L));
    }
    Integer __cfwr_aux343() {
        try {
            try {
            if ((96.64f + (-93.17f / null)) && false) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 5; __cfwr_i29++) {
            while (true) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        return null;
    }
}