/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        while (true) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 6; __cfwr_i82++) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 6; __cfwr_i51++) {
            try {
            if ((null | -98.96) || false) {
            return nul
        for (int __cfwr_i55 = 0; __cfwr_i55 < 9; __cfwr_i55++) {
            return null;
        }
l;
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      private static long __cfwr_aux272() {
        if (false || false) {
            try {
            if ((null << 55.78f) || false) {
            Float __cfwr_var4 = null;
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        while (true) {
            while ((-12.37 / -51.83f)) {
            if (((null * null) | 98.63) || false) {
            Boolean __cfwr_temp33 = null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        return -434L;
    }
}
