/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        while (false) {
            while ((57.08 >> (29.05 >> 556L))) {
            long __cfwr_node34 = -658L;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      Object __cfwr_aux144() {
        while (true) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 7; __cfwr_i73++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        Character __cfwr_var22 = null;
        if (((-15.36 - false) << 'R') || false) {
            while (((447L ^ 'G') & (-89.00f ^ 76.08f))) {
            while ((null ^ -18.74)) {
            return -94.77f;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    protected Float __cfwr_util619(Object __cfwr_p0, Object __cfwr_p1) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i46 = 0; __cfwr_i46 < 10; __cfwr_i46++) {
            while ((-27.78 % (-830L - 'v'))) {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 4; __cfwr_i68++) {
            if ((71.02 & (61.64 << -664)) && false) {
            while (true) {
            try {
            try {
            for (int __cfwr_i27 = 0; __cfwr_i27 < 8; __cfwr_i27++) {
            if (((null % null) % null) || true) {
            return null;
        }
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        while (false) {
            boolean __cfwr_data64 = (true + (null - 67.39f));
            break; // Prevent infinite loops
        }
        while (true) {
            while (false) {
            Integer __cfwr_val57 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    int __cfwr_helper278(char __cfwr_p0) {
        if (true || true) {
            return null;
        }
        while (true) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 8; __cfwr_i97++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i44 = 0; __cfwr_i44 < 10; __cfwr_i44++) {
            if (false && (-253L * 394)) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 9; __cfwr_i30++) {
            while (false) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 10; __cfwr_i20++) {
            try {
            if ((-36.30f * 318) || true) {
            while ((-345 & (null >> 'B'))) {
            long __cfwr_obj62 = -98L;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        while (true) {
            try {
            while (true) {
            if (false && true) {
            long __cfwr_entry85 = ('Y' & null);
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return -348;
    }
}
