/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        for (int __cfwr_i69 = 0; __cfwr_i69 < 3; __cfwr_i69++) {
            while (true) {
            try {
            try {
            try {
            return -623L;
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      protected static Float __cfwr_handle274(double __cfwr_p0) {
        try {
            for (int __cfwr_i83 = 0; __cfwr_i83 < 3; __cfwr_i83++) {
            try {
            try {
            boolean __cfwr_val2 = (-924 | -64.20);
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        while ((true % (977 + 434))) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 8; __cfwr_i1++) {
            try {
            return null;
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        if (false && false) {
            if (('z' << false) && true) {
            Float __cfwr_temp66 = null;
        }
        }
        return 4.60f;
        return null;
    }
}
