/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        Double __cfwr_temp65 = null;

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@
        return null;
IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      char __cfwr_calc913(boolean __cfwr_p0) {
        byte __cfwr_item41 = (null * 81L);
        for (int __cfwr_i54 = 0; __cfwr_i54 < 10; __cfwr_i54++) {
            return null;
        }
        if (true && ((null ^ 66.59) | null)) {
            try {
            Character __cfwr_result90 = null;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
        return '9';
    }
}
