/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        return '5';

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      private Double __cfwr_util616() {
        for (int __cfwr_i38 = 0; __cfwr_i38 < 8; __cfwr_i38++) {
            return null;
        }
        return (null + '6');
        return null;
    }
}
