/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        for (int __cfwr_i50 = 0; __cfwr_i50 < 1; __cfwr_i50++) {
            while (true) {
            return "hello98";
            break; // Prevent infinite loops
        }
        }

    int[] banana;
    int @SameLen("a2") [] b = samelen_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void u
        return ((57.49f | false) % 493L);
bc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      protected static Character __cfwr_compute779(Character __cfwr_p0) {
        try {
            try {
            return null;
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        for (int __cfwr_i57 = 0; __cfwr_i57 < 7; __cfwr_i57++) {
            while (true) {
            try {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 1; __cfwr_i88++) {
            try {
            while (false) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 2; __cfwr_i74++) {
            if (true && false) {
            if (true || (null * null)) {
            float __cfwr_var49 = -80.11f;
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    public byte __cfwr_aux586(Boolean __cfwr_p0, long __cfwr_p1) {
        try {
            try {
            return null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        for (int __cfwr_i95 = 0; __cfwr_i95 < 9; __cfwr_i95++) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 1; __cfwr_i53++) {
            Object __cfwr_node21 = null;
        }
        }
        for (int __cfwr_i79 = 0; __cfwr_i79 < 9; __cfwr_i79++) {
            short __cfwr_temp24 = null;
        }
        return null;
    }
}
