/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        float __cfwr_temp1 = ('H' << (568L / -99
        for (int __cfwr_i28 = 0; __cfwr_i28 < 6; __cfwr_i28++) {
            float __cfwr_val50 = 79.14f;
        }
2L));

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      private Object __cfwr_process886(double __cfwr_p0, long __cfwr_p1, byte __cfwr_p2) {
        String __cfwr_elem14 = "temp67";
        return null;
    }
}
