/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class SLSubtyping_slice {
  void subtype(int @SameLen("#2") [] a, int[] b) {
        float __cfwr_temp73 = -76.66f;

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
  }

    public static double __cfwr_compute73
        for (int __cfwr_i94 = 0; __cfwr_i94 < 9; __cfwr_i94++) {
            try {
            try {
            Boolean __cfwr_var51 = null;
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
0(Double __cfwr_p0, Boolean __cfwr_p1) {
        return "value7";
        Integer __cfwr_obj39 = null;
        try {
            Float __cfwr_val11 = null;
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        return ('n' | null);
    }
    static Boolean __cfwr_helper72(Character __cfwr_p0) {
        if (false && false) {
            if (true || true) {
            return null;
        }
        }
        for (int __cfwr_i95 = 0; __cfwr_i95 < 8; __cfwr_i95++) {
            if (true && ((-223 / 'A') | true)) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 7; __cfwr_i32++) {
            try {
            return null;
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
        }
        }
        return null;
    }
}