/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        Character __cfwr_item18 = null;

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

        double __cfwr_temp84 = (-154L ^ (135 + false));

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      public Boolean __cfwr_proc447(float __cfwr_p0) {
        if (true && true) {
            try {
            while (true) {
            if ((-92.43f / -111L) && (95.87 % null)) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 4; __cfwr_i91++) {
            return 293;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        return null;
        return null;
    }
    private static Double __cfwr_temp629() {
        byte __cfwr_entry95 = null;
        return null;
    }
    protected static Float __cfwr_aux683(long __cfwr_p0, Character __cfwr_p1, boolean __cfwr_p2) {
        for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            if (false && true) {
            return null;
        }
        }
        if (false && ((null * -6.38f) * 664L)) {
            if (false && true) {
            short __cfwr_elem37 = (('j' ^ false) / -842);
        }
        }
        char __cfwr_var19 = 'c';
        return null;
    }
}
