/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while (true) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 3; __cfwr_i34++) {
            while ((null / (null << -11.68f))) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 1; __cfwr_i14++) {
            double __cfwr_entry94 = ((433 + null) ^ null);
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected static Object __cfwr_proc198() {
        try {
            long __cfwr_entry85 = -556L;
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        return null;
    }
    private static boolean __cfwr_calc673(float __cfwr_p0) {
        for (int __cfwr_i38 = 0; __cfwr_i38 < 8; __cfwr_i38++) {
            return 84.54;
        }
        for (int __cfwr_i99 = 0; __cfwr_i99 < 7; __cfwr_i99++) {
            return null;
        }
        return false;
    }
}
