/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        while (true) {
            while (((false - 'S') & 92.21f)) {
            while (false) {
            try {
            if (true && ('M' & null)) {
        
        while (true) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 10; __cfwr_i4++) {
            return null;
        }
            break; // Prevent infinite loops
        }
    for (int __cfwr_i55 = 0; __cfwr_i55 < 1; __cfwr_i55++) {
            while (true) {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 10; __cfwr_i54++) {
            if (true || true) {
            try {
            return -35.93f;
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      protected int __cfwr_helper685(String __cfwr_p0, Boolean __cfwr_p1, short __cfwr_p2) {
        try {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            Object __cfwr_item42 = null;
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        return 793;
    }
    Object __cfwr_handle709(Long __cfwr_p0, Double __cfwr_p1) {
        try {
            String __cfwr_val55 = "result92";
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        return null;
    }
    Object __cfwr_proc606(boolean __cfwr_p0) {
        if (true || false) {
            if ((null & 18.46f) || true) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 2; __cfwr_i75++) {
            while (false) {
            if (true || (983L - 86.23)) {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 3; __cfwr_i77++) {
            if (false && (90.47f / null)) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 1; __cfwr_i80++) {
            while (false) {
            while ((258L + 96.24f)) {
            try {
            Float __cfwr_data39 = null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        boolean __cfwr_var50 = false;
        for (int __cfwr_i61 = 0; __cfwr_i61 < 5; __cfwr_i61++) {
            return "test59";
        }
        return null;
    }
}
