/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return 96.94;


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      protected static double __cfwr_temp775() {
        for (int __cfwr_i49 = 0; __cfwr_i49 < 6; __cfwr_i49++) {
            return null;
        }
        try {
            try {
            int __cfwr_var68 = (6.77 % -343L);
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        return null;
        return -29.78;
    }
}
