/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        for (int __cfwr_i19 = 0; __cfwr_i19 < 8; __cfwr_i19++) {
            try {
            while (('d' & false)) {
            if (true || true) {
            return -93L;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        }

    int[] banana;
    int @SameLen("a2") [] b = same
        for (int __cfwr_i74 = 0; __cfwr_i74 < 4; __cfwr_i74++) {
            try {
            if (true || false) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 10; __cfwr_i28++) {
            if (false && true) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 4; __cfwr_i59++) {
            if (true && (25.81f ^ true)) {
            char __cfwr_obj55 = '2';
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
len_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      Character __cfwr_util346(Object __cfwr_p0, short __cfwr_p1, long __cfwr_p2) {
        try {
            try {
            short __cfwr_result69 = null;
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        if ((null >> 162) || (null - 'i')) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 7; __cfwr_i22++) {
            while (true) {
            if (true && (-83.09f >> 890)) {
            return 390L;
        }
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
    static float __cfwr_util771(char __cfwr_p0, char __cfwr_p1, Double __cfwr_p2) {
        String __cfwr_elem43 = "hello2";
        return -74.83;
        try {
            if ((true & (36.88f << 'E')) || true) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 10; __cfwr_i32++) {
            char __cfwr_item56 = 'k';
        }
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        return 45.72f;
    }
    public Integer __cfwr_helper44(boolean __cfwr_p0) {
        try {
            long __cfwr_var39 = -168L;
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        return null;
    }
}
