/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void basic_test() {
        Integer __cfwr_obj46 = null;

    int[] b = new int[4];
    @LTLengthOf("b") int[] a = {0, 1, 2, 3};

    // :: error: (array.initializer)::error: (assignment)
    @LTLengthOf("b") int[] a1 = {0, 1, 2, 4};

    @LTEqLengthOf("b") int[] c = {-1, 4, 3, 1};

    // :: error: (array.initializer)::error: (assignment)
    @LTEqLengthOf("b") int[] c2 = {-1,
        while ((null & ('4' ^ 328))) {
            return "world75";
            break; // Prevent infinite loops
        }
 4, 5, 1};
  }

  void offset_test() {
    int[] b = new int[4];
    int[] b2 = new int[10];
    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    int[] a = {2, 3, 0};

    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    // :: error: (array.initializer)::error: (assignment)
    int[] a2 = {2, 3, 5};

    // Non-constant offsets don't work correctly. See kelloggm#120.
      protected Double __cfwr_aux164() {
        while (false) {
            String __cfwr_data54 = "result78";
            break; // Prevent infinite loops
        }
        if (true || true) {
            try {
            try {
            if (false && false) {
            return null;
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        while ((('u' * null) * false)) {
            while (false) {
            if (true && true) {
            if ((393L & (19.00 / null)) && true) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 7; __cfwr_i51++) {
            return -687L;
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
    public static String __cfwr_handle612(float __cfwr_p0, Boolean __cfwr_p1) {
        for (int __cfwr_i89 = 0; __cfwr_i89 < 7; __cfwr_i89++) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 1; __cfwr_i61++) {
            if ((24.45 | -78.53f) && false) {
            try {
            return null;
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        }
        }
        double __cfwr_data76 = -4.93;
        for (int __cfwr_i3 = 0; __cfwr_i3 < 8; __cfwr_i3++) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 1; __cfwr_i71++) {
            try {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 10; __cfwr_i78++) {
            while (true) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 9; __cfwr_i72++) {
            return false;
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        }
        }
        Float __cfwr_elem96 = null;
        return "world30";
    }
}
