/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        while (false) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 3; __cfwr_i51++) {
            if (true || (false * -537L)) {
            while (false) {
            try {
            while (false) {
            try {
            try {
            try {
            while ((true * (907L / true))) {
            while ((26.91f << ('b' | null))) {
            try {
            if (false && (459L * -14.90)) {
            try {
            if (false && true) {
            byte __cfwr_obj20 = null;
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.length);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
      private static boolean __cfwr_handle537(String __cfwr_p0) {
        while (true) {
            String __cfwr_node51 = "test35";
            break; // Prevent infinite loops
        }
        while (true) {
            Integer __cfwr_elem23 = null;
            break; // Prevent infinite loops
        }
        return (false & 92.69);
    }
    public static String __cfwr_util392(Integer __cfwr_p0) {
        for (int __cfwr_i64 = 0; __cfwr_i64 < 7; __cfwr_i64++) {
            return null;
        }
        return "item1";
    }
}
