/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 8; __cfwr_i55++) {
            Long __cfwr_val94 = null;
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      protected double __cfwr_func422(boolean __cfwr_p0, Long __cfwr_p1, Boolean __cfwr_p2) {
        Double __cfwr_node54 = null;
        if (true && false) {
            try {
            if ((('b' << null) - -122L) && true) {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 5; __cfwr_i79++) {
            while (true) {
            while (false) {
            if (true || false) {
            try {
            if (true && true) {
            return null;
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        if (false && (5.39 >> ('f' | -76.78))) {
            while (false) {
            while (false) {
            Float __cfwr_entry20 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return 79.83;
    }
    private String __cfwr_handle529(boolean __cfwr_p0, char __cfwr_p1, long __cfwr_p2) {
        return (null << (true / null));
        for (int __cfwr_i77 = 0; __cfwr_i77 < 3; __cfwr_i77++) {
            return 'm';
        }
        if (((-78.83f % null) - 909) && (null >> null)) {
            if (false && true) {
            while (true) {
            return -656;
            break; // Prevent infinite loops
        }
        }
        }
        if ((-76.72f | null) || false) {
            if (true && (null | -19.85)) {
            if (true && false) {
            if (false || false) {
            return null;
        }
        }
        }
        }
        return "result90";
    }
    public float __cfwr_aux178(float __cfwr_p0, String __cfwr_p1) {
        try {
            return (-98 + null);
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        return 40.41f;
    }
}
