/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        return -52.04;

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private Character __cfwr_handle45(double __cfwr_p0) {
        try {
            try {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 6; __cfwr_i8++) {
            try {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 1; __cfwr_i37++) {
            while ((null & -45)) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 5; __cfwr_i93++) {
            return ((469L + null) / 88.61);
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        while (false) {
            double __cfwr_item33 = 8.48;
            break; // Prevent infinite loops
        }
        return null;
    }
    public static boolean __cfwr_compute221(boolean __cfwr_p0, byte __cfwr_p1) {
        try {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 8; __cfwr_i97++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 7; __cfwr_i3++) {
            try {
            return (4.89 << (482 >> -914));
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        return false;
    }
    public static short __cfwr_util421() {
        while (true) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 9; __cfwr_i74++) {
            while (false) {
            try {
            return "hello30";
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return (null - 45.48f);
    }
}
