/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while ((false - 93.26)) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 4; __cfwr_i78++) {
            return null;
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: e
        while (false) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 7; __cfwr_i55++) {
            Boolean __cfwr_entry37 = null;
        }
            break; // Prevent infinite loops
        }
rror: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      public static boolean __cfwr_process87(Character __cfwr_p0, short __cfwr_p1, int __cfwr_p2) {
        for (int __cfwr_i95 = 0; __cfwr_i95 < 6; __cfwr_i95++) {
            if (false || (-27.75f | (-455L - 'G'))) {
            try {
            char __cfwr_elem92 = 'l';
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        }
        }
        if (false && true) {
            try {
            return null;
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        while (true) {
            float __cfwr_temp80 = -24.43f;
            break; // Prevent infinite loops
        }
        return true;
    }
}
