/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (((-49.78f / 315) ^ null)) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 6; __cfwr_i35++) {
            float __cfwr_data16 = 2.95f;
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      public long __cfwr_handle237(int __cfwr_p0, char __cfwr_p1, double __cfwr_p2) {
        return null;
        try {
            while ((-47.25 % 'o')) {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 1; __cfwr_i62++) {
            Long __cfwr_item70 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        while (false) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 2; __cfwr_i64++) {
            while (true) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 10; __cfwr_i33++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        try {
            while (true) {
            while (false) {
            char __cfwr_item10 = 'f';
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        return 463L;
    }
    public static short __cfwr_compute548() {
        for (int __cfwr_i75 = 0; __cfwr_i75 < 6; __cfwr_i75++) {
            Boolean __cfwr_entry28 = null;
        }
        if (false || false) {
            if (true || false) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 5; __cfwr_i16++) {
            while (false) {
            if (((null << -17.35) ^ 709L) || false) {
            double __cfwr_data10 = -74.38;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        return false;
        if ((43.63 % 429) || true) {
            while (false) {
            try {
            Integer __cfwr_data31 = null;
        } catch (Exception __cfwr_e49) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
