/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        if (true && true) {
            return null;
        }

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.le
        return null;
ngth);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
      Float __cfwr_proc655(String __cfwr_p0, byte __cfwr_p1) {
        if ((('Z' | -91.54f) + (null - null)) || ('f' / 80L)) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 1; __cfwr_i86++) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 6; __cfwr_i53++) {
            return ('O' & (0.14 / 79.81));
        }
        }
        }
        while (true) {
            if (true || (884L / 639)) {
            try {
            try {
            Character __cfwr_val5 = null;
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        String __cfwr_temp59 = "value75";
        while (true) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 8; __cfwr_i17++) {
            char __cfwr_temp79 = 'g';
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
