/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        return -140L;
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    Boolean __cfwr_calc545() {
        try {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        return null;
    }
    Double __cfwr_util321(byte __cfwr_p0, Object __cfwr_p1) {
        Boolean __cfwr_var77 = null;
        return -832L;
        byte __cfwr_node77 = null;
        return null;
    }
    private boolean __cfwr_process623(short __cfwr_p0, boolean __cfwr_p1) {
        if (('4' >> (744 % 'Q')) || true) {
            return null;
        }
        Object __cfwr_val68 = null;
        if (true && true) {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return true;
    }
}
