/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IntroAnd_slice {
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        while (false) {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 2; __cfwr_i52++) {
            while ((768L + -20.49f)) {
            return (-81.33 | -12.88f);
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
  }

    private static long __cfwr_process931(Float __cfwr_p0, boolean __cfwr_p1, float __cfwr_p2) {
        while ((878L & -338)) {
            while (true) {
            if (true && (('8' % -89.66) ^ (-356 / 52.80f))) {
            if ((('6' - 'r') + 'o') || false) {
            char __cfwr_var28 = ((null << 'D') + (-40.07 << 164L));
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        int __cfwr_result40 = (false | 18.70f);
        Integer __cfwr_var58 = null;
        return (-89.14f % (57.38f << -87.43));
    }
}