/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            try {
            while (((null << -54.58) * (-19.66 ^ 'r'))) {
            while ((('b' ^ 'p') ^ 327)) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 4; __cfwr_i67++) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 9; __cfwr_i24++) {
            try {
            try {
            return -541;
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected static Object __cfwr_aux893(String __cfwr_p0) {
        String __cfwr_temp24 = "item87";
        return null;
    }
    private Double __cfwr_handle910(byte __cfwr_p0) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        String __cfwr_entry6 = "item23";
        try {
            return -644L;
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        while ((51.90f & (-35.95f ^ -695))) {
            if (true || true) {
            return "data10";
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
