/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        return -18.95f;

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      int __cfwr_aux443(Boolean __cfw
        for (int __cfwr_i88 = 0; __cfwr_i88 < 9; __cfwr_i88++) {
            String __cfwr_var62 = "data26";
        }
r_p0, double __cfwr_p1) {
        try {
            try {
            try {
            while ((('x' - 'W') / (null << -825))) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        try {
            Character __cfwr_entry24 = null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        for (int __cfwr_i26 = 0; __cfwr_i26 < 4; __cfwr_i26++) {
            while ((51.93f * 897)) {
            return -54L;
            break; // Prevent infinite loops
        }
        }
        if (false && ((null - 85.62) << (null % false))) {
            try {
            byte __cfwr_result19 = null;
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
        return -680;
    }
    static Object __cfwr_proc307(int __cfwr_p0) {
        int __cfwr_temp40 = 773;
        try {
            try {
            while ((75.21f >> null)) {
            while (false) {
            while (true) {
            while (((null - 733) & (true | null))) {
            while (true) {
            boolean __cfwr_val58 = false;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        while (false) {
            try {
            byte __cfwr_result30 = null;
        } catch (Exception __cfwr_e89) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    char __cfwr_aux407(Float __cfwr_p0, short __cfwr_p1, Boolean __cfwr_p2) {
        char __cfwr_val34 = 'N';
        while (false) {
            while (false) {
            String __cfwr_entry45 = "hello51";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return 'f';
        return (null & 'l');
        return 'j';
    }
}
