/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (false) {
            double __cfwr_item75 = -32.96;
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");
        while (((48.93 % null) + (null * 57.44))) {
            if ((5.50f | (-650L << null)) || true) {
            return 'l';
        }
            break; // Prevent infinite loops
        }


    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      public static byte __cfwr_helper370(Boolean __cfwr_p0) {
        try {
            float __cfwr_elem90 = -46.25f;
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        Character __cfwr_obj37 = null;
        try {
            while (((-626L & null) - (false << null))) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 4; __cfwr_i25++) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 4; __cfwr_i2++) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 2; __cfwr_i15++) {
            try {
            if (true || (74.61f - (null / 0))) {
            return (-386L ^ false);
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        for (int __cfwr_i89 = 0; __cfwr_i89 < 1; __cfwr_i89++) {
            return (82.11f & (-6.52 & 'j'));
        }
        return ((null / 96.65) ^ (829 ^ '3'));
    }
    protected static int __cfwr_func415(float __cfwr_p0, char __cfwr_p1) {
        char __cfwr_temp54 = 'X';
        return 116;
    }
    private static Double __cfwr_helper475() {
        Object __cfwr_result62 = null;
        return null;
        while (true) {
            while ((-43.10 << null)) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}
