/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class UBSubtyping_slice {
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        return (null << ('l' & null));

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
  }

    private Object __cfwr_proc194(long __cfwr_p0, Boolean __cfwr_p1, int __cfwr_p2) {
        while (true) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 10; __cfwr_i41++) {
            try {
            try {
            if (true && false) {
            while (((true | true) % false)) {
            while (false) {
            try {
            if (('s' % 761L) || true) {
            return null;
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        Float __cfwr_elem58 = null;
        if ((true - null) && false) {
            if (true || true) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            while (true) {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 2; __cfwr_i88++) {
            Boolean __cfwr_temp9 = null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        byte __cfwr_result3 = ((660L + false) - ('Z' ^ -579L));
        return null;
    }
    private Boolean __cfwr_util690(String __cfwr_p0, long __cfwr_p1, long __cfwr_p2) {
        if (false && true) {
            boolean __cfwr_item53 = false;
        }
        return null;
    }
    public static Boolean __cfwr_aux330(int __cfwr_p0, float __cfwr_p1, Object __cfwr_p2) {
        for (int __cfwr_i77 = 0; __cfwr_i77 < 9; __cfwr_i77++) {
            return ((-449L | 84.36) ^ false);
        }
        Integer __cfwr_elem42 = null;
        if (false && false) {
            if (true && ('v' - (null * true))) {
            while (false) {
            if (((null ^ null) | (true | null)) || true) {
            return 'E';
        }
            break; // Prevent infinite loops
        }
        }
        }
        if (false && (-24.99 << (false % 'r'))) {
            while (true) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 1; __cfwr_i75++) {
            if (false || false) {
            Double __cfwr_data85 = null;
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}