/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LubIndex_slice {
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        try {
            short __cfwr_temp7 = null;
        } catch (Exception __cfwr_e21) {
            // ignore
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

    private static float __cfwr_helper695(Boolean __cfwr_p0, Integer __cfwr_p1, int __cfwr_p2) {
        double __cfwr_result91 = 74.19;
        long __cfwr_temp83 = 50L;
        if (((-9.64f + false) | (-598 >> 30.29f)) && true) {
            return 'x';
        }
        for (int __cfwr_i39 = 0; __cfwr_i39 < 6; __cfwr_i39++) {
            if (false && false) {
            try {
            boolean __cfwr_val72 = false;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        }
        }
        return (false / -653L);
    }
}