/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        if (true || true) {
            Long __cfwr_elem77 = null;
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @
        while (false) {
            while ((-31.42 ^ 527)) {
            try {
            return (('2' & -666) >> -959);
        } catch (Exception __cfwr_e5) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
LTEqLengthOf({"arr", "arr3"}) int j = test;
      String __cfwr_process760() {
        for (int __cfwr_i90 = 0; __cfwr_i90 < 8; __cfwr_i90++) {
            return null;
        }
        Long __cfwr_item60 = null;
        return "temp45";
    }
    Boolean __cfwr_compute4(Character __cfwr_p0) {
        char __cfwr_obj48 = 'g';
        return null;
    }
    public short __cfwr_util939(String __cfwr_p0, float __cfwr_p1) {
        if (false && ((null * -318L) | null)) {
            while (false) {
            if (true && false) {
            try {
            return null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i91 = 0; __cfwr_i91 < 6; __cfwr_i91++) {
            if (false && false) {
            try {
            while (false) {
            try {
            if (false || true) {
            if (true && true) {
            Character __cfwr_var36 = null;
        }
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        }
        if (false && false) {
            return "hello28";
        }
        if (false || true) {
            return null;
        }
        return null;
    }
}
