/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        boolean __cfwr_data86 = false;

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3
        for (int __cfwr_i55 = 0; __cfwr_i55 < 3; __cfwr_i55++) {
            while (((-972L % null) - null)) {
            while (true) {
            try {
            while (('F' / -364L)) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 6; __cfwr_i5++) {
            return true;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      protected long __cfwr_handle844() {
        for (int __cfwr_i13 = 0; __cfwr_i13 < 2; __cfwr_i13++) {
            while (false) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 6; __cfwr_i46++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        return -145L;
        return (-494L * null);
    }
}
