/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        try {
            if (
        if ((true << null) && true) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 1; __cfwr_i22++) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 10; __cfwr_i26++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 8; __cfwr_i7++) {
            return ((41.94f | null) % 222L);
        }
        }
        }
        }
true && true) {
            return (64.03f + (false - null));
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      protected static String __cfwr_process42() {
        byte __cfwr_node13 = ((53.26 / null) / 40.51f);
        while (true) {
            if (false && false) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 9; __cfwr_i41++) {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 7; __cfwr_i48++) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i78 = 0; __cfwr_i78 < 4; __cfwr_i78++) {
            try {
            try {
            if (false || false) {
            while ((false + (64.39f / 'r'))) {
            try {
            try {
            try {
            try {
            try {
            if (((-39.94f & 13.81f) ^ ('0' - null)) && false) {
            Double __cfwr_data8 = null;
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        }
        return "hello82";
    }
    static Object __cfwr_process43(boolean __cfwr_p0, Integer __cfwr_p1) {
        if (((null % 14.29) | (false | -733L)) && true) {
            while (true) {
            try {
            try {
            return (null % 'h');
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        while (true) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 10; __cfwr_i45++) {
            return -654;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
