/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      double __cfwr_temp204(float __cfwr_p0) {
        return -992;

        while (true) {
            try {
            return false;
        } catch (Exception __cfwr_e38) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            while (true) {
            while ((null * null)) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 10; __cfwr_i47++) {
            long __cfwr_temp26 = -133L;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        return (362L + null);
    }
}
