/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        while (true) {
            Boolean __cfwr_temp21 = null;
            break; // Prevent infinite loops
        }

    int x = a[i & k];
    int x1 = a[k & i]
        return null;
;
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      private Boolean __cfwr_func272(short __cfwr_p0, double __cfwr_p1) {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 1; __cfwr_i60++) {
            return -25.77;
        }
        for (int __cfwr_i69 = 0; __cfwr_i69 < 4; __cfwr_i69++) {
            return -74.93f;
        }
        for (int __cfwr_i46 = 0; __cfwr_i46 < 2; __cfwr_i46++) {
            Object __cfwr_val97 = null;
        }
        return null;
    }
    Float __cfwr_func11(int __cfwr_p0) {
        for (int __cfwr_i52 = 0; __cfwr_i52 < 1; __cfwr_i52++) {
            Integer __cfwr_elem62 = null;
        }
        return -84.43;
        return null;
    }
}
