/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineGTE_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i54 = 0; __cfwr_i54 < 6; __cfwr_i54++) {
            if (((670L ^ 'j') << (983 ^ null)) || false) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 9; __cfwr_i38++) {
            if (false && (null / 362L)) {
            try {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 10; __cfwr_i97++)
        for (int __cfwr_i47 = 0; __cfwr_i47 < 8; __cfwr_i47++) {
            return null;
        }
 {
            return (290L ^ (true >> null));
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        }
        }
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
  }

    private double __cfwr_temp215(Character __cfwr_p0) {
        for (int __cfwr_i96 = 0; __cfwr_i96 < 7; __cfwr_i96++) {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 7; __cfwr_i50++) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 4; __cfwr_i11++) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 10; __cfwr_i44++) {
            return null;
        }
        }
        }
        }
        double __cfwr_node57 = -69.56;
        return (null >> true);
    }
    private static char __cfwr_aux889() {
        while (false) {
            while (true) {
            try {
            return -658L;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return 441L;
        return 'Z';
    }
}