/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        while (((300L ^ -84.64) % 85.08f)) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 5; __cfwr_i7++) {
            Boolean __cfwr_result63 = null;
        }
            break; // Prevent infinite loops
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

   
        return null;
 // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      static Float __cfwr_util391(String __cfwr_p0) {
        if (true || false) {
            try {
            if (true || true) {
            long __cfwr_node92 = ((346L * null) ^ '4');
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        return null;
    }
    protected char __cfwr_aux147(Object __cfwr_p0, Float __cfwr_p1) {
        byte __cfwr_elem90 = null;
        for (int __cfwr_i89 = 0; __cfwr_i89 < 2; __cfwr_i89++) {
            if (false || false) {
            char __cfwr_val21 = 'i';
        }
        }
        if (true && true) {
            try {
            if ((-98.27 * (false + 606)) || false) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 9; __cfwr_i63++) {
            long __cfwr_obj26 = -633L;
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        try {
            char __cfwr_result72 = 'S';
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        return '5';
    }
    public static char __cfwr_func798(float __cfwr_p0, byte __cfwr_p1) {
        try {
            return -89.77f;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        try {
            return true;
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        return null;
        while (((true >> null) | false)) {
            return null;
            break; // Prevent infinite loops
        }
        return ('3' * 992);
    }
}
