/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        try {
            C
        try {
            try {
            try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 8; __cfwr_i38++) {
            try {
            try {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 9; __cfwr_i33++) {
            return (442 + false);
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
haracter __cfwr_entry77 = null;
        } catch (Exception __cfwr_e44) {
            // ignore
        }

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      private static Float __cfwr_helper541() {
        if (false || true) {
            try {
            String __cfwr_temp67 = "test28";
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        }
        for (int __cfwr_i25 = 0; __cfwr_i25 < 8; __cfwr_i25++) {
            int __cfwr_val60 = ((null * null) % (-70.60 / -932L));
        }
        return null;
    }
    protected static short __cfwr_process218() {
        try {
            if (true && ((true * -60.48) / -764L)) {
            return 278L;
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        return null;
    }
    String __cfwr_helper466(Float __cfwr_p0) {
        if (false && false) {
            try {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 2; __cfwr_i43++) {
            float __cfwr_entry77 = 13.47f;
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        }
        Character __cfwr_var97 = null;
        try {
            boolean __cfwr_obj93 = true;
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        return "item44";
        return "hello57";
    }
}
