/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            try {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 9; __cfwr_i37++) {
            try {
            if ((null / null) && true) {
            if (false && true) {
            if ((-769 << (88.80f | -35.44f)) || false) {
            char __cfwr_elem95 = 't';
        }
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      static int __cfwr_helper440(byte __cfwr_p0, Long __cfwr_p1) {
        while ((null >> 'G')) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 2; __cfwr_i69++) {
            try {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 10; __cfwr_i95++) {
            while (true) {
            try {
            try {
            while (false) {
            while (true) {
            return ((false / -803L) % '9');
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        long __cfwr_var41 = 930L;
        return null;
        return -250;
    }
    static int __cfwr_helper479() {
        for (int __cfwr_i94 = 0; __cfwr_i94 < 10; __cfwr_i94++) {
            return -104L;
        }
        Long __cfwr_obj78 = null;
        if (true || ('X' * -2.75)) {
            while (false) {
            try {
            Object __cfwr_var62 = null;
        } catch (Exception __cfwr_e81) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
        return 318;
    }
}
