/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        while (false) {
            if (true || true) {
            if (true && false) {
            try {
            return (220 % (false | null));
        } catch (Exception 
        if (true || true) {
            Integer __cfwr_obj86 = null;
        }
__cfwr_e41) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    public long __cfwr_func101(String __cfwr_p0) {
        char __cfwr_elem47 = 'O';
        while (true) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 10; __cfwr_i23++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 4; __cfwr_i3++) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        if (true || ((-192 % false) - -564)) {
            Float __cfwr_var56 = null;
        }
        return (-890L >> null);
    }
    protected Boolean __cfwr_aux871() {
        for (int __cfwr_i44 = 0; __cfwr_i44 < 5; __cfwr_i44++) {
            if (true && (37.22f >> 137L)) {
            float __cfwr_val28 = -5.74f;
        }
        }
        return 939;
        while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e58) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
