/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        try {
            if (true && true) {
            if ((-205L + (165 + null)) || ((-34.15f >> -166) * true)) {
            try {
   
        for (int __cfwr_i14 = 0; __cfwr_i14 < 8; __cfwr_i14++) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 2; __cfwr_i64++) {
            while (true) {
            if (((null & 663L) << (null | null)) && false) {
            try {
            while (true) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            Integer __cfwr_result19 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
         double __cfwr_result71 = (58.70 >> -8.36f);
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e50) {
            // ignore
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      private int __cfwr_util706(short __cfwr_p0, float __cfwr_p1) {
        while (true) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 7; __cfwr_i32++) {
            while (true) {
            try {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 4; __cfwr_i7++) {
            if (false || ('4' ^ '5')) {
            if (((494L | null) << (null << -599)) || (false ^ (903 - null))) {
            while ((42 + -20)) {
            if (true && true) {
            while (true) {
            if (true || (-538 >> (null * null))) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 2; __cfwr_i38++) {
            double __cfwr_val50 = 31.72;
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        while ((null | (null | -80.82))) {
            if ((613L & null) && ((-552 ^ -33.09f) >> 'Q')) {
            return null;
        }
            break; // Prevent infinite loops
        }
        try {
            return 59.02f;
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        return -161;
    }
}
