/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while (('s' | null)) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 9; __cfwr_i58++) {
            byte __cfwr_elem95 = null;
        }
  
        while ((null | -71.27)) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 9; __cfwr_i61++) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 10; __cfwr_i9++) {
            while (((-23.65 ^ -91.55f) | null)) {
            try {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 9; __cfwr_i64++) {
            return null;
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
          break; // Prevent infinite loops
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      public static Integer __cfwr_helper528(long __cfwr_p0, float __cfwr_p1) {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 10; __cfwr_i60++) {
            Long __cfwr_temp23 = null;
        }
        return 54.72;
        if (true && true) {
            return null;
        }
        for (int __cfwr_i65 = 0; __cfwr_i65 < 7; __cfwr_i65++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 2; __cfwr_i7++) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 8; __cfwr_i45++) {
            while (true) {
            while (false) {
            while (true) {
            if (false && ((791L % 90.42) * (-11.23 | null))) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 2; __cfwr_i25++) {
            short __cfwr_node83 = null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        return null;
    }
    protected Boolean __cfwr_compute312(Object __cfwr_p0) {
        for (int __cfwr_i30 = 0; __cfwr_i30 < 8; __cfwr_i30++) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 1; __cfwr_i37++) {
            if (false && true) {
            return null;
        }
        }
        }
        if (true && ('x' + (-698 << 'I'))) {
            if (true && false) {
            if (false && ('s' << true)) {
            try {
            return null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        }
        }
        return null;
    }
    private static float __cfwr_proc26(short __cfwr_p0, Boolean __cfwr_p1, Long __cfwr_p2) {
        for (int __cfwr_i8 = 0; __cfwr_i8 < 10; __cfwr_i8++) {
            return null;
        }
        return ((-350L ^ null) % -886);
    }
}
