/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Index167_slice {
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        if (((-668L << -14.04f) * null) && (null / (false >> -924L))) {
            short __cfwr_temp98 = null;
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
  }

    private Character __cfwr_process335() {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 3; __cfwr_i21++) {
            Integer __cfwr_data2 = null;
        }
        if (('Y' ^ -562L) && false) {
            while ((-832 ^ -600)) {
            if (false || false) {
            Boolean __cfwr_var10 = null;
        }
            break; // Prevent infinite loops
        }
        }
        while ((-824 << 'r')) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 6; __cfwr_i1++) {
            try {
            try {
            return -30.20;
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 2; __cfwr_i48++) {
            try {
            while (false) {
            try {
            for (int __cfwr_i27 = 0; __cfwr_i27 < 8; __cfwr_i27++) {
            char __cfwr_obj47 = 'w';
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        return null;
    }
    private static Boolean __cfwr_util403(char __cfwr_p0, Character __cfwr_p1) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 3; __cfwr_i72++) {
            while (false) {
            while (true) {
            boolean __cfwr_temp99 = true;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        try {
            try {
            byte __cfwr_data77 = null;
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        return "test87";
        if ((-800L + (null * -81.53f)) || true) {
            if (true || true) {
            if (true && ((98.37 ^ null) + -13L)) {
            return null;
        }
        }
        }
        return null;
    }
    protected int __cfwr_helper130(Double __cfwr_p0) {
        Integer __cfwr_entry48 = null;
        return null;
        return (-95L / null);
    }
}