/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        if (('J' << null) || true) {
           
        for (int __cfwr_i90 = 0; __cfwr_i90 < 4; __cfwr_i90++) {
            if ((-828 >> 62.15) || (null + -44.82)) {
            String __cfwr_elem37 = "value63";
        }
        }
 Character __cfwr_var89 = null;
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      protected char __cfwr_proc995() {
        Boolean __cfwr_obj90 = null;
        for (int __cfwr_i93 = 0; __cfwr_i93 < 7; __cfwr_i93++) {
            int __cfwr_temp25 = -364;
        }
        Boolean __cfwr_temp85 = null;
        for (int __cfwr_i27 = 0; __cfwr_i27 < 6; __cfwr_i27++) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 6; __cfwr_i37++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 5; __cfwr_i73++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 9; __cfwr_i73++) {
            while ((-2.57f | 206)) {
            int __cfwr_var24 = (71.75f + 41.48f);
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        return 'w';
    }
}
