/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (true || true) {
            while (false) {
            return ((-28.33f % -427L) + null);
            break; // Prevent infinite loops
        }
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      static Character __cfwr_handle442(Object __cfwr_p0) {
        try {
            return null;
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        for (int __cfwr_i33 = 0; __cfwr_i33 < 10; __cfwr_i33++) {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        }
        if (false && ((-60.69 << null) - 191L)) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 6; __cfwr_i1++) {
            if (false && false) {
            while ((69.77f >> ('K' >> false))) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 4; __cfwr_i61++) {
            try {
            if (false && true) {
            try {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 8; __cfwr_i74++) {
            while (false) {
            Long __cfwr_item23 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        return null;
    }
    private Float __cfwr_helper448() {
        String __cfwr_result22 = "data11";
        if (true || true) {
            while ((97L / ('E' / true))) {
            if ((776 & (false + 'z')) || true) {
            try {
            if (true && true) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 10; __cfwr_i7++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        while ((true / (29L | '5'))) {
            return true;
            break; // Prevent infinite loops
        }
        return null;
    }
}
