/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        return null;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
      protected Object __cfwr_handl
        return null;
e484(long __cfwr_p0) {
        return -19.50f;
        try {
            short __cfwr_data93 = null;
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        for (int __cfwr_i65 = 0; __cfwr_i65 < 3; __cfwr_i65++) {
            String __cfwr_entry14 = "result35";
        }
        try {
            try {
            try {
            try {
            return 12.14f;
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        return null;
    }
}
