/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        if (false || ('C' * (null + null))) {
            if (((-20.97f >> false) >> 939) && true) {
            return null;
        }
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      public static char __cfwr_aux659(String __cfwr_p0, Object __cfwr_p1) {
        while (false) {
            try {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 6; __cfwr_i68++) {
            return null;
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i39 = 0; __cfwr_i39 < 6; __cfwr_i39++) {
            String __cfwr_item16 = "item18";
        }
        try {
            return ((95L / false) | null);
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        return '8';
    }
}
