/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        return ((-163 % 'p') % (null & -629L));

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, 
        while (true) {
            if (((519 * null) * '2') || true) {
            short __cfwr_result20 = null;
        }
            break; // Prevent infinite loops
        }
@IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      public int __cfwr_temp591(String __cfwr_p0) {
        return null;
        while ((-945L | 856L)) {
            if (false && ('t' + -31.28)) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 9; __cfwr_i69++) {
            if (false && false) {
            try {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 10; __cfwr_i94++) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 2; __cfwr_i84++) {
            Float __cfwr_elem59 = null;
        }
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        return ((243L | null) % 'E');
    }
    protected static double __cfwr_helper585(Character __cfwr_p0, Boolean __cfwr_p1) {
        for (int __cfwr_i41 = 0; __cfwr_i41 < 1; __cfwr_i41++) {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 2; __cfwr_i77++) {
            Character __cfwr_item61 = null;
        }
        }
        if (false || true) {
            while (true) {
            try {
            while (false) {
            if (((true + null) & (714 % '0')) && true) {
            while (((24.43f >> null) & true)) {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 9; __cfwr_i76++) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 9; __cfwr_i80++) {
            return "value40";
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i56 = 0; __cfwr_i56 < 6; __cfwr_i56++) {
            while ((('8' + null) + (-144L + null))) {
            if (false && false) {
            while (((-141L >> null) - (null | false))) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 8; __cfwr_i91++) {
            while (((-73.79f - 459L) + (3.95 * 623))) {
            return 'n';
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        if (true || false) {
            try {
            while ((11.69 - 4.05f)) {
            Float __cfwr_obj13 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        return 5.09;
    }
    static long __cfwr_aux419(Character __cfwr_p0, Integer __cfwr_p1) {
        Long __cfwr_data99 = null;
        try {
            int __cfwr_elem19 = 556;
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        while ((591L % (-299 + 540))) {
            while (true) {
            long __cfwr_val21 = -873L;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return -728L;
    }
}
