/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        if (false && true) {
            boolean __cfwr_temp74 = false;
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
 
        short __cfwr_data38 = ((297L << 6.57f) & null);
   // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      private float __cfwr_func287(String __cfwr_p0, Character __cfwr_p1) {
        return -80.37;
        try {
            if (true && true) {
            while ((-663L << -477)) {
            Long __cfwr_node5 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        return -0.66f;
    }
}
