/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        while (true) {
     
        try {
            return 11L;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
       while (true) {
            while (false) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 6; __cfwr_i42++) {
            try {
            while (false) {
            while (false) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 8; __cfwr_i47++) {
            return false;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      int __cfwr_compute731() {
        if (false || true) {
            while (true) {
            return 933L;
            break; // Prevent infinite loops
        }
        }
        try {
            short __cfwr_result4 = null;
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        if (((-24.43 & 74.40) % -433) && true) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            Boolean __cfwr_result22 = null;
        }
        }
        for (int __cfwr_i50 = 0; __cfwr_i50 < 10; __cfwr_i50++) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 4; __cfwr_i45++) {
            return 80.32;
        }
        }
        return 568;
    }
    public static Long __cfwr_proc213() {
        if (((null >> 89.70f) / (913L << -417)) && true) {
            if (true || (-58.72 << (-9.49f % -73.07))) {
            return "item19";
        }
        }
        return null;
    }
}
