/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i50 = 0; __cfwr_i50 < 6; __cfwr_i50++) {
            while (false) {
            try {
            try {
            while (true) {
            while (true) {
            int __cfwr_result70 = (38.02f >> (41.39 & -77.28));
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private static long __cfwr_proc9(String __cfwr_p0) {
        return null;
        for (int __cfwr_i81 = 0; __cfwr_i81 < 2; __cfwr_i81++) {
            try {
            return ((-90.31 / null) | (false / null));
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        short __cfwr_node85 = null;
        for (int __cfwr_i31 = 0; __cfwr_i31 < 3; __cfwr_i31++) {
            byte __cfwr_data97 = null;
        }
        return 482L;
    }
    static Float __cfwr_helper244() {
        if (true || false) {
            return null;
        }
        if (((-65 - null) + ('Q' << 242)) || (null / (-378L - false))) {
            while (true) {
            try {
            while (false) {
            char __cfwr_elem55 = 'p';
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        while (((null | null) & 12L)) {
            if (true || true) {
            if (true || true) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 7; __cfwr_i39++) {
            while (((-802L - null) | ('o' - -20.64))) {
            try {
            return null;
        } catch (Exception __cfwr_e54) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
