/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 5; __cfwr_i21++) {
            if (true || false) {
            try {
            long __cfwr_temp30 = (694L + 571L);
        } cat
        try {
            short __cfwr_elem25 = null;
        } catch (Exception __cfwr_e60) {
            // ignore
        }
ch (Exception __cfwr_e46) {
            // ignore
        }
        }
        }
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    public static long __cfwr_func186(Integer __cfwr_p0, boolean __cfwr_p1, Character __cfwr_p2) {
        while (true) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 3; __cfwr_i17++) {
            String __cfwr_var6 = "world68";
        }
            break; // Prevent infinite loops
        }
        return -276L;
    }
    private static Character __cfwr_handle321(char __cfwr_p0, Double __cfwr_p1) {
        if (false && false) {
            while ((2.75 | 'j')) {
            try {
            try {
            try {
            float __cfwr_result8 = -10.70f;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        try {
            return null;
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        for (int __cfwr_i30 = 0; __cfwr_i30 < 7; __cfwr_i30++) {
            if (true && false) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 6; __cfwr_i80++) {
            return (null << (-69.18 | 11.35f));
        }
        }
        }
        return null;
    }
}
