/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      public static Character __cfwr_proc739(boolean __cfwr_p0) {
        while (true) {
            Long __cfwr_elem97 = null;
            break; // Prevent infinite loops
        }
        return null;
    }
}
