/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        char __cfwr_var43 = 'c';

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a + 1;
    @LTEqLengthOf({"arr"}) int c1 = a + 1;
    @LTLengthOf({"arr"}) int d = a + 0;
    @LTLengthOf({"arr"}) int e = a + (-7);
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a + 7;
      public static Long __cfwr_util182(double __cfwr_p0, Character __cfwr_p1) {
        try {
            if (true || false) {
            if (false || false) {
            return -14.35f;
        }
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        Character __cfwr_data39 = null;
        return null;
    }
    static Integer __cfwr_temp219() {
        while ((65.78 | null)) {
            if ((277L << -90.55f) || false) {
            try {
            try {
            try {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 3; __cfwr_i42++) {
            if (false && true) {
            return null;
        }
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        for (int __cfwr_i63 = 0; __cfwr_i63 < 5; __cfwr_i63++) {
            while ((813 >> -63.78)) {
            byte __cfwr_elem8 = (null >> null);
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
