/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        while (true) {
            double __cfwr_obj88 = -79.73;
            break; // Prevent infinite loops
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      public short __cfwr_helper230(Float __cfwr_p0) {
        try {
            try {
            if ((-833L >> (90L << -372L)) && true) {
            if (false || (null & (-661L % -54))) {
            try {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        for (int __cfwr_i39 = 0; __cfwr_i39 < 5; __cfwr_i39++) {
            long __cfwr_result72 = -147L;
        }
        try {
            try {
            Float __cfwr_obj59 = null;
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        return "hello50";
        return null;
    }
}

  void test4(@NonNegative int x1, int[] a) {
