/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 7; __cfwr_i13++) {
            try {
            try {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 10; __cfwr_i99++) {
            while ((556 >>
        Long __cfwr_entry55 = null;
 (-24.42f | false))) {
            if (false && false) {
            try {
            try {
            try {
            return 99.00f;
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      static long __cfwr_proc337(long __cfwr_p0) {
        try {
            if (true || true) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 5; __cfwr_i30++) {
            long __cfwr_data66 = (null | null);
        }
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        while (true) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 5; __cfwr_i33++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        while ((-4.98f << 'C')) {
            return ((false ^ null) & 590L);
            break; // Prevent infinite loops
        }
        if (true || false) {
            while (true) {
            while ((-92.71 ^ (null >> true))) {
            try {
            return null;
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return -513L;
    }
    private static float __cfwr_process512() {
        if (true && true) {
            return ((38.33 / -112) - -908L);
        }
        Character __cfwr_var16 = null;
        Object __cfwr_obj48 = null;
        Object __cfwr_val93 = null;
        return 85.71f;
    }
    public static long __cfwr_compute283(boolean __cfwr_p0, byte __cfwr_p1, Integer __cfwr_p2) {
        while (false) {
            return (-173 / null);
            break; // Prevent infinite loops
        }
        Long __cfwr_result43 = null;
        try {
            while (true) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 1; __cfwr_i3++) {
            while (false) {
            while ((-388 - null)) {
            return 78.36;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        return -997L;
    }
}
