/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LessThanLen_slice {
  public static void m4(int @MinLen(1) [] shorter) {
        try {
            while (('e' ^ (774 % 'x'))) {
            while (true) {
            if ((null | 42.93f) || (78.84 / null)) {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 8; __cfwr_i81++) {
            try {
  
        while (false) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 8; __cfwr_i82++) {
            while (true) {
            try {
            if (false || (null & 363)) {
            try {
            try {
            return false;
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
          char __cfwr_elem84 = '3';
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

    private boolean __cfwr_process699() {
        try {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 5; __cfwr_i31++) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 10; __cfwr_i90++) {
            try {
            Boolean __cfwr_item74 = null;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        try {
            return null;
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        try {
            Object __cfwr_result76 = null;
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        return ('g' & -757);
    }
    public Float __cfwr_func681(Integer __cfwr_p0, char __cfwr_p1, char __cfwr_p2) {
        float __cfwr_data85 = 43.80f;
        try {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 10; __cfwr_i71++) {
            return null;
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        try {
            if (true || false) {
            try {
            if (true || true) {
            if (false && (-4.87 | 322)) {
            float __cfwr_item86 = 22.04f;
        }
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        return true;
        return null;
    }
    protected char __cfwr_process779() {
        return (48.53f | 602L);
        if (((182L ^ null) * false) && (-661 ^ -41.97)) {
            while ((47.48 / true)) {
            try {
            return null;
        } catch (Exception __cfwr_e28) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        try {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 6; __cfwr_i2++) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 4; __cfwr_i22++) {
            return 'g';
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        return (null % (null - 'p'));
    }
}