/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        while (false) {
            if (true && false) {
            try {
            try {
            try {
            String __cfwr_obj57 = "item75";
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      static boolean __cfwr_aux694(Long __cfwr_p0, short __cfwr_p1, Boolean __cfwr_p2) {
        return false;
        for (int __cfwr_i98 = 0; __cfwr_i98 < 9; __cfwr_i98++) {
            while (('K' % false)) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 7; __cfwr_i67++) {
            while (false) {
            try {
            while ((null >> 'B')) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        return false;
    }
    private float __cfwr_aux161(short __cfwr_p0, Long __cfwr_p1) {
        for (int __cfwr_i5 = 0; __cfwr_i5 < 1; __cfwr_i5++) {
            return null;
        }
        try {
            while (true) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 8; __cfwr_i16++) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 1; __cfwr_i55++) {
            try {
            return null;
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        return -64.16f;
    }
    private static Long __cfwr_process403(float __cfwr_p0, String __cfwr_p1, double __cfwr_p2) {
        return (7.04f ^ 39.36);
        return null;
    }
}
