/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LessThanFloat_slice {
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;


    static int __cfwr_compute228() {
        while (true) {
            while (true) {
            Double __cfwr_data65 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

        return null;
        for (int __cfwr_i74 = 0; __cfwr_i74 < 9; __cfwr_i74++) {
            Boolean __cfwr_obj47 = null;
        }
        for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            try {
            return ((null * null) & (-613 | -563));
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        for (int __cfwr_i25 = 0; __cfwr_i25 < 1; __cfwr_i25++) {
            try {
            try {
            if (false && ((null * -175) * -35.15)) {
            return null;
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        return 86;
    }
}