/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return (-503 | 'T');

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      public s
        if (true && false) {
            try {
            if ((true * 'H') || true) {
            if (false && true) {
            Double __cfwr_entry47 = null;
        }
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        }
tatic float __cfwr_temp263() {
        for (int __cfwr_i78 = 0; __cfwr_i78 < 2; __cfwr_i78++) {
            while (true) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        while (false) {
            boolean __cfwr_elem18 = (-159 >> (76.81f ^ -127L));
            break; // Prevent infinite loops
        }
        return null;
        Integer __cfwr_elem84 = null;
        return (true * -93);
    }
    public short __cfwr_proc455(byte __cfwr_p0, int __cfwr_p1) {
        if (true && false) {
            if (true || false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        if (true || false) {
            return ((-80.42 * null) / -64.48f);
        }
        return ('l' & -15.39f);
    }
    static boolean __cfwr_aux232(long __cfwr_p0, Long __cfwr_p1) {
        while (false) {
            Double __cfwr_temp17 = null;
            break; // Prevent infinite loops
        }
        try {
            Double __cfwr_obj89 = null;
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        return null;
        return null;
        return true;
    }
}
