/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        boolean __cfwr_elem
        while (false) {
            try {
            if (false || (false * 675L)) {
            while (true) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 9; __cfwr_i22++) {
            try {
            while (true) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 9; __cfwr_i73++) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
            break; // Prevent infinite loops
        }
19 = true;

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      public int __cfwr_process952(double __cfwr_p0, float __cfwr_p1) {
        while ((('b' >> 33.18) << (28.76f >> 53.53))) {
            while (false) {
            if (true || true) {
            if (false || (null >> (-920L << 465))) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 3; __cfwr_i5++) {
            try {
            try {
            try {
            if (false && (-65L / 756)) {
            try {
            if (false && ((null + null) << 'l')) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 7; __cfwr_i97++) {
            if (true && false) {
            try {
            Boolean __cfwr_item57 = null;
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        while (false) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 5; __cfwr_i72++) {
            return -663;
        }
            break; // Prevent infinite loops
        }
        return -631;
    }
}
