/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        return -1.66f;

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      protected char __cfwr_func87(String __cfwr_p0, String __cfwr_p1, Boolean __cfwr_p2) {
        String __cfwr_var72 = "temp64";
        return -141L;
        return (830 + -71.31f);
    }
}
