/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (((412L << null) << -4.84f) || (16.14f ^ -83.34)) {
            long __cfwr_val56 = ((593L - 33.16f) / (false
        for (int __cfwr_i73 = 0; __cfwr_i73 < 4; __cfwr_i73++) {
            return null;
        }
 - false));
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      private static Boolean __cfwr_helper896(boolean __cfwr_p0, double __cfwr_p1, Boolean __cfwr_p2) {
        return null;
        while (false) {
            try {
            return (null << 61.16f);
        } catch (Exception __cfwr_e92) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
