/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        if ((null >> -77.75f) && false) {
            while (false) {
            try {
            while ((2.54f + -949)) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 2; __cfwr_i20++) {
            return '7';
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      private String __cfwr_aux421(float __cfwr_p0, Double __cfwr_p1) {
        Character __cfwr_obj63 = null;
        return "temp82";
    }
}
