/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        while ((-589 & (-597L & -77.26f))) {
            Boolean __cfwr_val21 = null;
            break; // Prevent infinite loops
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      private static Integer __cfwr_temp85(int __cfwr_p0, float __cfwr_p1, int __cfwr_p2) {
        while (false) {
            while ((-54.42 % (-147 - -73.27))) {
            while (true) {
            if (true && true) {
            String __cfwr_temp21 = "hello27";
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i50 = 0; __cfwr_i50 < 4; __cfwr_i50++) {
            if (true || ('9' * null)) {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        }
        return null;
    }
    protected byte __cfwr_process119(Character __cfwr_p0, Long __cfwr_p1) {
        Integer __cfwr_result29 = null;
        return null;
    }
    protected float __cfwr_process651() {
        while (false) {
            try {
            return 75.98f;
        } catch (Exception __cfwr_e67) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        Long __cfwr_data38 = null;
        while (((-168 - -151) >> false)) {
            while (false) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 6; __cfwr_i41++) {
            if (false && false) {
            while (false) {
            try {
            try {
            char __cfwr_obj36 = (-63.80f * 991L);
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        } catch (Exception __cfwr_e66) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i79 = 0; __cfwr_i79 < 2; __cfwr_i79++) {
            try {
            return null;
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        }
        return 9.73f;
    }
}
