/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        while ((793L / -60.49)) {
            return null;
            break; // Prevent infinite loops
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({
        byte __cfwr_data77 = null;
"a", "b", "c"}) int abl2 = identity(abl);
      private boolean __cfwr_calc927(Long __cfwr_p0, boolean __cfwr_p1) {
        try {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 10; __cfwr_i72++) {
            return 196;
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        while (true) {
            try {
            char __cfwr_node1 = (867 & (504L | 41.11f));
        } catch (Exception __cfwr_e55) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return true;
    }
    protected double __cfwr_func65(boolean __cfwr_p0, String __cfwr_p1) {
        while (((322L << null) % 727L)) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
        try {
            while (false) {
            Object __cfwr_obj99 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        if (true || (77.98 - null)) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 1; __cfwr_i95++) {
            return 'J';
        }
        }
        return -86.71;
    }
    boolean __cfwr_compute749(Integer __cfwr_p0, Integer __cfwr_p1) {
        try {
            try {
            if (((null & -61.11) & 'o') && false) {
            while (true) {
            try {
            while (true) {
            while (true) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 8; __cfwr_i36++) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 10; __cfwr_i58++) {
            return -897;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        return true;
    }
}
