/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Issue3224_slice {
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        if (false || true) {
            return null;
        }

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.length);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
  }

    double __cfwr_func378(boolean __cfwr_p0, Integer __cfwr_p1) {
        while (false) {
            if (false && false) {
            try {
            while (false) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 7; __cfwr_i56++) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 8; __cfwr_i15++) {
            if (false && false) {
            try {
            if ((null | null) || true) {
            return null;
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 7; __cfwr_i94++) {
            while (true) {
            Float __cfwr_temp79 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        char __cfwr_elem96 = (null * -32.90);
        return 65.99;
    }
    static double __cfwr_compute723(short __cfwr_p0, char __cfwr_p1) {
        byte __cfwr_temp41 = null;
        if (false || ((15.76f >> -956) + 'I')) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 2; __cfwr_i73++) {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 4; __cfwr_i79++) {
            return null;
        }
        }
        }
        for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            try {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 9; __cfwr_i16++) {
            return null;
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
        while ((-638 / (69.37f / 401L))) {
            Character __cfwr_item27 = null;
            break; // Prevent infinite loops
        }
        return 53.84;
    }
}