/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
      float __cfwr_helper209() {
        if ((-184L + -3.16) && (null & -109)) {
            while (false) {
            try {
    
        while (true) {
            return -57.10f;
            break; // Prevent infinite loops
        }
        float __cfwr_elem84 = 97.59f;
        } catch (Exception __cfwr_e6) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

        try {
            Integer __cfwr_result55 = null;
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        return -97.51f;
    }
}
