/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        Boolean __cfwr_var3 = null;

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private static long __cfwr_helper121(String __cfwr_p0, Object __cfwr_p1) {
        while (false) {
            if (false && false) {
            try {
            while (false) {
            double __cfwr_temp98 = (-59.41f | 98.88);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return "result5";
        while (true) {
            while (false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if ((null / (-629 + 's')) && (392L << (true / 231))) {
            if (false || ((-27.93f | true) ^ 'V')) {
            Boolean __cfwr_node81 = null;
        }
        }
        return (null | -406);
    }
    private static Long __cfwr_temp673(long __cfwr_p0, Character __cfwr_p1, Boolean __cfwr_p2) {
        try {
            while (true) {
            if ((null ^ 604) && true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        try {
            if (((null ^ true) >> -59.57f) || false) {
            if (true || false) {
            while ((-58.16 - (false % null))) {
            if (true && ('h' % true)) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 8; __cfwr_i55++) {
            if (true && true) {
            while (true) {
            while (true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 6; __cfwr_i10++) {
            long __cfwr_item39 = -515L;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        Integer __cfwr_var88 = null;
        return null;
    }
    private static Double __cfwr_helper520() {
        while ((364 / -47L)) {
            if (true && true) {
            Long __cfwr_temp42 = null;
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 6; __cfwr_i7++) {
            return null;
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        return null;
    }
}
