/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineNeq_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        return 'J';

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
  }

    public static float __cfwr_helper229(String __cfwr_p0, Double __cfwr_p1) {
        try {
            if (true && true) {
            try {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 9; __cfwr_i44++) {
            Integer __cfwr_node72 = null;
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        if (false || false) {
            return null;
        }
        String __cfwr_val44 = "data30";
        return -49.91f;
    }
    private Integer __cfwr_aux922(Double __cfwr_p0) {
        if (false && (('v' / null) - false)) {
            if (true && ((89 % -75.96) >> null)) {
            if (true || (null % (-893 & true))) {
            if (false && true) {
            if (((-60.97 - 91.77) ^ 28.27) || (-29.90f % true)) {
            return null;
        }
        }
        }
        }
        }
        for (int __cfwr_i82 = 0; __cfwr_i82 < 10; __cfwr_i82++) {
            if (true && (50.12 + null)) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 10; __cfwr_i72++) {
            try {
            try {
            return -223;
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        }
        }
        return null;
    }
    Boolean __cfwr_helper50(byte __cfwr_p0) {
        while ((740L % -39.83f)) {
            return null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i80 = 0; __cfwr_i80 < 10; __cfwr_i80++) {
            long __cfwr_node33 = -35L;
        }
        try {
            if (false || false) {
            if (((null & true) + -970) || false) {
            return null;
        }
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        return null;
    }
}