/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        while (true) {
            String __cfwr_data70 = "item92";
            break; // Prevent infinite loops
     
        while ((487 << false)) {
            Character __cfwr_node25 = null;
            break; // Prevent infinite loops
        }
   }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test4(@NonNegative int x1, int[] a) {
}    char __cfwr_helper316(Float __cfwr_p0) {
        return ((null + 28.81) ^ '5');
        return 'S';
    }
    private static Double __cfwr_compute47(Double __cfwr_p0, Float __cfwr_p1, double __cfwr_p2) {
        while (false) {
            try {
            while ((-795L ^ 60.80)) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 9; __cfwr_i12++) {
            try {
            String __cfwr_temp24 = "world95";
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        if (true && true) {
            try {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 3; __cfwr_i63++) {
            byte __cfwr_obj16 = null;
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
        return null;
    }
    public static Character __cfwr_util18(Object __cfwr_p0, Character __cfwr_p1, boolean __cfwr_p2) {
        Boolean __cfwr_val75 = null;
        return null;
    }
}