/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        while (true) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 10; __cfwr_i35++) {
            while (true) 
        long __cfwr_entry10 = (false / -81.16);
{
            if (false || false) {
            Long __cfwr_elem4 = null;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      private Object __cfwr_process817(Integer __cfwr_p0) {
        if (true && false) {
            char __cfwr_node14 = 'r';
        }
        short __cfwr_item89 = null;
        try {
            return null;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        return null;
    }
    static double __cfwr_temp633(double __cfwr_p0, Float __cfwr_p1) {
        if (true || true) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 6; __cfwr_i18++) {
            return null;
        }
        }
        return (54.19 >> -37.25);
    }
    protected double __cfwr_handle90(Boolean __cfwr_p0, Integer __cfwr_p1, boolean __cfwr_p2) {
        if (false && ((814 ^ -39.91f) | (-822 - 231L))) {
            int __cfwr_temp27 = -100;
        }
        Double __cfwr_data99 = null;
        return 'X';
        try {
            try {
            if (((-89.31 - 85.41) >> null) || false) {
            while ((null / null)) {
            while (false) {
            while ((70.25f ^ 81.67f)) {
            if (false && (266L >> ('6' << null))) {
            try {
            while (false) {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 4; __cfwr_i48++) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 4; __cfwr_i10++) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 10; __cfwr_i87++) {
            if (true || false) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        return 35.11;
    }
}

  void test4(@NonNegative int x1, int[] a) {
