/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 6; __cfwr_i94++) {
            if ((null >> 'K') || false) {
            try {
            return false;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private byte __cfwr_util814(float __cfwr_p0, Object __cfwr_p1) {
        try {
            double __cfwr_item31 = 38.71;
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        if (false && false) {
            while (true) {
            try {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 7; __cfwr_i71++) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            try {
            return null;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    static Integer __cfwr_compute958() {
        try {
            try {
            return -44L;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        if ((true | (9.96 ^ -78.22)) || true) {
            return null;
        }
        return null;
    }
}
