/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        for (int __cfwr_i95 = 0; __cfwr_i95 < 6; __cfwr_i95++) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 5; __cfwr_i67++) {
            while (true) {
            Long __cfwr_var99 = null;
            break; // Prevent infinite loops
        }
        }
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      protected double __cfwr_handle845(Integer __cfwr_p0, Object __cfwr_p1, float __cfwr_p2) {
        try {
            try {
            while (true) {
            if (true || true) {
            while (true) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 3; __cfwr_i51++) {
            try {
            return null;
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        for (int __cfwr_i97 = 0; __cfwr_i97 < 7; __cfwr_i97++) {
            return -666;
        }
        return 48.38;
    }
    protected Integer __cfwr_aux422() {
        try {
            while (('J' | null)) {
            return 166L;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        return null;
    }
}
