/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        while ((-23.33f / (-573 ^ true))) {
            int __cfwr_var28 = ('z' & (true + null));
            break; // Prevent infinite loops
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      public static short __cfwr_aux973() {
        return false;
        return null;
    }
    Integer __cfwr_proc492(int __cfwr_p0) {
        try {
            while (true) {
            short __cfwr_val94 = (('D' << -893) ^ null);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        for (int __cfwr_i1 = 0; __cfwr_i1 < 10; __cfwr_i1++) {
            if (false || true) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 7; __cfwr_i2++) {
            try {
            return true;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
        }
        }
        for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            while ((false ^ (true % true))) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 3; __cfwr_i70++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
