/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        float __cf
        if (true || (13.54f % (true / 176L))) {
            while (false) {
            Boolean __cfwr_item6 = null;
            break; // Prevent infinite loops
        }
        }
wr_result94 = ((896 - -494) ^ null);

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      public static long __cfwr_compute83(boolean __cfwr_p0, Object __cfwr_p1) {
        return null;
        Boolean __cfwr_elem3 = null;
        for (int __cfwr_i69 = 0; __cfwr_i69 < 7; __cfwr_i69++) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 3; __cfwr_i80++) {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 5; __cfwr_i81++) {
            while (((-61.46 << -991L) | 78.77f)) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 4; __cfwr_i72++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 2; __cfwr_i73++) {
            try {
            try {
            while (((-77.80 >> null) ^ (false + 486L))) {
            try {
            if (false && ((827 << null) & null)) {
            while (false) {
            try {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 1; __cfwr_i88++) {
            try {
            if (true && true) {
            return 1.06f;
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        return (452L & (-481 + 'a'));
    }
}
