/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while ((-21.58 / false)) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 8; __cfwr_i59++) {
            double __cfwr_elem15 = -7.75;
     
        if ((42.85 & (-180L << true)) && false) {
            while ((null / null)) {
            return (null - 136L);
            break; // Prevent infinite loops
        }
        }
   }
            break; // Prevent infinite loops
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected Boolean __cfwr_helper134() {
        return (443L + null);
        try {
            return null;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        for (int __cfwr_i26 = 0; __cfwr_i26 < 9; __cfwr_i26++) {
            Boolean __cfwr_entry17 = null;
        }
        return null;
        return null;
    }
}
