/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        Character __cfwr_node45 = null;

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      public static boolean __cfwr_compute738(byte __cfwr_p0) {
        for (int __cfwr_i81 = 0; __cfwr_i81 < 2; __cfwr_i81++) {
            return null;
        }
        try {
            if (false && false) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 3; __cfwr_i40++) {
            if (false || true) {
            while (((599L & -84.96) / -743)) {
            try {
            if (true || true) {
            Double __cfwr_result24 = null;
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        while ((-876L >> (true % 91.16f))) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return false;
    }
}
