/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Index167_slice {
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        for (int _
        for (int __cfwr_i9 = 0; __cfwr_i9 < 8; __cfwr_i9++) {
            if ((-80.76 & (-902 & -719)) || true) {
            if (false && false) {
            try {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 8; __cfwr_i34++) {
            char __cfwr_data69 = ('1' / false);
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
        }
        }
_cfwr_i96 = 0; __cfwr_i96 < 7; __cfwr_i96++) {
            if (true || (57.96 * -770)) {
            Integer __cfwr_data21 = null;
        }
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
  }

    protected static Double __cfwr_helper614(boolean __cfwr_p0, Long __cfwr_p1, Boolean __cfwr_p2) {
        for (int __cfwr_i29 = 0; __cfwr_i29 < 4; __cfwr_i29++) {
            try {
            return 42.58f;
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        if (true && true) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 7; __cfwr_i9++) {
            while (false) {
            if (true || true) {
            try {
            while (((-53.18 >> 87.97f) & null)) {
            Long __cfwr_elem39 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        return ('Y' & -923);
        return null;
        return null;
    }
    byte __cfwr_helper99(int __cfwr_p0) {
        if (('3' >> null) || (null / true)) {
            Double __cfwr_var53 = null;
        }
        return "hello56";
        try {
            if (false || true) {
            while (((-75 / true) >> (87.03f & -86.39))) {
            while ((-533L >> -458L)) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 6; __cfwr_i16++) {
            while (true) {
            while (false) {
            if (false || false) {
            String __cfwr_data61 = "value63";
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        while ((-80.74 - -28.72)) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 4; __cfwr_i4++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}