/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i24 = 0; __cfwr_i24 < 8; __cfwr_i24++) {
            while (true) {
            try {
            if (true && true) {
            long __cfwr_val33 = (63.93f * -36.54);
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      private Character __cfwr_calc556() {
        long __cfwr_data89 = -52L;
        try {
            try {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 9; __cfwr_i74++) {
            return null;
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        Character __cfwr_node9 = null;
        for (int __cfwr_i12 = 0; __cfwr_i12 < 8; __cfwr_i12++) {
            try {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 3; __cfwr_i18++) {
            return 663L;
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        }
        return null;
    }
}
