/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i18 = 0; __cfwr_i18 < 3; __cfwr_i18++) {
            return null;
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      protected char __cfwr_helper498(byte __cfwr_p0, short __cfwr_p1, long __cfwr_p2) {
        for (int __cfwr_i89 = 0; __cfwr_i89 < 5; __cfwr_i89++) {
            while (false) {
            return false;
            break; // Prevent infinite loops
        }
        }
        return 227;
        return (535 / (620 * -8.32f));
    }
    public String __cfwr_temp654() {
        try {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 7; __cfwr_i76++) {
            if (false && true) {
            if (('9' * -792) && false) {
            while (((935 << 61.70f) % (-531 & 'i'))) {
            boolean __cfwr_entry83 = ((-78.37f >> null) / null);
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        for (int __cfwr_i36 = 0; __cfwr_i36 < 3; __cfwr_i36++) {
            while (false) {
            while (true) {
            while ((null ^ null)) {
            while (false) {
            if (((null ^ -399L) ^ (false / null)) || true) {
            return 362;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i3 = 0; __cfwr_i3 < 3; __cfwr_i3++) {
            try {
            return false;
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        }
        return "temp72";
    }
}
