/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        for (int __cfwr_i87 = 0; __cfwr_i87 < 7; __cfwr_i87++) {
            try {
            if (false && ((628L + 61.29f) * -837)) {
            return null;
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      private short __cfwr_aux570(float __cfwr_p0) {
        try {
            if (((10.95 << 175L) >> 970L) || true) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        return (-928L | (-58.09 / -66.84));
    }
    public byte __cfwr_compute325(Character __cfwr_p0, boolean __cfwr_p1) {
        return -730L;
        return null;
        Boolean __cfwr_item6 = null;
        while (false) {
            while (false) {
            try {
            try {
            while ((true + (null + null))) {
            while (false) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 5; __cfwr_i57++) {
            try {
            try {
            if (true || false) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 1; __cfwr_i2++) {
            if (false && (-582 << (-570 + 'm'))) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 9; __cfwr_i89++) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 9; __cfwr_i84++) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 6; __cfwr_i24++) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 2; __cfwr_i51++) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 8; __cfwr_i9++) {
            try {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 8; __cfwr_i28++) {
            while ((-4.44f << 93.90)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        }
        }
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
