/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return ((4.31 >> 662) / 52L);

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      protected int __cfwr_handle25() {
        if (false && false) {
            try {
            double __cfwr_item48 = ((null & -47.70f) & (null % null));
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        }
        while ((130 - null)) {
            while (true) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 7; __cfwr_i5++) {
            if (false || (40L % true)) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 6; __cfwr_i18++) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 3; __cfwr_i56++) {
            if ((-620 + 13.36) && false) {
            try {
            try {
            while ((null + 439)) {
            return "hello51";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            if (false && true) {
            if ((4.58f / null) || true) {
            if (false && (-6.26f - 'p')) {
            float __cfwr_elem60 = (null * 'H');
        }
        }
        }
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        return -973;
    }
}
