/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        for (int __cfwr_i35 
        float __cfwr_item41 = 78.47f;
= 0; __cfwr_i35 < 5; __cfwr_i35++) {
            Boolean __cfwr_item97 = null;
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      public byte __cfwr_util291(String __cfwr_p0) {
        try {
            if (false || false) {
            Long __cfwr_obj70 = null;
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        if (false || (-245 | null)) {
            return (84.46f - -397);
        }
        return (false - (true + -22.41f));
    }
    public static long __cfwr_func31(Boolean __cfwr_p0, boolean __cfwr_p1) {
        while ((null & (30.52 & null))) {
            try {
            try {
            try {
            if (false && true) {
            while (true) {
            while (false) {
            Float __cfwr_item48 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i41 = 0; __cfwr_i41 < 8; __cfwr_i41++) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 2; __cfwr_i34++) {
            for (int __cfwr_i27 = 0; __cfwr_i27 < 10; __cfwr_i27++) {
            while (true) {
            try {
            while ((false % (null % null))) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        try {
            try {
            return -53.07;
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        if (false || true) {
            return null;
        }
        return -781L;
    }
    private static char __cfwr_handle373(byte __cfwr_p0, Boolean __cfwr_p1, Character __cfwr_p2) {
        return null;
        while (true) {
            Float __cfwr_item23 = null;
            break; // Prevent infinite loops
        }
        return (-505L - 76.69);
    }
}
