/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        for (int __cfwr_i71 = 0; __cfwr_i71 < 2; __cfwr_i71++) {
            return -94.51f;
        }

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      public static char __cfwr_aux248() {
        for (int __cfwr_i50 = 0; __cfwr_i50 < 8; __cfwr_i50++) {
            while ((78.38f + (null * -612L))) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 8; __cfwr_i23++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        if (false && false) {
            if (true || true) {
            try {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 1; __cfwr_i24++) {
            while (false) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 6; __cfwr_i15++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        }
        return 'j';
    }
    protected static Integer __cfwr_handle140(short __cfwr_p0, int __cfwr_p1) {
        for (int __cfwr_i89 = 0; __cfwr_i89 < 9; __cfwr_i89++) {
            while (true) {
            while ((null * -70.21f)) {
            try {
            return 36.20f;
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        if (true && true) {
            return ((35.28f & 'q') / 181L);
        }
        for (int __cfwr_i68 = 0; __cfwr_i68 < 5; __cfwr_i68++) {
            return ((72 * null) * 72.02);
        }
        return null;
    }
    protected static float __cfwr_aux492() {
        try {
            while (true) {
            while (true) {
            if (true || true) {
            if (true && true) {
            while ((68.49f - 463L)) {
            try {
            try {
            try {
            if (true || false) {
            return null;
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        String __cfwr_entry35 = "value47";
        for (int __cfwr_i40 = 0; __cfwr_i40 < 1; __cfwr_i40++) {
            while (true) {
            Integer __cfwr_obj46 = null;
            break; // Prevent infinite loops
        }
        }
        return 3.76f;
    }
}
