/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i41 = 0; __cfwr_i41 < 3; __cfwr_i41++) {
            return -704L;
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private byte __cfwr_compute360(int __cfwr_p0, Integer __cfwr_p1) {
        if (false || ((null >> 'D') * null)) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 5; __cfwr_i57++) {
            if (false || ((null + null) * null)) {
            return null;
        }
        }
        }
        return (null % (-36.47 / 27));
    }
    static int __cfwr_proc376(Float __cfwr_p0) {
        while (true) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 3; __cfwr_i64++) {
            try {
            try {
            if (((0.83 / null) | null) && true) {
            try {
            if ((-168 * 13.59) && false) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 9; __cfwr_i3++) {
            float __cfwr_obj88 = 32.63f;
        }
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        boolean __cfwr_result63 = false;
        while (false) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 9; __cfwr_i98++) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 8; __cfwr_i40++) {
            if (true || true) {
            try {
            while (false) {
            while ((792L & null)) {
            try {
            Character __cfwr_data51 = null;
        } catch (Exception __cfwr_e37) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        return ((14.67 ^ null) / '7');
    }
}
