/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IndexForTestLBC_slice {
  void callTest1(int x) {
        int __cfwr_data52 = 26;

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length 
        try {
            if (((null & null) + -18.37f) || true) {
            try {
            while (false) {
            try {
            try {
            int __cfwr_node12 = -673;
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
- 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
  }

    protected static Float __cfwr_process209(short __cfwr_p0, Character __cfwr_p1, long __cfwr_p2) {
        for (int __cfwr_i97 = 0; __cfwr_i97 < 6; __cfwr_i97++) {
            return (true ^ (829 >> false));
        }
        return null;
    }
}