/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        for (int __cfwr_i15 = 0; __cfwr_i15 < 9; __cfwr_i15++) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 7; __cfwr_i24++) {
            char __cfwr_obj6 = 'C';
        }
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      private static boolean __cfwr_handle868() {
        for (int __cfwr_i23 = 0; __cfwr_i23 < 6; __cfwr_i23++) {
            try {
            try {
            return "item7";
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
        for (int __cfwr_i40 = 0; __cfwr_i40 < 4; __cfwr_i40++) {
            double __cfwr_entry64 = (-39.95 << -93.06);
        }
        return ((-63.27 / 688) & (null * null));
    }
    public static char __cfwr_func350() {
        if (true || (true | (-834 ^ null))) {
            Integer __cfwr_val62 = null;
        }
        for (int __cfwr_i68 = 0; __cfwr_i68 < 3; __cfwr_i68++) {
            while ((true & 86.06f)) {
            return (40.48f * -21.45);
            break; // Prevent infinite loops
        }
        }
        while ((-845L << -464L)) {
            if (false || false) {
            if (((-646 / 38.05f) % null) || true) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 3; __cfwr_i32++) {
            while (false) {
            while (false) {
            Integer __cfwr_var29 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        return (-60.38 << null);
    }
}
