/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IndexOrLowTests_slice {
  void test() {
        if (false || tr
        return 'V';
ue) {
            try {
            return (673 >> (172L >> 933));
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
  }

    private static Integer __cfwr_util638(short __cfwr_p0) {
        while ((null * false)) {
            return null;
            break; // Prevent infinite loops
        }
        return null;
    }
    public static boolean __cfwr_func910(Object __cfwr_p0, int __cfwr_p1, Double __cfwr_p2) {
        return null;
        return (null % -64L);
    }
    protected Boolean __cfwr_calc880() {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        return null;
    }
}