/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return null;

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      public static b
        if (false || false) {
            while (false) {
            while (((null + 2.53) >> false)) {
            try {
            while (true) {
            int __cfwr_node97 = ((74.89 ^ null) % -423L);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
yte __cfwr_temp891() {
        if (false || true) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 6; __cfwr_i85++) {
            try {
            while ((false << false)) {
            try {
            boolean __cfwr_entry75 = false;
        } catch (Exception __cfwr_e69) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        }
        }
        long __cfwr_var45 = ((752L | null) + -73.28);
        return ((false - 466L) - (34.10 + true));
    }
    float __cfwr_aux963(String __cfwr_p0) {
        try {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 9; __cfwr_i31++) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 1; __cfwr_i58++) {
            char __cfwr_result35 = 'u';
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        Character __cfwr_data97 = null;
        return -16.44f;
    }
    private double __cfwr_func137() {
        try {
            if (false && false) {
            if (false && (null * (true / -130L))) {
            while ((-507L & null)) {
            return ((-699L + null) + null);
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        return 59.58;
    }
}
