/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        boolean __cfwr_
        Integer __cfwr_var86 = null;
item43 = (93.03f | null);


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      private static Boolean __cfwr_compute345(char __cfwr_p0, Boolean __cfwr_p1) {
        try {
            try {
            Integer __cfwr_result80 = null;
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        if ((null / (false % 'l')) || true) {
            return 18.08f;
        }
        try {
            while (true) {
            Double __cfwr_data71 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        while (((431 + 'b') & -451L)) {
            return 'e';
            break; // Prevent infinite loops
        }
        return null;
    }
}
