/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    protected static float __cfwr_temp572(Float __cfwr_p0, Double __cfwr_p1, byte __cfwr_p2) {
        while (false) {
            while (false) {
 
        return null;
           return ('l' << 'u');
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

        try {
            if (true || (null | 998)) {
            Long __cfwr_temp86 = null;
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        try {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 4; __cfwr_i15++) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 2; __cfwr_i39++) {
            if (true || (null / (66.43 ^ -96.13f))) {
            return "data33";
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        return -629L;
        return (-544L / ('x' - null));
    }
    private char __cfwr_temp381(Long __cfwr_p0, boolean __cfwr_p1, double __cfwr_p2) {
        return null;
        return (-64.93f / 40L);
    }
    protected static short __cfwr_proc731(Long __cfwr_p0) {
        while (false) {
            if (false || (null + '3')) {
            return null;
        }
            break; // Prevent infinite loops
        }
        short __cfwr_obj60 = null;
        return (93.86 + 547);
    }
