/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 1; __cfwr_i96++) {
            if (false || true) {
            if (false || true) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 5; __cfwr_i42++) {
            try {
            return "test66";
        } catch (Exception __cfwr_e44) {
            //
        for (int __cfwr_i12 = 0; __cfwr_i12 < 4; __cfwr_i12++) {
            while (true) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 9; __cfwr_i41++) {
            while (true) {
            if (true && true) {
            return ((null / null) - null);
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
 ignore
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      static int __cfwr_util791(String __cfwr_p0, Integer __cfwr_p1, String __cfwr_p2) {
        short __cfwr_obj18 = null;
        try {
            Float __cfwr_obj33 = null;
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        if ((true + 28.37f) || false) {
            if (true && true) {
            try {
            while (true) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 5; __cfwr_i1++) {
            while (true) {
            long __cfwr_var88 = 439L;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        }
        }
        try {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 9; __cfwr_i96++) {
            Double __cfwr_data68 = null;
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        return ((null | -843L) << 'J');
    }
    long __cfwr_aux835(Object __cfwr_p0, byte __cfwr_p1, Double __cfwr_p2) {
        if (false && false) {
            double __cfwr_item40 = ((-58.64f >> -22.74f) * 29.98);
        }
        return 636L;
    }
    Boolean __cfwr_aux480(Long __cfwr_p0, Float __cfwr_p1, Object __cfwr_p2) {
        Object __cfwr_temp18 = null;
        return null;
    }
}
