/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  ArrayAssignmentSameLen(int[] array, @IndexFor("#1") int index) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 5; __cfwr_i55++) {
            try {
            if (true || fal
        try {
            try {
            while ((-79.71 & -624L)) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 9; __cfwr_i18++) {
            if (true && ((null - -7.27f) % 282)) {
            try {
            while (true) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 4; __cfwr_i3++) {
            Integer __cfwr_val56 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
se) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 10; __cfwr_i31++) {
            if ((null / null) || false) {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 6; __cfwr_i97++) {
            return null;
        }
        }
        }
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }

    i_array = array;
    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {
    int[] c1 = a;
    // See useTest3 for an example of why this assignment should fail.
    @LTLengthOf(
        value = {"c1", "c1"},
        offset = {"0", "x"})
    // :: error: (assignment)
    int z = i;
      private long __cfwr_func749(boolean __cfwr_p0, float __cfwr_p1, char __cfwr_p2) {
        byte __cfwr_item86 = null;
        if (false || false) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 5; __cfwr_i31++) {
            try {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 2; __cfwr_i79++) {
            if (false && (502 >> 77)) {
            if (true && true) {
            Long __cfwr_val42 = null;
        }
        }
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
        }
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        float __cfwr_entry65 = ((null % 582) - 'T');
        return 31L;
    }
    public Boolean __cfwr_process841(long __cfwr_p0) {
        try {
            try {
            if (false || false) {
            try {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 10; __cfwr_i86++) {
            return 44;
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        return null;
    }
}
