/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        while (true) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 7; __cfwr_i98++) {
            if (false && ('z' + -14L)) {
            while (false) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 6; __cfwr_i32++) {
            if (true && false) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      protected byte __cfwr_aux749(byte __cfwr_p0, Object __cfwr_p1, Character __cfwr_p2) {
        return null;
        return null;
        try {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 3; __cfwr_i66++) {
            while (false) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 7; __cfwr_i39++) {
            try {
            try {
            Long __cfwr_node73 = null;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        if (false && false) {
            return true;
        }
        return null;
    }
    float __cfwr_handle22(Integer __cfwr_p0, int __cfwr_p1, Integer __cfwr_p2) {
        float __cfwr_temp99 = 60.65f;
        Character __cfwr_result77 = null;
        Double __cfwr_entry49 = null;
        Long __cfwr_node68 = null;
        return (null - 64.73);
    }
}
