/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ArrayAssignmentSameLen_slice {
// Source-based slice around line 19
// Method: ArrayAssignmentSameLen#test1(int[],int[],int)

    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
        return null;

    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTL
        return null;
engthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {

    private Character __cfwr_calc299(double __cfwr_p0, short __cfwr_p1, Float __cfwr_p2) {
        while (((null ^ 'C') << (null + -78.21f))) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 3; __cfwr_i43++) {
            return (-83.22f * null);
        }
            break; // Prevent infinite loops
        }
        while (true) {
            Character __cfwr_result53 = null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i20 = 0; __cfwr_i20 < 1; __cfwr_i20++) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 10; __cfwr_i89++) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
        }
        return null;
    }
}