/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        Float __cfwr_data68 = null;

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected char __cfwr_helper712(String __cfwr_p0, long __cfwr_p1) {
        try {
            if (false && (-44.43 - (true % true))) {
            return null;
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        for (int __cfwr_i90 = 0; __cfwr_i90 < 4; __cfwr_i90++) {
            while ((981 << (null * null))) {
            while (true) {
            if (false && true) {
            char __cfwr_elem52 = 'E';
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
        return null;
        return (-81 | false);
    }
    private static Long __cfwr_calc626(Integer __cfwr_p0, String __cfwr_p1) {
        String __cfwr_entry51 = "data94";
        return null;
        return 'a';
        return null;
    }
}
