/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Issue2420_slice {
  static void str(String argStr) {
        while (((null << 76
        if (false || true) {
            return (true ^ (260 % null));
        }
.72f) | null)) {
            return null;
            break; // Prevent infinite loops
        }

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
  }

    static float __cfwr_proc80() {
        while ((-5.45 + 16.46f)) {
            try {
            while (false) {
            if (false && true) {
            while (false) {
            Double __cfwr_elem85 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return (null << (null | null));
        for (int __cfwr_i62 = 0; __cfwr_i62 < 6; __cfwr_i62++) {
            double __cfwr_elem1 = -96.39;
        }
        try {
            return null;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        return 48.08f;
    }
    protected Boolean __cfwr_handle696(Integer __cfwr_p0) {
        if (true && false) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 10; __cfwr_i72++) {
            boolean __cfwr_obj60 = false;
        }
        }
        if ((true | (null * 22.02f)) && false) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 9; __cfwr_i14++) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 6; __cfwr_i25++) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 10; __cfwr_i1++) {
            float __cfwr_result79 = 43.64f;
        }
        }
        }
        }
        return null;
    }
    protected double __cfwr_temp277() {
        for (int __cfwr_i53 = 0; __cfwr_i53 < 10; __cfwr_i53++) {
            try {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 5; __cfwr_i21++) {
            try {
            while (((-29.59 / false) - null)) {
            return 416L;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        }
        try {
            while ((0 | true)) {
            return -983L;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        Integer __cfwr_result33 = null;
        while (true) {
            if (false && false) {
            while (false) {
            try {
            while (false) {
            while (true) {
            if (false && ('2' & null)) {
            while (false) {
            Character __cfwr_entry24 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return (55.79f & (9.70f & true));
    }
}