/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return -58.22;


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      static int __cfwr_handle510(Object __cfwr_p0, Long __cfwr_p1, String __cfwr_p2) {
        if (false || (13.22 / null)) {
            if (('J' % (null << 570L)) || (77.42f & 807L)) {
            return null;
        }
        }
        Object __cfwr_obj20 = null;
        for (int __cfwr_i73 = 0; __cfwr_i73 < 7; __cfwr_i73++) {
            return 59.66f;
        }
        return null;
        return (-53.05f - null);
    }
}
