/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testNewInstance(int i) {
        for (int __cfwr_i42 = 0; __cfwr_i42 < 7; __cfwr_i42++) {
            for (int
        for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            return -37.04;
        }
 __cfwr_i97 = 0; __cfwr_i97 < 3; __cfwr_i97++) {
            if ((null / true) || (true - false)) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 2; __cfwr_i78++) {
            if (true && (-92.05 << 101)) {
            if (((null >> null) + (10L - null)) && true) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 4; __cfwr_i11++) {
            return 1.41;
        }
        }
        }
        }
        }
        }
        }

    // :: error: (argument)
    Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
      private Boolean __cfwr_func276(Object __cfwr_p0) {
        for (int __cfwr_i32 = 0; __cfwr_i32 < 6; __cfwr_i32++) {
            if (true && false) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 6; __cfwr_i11++) {
            try {
            try {
            Long __cfwr_elem78 = null;
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        }
        }
        }
        try {
            if (true || true) {
            try {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 6; __cfwr_i37++) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 2; __cfwr_i16++) {
            while (true) {
            try {
            while (true) {
            if (true && ((null + -729) % 84L)) {
            while (false) {
            if (true || ((null & null) & 551)) {
            while (true) {
            return ('C' + (false >> 155L));
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        return null;
    }
}
