/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        Long __cfwr_item90 = null;

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public static int __cfwr_temp996() {
        for (int __cfwr_i17 = 0; __cfwr_i17 < 8; __cfwr_i17++) {
            return ((false | null) * (-21.52 - -629));
        }
        if (false || true) {
            return null;
        }
        while ((-81.21f + null)) {
            if (false || (-389L + (-13.17f + 626L))) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 2; __cfwr_i75++) {
            while (true) {
            Character __cfwr_entry20 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return (null * 985L);
    }
}
