/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        while (false) {
            while (false) {
            if (false && (-82.92 | -36.57f)) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      static short __cfwr_util887() {
        if ((null | -12.17f) || false) {
            if (false && false) {
            while ((false / null)) {
            while (((null << null) + -19.89f)) {
            if (true || true) {
            while (false) {
            if (true && (439 + -516)) {
            char __cfwr_var27 = 'q';
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        Character __cfwr_val27 = null;
        if (true && (-48.36 * (false & '6'))) {
            while (false) {
            try {
            while (true) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 8; __cfwr_i37++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
