/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        for (int __cfwr_i43 = 0; __cfwr_i43 < 1; __cfwr_i43++) {
            boolean __cfwr_item18 = ((-499 * 'K') >> 459L);
   
        if (((61.88 + false) >> (-24.37f | -42.10)) && false) {
            while ((null % -29.99)) {
            float __cfwr_item12 = ((-60.29f & false) & -545);
            break; // Prevent infinite loops
        }
        }
     }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      private static Integer __cfwr_helper454(Object __cfwr_p0) {
        for (int __cfwr_i2 = 0; __cfwr_i2 < 6; __cfwr_i2++) {
            Double __cfwr_node16 = null;
        }
        return (null * 'A');
        if ((-64.57 | (-258L << false)) && false) {
            try {
            try {
            while (true) {
            Object __cfwr_result50 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        }
        return null;
    }
}
