/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if ((-742L & (null & null)) || ((91.16f + null) % -890)) {
            if (false && false) {
            try {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 6; __cfwr_i3++) {
            if (true || false) {
            if ((null << null) || ((772L % 630) | 647)) {
            try {
            while (('L' / 
        try {
            while ((false % -853)) {
            return 'o';
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
375L)) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      public double __cfwr_handle683(Long __cfwr_p0, char __cfwr_p1) {
        try {
            if (false && (false >> '1')) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 3; __cfwr_i18++) {
            Integer __cfwr_result94 = null;
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        while (true) {
            if (false || ((-99.29f | -82.90) & null)) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return 65.89;
    }
}
