/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i24 = 0; __cfwr_i24 < 7; __cfwr_i24++) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 8; __cfwr_i98++) {
            long __cfwr_temp53 = (true % -93.49f);
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.
        try {
            double __cfwr_var6 = 19.08;
        } catch (Exception __cfwr_e53) {
            // ignore
        }
max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private long __cfwr_process54(boolean __cfwr_p0, short __cfwr_p1, Integer __cfwr_p2) {
        return null;
        if (true && true) {
            return ('n' >> 43.91f);
        }
        for (int __cfwr_i16 = 0; __cfwr_i16 < 3; __cfwr_i16++) {
            while (true) {
            return 829L;
            break; // Prevent infinite loops
        }
        }
        return -156L;
    }
    private static boolean __cfwr_process562(Object __cfwr_p0, double __cfwr_p1) {
        float __cfwr_item4 = (72.29 + null);
        return false;
    }
    public Boolean __cfwr_util662(double __cfwr_p0, long __cfwr_p1) {
        for (int __cfwr_i18 = 0; __cfwr_i18 < 9; __cfwr_i18++) {
            try {
            while (false) {
            try {
            try {
            int __cfwr_result80 = -681;
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        }
        return 48.24f;
        for (int __cfwr_i38 = 0; __cfwr_i38 < 7; __cfwr_i38++) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 4; __cfwr_i13++) {
            while (((92L / true) << 32L)) {
            char __cfwr_entry41 = '6';
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
}
