/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        long __cfwr_val47 = -370L;

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      static byte __cfwr_helper312(long __cfwr_p0, String __cfwr_p1) {
        try {
            String __cfwr_result91 = "result26";
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        return null;
        Integer __cfwr_val78 = null;
        while (true) {
            while (true) {
            if (true && false) {
            try {
            int __cfwr_node77 = -494;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return (null | 665);
    }
    Float __cfwr_util263(long __cfwr_p0, long __cfwr_p1) {
        boolean __cfwr_entry62 = false;
        return null;
    }
}
