/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        return null;
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d
        String __cfwr_item3 = "item98";
"})
          int x) {    protected static char __cfwr_aux436(Double __cfwr_p0, long __cfwr_p1, char __cfwr_p2) {
        if ((('U' >> 36.47) & (-177 ^ -79.46f)) && true) {
            if (((true - -103L) + 41.72f) || false) {
            return -11.68f;
        }
        }
        if (true && (false & null)) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 10; __cfwr_i95++) {
            Float __cfwr_var46 = null;
        }
        }
        return 'f';
    }
    static Double __cfwr_proc158() {
        while ((null - 722L)) {
            if (true && false) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 2; __cfwr_i26++) {
            try {
            while (false) {
            Double __cfwr_result38 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        try {
            return -451;
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return null;
    }
}
