/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        return 'p';
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"
        for (int __cfwr_i26 = 0; __cfwr_i26 < 4; __cfwr_i26++) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
})
          int x) {    static double __cfwr_proc119() {
        for (int __cfwr_i71 = 0; __cfwr_i71 < 6; __cfwr_i71++) {
            try {
            return '7';
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        }
        return 75.72;
    }
}
