/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        if (false || false) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 6; __cfwr_i22++) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 2; __cfwr_i63++) {
            Object __cfwr_result18 = null;
        }
        }
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      private Double __cfwr_process143() {
        return -7.74;
        if (false && true) {
            if (false || true) {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 5; __cfwr_i60++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 1; __cfwr_i47++) {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 2; __cfwr_i52++) {
            try {
            try {
            float __cfwr_data45 = ('T' & null);
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
        }
        }
        }
        }
        try {
            try {
            if (true && false) {
            try {
            return (-817L << null);
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        while (false) {
            if (false || false) {
            return ((-50.63 / null) / null);
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
