/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            return null;
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error:
        if ((('X' | 'm') - (null & 39.70f)) || ((null ^ -124L) + (521 | -72.20))) {
            char __cfwr_elem18 = 'w';
        }
 (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      static boolean __cfwr_handle484(double __cfwr_p0) {
        char __cfwr_item56 = 'c';
        try {
            Object __cfwr_var96 = null;
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        for (int __cfwr_i43 = 0; __cfwr_i43 < 1; __cfwr_i43++) {
            while (false) {
            while (false) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 8; __cfwr_i6++) {
            try {
            short __cfwr_elem4 = null;
        } catch (Exception __cfwr_e81) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return (35.19 / null);
    }
}
