/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        if ((null % (-18.92f % null)) || ((-27.30 ^ null) % 105L)) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 10; __cfwr_i26++) {
            if ((14.33f | (-98.04f ^ 0.10)) || (null ^ ('d' * true))) {
            if (true || true) {
            Long __cfwr_elem2 = null;
        }
        }
        }
        }
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    static byte __cfwr_handle901(Integer __cfwr_p0) {
        try {
            try {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 7; __cfwr_i55++) {
            while (((false >> null) ^ (465L * 'M'))) {
            while (false) {
            while (false) {
            try {
            Object __cfwr_elem46 = null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        if (('Z' * true) && true) {
            if (false || false) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 1; __cfwr_i39++) {
            if (false || ((69.35f % null) * null)) {
            try {
            return 'c';
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        }
        }
        }
        return null;
    }
    static short __cfwr_process815() {
        Character __cfwr_val33 = null;
        double __cfwr_val44 = -98.27;
        try {
            return null;
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        return null;
    }
}
