/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    protected short __cfwr_func726(char __cfwr_p0, Boolean __cfwr_p1, char __cfwr_p2) {
        for (int __cfwr_i23 
        short __cfwr_result62 = null;
= 0; __cfwr_i23 < 10; __cfwr_i23++) {
            try {
            try {
            while (((null >> 'h') ^ '2')) {
            long __cfwr_elem41 = -466L;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }

        if (false || true) {
            if (false && true) {
            try {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        }
        }
        for (int __cfwr_i35 = 0; __cfwr_i35 < 3; __cfwr_i35++) {
            return null;
        }
        return null;
    }
    protected Long __cfwr_temp728(double __cfwr_p0) {
        return null;
        if (false && false) {
            Double __cfwr_item8 = null;
        }
        while (true) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 2; __cfwr_i8++) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 3; __cfwr_i89++) {
            Long __cfwr_var89 = null;
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
    public char __cfwr_process570(char __cfwr_p0, byte __cfwr_p1, byte __cfwr_p2) {
        while ((true % '4')) {
            return null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i19 = 0; __cfwr_i19 < 2; __cfwr_i19++) {
            float __cfwr_elem44 = (false >> false);
        }
        if (true || (null % -85.08f)) {
            try {
            float __cfwr_var35 = 28.34f;
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        return 'Q';
    }
