/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i89 = 0; __cfwr_i89 < 2; __cfwr_i89++) {
            String __cfwr_item12 = "result29";
        
        return null;
}

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected static Double __cfwr_func678(Object __cfwr_p0, Object __cfwr_p1, Character __cfwr_p2) {
        for (int __cfwr_i77 = 0; __cfwr_i77 < 10; __cfwr_i77++) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        for (int __cfwr_i95 = 0; __cfwr_i95 < 7; __cfwr_i95++) {
            Integer __cfwr_temp95 = null;
        }
        if (false && false) {
            if (true && (true ^ (-645 | 92.72f))) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 7; __cfwr_i74++) {
            return null;
        }
        }
        }
        return null;
    }
    static String __cfwr_calc358(Boolean __cfwr_p0, float __cfwr_p1, Object __cfwr_p2) {
        while (false) {
            if (false || false) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 5; __cfwr_i13++) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 5; __cfwr_i59++) {
            if (((null & -178L) + 744L) || false) {
            while ((55.85 ^ false)) {
            try {
            double __cfwr_entry64 = (-98 * (-187 | 68.91f));
        } catch (Exception __cfwr_e12) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i16 = 0; __cfwr_i16 < 9; __cfwr_i16++) {
            return null;
        }
        return (-78 / (null & 985));
        if (true && true) {
            if (false && ((null & null) | (-42.43 - 'U'))) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 3; __cfwr_i20++) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 1; __cfwr_i90++) {
            try {
            if (true || true) {
            return null;
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }
        }
        }
        }
        return "value80";
    }
}
