/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        while ((true / null)) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 7; __cfwr_i17++) {
            short __cfwr_item46 = (4.39f << (52.28 | null));
        }
            break; // Prevent infinite loops
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      public static long __cfwr_aux790(int __cfwr_p0, Object __cfwr_p1) {
        Object __cfwr_entry20 = null;
        return -79L;
    }
    private Boolean __cfwr_temp32() {
        for (int __cfwr_i79 = 0; __cfwr_i79 < 7; __cfwr_i79++) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 2; __cfwr_i99++) {
            try {
            try {
            if (true || false) {
            try {
            return null;
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        }
        }
        Boolean __cfwr_node26 = null;
        return null;
    }
}
