/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BinomialTest_slice {
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        if (false && true) {
            try {
            try {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 4; __cfwr_i41++) {
     
        boolean __cfwr_item56 = ((-55.57 + 12.08) >> (408L - true));
       try {
            while (((-82.62f ^ 'T') * -18.04)) {
            char __cfwr_data95 = '4';
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
  }

    static double __cfwr_compute44(long __cfwr_p0, Integer __cfwr_p1) {
        while (true) {
            char __cfwr_result15 = (null / null);
            break; // Prevent infinite loops
        }
        while ((-27.60f | (null << true))) {
            if (false && true) {
            Float __cfwr_item87 = null;
        }
            break; // Prevent infinite loops
        }
        return 10.38;
    }
    static Float __cfwr_proc660() {
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        if (true || false) {
            try {
            try {
            return ((-921 % -72.00f) * 756L);
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        if ((null % (null & -61.25)) || ((95 - -73.22f) - (-582L << null))) {
            try {
            if (true && false) {
            int __cfwr_elem5 = ((null * -752L) % (-543L - -6.41));
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        }
        return null;
    }
}