/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        if (true && true) {
            long __cfwr_item30 = 112L;
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      private Character __cfwr_compute603(float __cfwr_p0, Integer __cfwr_p1, double __cfwr_p2) {
        try {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e17) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        for (int __cfwr_i78 = 0; __cfwr_i78 < 3; __cfwr_i78++) {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 5; __cfwr_i96++) {
            return -6.71;
        }
        }
        return null;
    }
    static Character __cfwr_aux302(Integer __cfwr_p0, int __cfwr_p1, String __cfwr_p2) {
        int __cfwr_result92 = 573;
        return "test98";
        return null;
    }
    protected static String __cfwr_proc54(Long __cfwr_p0, short __cfwr_p1, boolean __cfwr_p2) {
        Integer __cfwr_var43 = null;
        if (true || false) {
            if ((621L * 791) || (null >> 18.96f)) {
            Double __cfwr_temp76 = null;
        }
        }
        return "result42";
    }
}
