/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        float __cfwr_elem7 
        while (true) {
            while (true) {
            long __cfwr_node82 = 968L;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
= 95.47f;

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      private static Object __cfwr_func42(Double __cfwr_p0, int __cfwr_p1, Object __cfwr_p2) {
        for (int __cfwr_i53 = 0; __cfwr_i53 < 7; __cfwr_i53++) {
            Integer __cfwr_val30 = null;
        }
        while (((null ^ 40.17f) | -75.95f)) {
            if ((null * null) || true) {
            try {
            while (false) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 9; __cfwr_i28++) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 10; __cfwr_i87++) {
            Double __cfwr_elem97 = null;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    public Double __cfwr_helper760(Integer __cfwr_p0, int __cfwr_p1) {
        try {
            if (((null << 481L) - -167) || (183L + null)) {
            return 932;
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        try {
            if (true && true) {
            while (true) {
            if (false && false) {
            try {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 7; __cfwr_i22++) {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 1; __cfwr_i21++) {
            if (true && true) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 10; __cfwr_i39++) {
            return ((63.91 | -996L) & null);
        }
        }
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        for (int __cfwr_i30 = 0; __cfwr_i30 < 9; __cfwr_i30++) {
            try {
            return "data7";
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        return null;
    }
    private float __cfwr_calc483() {
        return null;
        for (int __cfwr_i85 = 0; __cfwr_i85 < 4; __cfwr_i85++) {
            return 29.04f;
        }
        try {
            float __cfwr_var84 = 76.50f;
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        return true;
        return -60.06f;
    }
}
