/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ReflectArray_slice {
  void testNewInstance(int i) {
        char __cfwr_obj18 = ((false & false) & null);

    // :: error: (argument)
  
        try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 7; __cfwr_i62++) {
            Float __cfwr_var97 = null;
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
  Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
  }

    protected boolean __cfwr_util872(long __cfwr_p0) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 2; __cfwr_i55++) {
            while (false) {
            while (true) {
            return ('I' << -901);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        short __cfwr_result48 = null;
        try {
            if (true && true) {
            try {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 3; __cfwr_i80++) {
            int __cfwr_val29 = -467;
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        try {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 10; __cfwr_i55++) {
            return null;
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        return true;
    }
    static Character __cfwr_helper496(char __cfwr_p0, float __cfwr_p1, byte __cfwr_p2) {
        try {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 10; __cfwr_i45++) {
            Long __cfwr_data78 = null;
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        return null;
        if (false && false) {
            while (((true + -829) | -97.70)) {
            if (true && true) {
            return 56.28f;
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i74 = 0; __cfwr_i74 < 5; __cfwr_i74++) {
            Integer __cfwr_item34 = null;
        }
        return null;
    }
    public static Float __cfwr_aux583() {
        return false;
        if ((-512 | -339) || true) {
            if ((null / null) || true) {
            Character __cfwr_var14 = null;
        }
        }
        try {
            return null;
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        return null;
    }
}