/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        if ((783 + (616L - 67)) && true) {
            if (false || true) {
            if (((-46.43f * null) - 27.87) && (false - 422L)) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 7; __cfwr_i12++) {
            char __cfwr_elem88 = '2';
        }
        }
        }
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      public Object __cfwr_aux701(byte __cfwr_p0, boolean __cfwr_p1, Double __cfwr_p2) {
        try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 1; __cfwr_i38++) {
            char __cfwr_elem75 = ((null / -37L) & ('p' % null));
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        for (int __cfwr_i65 = 0; __cfwr_i65 < 3; __cfwr_i65++) {
            return ((-37.28 * -703) + (-605L / -410));
        }
        return null;
    }
}
