/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        if (false || false) {
            if (false || false) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 3; __cfwr_i14++) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        }
        }
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      private static byte __cfwr_calc376(Object __cfwr_p0, long __cfwr_p1, char __cfwr_p2) {
        for (int __cfwr_i23 = 0; __cfwr_i23 < 7; __cfwr_i23++) {
            long __cfwr_var17 = 322L;
        }
        try {
            return (false << 23.20f);
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        return null;
        return null;
    }
}
