/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        int __cfwr_temp11 = 903;

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      Double 
        return -121;
__cfwr_util931(double __cfwr_p0) {
        for (int __cfwr_i78 = 0; __cfwr_i78 < 6; __cfwr_i78++) {
            if (('j' ^ 89.30f) || true) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 7; __cfwr_i8++) {
            return ((true % 952) * (-79.72 << false));
        }
        }
        }
        try {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 3; __cfwr_i79++) {
            if (false && (null * true)) {
            Float __cfwr_node14 = null;
        }
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }
        for (int __cfwr_i65 = 0; __cfwr_i65 < 8; __cfwr_i65++) {
            if (true || true) {
            if (false || false) {
            if (false || ((null ^ 'k') | 'g')) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 5; __cfwr_i6++) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 6; __cfwr_i70++) {
            return ('o' - true);
        }
        }
        }
        }
        }
        }
        return null;
        return null;
    }
}
