/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        return null;

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private short __cfwr_func860() {
        try {
            Character __cfwr_var29 = null;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        return null;
    }
    protected static short __cfwr_process69() {
        if (((null | null) ^ true) && true) {
            while (true) {
            if (true || true) {
            return -924L;
        }
            break; // Prevent infinite loops
        }
        }
        try {
            if ((null + 'L') && false) {
            if (true || ('0' >> -109L)) {
            try {
            if (false || true) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 5; __cfwr_i63++) {
            while ((null >> (837L % -345L))) {
            if (true || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        while ((45.88 + 'm')) {
            if ((-597L - -686L) || true) {
            return (-49.91f + 833);
        }
            break; // Prevent infinite loops
        }
        return (null >> (26.93f | 3.03));
    }
}
