/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        return null;

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @In
        while ((551 - (null ^ false))) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 10; __cfwr_i66++) {
            try {
            try {
            if ((-12L >> (-757L >> false)) || true) {
            if (true && true) {
            try {
            long __cfwr_obj58 = (null % -480);
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
dexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private Long __cfwr_util867(double __cfwr_p0) {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 10; __cfwr_i21++) {
            while (true) {
            Boolean __cfwr_result51 = null;
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
