/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineEq_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i5 = 0; __cfwr_i5 < 4; __cfwr_i5++) {
            try {
            return 256;
        } catch 
        for (int __cfwr_i97 = 0; __cfwr_i97 < 5; __cfwr_i97++) {
            return (79.16 >> 95.85);
        }
(Exception __cfwr_e50) {
            // ignore
        }
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
  }

    static Character __cfwr_helper622(Boolean __cfwr_p0, boolean __cfwr_p1, long __cfwr_p2) {
        for (int __cfwr_i12 = 0; __cfwr_i12 < 3; __cfwr_i12++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 9; __cfwr_i3++) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 2; __cfwr_i19++) {
            if (false && true) {
            while (false) {
            return (null + (-1.94f - null));
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        for (int __cfwr_i66 = 0; __cfwr_i66 < 4; __cfwr_i66++) {
            while (true) {
            try {
            try {
            Object __cfwr_temp13 = null;
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    protected static char __cfwr_process880() {
        try {
            try {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        for (int __cfwr_i83 = 0; __cfwr_i83 < 2; __cfwr_i83++) {
            if ((null - 77.00f) && false) {
            float __cfwr_elem28 = 41.01f;
        }
        }
        float __cfwr_item89 = -28.69f;
        return 'm';
    }
}