/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        if (((-421 ^ -81.45f) << 'e') && (null & 336)) {
            if (false || (false ^ ('3' << 87.85f))) {
            try {
            while (false) {
            try {
            return ((false | -117L) - 's');
        } catch (Exception __cfwr_e28) {
            // ignore
        }
            break; // Prevent infini
        if (true || ((11.60 << null) | 'W')) {
            if (true && (-85.48 * true)) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 4; __cfwr_i46++) {
            while (true) {
            Character __cfwr_result21 = null;
            break; // Prevent infinite loops
        }
        }
        }
        }
te loops
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      public static Long __cfwr_aux26(long __cfwr_p0, short __cfwr_p1, short __cfwr_p2) {
        if ((null ^ null) || false) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 1; __cfwr_i12++) {
            try {
            double __cfwr_val61 = 79.38;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        }
        return null;
    }
}
