/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        return (('F' >> 's') * false);

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNe
        for (int __cfwr_i40 = 0; __cfwr_i40 < 5; __cfwr_i40++) {
            Object __cfwr_temp49 = null;
        }
gative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      private long __cfwr_compute973() {
        if ((592 >> 34.86f) || false) {
            Double __cfwr_result40 = null;
        }
        while (true) {
            if (false || (('E' | -789L) | (-574 | 's'))) {
            while (false) {
            try {
            try {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 7; __cfwr_i4++) {
            return null;
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i2 = 0; __cfwr_i2 < 2; __cfwr_i2++) {
            while (true) {
            Character __cfwr_result77 = null;
            break; // Prevent infinite loops
        }
        }
        return ((-58.87 << 'K') & null);
    }
    private static String __cfwr_temp693(float __cfwr_p0, short __cfwr_p1, Object __cfwr_p2) {
        if (false || ((true % -39.16) * true)) {
            return null;
        }
        if (true && true) {
            return "result16";
        }
        try {
            if ((null - null) || false) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 4; __cfwr_i2++) {
            Boolean __cfwr_val73 = null;
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        if (((55.64 - 20.05f) >> false) || true) {
            return (true * 'F');
        }
        return "result69";
    }
    public static char __cfwr_process339(Double __cfwr_p0) {
        while (false) {
            if ((true - (23.08f | null)) || true) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 3; __cfwr_i56++) {
            Float __cfwr_item86 = null;
        }
        }
            break; // Prevent infinite loops
        }
        return 'U';
    }
}

  void test4(@NonNegative int x1, int[] a) {
