/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LubIndex_slice {
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        for (int __cfwr_i65 = 0; __cfwr_i65 < 9; __cfwr_i65++) {
            return "world25";
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

    private static String __cfwr_util967(Character __cfwr_p0, Object __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i68 = 0; __cfwr_i68 < 5; __cfwr_i68++) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 7; __cfwr_i21++) {
            if ((-38.18f - null) && true) {
            return 163;
        }
        }
        }
        while (true) {
            try {
            if (true || false) {
            int __cfwr_obj84 = 645;
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "item68";
    }
    private static double __cfwr_compute78(byte __cfwr_p0) {
        return null;
        for (int __cfwr_i75 = 0; __cfwr_i75 < 8; __cfwr_i75++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 6; __cfwr_i7++) {
            char __cfwr_var36 = (null | 852);
        }
        }
        return -21.15;
    }
    boolean __cfwr_temp152() {
        while (false) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            try {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 3; __cfwr_i59++) {
            return null;
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i88 = 0; __cfwr_i88 < 1; __cfwr_i88++) {
            try {
            return null;
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        }
        try {
            Float __cfwr_item39 = null;
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        return 'g';
        return (560 ^ 'y');
    }
}