/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i48 = 0; __cfwr_i48 < 4; __cfwr_i48++) {
            try {
            return (true ^ (null % -28));
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      private String __cfwr_proc465() {
        try {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 1; __cfwr_i28++) {
            try {
            while (true) {
            while (false) {
            return -646;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        for (int __cfwr_i79 = 0; __cfwr_i79 < 9; __cfwr_i79++) {
            while (false) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 4; __cfwr_i15++) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 8; __cfwr_i69++) {
            if (true || (965 / null)) {
            float __cfwr_var35 = 53.13f;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        return "value27";
    }
    private byte __cfwr_process482(Long __cfwr_p0, short __cfwr_p1, short __cfwr_p2) {
        while ((null ^ (-286 + 691L))) {
            return null;
            break; // Prevent infinite loops
        }
        if (false && true) {
            String __cfwr_data84 = "data11";
        }
        return null;
    }
    protected Boolean __cfwr_temp8(String __cfwr_p0, double __cfwr_p1) {
        Character __cfwr_entry91 = null;
        return null;
    }
}
