/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class MinMaxIndex_slice {
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            while (true) {
            return "item91";
            break; // Prevent infinite loops
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
  }

    public static int __cfwr_proc737() {
        Integer __cfwr_obj7 = null;
        return ((-8.51 | null) % null);
    }
    public float __cfwr_helper6(Long __cfwr_p0) {
        while ((false - true)) {
            return "test21";
            break; // Prevent infinite loops
        }
        while (false) {
            long __cfwr_item13 = 787L;
            break; // Prevent infinite loops
        }
        return 27.26f;
    }
}