/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        try {
            while (true) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 10; __cfwr_i44++) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 1; __cfwr_i41++) {
            float __cfwr_entry18 = 35.90f;
  
        return null;
      }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      public static short __cfwr_proc548(short __cfwr_p0, long __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i51 = 0; __cfwr_i51 < 8; __cfwr_i51++) {
            if (false && true) {
            Object __cfwr_data28 = null;
        }
        }
        short __cfwr_temp85 = null;
        return null;
    }
    public static Float __cfwr_util239(Character __cfwr_p0) {
        Integer __cfwr_item32 = null;
        for (int __cfwr_i42 = 0; __cfwr_i42 < 9; __cfwr_i42++) {
            try {
            Double __cfwr_temp90 = null;
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        }
        for (int __cfwr_i36 = 0; __cfwr_i36 < 5; __cfwr_i36++) {
            while ((-936L - null)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        try {
            while (true) {
            while (false) {
            return -74.64;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        return null;
    }
    private static Double __cfwr_compute854(float __cfwr_p0) {
        Boolean __cfwr_result53 = null;
        short __cfwr_temp80 = (null | true);
        return null;
        return null;
    }
}
