/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            while ((-65.06f | -513)) {
            while (false) {
            try {
            if (true || true) {
            return null;
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      float __cfwr_helper613(boolean __cfwr_p0) {
        try {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        return -46.83f;
    }
    protected static Long __cfwr_aux756(Float __cfwr_p0, int __cfwr_p1) {
        try {
            return ((-101L | -40.36f) >> (624L >> 42.62));
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        for (int __cfwr_i71 = 0; __cfwr_i71 < 7; __cfwr_i71++) {
            return "data25";
        }
        float __cfwr_result48 = 20.82f;
        return null;
    }
    public static Character __cfwr_compute64(Long __cfwr_p0) {
        Float __cfwr_obj42 = null;
        Object __cfwr_data82 = null;
        for (int __cfwr_i89 = 0; __cfwr_i89 < 5; __cfwr_i89++) {
            try {
            if ((null / -234) && (5.55 - (null | null))) {
            Object __cfwr_entry2 = null;
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        return null;
    }
}
