/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        return ((null & true) & null);

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      private Double __cfwr_func898(Long __cfwr_p0) {
        Object __cfwr_var19 = null;
        try {
            Long __cfwr_temp71 = null;
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        long __cfwr_node36 = ((null ^ -804) >> ('L' % '7'));
        for (int __cfwr_i59 = 0; __cfwr_i59 < 3; __cfwr_i59++) {
            try {
            if (false && true) {
            return null;
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        return null;
    }
}
