/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        return -79.06;

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      public Boolean __cfwr_helper68(boolean __cfwr_p0, double __cfwr_p1, Long __cfwr_p2) {
        try {
            for (int __cfwr_i83 = 0; __cfwr_i83 < 8; __cfwr_i83++) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 1; __cfwr_i34++) {
            long __cfwr_temp33 = -489L;
        }
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        return null;
    }
    public static char __cfwr_calc307(short __cfwr_p0) {
        while (('o' % 839)) {
            if (false || false) {
            if (true && true) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i24 = 0; __cfwr_i24 < 3; __cfwr_i24++) {
            return null;
        }
        return (true % (null % -54.24));
    }
    protected Long __cfwr_process513(float __cfwr_p0, long __cfwr_p1) {
        for (int __cfwr_i61 = 0; __cfwr_i61 < 8; __cfwr_i61++) {
            return "temp72";
        }
        for (int __cfwr_i28 = 0; __cfwr_i28 < 10; __cfwr_i28++) {
            float __cfwr_data29 = 61.01f;
        }
        return null;
    }
}
