/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test1(@Positive int x, @Positive int y) {
        while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Preven
        if (((true + -493L) - 678L) || false) {
            while (true) {
            if (true || (431L % -96.87f)) {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 8; __cfwr_i50++) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 5; __cfwr_i29++) {
            while ((17.60f ^ false)) {
            while (true) {
            try {
            return 'K';
        } catch (Exception __cfwr_e87) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
t infinite loops
        }

    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexFor("newArray") int j = y;
  }

  void test2(@NonNegative int x, @Positive int y) {
    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test3(@NonNegative int x, @NonNegative int y) {
    int[] newArray = new int[x + y];
    @IndexOrHigh("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test4(@GTENegativeOne int x, @NonNegative int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    @LTEqLengthOf("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test5(@GTENegativeOne int x, @GTENegativeOne int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexOrHigh("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test6(int x, int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexFor("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
      protected Long __cfwr_func411() {
        while (false) {
            try {
            return null;
        } catch (Exception __cfwr_e86) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    public static Boolean __cfwr_temp525(Double __cfwr_p0) {
        for (int __cfwr_i58 = 0; __cfwr_i58 < 1; __cfwr_i58++) {
            while (false) {
            while ((-594 / (749L + 634L))) {
            return "test34";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        while (false) {
            String __cfwr_entry62 = "value39";
            break; // Prevent infinite loops
        }
        if (false && ((null + -84) ^ (true * -593L))) {
            while (false) {
            Float __cfwr_entry42 = null;
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
