/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        if (((162 / '9') % (18.90 >> null)) && false) {
            try {
            while ((-827L >> -59.06f)) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 7; __cfwr_i73++) {
            boolean __cfwr_val94 = (('L' & false) | false);
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      public static Object __cfwr_handle994(Long __cfwr_p0) {
        while (false) {
            boolean __cfwr_obj32 = false;
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}
