/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        for (int __cfwr_i13 = 0; __cfwr_i13 < 7; __cfwr_i13++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 2; __cfwr_i86++) {
            return -949L;
        }
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a + 1;
    @LTEqLengthOf({"arr"}) int c1 = a + 1;
    @LTLe
        return 61.11;
ngthOf({"arr"}) int d = a + 0;
    @LTLengthOf({"arr"}) int e = a + (-7);
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a + 7;
      public float __cfwr_calc273() {
        Boolean __cfwr_data94 = null;
        for (int __cfwr_i59 = 0; __cfwr_i59 < 5; __cfwr_i59++) {
            if (false && (-16.68 | -28.32)) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 8; __cfwr_i71++) {
            try {
            try {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 5; __cfwr_i65++) {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 2; __cfwr_i54++) {
            if (true || (909 + -658L)) {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 6; __cfwr_i48++) {
            if (true && (('x' | null) + (49.72 ^ 'd'))) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 8; __cfwr_i82++) {
            while (((null ^ -781) << 652)) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 5; __cfwr_i5++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        }
        }
        while (false) {
            try {
            if (false || true) {
            if (((-56.95 | 23.65f) >> -26.63) && true) {
            return ((null << 37.80f) & (12.70 | 90.18f));
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        while (true) {
            if (false && true) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 7; __cfwr_i11++) {
            return ((null + -276L) - (53.94f & true));
        }
        }
            break; // Prevent infinite loops
        }
        return 28.06f;
    }
}
