/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        if (false || false) {

        if (true || (28.18 ^ (-735L << -936))) {
            if ((712L / 3) || (-573L / 901)) {
            Boolean __cfwr_val47 = null;
        }
        }
            try {
            try {
            if ((null * false) && false) {
            return 'N';
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public short __cfwr_func572(Boolean __cfwr_p0, byte __cfwr_p1) {
        return null;
        return (null + null);
    }
}
