/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        while (true) {
     
        try {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 7; __cfwr_i52++) {
            return 95.95f;
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
       while (false) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 2; __cfwr_i25++) {
            while (false) {
            Object __cfwr_obj39 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      protected long __cfwr_func143(Boolean __cfwr_p0, double __cfwr_p1, Float __cfwr_p2) {
        while (true) {
            try {
            while (true) {
            Object __cfwr_node12 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 1; __cfwr_i55++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 1; __cfwr_i86++) {
            return false;
        }
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        Integer __cfwr_elem18 = null;
        if (((null * 11.55f) - 'M') && true) {
            while (((-779 - 609L) | 'I')) {
            if (true || true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        return -715L;
    }
    protected Long __cfwr_helper493() {
        try {
            while (true) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 9; __cfwr_i70++) {
            while ((98.52 * null)) {
            try {
            int __cfwr_temp51 = -860;
        } catch (Exception __cfwr_e67) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        return null;
        return null;
    }
    private String __cfwr_process209() {
        if (true && false) {
            return null;
        }
        try {
            while (false) {
            Integer __cfwr_elem33 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        try {
            for (int __cfwr_i27 = 0; __cfwr_i27 < 3; __cfwr_i27++) {
            Long __cfwr_entry54 = null;
        }
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        try {
            if (false && (87.55f ^ -57.92f)) {
            try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 7; __cfwr_i98++) {
            while (false) {
            return 454;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        return "data57";
    }
}
