/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        boolean __cfwr_result45 = true;

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt(
        while (false) {
            try {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 3; __cfwr_i41++) {
            try {
            if (true || (('l' + -35.41) / ('D' >> null))) {
            return null;
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
            break; // Prevent infinite loops
        }
"3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      Long __cfwr_temp625(char __cfwr_p0) {
        while (true) {
            return (87.65f * 544);
            break; // Prevent infinite loops
        }
        double __cfwr_data52 = (null << (true % 4.56));
        try {
            return (false * true);
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        if (true && false) {
            while (true) {
            try {
            if (true && (false % (null / 437L))) {
            try {
            if ((66.04f * false) && false) {
            float __cfwr_obj24 = -97.12f;
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    Long __cfwr_process51(char __cfwr_p0, Boolean __cfwr_p1) {
        return null;
        if (false && true) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 8; __cfwr_i93++) {
            if (((-66.03f ^ null) << 482L) && (-744 << null)) {
            return null;
        }
        }
        }
        return null;
        for (int __cfwr_i25 = 0; __cfwr_i25 < 6; __cfwr_i25++) {
            if (false && (null | 484L)) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 8; __cfwr_i43++) {
            return "value74";
        }
        }
        }
        return null;
    }
    static double __cfwr_helper676(Character __cfwr_p0) {
        try {
            while (true) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 9; __cfwr_i9++) {
            while (false) {
            if (false || false) {
            return -759L;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        return (-554 / ('A' - null));
    }
}
