/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        return null;

    i
        Double __cfwr_entry91 = null;
f (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      protected static short __cfwr_process634(char __cfwr_p0, Character __cfwr_p1, byte __cfwr_p2) {
        try {
            while (true) {
            if ((false % null) && false) {
            byte __cfwr_val67 = (239 << -959);
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        while (false) {
            while (((true ^ -979L) | 'U')) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if (false && false) {
            Character __cfwr_elem65 = null;
        }
        return null;
    }
    protected static boolean __cfwr_func100(Boolean __cfwr_p0) {
        Character __cfwr_var85 = null;
        return (('P' << null) | (-93.44f + 664L));
    }
}
