/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        try {
            try {
            byte __cfwr_entry88 = null;
        } catch (Exception __cfwr_e46) {
            // ignore
        }
      
        if (('1' + (null >> 50.31)) || false) {
            while (('t' / (-92.52 % -811L))) {
            while (false) {
            try {
            if (((-8.13 - 4.47f) ^ (-906L + 'X')) || ((true % 'k') % (125L + -71.69f))) {
            return null;
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
  } catch (Exception __cfwr_e9) {
            // ignore
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      static Integer __cfwr_func574(int __cfwr_p0) {
        return null;
        Object __cfwr_result65 = null;
        return null;
    }
    private Integer __cfwr_calc742() {
        if (false || true) {
            return false;
        }
        try {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 7; __cfwr_i60++) {
            return -387L;
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        return null;
    }
}
