/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class SLSubtyping_slice {
  void subtype(int @SameLen("#2") [] a, int[] b) {
        for (int __cfwr_i2 = 0; __cfwr_i2 < 3; __cfwr_i2++) {
            float __cfwr_var25 = 83.28f;
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
  }

    Integer __cfwr_compute145(long __cfwr_p0, Boolean __cfwr_p1, String __cfwr_p2) {
        try {
            try {
            if (true && false) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 9; __cfwr_i12++) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 8; __cfwr_i4++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 5; __cfwr_i47++) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 1; __cfwr_i61++) {
            byte __cfwr_result1 = (('u' - 406) << (false - -206L));
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        Float __cfwr_temp84 = null;
        return null;
    }
}