/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            byte __cfwr_node53 = null;
            break; // Prevent infinite loops
        }

  
        if (true || true) {
            try {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 4; __cfwr_i84++) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
  // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      static Double __cfwr_util25(int __cfwr_p0) {
        if ((-686 | null) && ((50.26f & 84.68) - 66.48f)) {
            while (true) {
            while (false) {
            try {
            try {
            while (false) {
            while (((null % 98.36f) * null)) {
            try {
            while (true) {
            try {
            while (false) {
            return 761;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        try {
            while ((32.92f >> null)) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        while (true) {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 6; __cfwr_i76++) {
            try {
            while ((-86.95f * 'c')) {
            return 604;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        float __cfwr_elem17 = ('C' * 99.27);
        return null;
    }
    public static Integer __cfwr_helper126() {
        return -84.26f;
        Character __cfwr_data9 = null;
        return null;
    }
    static int __cfwr_calc393(int __cfwr_p0) {
        for (int __cfwr_i88 = 0; __cfwr_i88 < 9; __cfwr_i88++) {
            while (false) {
            Float __cfwr_entry32 = null;
            break; // Prevent infinite loops
        }
        }
        return -828;
    }
}
