/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        try {
            while ((706 + true)) {
            if (true || false) {
            byte __cfwr_result36 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      Double __cfwr_process978() {
        return 5.53f;
        for (int __cfwr_i51 = 0; __cfwr_i51 < 10; __cfwr_i51++) {
            while (false) {
            while (false) {
            Long __cfwr_elem79 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i43 = 0; __cfwr_i43 < 3; __cfwr_i43++) {
            return null;
        }
        for (int __cfwr_i45 = 0; __cfwr_i45 < 2; __cfwr_i45++) {
            while (true) {
            while (false) {
            Double __cfwr_entry24 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
