/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        while (false) {
            if (true || (0.07f ^ (-91.53f - null))) {
            try {
            while (true) {
            while ((-49.81 + null)) {
            Long __cfwr_node43 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      Double __cfwr_helper723(boolean __cfwr_p0, byte __cfwr_p1, byte __cfwr_p2) {
        if (false || ((null & -92.41f) | 792)) {
            if (((-591 | true) >> -822) && true) {
            String __cfwr_obj70 = "hello32";
        }
        }
        if (false || false) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 3; __cfwr_i94++) {
            try {
            if (((null - null) * (null ^ null)) && false) {
            return -490;
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        }
        for (int __cfwr_i76 = 0; __cfwr_i76 < 6; __cfwr_i76++) {
            return null;
        }
        return null;
    }
}
