/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        try {
            Long __cfwr_val85 = null;
        } catch (Exception __cfwr_e43) {
            // ignore
        }

    int[] banana;
    int @SameLen("a2") [] b = samelen_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") i
        while (true) {
            while (true) {
            Character __cfwr_item92 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
nt ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      public Float __cfwr_aux18(float __cfwr_p0, Float __cfwr_p1, boolean __cfwr_p2) {
        return null;
        return null;
    }
    public boolean __cfwr_helper895() {
        while (false) {
            Object __cfwr_item8 = null;
            break; // Prevent infinite loops
        }
        try {
            return null;
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        return null;
        while (false) {
            if (false && true) {
            return (true - null);
        }
            break; // Prevent infinite loops
        }
        return false;
    }
    private static long __cfwr_proc819(float __cfwr_p0, Double __cfwr_p1, short __cfwr_p2) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        return -277L;
    }
}
