/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LessThanLen_slice {
  public static void m4(int @MinLen(1) [] shorter) {
        while (true) {
            while (((74.79f | -62.62) >> 'j')) {
            while ((69.33 * -389L)) {
            Long __cfwr_val31 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

    private static short __cfwr_helper718(String __cfwr_p0, char __cfwr_p1, int __cfwr_p2) {
        return null;
        return null;
        float __cfwr_data11 = (null << -744);
        return null;
    }
    protected static short __cfwr_helper108(double __cfwr_p0) {
        for (int __cfwr_i12 = 0; __cfwr_i12 < 3; __cfwr_i12++) {
            while (false) {
            byte __cfwr_entry32 = (-35.89f & (true | -82));
            break; // Prevent infinite loops
        }
        }
        while (true) {
            return (-380L - -95.49);
            break; // Prevent infinite loops
        }
        return null;
    }
}