/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LessThanFloat_slice {
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;


    protected static double __cfwr_handle122() {
        Integer __cfwr
        try {
            return null;
        } catch (Exception __cfwr_e66) {
            // ignore
        }
_elem93 = null;

        while (true) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 7; __cfwr_i49++) {
            while (true) {
            while (true) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 5; __cfwr_i87++) {
            return -64.60f;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        if (false || true) {
            while ((6.29 & null)) {
            return "result65";
            break; // Prevent infinite loops
        }
        }
        return (null * false);
    }
    private static int __cfwr_calc737(String __cfwr_p0, char __cfwr_p1) {
        while (true) {
            try {
            try {
            return false;
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i17 = 0; __cfwr_i17 < 10; __cfwr_i17++) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 1; __cfwr_i99++) {
            while (((null % -12.67) + 'j')) {
            while (false) {
            String __cfwr_temp44 = "item5";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        return ((-458 - 1.26f) / ('Y' >> null));
    }
    Double __cfwr_helper706(int __cfwr_p0) {
        while (false) {
            try {
            if (false || false) {
            double __cfwr_temp17 = ((206L / 'T') ^ (null / true));
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i72 = 0; __cfwr_i72 < 9; __cfwr_i72++) {
            if ((89.27f / 'R') || false) {
            try {
            if (((true / 'K') & null) || true) {
            try {
            if (((682L >> 44.69f) >> -125L) && (('I' << null) << -96.11f)) {
            while (true) {
            while (false) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 7; __cfwr_i1++) {
            try {
            char __cfwr_result13 = '9';
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
        }
        return null;
    }
}