/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        if (false && (564L / null)) {
            try {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 4; __cfwr_i65++) {
            try {
            while (false) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 7; __cfwr_i33++) {
            boolean __cfwr_val75 = ((null << -481) + -404L);
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      protected short __cfwr_process123(Character __cfwr_p0, Double __cfwr_p1) {
        try {
            if ((-15.43f + 405) && false) {
            if (false && true) {
            Object __cfwr_entry74 = null;
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        return null;
    }
}
