/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        return -56.66f;
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c",
        for (int __cfwr_i31 = 0; __cfwr_i31 < 1; __cfwr_i31++) {
            try {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 10; __cfwr_i69++) {
            while (true) {
            if (false && false) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 1; __cfwr_i63++) {
            try {
            if ((null & (237 + -50.48f)) || false) {
            return false;
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
 "d"})
          int x) {    public static long __cfwr_aux977() {
        return null;
        return 268L;
    }
    protected char __cfwr_helper422() {
        for (int __cfwr_i14 = 0; __cfwr_i14 < 3; __cfwr_i14++) {
            while (true) {
            Double __cfwr_temp88 = null;
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i64 = 0; __cfwr_i64 < 6; __cfwr_i64++) {
            while (false) {
            while (false) {
            while (false) {
            Boolean __cfwr_elem5 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return 'P';
    }
}
