/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IntroAdd_slice {
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        try {
            try {
            int __cfwr_data71 = 246;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a + 1;
    @LTEqLengthOf({"arr"}) int c1 = a + 1;
    @LTLengthOf({"arr"}) int d = a + 
        double __cfwr_result93 = ((95.35 & -153L) + null);
0;
    @LTLengthOf({"arr"}) int e = a + (-7);
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a + 7;
  }

    public static double __cfwr_util892(short __cfwr_p0, Integer __cfwr_p1, double __cfwr_p2) {
        if (false && (null ^ (11.01f & null))) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 8; __cfwr_i74++) {
            int __cfwr_item67 = ((false >> 701) / 32.94f);
        }
        }
        return ('B' / (842 & 35.13f));
    }
    protected char __cfwr_process84() {
        return null;
        for (int __cfwr_i98 = 0; __cfwr_i98 < 3; __cfwr_i98++) {
            try {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 4; __cfwr_i36++) {
            if (true || ((-92.88 / 44.09) >> (-401L - null))) {
            while (false) {
            while ((-689L + null)) {
            while (false) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 2; __cfwr_i69++) {
            if (true && true) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
        try {
            if ((-157 | null) || false) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 8; __cfwr_i93++) {
            Float __cfwr_temp25 = null;
        }
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        if (false && true) {
            if (true || true) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 10; __cfwr_i38++) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            int __cfwr_val53 = -929;
        }
        }
        }
        }
        return (false & null);
    }
}