/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        return null;

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      public long __cfwr_aux
        return 3.35f;
977(Float __cfwr_p0) {
        boolean __cfwr_elem48 = true;
        String __cfwr_elem60 = "test1";
        return -766L;
    }
    private char __cfwr_aux731() {
        if (true && true) {
            Long __cfwr_node16 = null;
        }
        while (false) {
            try {
            Float __cfwr_data99 = null;
        } catch (Exception __cfwr_e56) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return ('4' & 535L);
    }
}
