/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        for (int __cfwr_i24 = 0; __cfwr_i24 < 7; __cfwr_i24++) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 10; __cfwr_i36++) {
            while (((null / 'W') << (-820 & null))) {
            return (322 & (33.34f / -793));
            break; // Prevent infinite loops
        }
        }
        }

    int[] banana;
    int @SameLen("a2") [] b = samelen_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      public static Double __cfwr_proc32(Float __cfwr_p0) {
        try {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 9; __cfwr_i28++) {
            if (true || (null << ('8' >> 'E'))) {
            while (('v' * null)) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 9; __cfwr_i71++) {
            try {
            try {
            if (false && true) {
            if (true || false) {
            try {
            try {
            Character __cfwr_item44 = null;
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        return null;
    }
    public static double __cfwr_func297(Float __cfwr_p0, byte __cfwr_p1) {
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        return null;
        Object __cfwr_node73 = null;
        return ((-87.23f * 145) * null);
        return (false >> null);
    }
    private float __cfwr_util902() {
        try {
            if (true && true) {
            byte __cfwr_var94 = null;
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        for (int __cfwr_i37 = 0; __cfwr_i37 < 1; __cfwr_i37++) {
            Double __cfwr_node16 = null;
        }
        while (false) {
            try {
            if ((-42.40f | true) && false) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 7; __cfwr_i61++) {
            while (((-49.40f << 350) % ('T' * -42.35))) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
        return 17.56f;
    }
}
