/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void samelen_id(int @SameLen("#2") [] a, int[] a2) {
        Object __cfwr_entry22 = null;

    int[] banana;
    int @SameLen("a2") [] b = samelen_identity(a);
    // :: error: (assignment)
    int @SameLen("banana") [] c = samelen_identity(b);
  }

  // UpperBound tests
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
     
        while (false) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 2; __cfwr_i90++) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 5; __cfwr_i24++) {
            try {
            while (false) {
            if (false && true) {
            return ((null | null) >> (24.53f % 'F'));
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
 @LTEqLengthOf({"#1", "#2"}) int abl) {
    int[] c;

    @LTLengthOf("a") int ai1 = ubc_identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = ubc_identity(ai);

    @LTEqLengthOf("a") int al1 = ubc_identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = ubc_identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = ubc_identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = ubc_identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = ubc_identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = ubc_identity(abl);
      private static boolean __cfwr_temp126(double __cfwr_p0) {
        double __cfwr_entry92 = 12.04;
        while (true) {
            try {
            while (false) {
            if (false && true) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 5; __cfwr_i21++) {
            try {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 3; __cfwr_i53++) {
            try {
            Character __cfwr_entry51 = null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        if (((-2L & 402) * (31.22 + 44.63)) || false) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 7; __cfwr_i9++) {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 8; __cfwr_i68++) {
            try {
            if (true && false) {
            while (false) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 1; __cfwr_i84++) {
            Long __cfwr_item52 = null;
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        }
        }
        while ((-185 * ('7' << true))) {
            if ((null & -56.77) && true) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 8; __cfwr_i43++) {
            try {
            if ((('D' << -253L) | (null - null)) || true) {
            try {
            if (true && ((66.27 / null) << 'W')) {
            while (true) {
            try {
            Integer __cfwr_node81 = null;
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        return true;
    }
}
