/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (true || (7.79 << null)) {
            return -221L;
        }

    // :: error: (assignment)
    @LTLengthOf
        return (-747 | -70.46);
("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected static Integer __cfwr_temp692() {
        return (-68L - -827);
        try {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 1; __cfwr_i54++) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 5; __cfwr_i78++) {
            while ((null & (-42.42 / 510))) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 6; __cfwr_i9++) {
            long __cfwr_node93 = 351L;
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        return null;
    }
    protected float __cfwr_aux132() {
        return (-673L << (null - -9.34));
        for (int __cfwr_i19 = 0; __cfwr_i19 < 7; __cfwr_i19++) {
            try {
            return null;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        }
        return -40.05f;
    }
    private static boolean __cfwr_helper103() {
        try {
            try {
            try {
            if (false || (null & 311)) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 7; __cfwr_i91++) {
            if (false && false) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 3; __cfwr_i75++) {
            while (false) {
            return (('E' - 'k') >> null);
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        try {
            while ((940 - null)) {
            while (true) {
            if (false || false) {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 1; __cfwr_i50++) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 10; __cfwr_i26++) {
            if (false || true) {
            while (((-9.19 * -458) - -68.75)) {
            while (false) {
            try {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 9; __cfwr_i30++) {
            while (true) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 2; __cfwr_i85++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 9; __cfwr_i47++) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 2; __cfwr_i64++) {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 3; __cfwr_i5++) {
            try {
            return (false % null);
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        return true;
    }
}
