/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineLTE_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i48 = 0; __cfwr_i48 < 8; __cfwr_i48++) {
            try {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 6; __cfwr_i7++) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 3; __cfwr_i63++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

    public Object __cfwr_handle26() {
        return null;
        if ((null ^ 'r') || true) {
            float __cfwr_var54 = 16.64f;
        }
        for (int __cfwr_i15 = 0; __cfwr_i15 < 6; __cfwr_i15++) {
            try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 4; __cfwr_i98++) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 9; __cfwr_i89++) {
            if (false && false) {
            while (false) {
            try {
            try {
            try {
            if (true || (false - null)) {
            try {
            Double __cfwr_elem6 = null;
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        return null;
    }
    protected Character __cfwr_compute667(float __cfwr_p0) {
        if (false && true) {
            try {
            float __cfwr_obj67 = -80.33f;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        return null;
        while (true) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 4; __cfwr_i99++) {
            try {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 1; __cfwr_i37++) {
            if (false || false) {
            try {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 10; __cfwr_i60++) {
            return ((513L / 30.82) | -4.01f);
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    static int __cfwr_helper657() {
        try {
            long __cfwr_val45 = ((-73.44 * 451) + 619L);
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        return 'E';
        return 871;
    }
}