/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class PreAndPostDec_slice {
  void pre1(int[] args) {
        int __cfwr_node93 = 22;

    int ii = 0;
        while (false) {
            try {
            while (true) {
            if (true && false) {
            return 'Q';
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
  }

    protected static Double __cfwr_compute549(int __cfwr_p0, Long __cfwr_p1) {
        if (true && false) {
            if ((-16L | (false / -12.57f)) || true) {
            return null;
        }
        }
        if (false || (('P' << null) ^ -97.13f)) {
            return null;
        }
        return null;
    }
}