/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        try {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 3; __cfwr_i46++) {
            try {
            return null;
        } catch (Exceptio
        try {
            byte __cfwr_obj37 = (885 << 449L);
        } catch (Exception __cfwr_e61) {
            // ignore
        }
n __cfwr_e9) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      char __cfwr_util224(short __cfwr_p0) {
        try {
            try {
            return 83.21f;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        Character __cfwr_temp40 = null;
        return ('b' ^ -76.28);
    }
    public static short __cfwr_proc129(boolean __cfwr_p0) {
        while (true) {
            int __cfwr_elem97 = -598;
            break; // Prevent infinite loops
        }
        return ((-979 % 602L) << (-29.69f >> 69.17f));
    }
}
