/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        if (false && (null >> '5')) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 8; __cfwr_i56++) {
            if (true || ((null << 357L) | (null ^ 0.41f))) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 3; __cfwr_i71++) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 1; __cfwr_i29++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 9; __cfwr_i47++) {
            Double __cfwr_obj65 = null;
        }
        }
        }
        }
        }
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      protected short __cfwr_aux870() {
        for (int __cfwr_i12 = 0; __cfwr_i12 < 10; __cfwr_i12++) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 7; __cfwr_i37++) {
            try {
            try {
            byte __cfwr_val14 = (93.68f << 367);
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
        }
        if (false && false) {
            Float __cfwr_item19 = null;
        }
        for (int __cfwr_i74 = 0; __cfwr_i74 < 4; __cfwr_i74++) {
            while (false) {
            if (false || true) {
            if (true || true) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 6; __cfwr_i38++) {
            while (false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
