/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i8 = 0; __cfwr_i8 < 6; __cfwr_i8++) {
            return 89L;
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assign
        for (int __cfwr_i52 = 0; __cfwr_i52 < 8; __cfwr_i52++) {
            if (true && false) {
            if (true && false) {
            if (false && false) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 6; __cfwr_i82++) {
            return (('R' << 427L) + null);
        }
        }
        }
        }
        }
ment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      static Character __cfwr_proc176() {
        return null;
        if ((null / -96.86f) && true) {
            while (true) {
            if (true || false) {
            if (((601 % 'Z') >> 'r') || false) {
            while (false) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        if ((false - null) && false) {
            try {
            if (true || false) {
            return '1';
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        if (false || true) {
            if ((null ^ (false << 22.02f)) && false) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 7; __cfwr_i32++) {
            try {
            return null;
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        }
        }
        }
        return null;
    }
}
