/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        if (false 
        try {
            while (false) {
            int __cfwr_result9 = -481;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
|| (-872L >> (7.12 << null))) {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      static boolean __cfwr_aux841() {
        for (int __cfwr_i65 = 0; __cfwr_i65 < 7; __cfwr_i65++) {
            while (false) {
            while ((341L * null)) {
            while ((66.48f << (null << 'k'))) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 2; __cfwr_i33++) {
            while (true) {
            byte __cfwr_temp83 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            Double __cfwr_elem9 = null;
        }
        Long __cfwr_result82 = null;
        while (true) {
            try {
            while (true) {
            while (false) {
            if (true || (null / 818L)) {
            if (true || false) {
            return 'n';
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return true;
    }
}
