/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        if ((-46.85 & (869L % -2.0
        while (false) {
            Object __cfwr_elem65 = null;
            break; // Prevent infinite loops
        }
8f)) || false) {
            try {
            Double __cfwr_item27 = null;
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private short __cfwr_util148(int __cfwr_p0, boolean __cfwr_p1) {
        Long __cfwr_var32 = null;
        return null;
    }
    private static Boolean __cfwr_process952(Boolean __cfwr_p0) {
        for (int __cfwr_i79 = 0; __cfwr_i79 < 3; __cfwr_i79++) {
            return (-793 * 97.79);
        }
        for (int __cfwr_i27 = 0; __cfwr_i27 < 3; __cfwr_i27++) {
            try {
            byte __cfwr_val72 = null;
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        }
        return null;
    }
    static String __cfwr_compute204(char __cfwr_p0, short __cfwr_p1) {
        while (false) {
            char __cfwr_var76 = '0';
            break; // Prevent infinite loops
        }
        return "result86";
    }
}
