/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        long __cfwr_item69 = ((null / -997) & false);

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      String __cfwr_aux865(Boolean __cfwr_p0) {
        Double __cfwr_result20 = null;
        return "result64";
    }
}
