/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        Float __cfwr_temp25 = null;

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      public boolean __cfwr_compute640(Double __cfwr_p0) {
        return null;
        short __cfwr_elem85 = null;
        while (false) {
            while (true) {
            return ('j' << -987L);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return false;
    }
    private static Long __cfwr_aux187(Integer __cfwr_p0, Character __cfwr_p1, Object __cfwr_p2) {
        try {
            Double __cfwr_entry37 = null;
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        try {
            Float __cfwr_item30 = null;
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        if ((false ^ 46.28) || false) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 7; __cfwr_i71++) {
            try {
            byte __cfwr_result93 = null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        }
        return null;
    }
    protected byte __cfwr_aux391(Boolean __cfwr_p0, Float __cfwr_p1, Double __cfwr_p2) {
        if (false || true) {
            while (false) {
            float __cfwr_temp62 = ((-779 % 373L) / (null % -71L));
            break; // Prevent infinite loops
        }
        }
        while (true) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 8; __cfwr_i6++) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 6; __cfwr_i14++) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 5; __cfwr_i45++) {
            while (false) {
            if (false && (null - (-43.86 - -16.93))) {
            double __cfwr_result85 = -13.32;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        return (false | ('L' + -182));
    }
}
