/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m5(String @MinLen(1) [] args, String[] otherArray) {
        short __cfwr_val73 = null;

    // :: error: (assignment)
    String @MinLen(1) [] args2 = java.util.Arrays.copyOf(args, otherArray.length);
  }

  public static void m6(String @MinLen(1) [] args) {
    // :: error: (assignment)
    String @MinLen(1) [] args2 = Arrays.copyOf(args, args.length);
      private static Double __cfwr_util984() {
        return null;
        for (int __cfwr_i2 = 0; __cfwr_i2 < 3; __cfwr_i2++) {
            try {
            int __cfwr_item22 = 928;
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
        return 'f';
        return null;
    }
}
