/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        if (false || ((true ^ null) - -788L)) {
            double __cfwr_var12 = -58.65;
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) in
        return true;
t i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      protected Float __cfwr_util588() {
        return null;
        try {
            Float __cfwr_elem75 = null;
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        return null;
    }
    private Long __cfwr_helper492(Long __cfwr_p0, short __cfwr_p1) {
        Integer __cfwr_item49 = null;
        try {
            long __cfwr_entry69 = 709L;
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        return null;
    }
    public Integer __cfwr_func590(String __cfwr_p0) {
        while ((84.75f & (385L >> -36.28))) {
            Character __cfwr_var26 = null;
            break; // Prevent infinite loops
        }
        while (false) {
            try {
            while (true) {
            try {
            return "temp15";
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
