/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        Double __cfwr_obj82 = null;

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      p
        while (false) {
            while (true) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
rivate static long __cfwr_util681() {
        while (true) {
            if (false || (-1.25 % 823L)) {
            long __cfwr_obj82 = 918L;
        }
            break; // Prevent infinite loops
        }
        return -781L;
    }
    protected short __cfwr_handle603(Object __cfwr_p0, short __cfwr_p1, int __cfwr_p2) {
        while (false) {
            while ((66.01f - null)) {
            short __cfwr_temp78 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        while (((-41.82 & 88.96) & null)) {
            if (true || (-488L << null)) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 10; __cfwr_i90++) {
            short __cfwr_item61 = null;
        }
        }
            break; // Prevent infinite loops
        }
        if (false || (-249 / (-205 / null))) {
            if ((-31.03f * true) && (135L + 174L)) {
            if (false && true) {
            return ((null % 39.88) >> -52.68);
        }
        }
        }
        return null;
    }
    private int __cfwr_temp405(boolean __cfwr_p0, byte __cfwr_p1) {
        try {
            while (true) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 8; __cfwr_i85++) {
            try {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 7; __cfwr_i14++) {
            if (((null >> 20.00) / -81.40f) && true) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 7; __cfwr_i86++) {
            while (true) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 5; __cfwr_i55++) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 5; __cfwr_i33++) {
            try {
            return ((null & null) + 'U');
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        return -93;
    }
}
