/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        while (false) {
            try {
            Boolean __cfwr_temp29 = null;
        } catch (Exception __cfwr_e26) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      protected Integer __cfwr_func254(Double __cfwr_p0) {
        while ((true & 8.73)) {
            return null;
            break; // Prevent infinite loops
        }
        while (true) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 9; __cfwr_i24++) {
            double __cfwr_data44 = ((null ^ -470) - -77.24);
        }
            break; // Prevent infinite loops
        }
        return "result48";
        return "result46";
        return null;
    }
}
