/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i4 = 0; __cfwr_i4 < 8; __cfwr_i4++) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 8; __cfwr_i6++) {
            return 76.50;
        }
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;

    } else {

      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test != b) {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    } else {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      public boolean __cfwr_func794(float __cfwr_p0, float __cfwr_p1, Float __cfwr_p2) {
        return null;
        return false;
    }
    protected static Float __cfwr_util547(Integer __cfwr_p0, Integer __cfwr_p1) {
        while (false) {
            try {
            if (true || (true - true)) {
            if (false && true) {
            return null;
        }
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}
