/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        if (((null - null) >> 15.99) && false) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 3; __cfwr_i38++) {
            return (-712L * (false * null));
        }
        }
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String
        try {
            while (true) {
            while (((47.29f + '2') & 775L)) {
            try {
            return 36;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      protected Character __cfwr_proc10() {
        Object __cfwr_obj58 = null;
        for (int __cfwr_i9 = 0; __cfwr_i9 < 2; __cfwr_i9++) {
            try {
            for (int __cfwr_i68 = 0; __cfwr_i68 < 8; __cfwr_i68++) {
            return null;
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        }
        while (true) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 6; __cfwr_i6++) {
            byte __cfwr_val6 = null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
