/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        while (true) {
            while (false) {
            if ((5.36f ^ ('4' & 583)) || true) {
            long __cfwr_val14 = 826L;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
       
        long __cfwr_data62 = (-44.97 & 'i');
 }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      protected byte __cfwr_proc250(byte __cfwr_p0) {
        for (int __cfwr_i45 = 0; __cfwr_i45 < 1; __cfwr_i45++) {
            while (false) {
            if (true || false) {
            while ((70.44 * 166)) {
            boolean __cfwr_data21 = ('v' - true);
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i57 = 0; __cfwr_i57 < 7; __cfwr_i57++) {
            Object __cfwr_obj25 = null;
        }
        while (false) {
            try {
            if (((-291L & null) & -742) && true) {
            try {
            if (true && false) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 3; __cfwr_i9++) {
            return -224L;
        }
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return ((-33.30f / 813L) * null);
    }
    private Integer __cfwr_compute161() {
        try {
            return null;
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        for (int __cfwr_i29 = 0; __cfwr_i29 < 2; __cfwr_i29++) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 7; __cfwr_i43++) {
            Object __cfwr_obj14 = null;
        }
        }
        return null;
    }
    public Integer __cfwr_func832(Double __cfwr_p0, int __cfwr_p1, Object __cfwr_p2) {
        for (int __cfwr_i47 = 0; __cfwr_i47 < 9; __cfwr_i47++) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            Object __cfwr_elem31 = null;
        }
        }
        return null;
    }
}
