/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        for (int __cfwr_i46 = 0; __cfwr_i46 < 10; __cfwr_i46++) {
            while (false) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 6; __cfwr_i63++) {
            try {
            try {
            if (true && ((null | -394) + (863L | 'C'))) {
            return null;
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      protected static Float __cfwr_temp155(byte __cfwr_p0) {
        Double __cfwr_elem20 = null;
        return null;
    }
    char __cfwr_proc850(short __cfwr_p0, byte __cfwr_p1) {
        try {
            return null;
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        while (true) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 2; __cfwr_i14++) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 8; __cfwr_i91++) {
            try {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 4; __cfwr_i19++) {
            try {
            while (false) {
            while ((36.81 % 'h')) {
            try {
            if (false || true) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 8; __cfwr_i73++) {
            int __cfwr_elem48 = -572;
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i82 = 0; __cfwr_i82 < 8; __cfwr_i82++) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 6; __cfwr_i72++) {
            try {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 4; __cfwr_i90++) {
            if (true && true) {
            while (true) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 2; __cfwr_i66++) {
            while (false) {
            Character __cfwr_elem78 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        }
        for (int __cfwr_i9 = 0; __cfwr_i9 < 8; __cfwr_i9++) {
            while (false) {
            return true;
            break; // Prevent infinite loops
        }
        }
        return 'o';
    }
}
