/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class OffsetAnnotations_slice {
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
  }

    private static Float __cfwr_compute964(Integer __cfwr_p0, short __cfwr_p1, Character __cfwr_p2) {
        try {
            while ((-45.61f + nu
        for (int __cfwr_i40 = 0; __cfwr_i40 < 2; __cfwr_i40++) {
            while (((-334L | 488L) | null)) {
            short __cfwr_data48 = null;
            break; // Prevent infinite loops
        }
        }
ll)) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 1; __cfwr_i92++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }

        if (true && ('J' * (-6.69f - null))) {
            Double __cfwr_entry86 = null;
        }
        return null;
        return null;
    }
}