/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        try {
            if (true || true) {
            try {
            while (((-63.85f & '2') & (-21.05 % -591))) {
            char __cfwr_val39 = 'W';
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr
        for (int __cfwr_i74 = 0; __cfwr_i74 < 3; __cfwr_i74++) {
            return 707L;
        }
_e60) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      public int __cfwr_util931() {
        while (false) {
            try {
            if (false && false) {
            byte __cfwr_elem61 = null;
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i19 = 0; __cfwr_i19 < 9; __cfwr_i19++) {
            try {
            try {
            Long __cfwr_elem65 = null;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        }
        Float __cfwr_elem47 = null;
        return -45;
    }
}
