/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        for (int __cfwr_i90 = 0; __cfwr_i90 < 9; __cfwr_i90++) {
            if (true && false) {
            try {
  
        for (int __cfwr_i63 = 0; __cfwr_i63 < 1; __cfwr_i63++) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 10; __cfwr_i98++) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 9; __cfwr_i90++) {
            if (true || true) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 2; __cfwr_i2++) {
            while (true) {
            try {
            return (949L | 'l');
        } catch (Exception __cfwr_e56) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
          Double __cfwr_data25 = null;
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      private static short __cfwr_func255() {
        try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 7; __cfwr_i62++) {
            try {
            while (true) {
            try {
            while ((true | null)) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 3; __cfwr_i19++) {
            if (true && true) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        if (true || false) {
            if (true && true) {
            try {
            while ((false & 458)) {
            return 23.23f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
        }
        if (true || false) {
            String __cfwr_node81 = "world21";
        }
        return null;
    }
}

  void test4(@NonNegative int x1, int[] a) {
