/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while (((95.00 + null) * (null | null))) {
            while (false) {
            while (true) {
            while (false) {
            retur
        for (int __cfwr_i53 = 0; __cfwr_i53 < 9; __cfwr_i53++) {
            try {
            for (int __cfwr_i5 = 0; __cfwr_i5 < 1; __cfwr_i5++) {
            try {
            while (true) {
            try {
            while (true) {
            if (false || true) {
            if (true || false) {
            try {
            Character __cfwr_elem66 = null;
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
n null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      static Object __cfwr_calc72() {
        for (int __cfwr_i64 = 0; __cfwr_i64 < 1; __cfwr_i64++) {
            byte __cfwr_item75 = null;
        }
        return 353;
        if (false || false) {
            if ((false & false) && false) {
            if (true || true) {
            Long __cfwr_data37 = null;
        }
        }
        }
        if (false || false) {
            while (true) {
            return 'm';
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
