/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr
        while ((-69L * (null / -539))) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 3; __cfwr_i38++) {
            if (true || (-46.02 & (false + false))) {
            if (true && false) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
_i91 = 0; __cfwr_i91 < 5; __cfwr_i91++) {
            return null;
        }


    if (index != -1) {
      array[index] = 1;
    }

    @IndexOrHigh("array") int y = index + 1;
    // :: error: (array.access.unsafe.high)
    array[y] = 1;
    if (y < array.length) {
      array[y] = 1;
    }
    // :: error: (assignment)
    index = array.length;
      Float __cfwr_aux188(Long __cfwr_p0) {
        while (true) {
            if ((-78.42f % -89.68) && false) {
            Double __cfwr_obj7 = null;
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    char __cfwr_helper957(long __cfwr_p0) {
        try {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 7; __cfwr_i31++) {
            while (true) {
            try {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 7; __cfwr_i26++) {
            try {
            try {
            int __cfwr_temp60 = -359;
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        return (-558 & -817);
    }
    public static Boolean __cfwr_temp165() {
        if (false && false) {
            Double __cfwr_elem19 = null;
        }
        return null;
    }
}
