/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i45 = 0; __cfwr_i45 < 1; __cfwr_i45++) {
            try {
            while (false) {
            if (false && true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        }

  
        try {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 3; __cfwr_i21++) {
            return -612L;
        }
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
  // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      protected boolean __cfwr_util684(long __cfwr_p0, Long __cfwr_p1, Double __cfwr_p2) {
        return null;
        return false;
    }
    private String __cfwr_process258(Float __cfwr_p0) {
        while (true) {
            if ((449 - (false * 'W')) && ((893L + -33.09f) << -50.94)) {
            if (false || true) {
            if (true && ((null << null) % null)) {
            Float __cfwr_val92 = null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        return (null & -975L);
        return "value26";
    }
    static Boolean __cfwr_helper572(short __cfwr_p0, Boolean __cfwr_p1, Double __cfwr_p2) {
        return -214L;
        return null;
    }
}
