/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        Character __cfwr_result26 = null;

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      static float __cfwr_temp718(short __cfwr_p0) {
        return null;
        return false;
        return 93.51f;
    }
    int __cfwr_helper252(Boolean __cfwr_p0, long __cfwr_p1, Float __cfwr_p2) {
        for (int __cfwr_i11 = 0; __cfwr_i11 < 3; __cfwr_i11++) {
            Object __cfwr_node25 = null;
        }
        if (false && false) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 1; __cfwr_i26++) {
            String __cfwr_item54 = "value49";
        }
        }
        return 962;
    }
    protected long __cfwr_proc788(short __cfwr_p0, Double __cfwr_p1) {
        return null;
        return (-1.26f >> (-21.95 % null));
    }
}
