/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        if (true || (-95.40f - null)) {
            while (true) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 3; __cfwr_i45++) {
            while (('W' << -57.01)) {
            while (false) {
            try {
            return null;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      public static double __cfwr_proc656(Integer __cfwr_p0, short __cfwr_p1) {
        while ((-406L - null)) {
            long __cfwr_item1 = -158L;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i56 = 0; __cfwr_i56 < 7; __cfwr_i56++) {
            long __cfwr_val21 = ((-392 ^ 'V') | 488L);
        }
        return -84.53;
    }
    protected Long __cfwr_process168(String __cfwr_p0, Double __cfwr_p1) {
        while (true) {
            int __cfwr_item37 = -337;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i1 = 0; __cfwr_i1 < 5; __cfwr_i1++) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 6; __cfwr_i35++) {
            while ((-59.16f & 359)) {
            return (null | true);
            break; // Prevent infinite loops
        }
        }
        }
        for (int __cfwr_i8 = 0; __cfwr_i8 < 7; __cfwr_i8++) {
            try {
            if (true || false) {
            if (((-18.88f % 'E') ^ null) && ((null | null) * -192L)) {
            String __cfwr_temp38 = "test54";
        }
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        }
        return null;
    }
}
