/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i40 = 0; __cfwr_i40 < 4; __cfwr_i40++) {
            while (true) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 10; __cfwr_i55++) {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 3; __cfwr_i25++) {
            while (((null | 195L) / 101L)) {
            if (false || (41.81 ^ (350L % -32))) {
 
        while (true) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 8; __cfwr_i58++) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 8; __cfwr_i69++) {
            if (true || (('m' / -78.48f) / 13.08f)) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 3; __cfwr_i3++) {
            return 3.47f;
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
           while (false) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 4; __cfwr_i40++) {
            try {
            Double __cfwr_temp67 = null;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      protected Double __cfwr_handle81(float __cfwr_p0, short __cfwr_p1, long __cfwr_p2) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 7; __cfwr_i72++) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 9; __cfwr_i22++) {
            if (false || true) {
            try {
            return "item28";
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        }
        }
        }
        while (((false & -59.98f) << (-654 * true))) {
            return null;
            break; // Prevent infinite loops
        }
        Integer __cfwr_obj56 = null;
        return null;
    }
}
