/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        for (int __cfwr_i59 = 0; __cfwr_i59 < 7; __cfwr_i59++) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 1; __cfwr_i29++) {
            while (false) {
            try {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 10; __cfwr_i42++) {
            return null;
        }
        } catch (Exception __cfwr_e23) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }

    int @MinLen(5) [] arr = new int[5];
    int a = 9;
    a += 5;
    a -= 2;
    int @MinLen(12) [] arr1 = new int[a];
    int @MinLen(3) [] arr2 = {1, 2, 3};
    // :: error: (assignment)
    int @MinLen(4) [] arr3 = {4, 5, 6};
    // :: error: (assignment)
    int @MinLen(7) [] arr4 = new int[4];
    // :: error: (assignment)
    int @MinLen(16) [] arr5 = new int[a];
      Integer __cfwr_func872(Integer __cfwr_p0) {
        return -31.54f;
        while (true) {
            while ((('C' + -86.32f) * 51.79)) {
            try {
            if (false && false) {
            if (false || true) {
            return true;
        }
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        while (true) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 2; __cfwr_i30++) {
            while (true) {
            String __cfwr_data38 = "data71";
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    private Float __cfwr_temp249() {
        Character __cfwr_result99 = null;
        return null;
    }
}
