/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (true || false) {
            boolean __cfwr_val13 = true;
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLen
        while (((89.19f / -154L) & -76.18f)) {
            if (true || true) {
            try {
            try {
            return "hello95";
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
gthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private String __cfwr_handle424(char __cfwr_p0, char __cfwr_p1) {
        return null;
        if (false && false) {
            return null;
        }
        for (int __cfwr_i55 = 0; __cfwr_i55 < 8; __cfwr_i55++) {
            if (true && true) {
            if (true && false) {
            if (false || true) {
            if (((null % true) + 'T') || true) {
            try {
            return null;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        }
        }
        }
        }
        return "result74";
    }
    public byte __cfwr_proc812(Object __cfwr_p0) {
        try {
            try {
            while ((-41.23 >> (-111 >> false))) {
            Object __cfwr_val45 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        while (false) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 7; __cfwr_i3++) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 10; __cfwr_i34++) {
            while (false) {
            if (false || ((145 ^ true) * null)) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i1 = 0; __cfwr_i1 < 6; __cfwr_i1++) {
            boolean __cfwr_data88 = (null / 70.20);
        }
        return null;
    }
    public static int __cfwr_temp43(Long __cfwr_p0, int __cfwr_p1) {
        while (((571L >> 186L) % -93L)) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i8 = 0; __cfwr_i8 < 8; __cfwr_i8++) {
            return ((-359 << null) - ('x' | null));
        }
        return ('0' / null);
    }
}
