/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        return null;
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    private Character __cfwr_helper581(String __cfwr_p0, short __cfwr_p1, Long __cfwr_p2) {
        while (true) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 5; __cfwr_i59++) {
            int __cfwr_obj97 = 37;
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}
