/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        while (false) {
            try {
            return ('u' & 69L);
        } catch (Exception __cfwr_e67) {
            /
        if (true || ((null % null) << (84.32 ^ 953L))) {
            if (((null & -37.61) * 'U') && ((328L - 'Q') & null)) {
            while (false) {
            try {
            return (-598L + null);
        } catch (Exception __cfwr_e15) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
/ ignore
        }
            break; // Prevent infinite loops
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      Object __cfwr_process976(String __cfwr_p0, byte __cfwr_p1, double __cfwr_p2) {
        while (((982 / 993L) | false)) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 8; __cfwr_i9++) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 10; __cfwr_i95++) {
            while (false) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 9; __cfwr_i49++) {
            try {
            if (false || false) {
            char __cfwr_node60 = 'w';
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        if (true || true) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 4; __cfwr_i98++) {
            try {
            return null;
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }
        }
        return null;
    }
}
