/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test1(@Positive int x, @Positive int y) {
        for (int __cfwr_i3 = 0; __cfwr_i3 < 2; __cfwr_i3++) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            Character __cfwr_temp66 
        for (int __cfwr_i54 = 0; __cfwr_i54 < 2; __cfwr_i54++) {
            try {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 4; __cfwr_i45++) {
            if (false && false) {
            if ((('2' & true) & null) || false) {
            while (true) {
            try {
            try {
            try {
            if (false || (-16.36 ^ -522L)) {
            short __cfwr_val73 = null;
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        }
= null;
        }
        }

    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexFor("newArray") int j = y;
  }

  void test2(@NonNegative int x, @Positive int y) {
    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test3(@NonNegative int x, @NonNegative int y) {
    int[] newArray = new int[x + y];
    @IndexOrHigh("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test4(@GTENegativeOne int x, @NonNegative int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    @LTEqLengthOf("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test5(@GTENegativeOne int x, @GTENegativeOne int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexOrHigh("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test6(int x, int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexFor("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
      private static String __cfwr_func890(int __cfwr_p0) {
        try {
            short __cfwr_node15 = null;
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        while (false) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 9; __cfwr_i67++) {
            try {
            while ((-44L << (62.54 / null))) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 10; __cfwr_i55++) {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 9; __cfwr_i16++) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 10; __cfwr_i38++) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 8; __cfwr_i19++) {
            try {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 1; __cfwr_i6++) {
            if ((('l' / null) / null) || (121L / 703)) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return "temp67";
    }
    public Character __cfwr_aux97(Boolean __cfwr_p0, char __cfwr_p1) {
        if (false && false) {
            return null;
        }
        return null;
        for (int __cfwr_i4 = 0; __cfwr_i4 < 9; __cfwr_i4++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 4; __cfwr_i73++) {
            if (((-64.76f + -68.59) / 888) || true) {
            if (true && true) {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 1; __cfwr_i65++) {
            byte __cfwr_temp78 = null;
        }
        }
        }
        }
        }
        Object __cfwr_node88 = null;
        return null;
    }
}
