/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    private Character __cfwr_temp36() {
        long __cfwr_entry31 = 500L;

        try {
            for (int __cfwr_i78
        return 116;
 = 0; __cfwr_i78 < 1; __cfwr_i78++) {
            if ((null ^ 'S') && (-230L + 726L)) {
            if (true || true) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 9; __cfwr_i28++) {
            while (('t' - true)) {
            Boolean __cfwr_val92 = null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        boolean __cfwr_elem19 = true;
        return null;
    }
