/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        for (int __cfwr_i40 = 0; __cfwr_i40 < 8; __cfwr_i40++) {
            return null;
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

 
        if ((430L + -98.30f) && false) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            if (false || true) {
            while (false) {
            while (false) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 9; __cfwr_i55++) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 4; __cfwr_i91++) {
            try {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 2; __cfwr_i56++) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 3; __cfwr_i75++) {
            try {
            while (true) {
            while ((true * 485L)) {
            return (75.07 << (-17.10f | -34.25));
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
 int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      static Float __cfwr_compute584(short __cfwr_p0) {
        byte __cfwr_temp42 = null;
        return null;
        char __cfwr_item17 = 'f';
        return null;
    }
    protected Object __cfwr_temp238() {
        Object __cfwr_data52 = null;
        return null;
    }
}
