/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i55 = 0; __cfwr_i55 < 4; __cfwr_i55++) {
            while (true) {
            while (((-229L ^ -62.96) ^ -72.54)) {
            if (false && false) {
            try {
            return null;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
            break;
        while (false) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            Long __cfwr_temp65 = null;
        }
            break; // Prevent infinite loops
        }
 // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      protected Double __cfwr_aux397(Double __cfwr_p0, Boolean __cfwr_p1, byte __cfwr_p2) {
        if (false && false) {
            if ((null >> (69.54 - 689L)) && true) {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 8; __cfwr_i21++) {
            if (true && false) {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 10; __cfwr_i96++) {
            while (((null ^ -579) + (49.21 ^ -53.64))) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 7; __cfwr_i6++) {
            try {
            try {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 10; __cfwr_i28++) {
            if (true || (null * (-37L >> 49.01))) {
            return null;
        }
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
        }
        return null;
    }
    private static float __cfwr_process422(Long __cfwr_p0, char __cfwr_p1, Character __cfwr_p2) {
        try {
            char __cfwr_entry83 = 'u';
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        try {
            if (true || true) {
            try {
            return "data52";
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        return null;
        return -76.13f;
    }
    protected long __cfwr_util817(Boolean __cfwr_p0, int __cfwr_p1) {
        if ((null >> (83.03 << false)) && true) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 6; __cfwr_i47++) {
            try {
            char __cfwr_node78 = 'h';
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        }
        }
        Character __cfwr_result97 = null;
        return 660L;
    }
}
