/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        if (('8' / false) || (null >> (true << -511))) {
            return null;
        }

    return factorials[k];
  }

  public static
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
 void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      public String __cfwr_func652(Character __cfwr_p0, boolean __cfwr_p1, Boolean __cfwr_p2) {
        if ((null * true) || true) {
            Boolean __cfwr_var91 = null;
        }
        if (true || (null | null)) {
            while (false) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 7; __cfwr_i86++) {
            if (true && (9.64f / 'A')) {
            for (int __cfwr_i83 = 0; __cfwr_i83 < 5; __cfwr_i83++) {
            if (false && false) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 7; __cfwr_i80++) {
            byte __cfwr_obj44 = null;
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        if (false && true) {
            return null;
        }
        return "value96";
    }
    public byte __cfwr_temp422(Boolean __cfwr_p0, short __cfwr_p1) {
        while (true) {
            return -82.85f;
            break; // Prevent infinite loops
        }
        long __cfwr_temp12 = 846L;
        return null;
    }
    public static Object __cfwr_util372(Boolean __cfwr_p0, Character __cfwr_p1) {
        return ((null - 39.97) - (-802L / -72.88));
        while (false) {
            if (false || true) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        Long __cfwr_data23 = null;
        return null;
    }
}
