/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineLT_slice {
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        Object __cfwr_node73 = null;

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
  }

    protected static Boolean __cfwr_aux963(Integer __cfwr_p0, float __cfwr_p1, Integer __cfwr_p2) {
        for (int __cfwr_i69 = 0; __cfwr_i69 < 7; __cfwr_i69++) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 4; __cfwr_i63++) {
            if (false || false) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 6; __cfwr_i7++) {
            if (false || false) {
            while ((-79L + -14)) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 6; __cfwr_i33++) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 7; __cfwr_i67++) {
            if (false || ('Y' & null)) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 10; __cfwr_i28++) {
            return "value90";
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }
        try {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 2; __cfwr_i84++) {
            return null;
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        Character __cfwr_result5 = null;
        return null;
        return null;
    }
    private static long __cfwr_helper394(Boolean __cfwr_p0) {
        try {
            long __cfwr_entry22 = ((-39.19 << true) << -70.49);
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        if (((-12.01f >> 7.08f) - true) && true) {
            Boolean __cfwr_entry4 = null;
        }
        try {
            String __cfwr_elem17 = "data66";
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        return 102L;
    }
    protected static Boolean __cfwr_func626(int __cfwr_p0, Object __cfwr_p1, boolean __cfwr_p2) {
        if (false || true) {
            try {
            String __cfwr_temp27 = "value99";
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        }
        return null;
        return null;
    }
}