/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    static double __cfwr_compute51(String __cfwr_p0, Long __cfwr_p1, Boolean __cfwr_p2) {
        Long __cfwr_entry83 = null;

        try {
            try {
            while (true) {
            if (false && true) {
            for (int __cfwr_i82 = 0; __cfwr_i82 < 5; __cfwr_i82++) {
            try {
            return 8.48;
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        } catch (Exception __cfwr_e16) {
            // ignore
        }
        try {
            float __cfwr_data97 = -45.13f;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        return null;
        return null;
        return -85.75;
    }
    protected static String __cfwr_handle409() {
        while ((-322L * (null ^ -579))) {
            try {
            String __cfwr_temp30 = "test57";
        } catch (Exception __cfwr_e59) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "temp25";
    }
