/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    public byte __cfwr_proc969(Character __cfwr_p0, int __cfwr_p1) {
        if (false && (-21.21 | null)) {
            if (false && false) {
            float __cfwr_temp87 = 59.10f;
        }
        }

        try {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 1; __cfwr_i6++) {
            return null;
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        try {
            while (false) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 4; __cfwr_i51++) {
            while (false) {
            try {
            return null;
        } catch (Exception __cfwr_e97) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        return null;
    }
