/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        for (int __cfwr_i62 = 0; __cfwr_i62 < 2; __cfwr_i62++) {
            if (('9' - 'o') && ((null + 'm') + (true << -805))) {
            try {
            if (true || (null - false)) {
            while (true) {
            if (true && true) {
            try {
            while (((727L % 'n') & -69)) {
            if (true && true) {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 9; __cfwr_i81++) {
            try {
            try {
            byte __cfwr_entry82 = ('e' ^ true);
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        }

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      protected static boolean __cfwr_calc404(char __cfwr_p0, int __cfwr_p1, Long __cfwr_p2) {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 9; __cfwr_i60++) {
            for (int __cfwr_i80 = 0; __cfwr_i80 < 3; __cfwr_i80++) {
            Integer __cfwr_obj74 = null;
        }
        }
        return false;
    }
}
