/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 4; __cfwr_i21++) {
            if (false && false) {
            while (true) {
            return "item76";
            break; // Prevent infinite loops
        }
        }
        }

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      Double __cfwr_compute21(Character __cfwr_p0, Object __cfwr_p1) {
        Long __cfwr_val83 = null;
        int __cfwr_val9 = -689;
        return null;
    }
    private char __cfwr_helper977() {
        try {
            try {
            Double __cfwr_node72 = null;
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        return 'c';
    }
}
