/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        if (((515L * null) + null) || (null | (110 << false))) {
            for (int __cfwr_i51 = 0; __cfwr_i51 < 1; __cfwr_i51++) {
            return null;
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignme
        try {
            try {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 10; __cfwr_i65++) {
            try {
            if (true && true) {
            if (false || (('w' << null) ^ (55.08 | false))) {
            Double __cfwr_obj12 = null;
        }
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
nt)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private String __cfwr_proc262(short __cfwr_p0, boolean __cfwr_p1) {
        while (true) {
            return 57.11;
            break; // Prevent infinite loops
        }
        double __cfwr_val73 = (null * '7');
        if (('B' & (49 + 67)) || false) {
            return -8.43f;
        }
        return "test23";
    }
    protected Float __cfwr_handle988(double __cfwr_p0) {
        try {
            if (true && false) {
            if (((null ^ -66.00f) | 413) && false) {
            try {
            if (('w' << 281L) || true) {
            return 889;
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        if ((33L ^ (null + null)) && false) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 8; __cfwr_i69++) {
            String __cfwr_item48 = "item60";
        }
        }
        while (false) {
            while (true) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 10; __cfwr_i35++) {
            while (false) {
            char __cfwr_node82 = '8';
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
