/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        return -80.05f;

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
      public Float __cfwr_handle569(Float __cfwr_p0) {
        if (false || false) {
            return 58.61;
        }
        try {
            if (true || true) {
            if (true && (32.80 + 'R')) {
            if ((4.80f % 'p') || true) {
            while (false) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            Float __cfwr_elem25 = null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        return "data71";
        return null;
    }
    public short __cfwr_calc944(Double __cfwr_p0, boolean __cfwr_p1) {
        for (int __cfwr_i43 = 0; __cfwr_i43 < 1; __cfwr_i43++) {
            while (false) {
            return -81L;
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    protected static Long __cfwr_calc164(char __cfwr_p0) {
        try {
            Integer __cfwr_temp14 = null;
        } catch (Exception __cfwr_e81) {
            // ignore
        }
        return null;
    }
}
