/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class OffsetAnnotations_slice {
  public static void OffsetAnnotationsReader() throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedReader.read(buffer, 5, 7);
  }

  public static void OffsetAnnotationsWriter() throws IOException {
    BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
    char[] buffer = new char[10];
    // :: error: (argument)
    bufferedWriter.write(buffer, 5, 7);
  }

    protected static byte __cfwr_proc161(byte __cfwr_p0, Object __cfwr_p1) {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 6; __cfwr_i36++) {
            try {
            return ((true ^ 501L) << 'H');
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }

        for (int __cfwr_i47 = 0; __cfwr_i47 < 6; __cfwr_i47++) {
            for (int __cfwr_i91 = 0; __cfwr_i91 < 4; __cfwr_i91++) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 9; __cfwr_i53++) {
            return null;
        }
        }
        }
        return null;
    }
    static Float __cfwr_compute772(boolean __cfwr_p0) {
        boolean __cfwr_data2 = false;
        return null;
    }
}