/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 2; __cfwr_i36++) {
            return (null << -2.62);
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different seq
        if (false || (-13.19 - 757L)) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 1; __cfwr_i90++) {
            return 141;
        }
        }
uences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      protected static float __cfwr_handle945(Float __cfwr_p0) {
        while ((null >> 269)) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 2; __cfwr_i47++) {
            if (((true >> -33L) / null) || false) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 8; __cfwr_i32++) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        while ((1.65 * 'B')) {
            byte __cfwr_data20 = ((-436L / true) >> false);
            break; // Prevent infinite loops
        }
        return (('F' & 24.74) % true);
    }
}
