/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        for (int __cfwr_i41 = 0; __cfwr_i41 < 2; __cfwr_i41++) {
            while ((36.03f / (89.29f + '1'))) {
            return -43.50;
            break; // Prevent infinite loops
        }
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      double __cfwr_calc473(String __cfwr_p0, boolean __cfwr_p1) {
        while ((54.46 & (false >> '5'))) {
            if (true || false) {
            Boolean __cfwr_node78 = null;
        }
            break; // Prevent infinite loops
        }
        return 89.06;
    }
    public static Integer __cfwr_temp236(Boolean __cfwr_p0, long __cfwr_p1) {
        Character __cfwr_item50 = null;
        double __cfwr_val7 = (-70.31 % 'k');
        for (int __cfwr_i98 = 0; __cfwr_i98 < 1; __cfwr_i98++) {
            while (true) {
            return -274;
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    private static Boolean __cfwr_func614(Object __cfwr_p0, Boolean __cfwr_p1, Boolean __cfwr_p2) {
        try {
            while (('J' ^ -236)) {
            while (false) {
            if (true && true) {
            if (false || false) {
            return (null & (97.46 / 'o'));
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e50) {
            // ignore
        }
        return null;
    }
}

  void test4(@NonNegative int x1, int[] a) {
