/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        for (int __cfwr_i45 = 0; __cfwr_i45 < 3; __cfwr_i45++) {
            if (false || true) {
            if (false || true) {
            if (false && true) {
            String __cfwr_var66 = "hello40";
        }
        }
        }
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    
        while (false) {
            if (false || false) {
            while (true) {
            if (false && true) {
            return (15.51 - null);
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
// :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      public String __cfwr_helper340() {
        if (false || true) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 6; __cfwr_i94++) {
            byte __cfwr_item95 = null;
        }
        }
        try {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 4; __cfwr_i32++) {
            Float __cfwr_entry59 = null;
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        try {
            while ((false ^ 397L)) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        if (true || true) {
            return null;
        }
        return "hello92";
    }
}
