/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IntroAnd_slice {
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        for (int __cfwr_i46 = 0; __cfwr_i46 < 4; __cfwr_i46++) {
            while (true) {
            while (true) {
            for (int __cfwr_i81 = 0; __cfwr_i81 < 7; __cfwr_i81++) {
            try {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 9; __cfwr_i35++) {
            return null;
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
  }

    protected String __cfwr_handle939(int __cfwr_p0, short __cfwr_p1, short __cfwr_p2) {
        for (int __cfwr_i10 = 0; __cfwr_i10 < 9; __cfwr_i10++) {
            return null;
        }
        return null;
        return "item50";
    }
    protected static char __cfwr_handle742(long __cfwr_p0, Object __cfwr_p1) {
        for (int __cfwr_i46 = 0; __cfwr_i46 < 9; __cfwr_i46++) {
            return (false | (-9.36 % null));
        }
        while ((null - -97.33)) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 1; __cfwr_i6++) {
            if (true && true) {
            return "data42";
        }
        }
            break; // Prevent infinite loops
        }
        return (-323L & 31.52f);
    }
}