/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        while (false) {
            String __cfwr_elem70 = "temp29";
            break; // Prevent infinite loops
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      static Boolean __cfwr_process90(char __cfwr_p0) {
        Double __cfwr_node85 = null;
        Boolean __cfwr_data29 = null;
        return 805L;
        while (true) {
            Long __cfwr_entry47 = null;
            break; // Prevent infinite loops
        }
        return null;
    }
}
