/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  ArrayAssignmentSameLen(int[] array, @IndexFor("#1") int index) {
        while (true) {
            if (true && (false + -586)) {
            while (true) {
            for (i
        while (false) {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 9; __cfwr_i79++) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            return ('u' & 614L);
        }
        }
            break; // Prevent infinite loops
        }
nt __cfwr_i28 = 0; __cfwr_i28 < 1; __cfwr_i28++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }

    i_array = array;
    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {
    int[] c1 = a;
    // See useTest3 for an example of why this assignment should fail.
    @LTLengthOf(
        value = {"c1", "c1"},
        offset = {"0", "x"})
    // :: error: (assignment)
    int z = i;
      String __cfwr_handle216(Double __cfwr_p0, Integer __cfwr_p1, short __cfwr_p2) {
        try {
            while (true) {
            String __cfwr_elem58 = "temp48";
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        double __cfwr_item80 = 52.71;
        try {
            if (true || true) {
            if (true || (null - -2.30f)) {
            int __cfwr_data57 = 734;
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        return "result99";
    }
    protected static Character __cfwr_aux480(Character __cfwr_p0, Character __cfwr_p1, Integer __cfwr_p2) {
        if (true || true) {
            Float __cfwr_obj99 = null;
        }
        return null;
    }
    static int __cfwr_calc478(Float __cfwr_p0) {
        if (((null ^ 64.36f) | -50.18) || false) {
            try {
            Object __cfwr_entry21 = null;
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        }
        if (false || false) {
            if ((-604 - 470L) && true) {
            while (false) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 6; __cfwr_i9++) {
            if (false && (-80.71 / (23L | null))) {
            byte __cfwr_data88 = (-52 - 586L);
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        Character __cfwr_var86 = null;
        if ((true - -24.92f) || true) {
            while (true) {
            try {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 1; __cfwr_i89++) {
            return 233L;
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return 934;
    }
}
