/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        for (int __cfwr_i49 = 
        return (null * 963L);
0; __cfwr_i49 < 6; __cfwr_i49++) {
            Double __cfwr_val30 = null;
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public Long __cfwr_handle817(Object __cfwr_p0, int __cfwr_p1, Long __cfwr_p2) {
        for (int __cfwr_i12 = 0; __cfwr_i12 < 4; __cfwr_i12++) {
            try {
            return 47.45;
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        }
        if (true || false) {
            try {
            Float __cfwr_obj77 = null;
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
        return null;
    }
    private Long __cfwr_func365(Float __cfwr_p0, Double __cfwr_p1, int __cfwr_p2) {
        return ((411 << 645) % true);
        return null;
        return null;
    }
    private static int __cfwr_proc438(Integer __cfwr_p0, float __cfwr_p1, double __cfwr_p2) {
        try {
            return null;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        for (int __cfwr_i97 = 0; __cfwr_i97 < 5; __cfwr_i97++) {
            return null;
        }
        return null;
        return ('U' - 98.78);
    }
}
