/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        long __cfwr_elem94 = 141L;

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");
        while (true) {
            try {
            while (false) {
            Float __cfwr_result42 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
            break; // Prevent infinite loops
        }


    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private static Long __cfwr_calc762() {
        return false;
        return null;
    }
    protected static Boolean __cfwr_util16() {
        if (true && (('Z' ^ -511L) | -105L)) {
            Character __cfwr_val19 = null;
        }
        if (('e' >> false) && true) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 6; __cfwr_i8++) {
            return null;
        }
        }
        return null;
        return null;
    }
    protected static int __cfwr_handle163(Object __cfwr_p0) {
        for (int __cfwr_i9 = 0; __cfwr_i9 < 9; __cfwr_i9++) {
            return 'u';
        }
        return 304;
    }
}
