/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class InvalidSubsequence_slice {
  void assignB(int[] d) {
        if (true || false) {
            try {
            return true;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
  }

    protected static Float __cfwr_process523(Object __cfwr_p0) {
        return null;
        if (true && (null / -10.79)) {
            try {
            Float __cfwr_data98 = null;
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        return null;
    }
    public Double __cfwr_util32(Integer __cfwr_p0, int __cfwr_p1, int __cfwr_p2) {
        return "data34";
        while (false) {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 10; __cfwr_i34++) {
            try {
            if (false || true) {
            try {
            while (false) {
            try {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 9; __cfwr_i79++) {
            return 'm';
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    short __cfwr_proc644(Integer __cfwr_p0, long __cfwr_p1, short __cfwr_p2) {
        while (((-971L % 362L) + null)) {
            return null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i26 = 0; __cfwr_i26 < 4; __cfwr_i26++) {
            if (((585 ^ 638L) & (null * null)) || ('g' & false)) {
            return null;
        }
        }
        return (null >> 632L);
    }
}