/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class MinLenIndexFor_slice {
  void callTest(int x) {
        if (false && false) {
            try {
            short __cfwr_node54 = null;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
  }

    Integer __cfwr_handle303(Boolean __cfwr_p0, boolean __cfwr_p1, byte __cfwr_p2) {
        long __cfwr_elem32 = 475L;
        if (true && (null | -480)) {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 10; __cfwr_i26++) {
            return 'Q';
        }
        }
        for (int __cfwr_i59 = 0; __cfwr_i59 < 6; __cfwr_i59++) {
            return true;
        }
        for (int __cfwr_i42 = 0; __cfwr_i42 < 3; __cfwr_i42++) {
            while ((776 << 412L)) {
            short __cfwr_result60 = null;
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    protected Character __cfwr_helper659(int __cfwr_p0, Long __cfwr_p1, char __cfwr_p2) {
        if (true || (null - -50L)) {
            try {
            return null;
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        while ((null + (-74.79 / false))) {
            while (false) {
            while (true) {
            try {
            Boolean __cfwr_entry6 = null;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    static String __cfwr_calc896(Character __cfwr_p0) {
        try {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 2; __cfwr_i23++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 5; __cfwr_i86++) {
            while (true) {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 7; __cfwr_i60++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        return "item58";
        return 95.59f;
        return "data35";
    }
}