/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        try {
            if (((-64.57f / -60.95f) << ('k' | 210)) && ((800 % null) - 3.49f)) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positi
        Character __cfwr_node68 = null;
ve int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      private static String __cfwr_process878(float __cfwr_p0, boolean __cfwr_p1, Integer __cfwr_p2) {
        boolean __cfwr_data26 = true;
        return null;
        for (int __cfwr_i41 = 0; __cfwr_i41 < 9; __cfwr_i41++) {
            boolean __cfwr_data67 = false;
        }
        return null;
        return "value95";
    }
}
