/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        if (false && true) {
            long __cfwr_item63 = -35L;
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      Float __cfwr_calc865() {
        Double __cfwr_elem16 = null;
        for (int __cfwr_i43 = 0; __cfwr_i43 < 5; __cfwr_i43++) {
            while (((true ^ 32.15f) - (812L | 'V'))) {
            if ((-871 / 28.68f) || false) {
            long __cfwr_temp8 = 476L;
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
