/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        return -833;

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private static Long __cfwr_helper459(int __cfwr_p0, long __cfwr_p1) {
        for (int __cfwr_i98 = 0; __cfwr_i98 < 10; __cfwr_i98++) {
            while ((false | null)) {
            while ((null % 21.39)) {
            try {
            try {
            for (int __cfwr_i16 = 0; __cfwr_i16 < 2; __cfwr_i16++) {
            if (true || ((388 & null) ^ 943)) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            Object __cfwr_entry32 = null;
        }
        }
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    short __cfwr_func687() {
        try {
            while (false) {
            while (true) {
            while (true) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 6; __cfwr_i95++) {
            return null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        for (int __cfwr_i47 = 0; __cfwr_i47 < 9; __cfwr_i47++) {
            try {
            try {
            while (((24.53f & -304) >> 'h')) {
            try {
            while (false) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 9; __cfwr_i58++) {
            try {
            return (-74.81f ^ ('n' + 82.50));
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        }
        return -72.85f;
        byte __cfwr_obj43 = null;
        return null;
    }
}
