/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        while ((null & (-68.08f / 213L))) {
            try {
            if (((true * false) - (690 ^ -43.68f)) || true) {
            Integer __cfwr_val18 = null;
        }
        } catch (Exception __cfwr_e81) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d",
        while (true) {
            try {
            if (true && false) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 6; __cfwr_i36++) {
            try {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 1; __cfwr_i70++) {
            while (('F' >> 0.09)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
            break; // Prevent infinite loops
        }
 "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      private short __cfwr_aux703(int __cfwr_p0, float __cfwr_p1) {
        for (int __cfwr_i88 = 0; __cfwr_i88 < 8; __cfwr_i88++) {
            try {
            return null;
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        while (true) {
            if (false || false) {
            int __cfwr_item91 = 951;
        }
            break; // Prevent infinite loops
        }
        return -42.22;
        for (int __cfwr_i34 = 0; __cfwr_i34 < 9; __cfwr_i34++) {
            byte __cfwr_item22 = null;
        }
        return ((-839 & 232L) | 'T');
    }
    int __cfwr_proc478() {
        while ((667 & 251L)) {
            Boolean __cfwr_result70 = null;
            break; // Prevent infinite loops
        }
        return (57.13 >> (false + true));
    }
}
