/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        if (false && false) {
            if (false && false) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 5; __cfwr_i57++) {
            while (false) {
            while (true) {
            if (true && false) {
            while (false) {
            if (true && true) {
            return "temp11";
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assi
        while (false) {
            if ((23.75 >> null) || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
gnment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"}) int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      public long __cfwr_temp641(Double __cfwr_p0, char __cfwr_p1, Double __cfwr_p2) {
        return (-93.19f - 'M');
        while (false) {
            Object __cfwr_data23 = null;
            break; // Prevent infinite loops
        }
        return -849L;
    }
    private Character __cfwr_compute236(Integer __cfwr_p0, char __cfwr_p1, short __cfwr_p2) {
        return null;
        while (true) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 9; __cfwr_i44++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return null;
        try {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 1; __cfwr_i2++) {
            try {
            double __cfwr_val46 = -81.78;
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        return null;
    }
    Float __cfwr_compute536(float __cfwr_p0, Double __cfwr_p1, Double __cfwr_p2) {
        for (int __cfwr_i26 = 0; __cfwr_i26 < 4; __cfwr_i26++) {
            try {
            if (true && (true ^ (null ^ null))) {
            if (false && (561L % 63.26)) {
            try {
            try {
            try {
            char __cfwr_elem97 = ((265L & 97.42) >> 't');
        } catch (Exception __cfwr_e96) {
            // ignore
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        }
        int __cfwr_result61 = 121;
        while ((-8.23 - (-143L - null))) {
            int __cfwr_node26 = (34.88f & (-331 ^ -924));
            break; // Prevent infinite loops
        }
        return null;
    }
}
