/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    public static double __cfwr_process142(Float __cfwr_p0) {
        for (int __cfw
        try {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 9; __cfwr_i28++) {
            try {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 7; __cfwr_i92++) {
            while (false) {
            try {
            Object __cfwr_elem40 = null;
        } catch (Exception __cfwr_e90) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
r_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 6; __cfwr_i6++) {
            try {
            Double __cfwr_val81 = null;
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        }

        while (true) {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 7; __cfwr_i78++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        return 327;
        try {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 5; __cfwr_i53++) {
            Double __cfwr_obj45 = null;
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return (null + 694);
    }
    private static Double __cfwr_compute771(float __cfwr_p0, Character __cfwr_p1) {
        if (false || true) {
            while (true) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 7; __cfwr_i8++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        while ((null >> 8L)) {
            Boolean __cfwr_temp10 = null;
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
    static int __cfwr_calc658(short __cfwr_p0) {
        if (true && false) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 2; __cfwr_i24++) {
            try {
            try {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 7; __cfwr_i48++) {
            if (false && false) {
            return null;
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        }
        }
        while (((null << null) * null)) {
            short __cfwr_item66 = null;
            break; // Prevent infinite loops
        }
        return -317;
    }
