/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        for (int __cfwr_i74 
        Boolean __cfwr_var34 = null;
= 0; __cfwr_i74 < 6; __cfwr_i74++) {
            try {
            while (((-887 << -81.16f) >> (257 | -31L))) {
            if ((939L - 472L) && (-802L / -211)) {
            try {
            Object __cfwr_var97 = null;
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      protected static int __cfwr_temp287(Float __cfwr_p0, Object __cfwr_p1) {
        try {
            return null;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        if (true && false) {
            if (true || ((999L >> 10.95f) & -73.07f)) {
            for (int __cfwr_i33 = 0; __cfwr_i33 < 2; __cfwr_i33++) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 7; __cfwr_i98++) {
            if ((true ^ (null + null)) && false) {
            if ((0.81 >> 'b') && false) {
            try {
            while ((-163 >> (53.99f * null))) {
            Integer __cfwr_temp9 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        }
        }
        }
        }
        }
        return 945;
    }
    private Character __cfwr_func482(double __cfwr_p0, boolean __cfwr_p1, short __cfwr_p2) {
        return null;
        if (true && (null >> ('B' * 75.27f))) {
            float __cfwr_obj86 = -75.20f;
        }
        while (false) {
            try {
            short __cfwr_obj85 = null;
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
