/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class PlusPlusBug_slice {
  void test(@LTLengthOf("array") int x) {
        return (null >> 21.87);

    // :: error: (unary.increment)
    x++;
    // :: error: (unary.increment)
    ++x;
    // :: error: (assignment)
    x = x + 1;
  }

    private Integer __cfwr_aux678(Long __cfwr_p0, do
        while (true) {
            try {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 4; __cfwr_i89++) {
            float __cfwr_var57 = -79.99f;
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
            break; // Prevent infinite loops
        }
uble __cfwr_p1, byte __cfwr_p2) {
        while (true) {
            return (60.48 & -46L);
            break; // Prevent infinite loops
        }
        try {
            try {
            long __cfwr_val32 = ((true & null) << 1000L);
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        try {
            if ((null & (true | 59.34)) || false) {
            try {
            return "temp52";
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return null;
    }
    public static Integer __cfwr_compute153() {
        for (int __cfwr_i84 = 0; __cfwr_i84 < 1; __cfwr_i84++) {
            try {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 4; __cfwr_i95++) {
            while (((-754 & 99.33) - 75L)) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 1; __cfwr_i38++) {
            while (((17.23 & -19.57f) >> (true + null))) {
            Boolean __cfwr_obj10 = null;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
        Float __cfwr_elem95 = null;
        try {
            while ((-918 - -524L)) {
            if (false || false) {
            Boolean __cfwr_val37 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        return null;
    }
}