/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        try {
            while (true) {
            if (false || true) {
            if (false && ((null >> -0.27f) * (null | -628))) {
            while (false) {
            if ((388L / null) || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      static short __cfwr_helper188(boolean __cfwr_p0, double __cfwr_p1) {
        return (-66.90 ^ 'l');
        for (int __cfwr_i41 = 0; __cfwr_i41 < 5; __cfwr_i41++) {
            try {
            if (((97L - null) * ('J' % 276)) && (null % (-516L + 'n'))) {
            while (true) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 9; __cfwr_i28++) {
            try {
            if (false || false) {
            try {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 2; __cfwr_i17++) {
            long __cfwr_data64 = 231L;
        }
        } catch (Exception __cfwr_e91) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        }
        return (-21 ^ '9');
    }
}
