/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 4; __cfwr_i63++) {
            try {
            int __cfwr_node66 = 3;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      protected char __cfwr_compute927(int __cfwr_p0, char __cfwr_p1, byte __cfwr_p2) {
        if (true || false) {
            while (true) {
            if ((false ^ 27.59) && ((-79.20 % true) + (null ^ '7'))) {
            while (false) {
            if ((40.65 >> ('m' & 'M')) || false) {
            if ((-271L >> null) || false) {
            while (true) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 2; __cfwr_i86++) {
            return true;
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        Character __cfwr_elem61 = null;
        return (0.60 - false);
        return null;
        return 'D';
    }
    Float __cfwr_func547() {
        if ((595 + 211L) || ((49.72f & 31.75f) / (-29 * false))) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 2; __cfwr_i92++) {
            double __cfwr_entry9 = (null - ('M' & -69.64));
        }
        }
        try {
            return null;
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        if ((92.50f | (54.19f ^ -847L)) || ((true ^ -67.18) % null)) {
            while ((3.52f >> null)) {
            if (true && false) {
            if (true || false) {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 1; __cfwr_i76++) {
            long __cfwr_val18 = 355L;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
