/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        short __cfwr_node21 = null;
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              of
        if (false && false) {
            try {
            while (true) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 5; __cfwr_i40++) {
            while (false) {
            while ((null % (-238 - -662L))) {
            return 25.01;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
fset = {"c", "d"})
          int x) {    protected boolean __cfwr_helper37(char __cfwr_p0) {
        char __cfwr_val15 = 'M';
        if (false && false) {
            if (false || true) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 9; __cfwr_i13++) {
            Boolean __cfwr_var12 = null;
        }
        }
        }
        return false;
    }
    public Object __cfwr_aux422(Character __cfwr_p0, int __cfwr_p1, String __cfwr_p2) {
        int __cfwr_node6 = (null * (null & false));
        return null;
    }
    protected static Long __cfwr_temp504(byte __cfwr_p0, double __cfwr_p1) {
        while (true) {
            Long __cfwr_obj57 = null;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i53 = 0; __cfwr_i53 < 6; __cfwr_i53++) {
            while ((null / (702L % -937))) {
            char __cfwr_result23 = (58.38 - null);
            break; // Prevent infinite loops
        }
        }
        while (true) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 10; __cfwr_i86++) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 8; __cfwr_i98++) {
            try {
            while ((-364L - -65.07)) {
            while (true) {
            while (((null % -106L) * (false | 413L))) {
            try {
            return false;
        } catch (Exception __cfwr_e12) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i83 = 0; __cfwr_i83 < 6; __cfwr_i83++) {
            Boolean __cfwr_obj98 = null;
        }
        return null;
    }
}
