/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void subtype(int @SameLen("#2") [] a, int[] b) {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 1; __cfwr_i36++) {
            while ((13.70 << 61.67f)) {
            try {
            while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e98) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }

    int @SameLen({"a", "b"}) [] c = a;

    // :: error: (assignment)
    int @SameLen("c") [] q = {1, 2};
    int @SameLen("c") [] d = q;

    // :: error: (assignment)
    int @SameLen("f") [] e = a;
      protected long __cfwr_util911(Long __cfwr_p0) {
        return null;
        while (true) {
            Integer __cfwr_val66 = null;
            break; // Prevent infinite loops
        }
        try {
            while (true) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 3; __cfwr_i87++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        return 991L;
    }
}
