/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        try {
            while (true) {
            try {
            try {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 1; __cfwr_i60++) {
            return null;
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTLengthOf("arr") int c = b;

    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int d = b;
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    int b = 1;
    if (test == b) {
      @LTEqLengthOf("arr") int c = b;

      @LTLengthOf("arr") int g = b;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int e = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int d = b;
      private Double __cfwr_temp938(long __cfwr_p0) {
        boolean __cfwr_val34 = false;
        while (((42.18 | 15.49f) + (79.68f | 35L))) {
            try {
            while (false) {
            while ((95.51 | (-26.88f + '4'))) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 5; __cfwr_i4++) {
            if (false && (-80.27 & (-741L | null))) {
            if (false || (null + 'P')) {
            if (true || false) {
            try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 5; __cfwr_i98++) {
            return (-936L - false);
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
        try {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 1; __cfwr_i64++) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        return null;
    }
}
