/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        try {
            while ((
        while (true) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 2; __cfwr_i67++) {
            try {
            if ((84.22f + -98L) && true) {
            short __cfwr_elem68 = null;
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
('1' % 23.26) / (-23.64 ^ 'Q'))) {
            for (int __cfwr_i61 = 0; __cfwr_i61 < 3; __cfwr_i61++) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 10; __cfwr_i23++) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      float __cfwr_util112() {
        Integer __cfwr_node3 = null;
        try {
            return (-157 | -68.26f);
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        return 42.32f;
    }
    protected static char __cfwr_aux224(Character __cfwr_p0, float __cfwr_p1, Double __cfwr_p2) {
        if (false && (53.10 ^ ('H' >> 72.31f))) {
            for (int __cfwr_i65 = 0; __cfwr_i65 < 7; __cfwr_i65++) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 3; __cfwr_i90++) {
            char __cfwr_entry4 = 'l';
        }
        }
        }
        try {
            byte __cfwr_result88 = null;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        return null;
        return 'c';
    }
    String __cfwr_proc248(int __cfwr_p0, long __cfwr_p1, boolean __cfwr_p2) {
        return null;
        return (-0.14 + 'f');
        return "value19";
    }
}
