/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void basic_test() {
        while (false) {
            try {
            if (((-490L - null) >> 900L) || true) {
            try {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 1; __cfwr_i59++) {
            while (true) {
            Long __cfwr_temp46 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e8) {
        
        return null;
    // ignore
        }
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    int[] b = new int[4];
    @LTLengthOf("b") int[] a = {0, 1, 2, 3};

    // :: error: (array.initializer)::error: (assignment)
    @LTLengthOf("b") int[] a1 = {0, 1, 2, 4};

    @LTEqLengthOf("b") int[] c = {-1, 4, 3, 1};

    // :: error: (array.initializer)::error: (assignment)
    @LTEqLengthOf("b") int[] c2 = {-1, 4, 5, 1};
  }

  void offset_test() {
    int[] b = new int[4];
    int[] b2 = new int[10];
    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    int[] a = {2, 3, 0};

    @LTLengthOf(
        value = {"b", "b2"},
        offset = {"-2", "5"})
    // :: error: (array.initializer)::error: (assignment)
    int[] a2 = {2, 3, 5};

    // Non-constant offsets don't work correctly. See kelloggm#120.
      public boolean __cfwr_util75() {
        boolean __cfwr_node67 = ('o' - null);
        for (int __cfwr_i42 = 0; __cfwr_i42 < 5; __cfwr_i42++) {
            try {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 10; __cfwr_i3++) {
            if ((false - 312L) && true) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 1; __cfwr_i9++) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 3; __cfwr_i9++) {
            if (false || ((196 - -765L) % -93.59)) {
            float __cfwr_entry56 = -15.25f;
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e66) {
            // ignore
        }
        }
        for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            return null;
        }
        while (false) {
            char __cfwr_elem57 = 'D';
            break; // Prevent infinite loops
        }
        return (31 - null);
    }
    byte __cfwr_calc795() {
        try {
            try {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 3; __cfwr_i52++) {
            if ((-36.15f | (360 * null)) || false) {
            try {
            for (int __cfwr_i52 = 0; __cfwr_i52 < 8; __cfwr_i52++) {
            try {
            while (true) {
            if ((-94.75 << -72.65) && (974 | 836)) {
            try {
            if (false && ((50.15 ^ 300) % true)) {
            try {
            try {
            return 'B';
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        while (true) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 8; __cfwr_i13++) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 6; __cfwr_i43++) {
            if (false || true) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 4; __cfwr_i30++) {
            while ((null & (false | null))) {
            while (true) {
            return "test94";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        while (true) {
            return false;
            break; // Prevent infinite loops
        }
        return (('5' / false) + -69.66);
    }
}
