/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test1(@Positive int x, @Positive int y) {
        while ((null | 57.15)) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 9; __cfwr_i1++) {
            return null;
        }
            break; // Prevent inf
        for (int __cfwr_i32 = 0; __cfwr_i32 < 9; __cfwr_i32++) {
            try {
            double __cfwr_item51 = ((-7.79 | 87.03) << '0');
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
inite loops
        }

    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexFor("newArray") int j = y;
  }

  void test2(@NonNegative int x, @Positive int y) {
    int[] newArray = new int[x + y];
    @IndexFor("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test3(@NonNegative int x, @NonNegative int y) {
    int[] newArray = new int[x + y];
    @IndexOrHigh("newArray") int i = x;
    @IndexOrHigh("newArray") int j = y;
  }

  void test4(@GTENegativeOne int x, @NonNegative int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    @LTEqLengthOf("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test5(@GTENegativeOne int x, @GTENegativeOne int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexOrHigh("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
  }

  void test6(int x, int y) {
    // :: error: (array.length.negative)
    int[] newArray = new int[x + y];
    // :: error: (assignment)
    @IndexFor("newArray") int i = x;
    // :: error: (assignment)
    @IndexOrHigh("newArray") int j = y;
      public boolean __cfwr_process3(boolean __cfwr_p0, byte __cfwr_p1) {
        if ((null * (null + null)) && false) {
            try {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 8; __cfwr_i67++) {
            String __cfwr_node60 = "value97";
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        try {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 3; __cfwr_i20++) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 5; __cfwr_i13++) {
            byte __cfwr_data30 = null;
        }
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        return (-13.67f ^ 40.20f);
        while (true) {
            while ((-83L & -812)) {
            return "hello83";
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return false;
    }
    public Float __cfwr_handle31(String __cfwr_p0) {
        try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        return 'h';
        if (true || false) {
            return null;
        }
        return null;
    }
    public Float __cfwr_handle763() {
        if (('k' | 'J') && ((15L / null) % 30.78)) {
            try {
            while (false) {
            return false;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        }
        while ((-212L + ('G' | 32L))) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 3; __cfwr_i3++) {
            return ('c' << (null * 678));
        }
            break; // Prevent infinite loops
        }
        while (true) {
            if ((-475L / -3.97f) && false) {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 9; __cfwr_i1++) {
            short __cfwr_var82 = null;
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
}
