/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class StringIndexOf_slice {
  public static String nocheck(String l, String s) {
        if (false && true) {
            try {
            if (false && ('M' | -528)) {
            if ((null ^ 33.23f) && false) {
            while (('N' | null)) {
            try {
         
        Float __cfwr_var13 = null;
   return "result80";
        } catch (Exception __cfwr_e3) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e27) {
            // ignore
        }
        }

    int i = l.indexOf(s);
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + s.length());
  }

  public static String remove(String l, String s, int from, boolean last) {
    int i = last ? l.lastIndexOf(s, from) : l.indexOf(s, from);
    if (i >= 0) {
      return l.substring(0, i) + l.substring(i + s.length());
    }
    return l;
  }

  public static String stringLiteral(String l) {
    int i = l.indexOf("constant");
    if (i != -1) {
      return l.substring(0, i) + l.substring(i + "constant".length());
    }
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + "constant".length());
  }

  public static char character(String l, char c) {
    int i = l.indexOf(c);
    if (i > -1) {
      return l.charAt(i);
    }
    // :: error: (argument)
    return l.charAt(i);
  }

    private static Long __cfwr_proc506(float __cfwr_p0, float __cfwr_p1, double __cfwr_p2) {
        Character __cfwr_entry1 = null;
        while (((false >> null) | (-554 << 56))) {
            while (((-27.13f * -597) >> (-85.55f ^ -790L))) {
            if (((-89.16f % 512L) ^ null) || ((907L * 'w') % ('v' & 'B'))) {
            try {
            if (false || false) {
            return -415L;
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        double __cfwr_item16 = -10.32;
        while ((717 - (null - 165L))) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 5; __cfwr_i38++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 1; __cfwr_i47++) {
            while (false) {
            if (false || true) {
            if (false && false) {
            boolean __cfwr_elem18 = ((false | 550L) / ('5' & true));
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    Long __cfwr_helper312(Integer __cfwr_p0, Boolean __cfwr_p1) {
        if (false || true) {
            try {
            float __cfwr_elem46 = -40.30f;
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        }
        while ((28.46 & (-17.95f << null))) {
            while (true) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 6; __cfwr_i95++) {
            try {
            while (false) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 2; __cfwr_i2++) {
            while ((null | true)) {
            while (false) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 9; __cfwr_i69++) {
            try {
            char __cfwr_node95 = 'E';
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if ((849 * null) || ((null & -31.92) - null)) {
            boolean __cfwr_result34 = true;
        }
        return null;
    }
    protected static int __cfwr_calc889(Integer __cfwr_p0, short __cfwr_p1, Boolean __cfwr_p2) {
        try {
            while (true) {
            return 81.13;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        try {
            try {
            Character __cfwr_item45 = null;
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        if (false && ((null << true) % (true >> 381))) {
            try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 1; __cfwr_i62++) {
            if (true || false) {
            return (null << null);
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        }
        return (null & null);
        return -442;
    }
}