/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        for (int __cfwr_i85 
        short __cfwr_entry45 = (-28.33f + null);
= 0; __cfwr_i85 < 4; __cfwr_i85++) {
            for (int __cfwr_i45 = 0; __cfwr_i45 < 5; __cfwr_i45++) {
            while (true) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 8; __cfwr_i17++) {
            for (int __cfwr_i12 = 0; __cfwr_i12 < 1; __cfwr_i12++) {
            short __cfwr_temp49 = null;
        }
        }
            break; // Prevent infinite loops
        }
        }
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      private static boolean __cfwr_util228() {
        for (int __cfwr_i83 = 0; __cfwr_i83 < 9; __cfwr_i83++) {
            try {
            if (true || (-24.26 & (73.41f / true))) {
            if (('l' / 55L) || ((20.06 >> true) * false)) {
            if ((true - (null % 108)) && (-880 | (-696L << true))) {
            while (('4' % ('N' << 618L))) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        }
        if (true || (null & 441)) {
            if (false && false) {
            while (false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        return true;
    }
    public float __cfwr_handle393(Long __cfwr_p0, Double __cfwr_p1) {
        return null;
        for (int __cfwr_i87 = 0; __cfwr_i87 < 10; __cfwr_i87++) {
            try {
            while (true) {
            byte __cfwr_result32 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        }
        return (72L << -30.34f);
    }
}
