/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        while (false) {
    
        if (true || false) {
            if (true || false) {
            while (false) {
            if (false && true) {
            while (true) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 3; __cfwr_i21++) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 6; __cfwr_i21++) {
            return null;
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        return null;
            break; // Prevent infinite loops
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      private static Long __cfwr_proc888(Double __cfwr_p0) {
        Boolean __cfwr_data41 = null;
        while (true) {
            return "data67";
            break; // Prevent infinite loops
        }
        return null;
    }
    public int __cfwr_compute152(Integer __cfwr_p0) {
        if (((-30.03 - null) << (null << 882)) && (96.91 + 5.49)) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        return null;
        return (true | -18.72f);
    }
}
