/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        char __cfwr_data5 =
        if (false || true) {
            for (int __cfwr_i42 = 0; __cfwr_i42 < 8; __cfwr_i42++) {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 3; __cfwr_i93++) {
            return 1.92;
        }
        }
        }
 (-77.39 >> null);

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      protected static Float __cfwr_temp789(Integer __cfwr_p0, Float __cfwr_p1, Float __cfwr_p2) {
        try {
            Float __cfwr_entry89 = null;
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        Integer __cfwr_val26 = null;
        return null;
    }
    private double __cfwr_proc943(Double __cfwr_p0) {
        Double __cfwr_var66 = null;
        for (int __cfwr_i45 = 0; __cfwr_i45 < 5; __cfwr_i45++) {
            for (int __cfwr_i32 = 0; __cfwr_i32 < 2; __cfwr_i32++) {
            try {
            byte __cfwr_elem28 = (true & (null * 'U'));
        } catch (Exception __cfwr_e53) {
            // ignore
        }
        }
        }
        return (null & null);
    }
    protected static Double __cfwr_process461() {
        try {
            while (false) {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 6; __cfwr_i62++) {
            while (false) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        while (false) {
            String __cfwr_data77 = "test69";
            break; // Prevent infinite loops
        }
        Long __cfwr_var47 = null;
        return null;
    }
}
