/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        for (int __cfwr_i60 = 0; __cfwr_i60 < 4; __cfwr_i60++) {
            byte __cfwr_val23 = (-50.83f + null);
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      protected Character __cfwr_proc847(Object __cfwr_p0) {
        for (int __cfwr_i76 = 0; __cfwr_i76 < 6; __cfwr_i76++) {
            return ((-12.95 / 131L) * 'i');
        }
        long __cfwr_val91 = -312L;
        return null;
    }
    private static char __cfwr_func870(int __cfwr_p0, Object __cfwr_p1, Object __cfwr_p2) {
        for (int __cfwr_i66 = 0; __cfwr_i66 < 3; __cfwr_i66++) {
            if ((258 / 4.43) || false) {
            while (((-723 | 'z') / (968L & true))) {
            for (int __cfwr_i28 = 0; __cfwr_i28 < 10; __cfwr_i28++) {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 6; __cfwr_i77++) {
            while (true) {
            while (true) {
            while ((null | 79.71f)) {
            float __cfwr_temp84 = ((-437L << 93L) ^ null);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        return ((-657 >> -342) * false);
    }
}
