/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        double __cfwr_result72 = -9.17;

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      private short __cfwr_helper705(double __cfwr_p0, short __cfwr_p1) {
        try {
            return null;
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        try {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 7; __cfwr_i17++) {
            while (((-480 ^ -79.44f) / 'J')) {
            return (null - false);
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        for (int __cfwr_i12 = 0; __cfwr_i12 < 1; __cfwr_i12++) {
            return -575L;
        }
        return null;
    }
    static char __cfwr_proc936(double __cfwr_p0) {
        try {
            while (false) {
            try {
            int __cfwr_var36 = 425;
        } catch (Exception __cfwr_e53) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e11) {
            // ignore
        }
        return 'i';
    }
}
