/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        return null;

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    i
        char __cfwr_temp30 = (true * null);
f (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected Character __cfwr_helper395(Long __cfwr_p0, long __cfwr_p1) {
        for (int __cfwr_i65 = 0; __cfwr_i65 < 9; __cfwr_i65++) {
            Character __cfwr_temp66 = null;
        }
        for (int __cfwr_i68 = 0; __cfwr_i68 < 1; __cfwr_i68++) {
            try {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 8; __cfwr_i20++) {
            if ((572L | (-56.99 % null)) || true) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            try {
            while (false) {
            if (true || (628L * true)) {
            Integer __cfwr_temp22 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
        return null;
    }
    static Boolean __cfwr_util196(Boolean __cfwr_p0) {
        while (true) {
            try {
            return -521L;
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i18 = 0; __cfwr_i18 < 2; __cfwr_i18++) {
            Integer __cfwr_result10 = null;
        }
        Boolean __cfwr_elem28 = null;
        return null;
    }
    private static Double __cfwr_proc725(byte __cfwr_p0) {
        try {
            byte __cfwr_result40 = null;
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        if (true || true) {
            while (false) {
            try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 7; __cfwr_i38++) {
            return -30.06f;
        }
        } catch (Exception __cfwr_e59) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return 94.44;
        return null;
    }
}
