/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static String nocheck(String l, String s) {
        if ((null + true) && false) {
            if (true || (84.10 - 237L)) {
            while (false) {
            try {
            while (false) {
            if ((false ^ 10.45f) || true) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 7; __cfwr_i13++) {
            if (((-87.98 / true) << false) && true) {
            while (false) {
            if ((null & 292) || true) {
            return true;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }

    int i = l.indexOf(s);
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + s.length());
  }

  public static String remove(String l, String s, int from, boolean last) {
    int i = last ? l.lastIndexOf(s, from) : l.indexOf(s, from);
    if (i >= 0) {
      return l.substring(0, i) + l.substring(i + s.length());
    }
    return l;
  }

  public static String stringLiteral(String l) {
    int i = l.indexOf("constant");
    if (i != -1) {
      return l.substring(0, i) + l.substring(i + "constant".length());
    }
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + "constant".length());
  }

  public static char character(String l, char c) {
    int i = l.indexOf(c);
    if (i > -1) {
      return l.charAt(i);
    }
    // :: error: (argument)
    return l.charAt(i);
      protected static long __cfwr_proc949() {
        try {
            if (((false | false) & -76.99f) || true) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 8; __cfwr_i85++) {
            if (true || (73.31 ^ 56.55)) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        return 984L;
    }
    char __cfwr_aux723(Object __cfwr_p0, char __cfwr_p1, Long __cfwr_p2) {
        if (true && false) {
            return null;
        }
        return '0';
    }
    protected short __cfwr_handle107(float __cfwr_p0, Double __cfwr_p1) {
        long __cfwr_node81 = -882L;
        try {
            if (true || (586L << (29.88 | 50.46f))) {
            while (true) {
            if (true || false) {
            try {
            while (((-21.59f / null) >> null)) {
            if ((null - (-64.24f / null)) || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        if (false && ('q' % -181L)) {
            int __cfwr_entry16 = (null & -176L);
        }
        return ((null + -133) + false);
    }
}
