/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class CharSequenceTest_slice {
  void testAppend(Appendable app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

  void testAppend(StringWriter app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

    protected Float __cfwr_func678() {
        while (false) {
            try {
            try {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 7; __cfwr_i6++) {
            for (int __cfwr_i77 = 0; __cfwr_i77 < 4; __cfwr_i77++) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 6; __cfwr_i8++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 10; __cfwr_i73++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 8; __cfwr_i3++) {
            return null;
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
            break; // Prevent infinite loops
        }

        while (((true << null) * 166)) {
            while (((-57.21 - 'T') | (true | -842L))) {
            try {
            try {
            if (false && ('3' & (36.93 / -38.17))) {
            while ((-615L >> 59.44f)) {
            short __cfwr_obj69 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e13) {
            // ignore
        }
        } catch (Exception __cfwr_e69) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i91 = 0; __cfwr_i91 < 2; __cfwr_i91++) {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 8; __cfwr_i62++) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 3; __cfwr_i92++) {
            return 145;
        }
        }
        }
        return null;
    }
}