/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        while (false) {
            while (true) {
            return (-27L >> null);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
      protected static float __cfwr_func321(Boolean __cfwr_p0, Character __cfwr_p1, Long __cfwr_p2) {
        return "world1";
        if (true || ((null >> 88.86) / 'i')) {
            while ((null - -38.52)) {
            char __cfwr_node85 = 'J';
            break; // Prevent infinite loops
        }
        }
        return -17.51f;
    }
}
