/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    static float __cfwr_aux941(short __cfwr_p0) {
        try {
            try {
            while (true) {
            try {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 5; __cfwr_i54++) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 3; __cfwr_i75++) {
            try {
            return null;
        } catch (Exception __cfwr_e15) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e43) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e71) {
            // ignore
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }

        if ((null >> (-82 / false)) && false) {
            if (false && true) {
            byte __cfwr_temp68 = null;
        }
        }
        Object __cfwr_data35 = null;
        return null;
        return null;
        return 68.94f;
    }
    private boolean __cfwr_compute538(long __cfwr_p0) {
        Long __cfwr_var16 = null;
        return (717L & 396L);
    }
