/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        try {
            if (true && true) {
            try {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 5; __cfwr_i41++) {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 6; __cfwr_i88++) {
            Long __cfwr_item2 = null;
        }
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e48) {
            // ignore
        }

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      protected Character __cfwr_util670(Float __cfwr_p0, char __cfwr_p1) {
        return (null % -16L);
        try {
            while (false) {
            return 46.59f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        for (int __cfwr_i21 = 0; __cfwr_i21 < 10; __cfwr_i21++) {
            boolean __cfwr_temp60 = true;
        }
        for (int __cfwr_i21 = 0; __cfwr_i21 < 1; __cfwr_i21++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 7; __cfwr_i47++) {
            return null;
        }
        }
        return null;
    }
}
