/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        for (int _
        return 'v';
_cfwr_i23 = 0; __cfwr_i23 < 4; __cfwr_i23++) {
            for (int __cfwr_i88 = 0; __cfwr_i88 < 3; __cfwr_i88++) {
            if (true && false) {
            int __cfwr_obj54 = -484;
        }
        }
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      static double __cfwr_compute423() {
        while (true) {
            return "data13";
            break; // Prevent infinite loops
        }
        for (int __cfwr_i59 = 0; __cfwr_i59 < 9; __cfwr_i59++) {
            Long __cfwr_item54 = null;
        }
        while (true) {
            short __cfwr_node79 = (false ^ false);
            break; // Prevent infinite loops
        }
        Object __cfwr_elem81 = null;
        return -82.21;
    }
    protected int __cfwr_temp255(int __cfwr_p0) {
        if (('j' >> (null ^ null)) && ('0' * (354 & -576L))) {
            try {
            while (true) {
            try {
            try {
            return 976L;
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        try {
            Float __cfwr_var69 = null;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        try {
            if (((null & -28.50f) - 19.51) || false) {
            if (true || false) {
            if ((168L >> 513L) && ((1.03f << 385L) * 74.92)) {
            if (true || (-42.44f + false)) {
            try {
            if (true && false) {
            byte __cfwr_result65 = null;
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        return 68;
    }
}
