/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static String nocheck(String l, String s) {
        while (false) {
            if ((82.97f << 944L) || ((-24.08 % null) ^ (-68.92 * null))) {
            return null;
        }
            break; // Prevent infinite loops
        }

    int i = l.indexOf(s);
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + s.length());
  }

  public static String remove(String l, String s, int from, boolean last) {
    int i = last ? l.lastIndexOf(s, from) : l.indexOf(s, from);
    if (i >= 0) {
      return l.substring(0, i) + l.substring(i + s.length());
    }
    return l;
  }

  public static String stringLiteral(String l) {
    int i = l.indexOf("constant");
    if (i != -1) {
      return l.substring(0, i) + l.substring(i + "constant".length());
    }
    // :: error: (argument)
    return l.substring(0, i) + l.substring(i + "constant".length());
  }

  public static char character(String l, char c) {
    int i = l.indexOf(c);
    if (i > -1) {
      return l.charAt(i);
    }
    // :: error: (argument)
    return l.charAt(i);
      private static long __cfwr_handle226(long __cfwr_p0, Double __cfwr_p1, Integer __cfwr_p2) {
        while (true) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 3; __cfwr_i20++) {
            try {
            if (((null ^ 800) - 'N') || true) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 5; __cfwr_i46++) {
            char __cfwr_elem97 = 'b';
        }
        }
        }
        } catch (Exception __cfwr_e58) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        try {
            try {
            int __cfwr_data80 = -668;
        } catch (Exception __cfwr_e82) {
            // ignore
        }
        } catch (Exception __cfwr_e36) {
            // ignore
        }
        return -859L;
    }
    private static Character __cfwr_util16(Long __cfwr_p0, Long __cfwr_p1, Float __cfwr_p2) {
        if ((null << (70.23 << 460L)) || true) {
            return (60.40f + -419);
        }
        return null;
        return null;
    }
}
