/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testAppend(Appendable app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
  }

  void testAppend(StringWriter app, CharSequence cs, @IndexFor("#2") int i) throws IOException {
    app.append(cs, i, i);
    // :: error: (argument)
    app.append(cs, 1, 2);
      static short __cfwr_util138(double __cfwr_p0, char __cfwr_p1) {
        return 'D';

        while (false) {
            Character __cfwr_obj18 = null;
            break; // Prevent infinite loops
        }
        try {
            try {
            Integer __cfwr_val35 = null;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        } catch (Exception __cfwr_e37) {
            // ignore
        }
        float __cfwr_data22 = 36.11f;
        for (int __cfwr_i37 = 0; __cfwr_i37 < 4; __cfwr_i37++) {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 6; __cfwr_i30++) {
            if (true || ((37.43f & 'q') >> (-820 | -14.46))) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 8; __cfwr_i72++) {
            while (false) {
            if (false || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        return null;
    }
    public char __cfwr_aux813(Object __cfwr_p0) {
        return null;
        while (((-807L + -86.90f) >> -866L)) {
            while ((null / null)) {
            Integer __cfwr_temp13 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i23 = 0; __cfwr_i23 < 5; __cfwr_i23++) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 10; __cfwr_i56++) {
            if ((true & (null >> null)) || false) {
            if ((67 | (-89.81f * true)) || false) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 9; __cfwr_i29++) {
            return null;
        }
        }
        }
        }
        }
        for (int __cfwr_i31 = 0; __cfwr_i31 < 8; __cfwr_i31++) {
            return null;
        }
        return (true % null);
    }
    private static char __cfwr_handle271(short __cfwr_p0, Boolean __cfwr_p1) {
        if (false || (null - -69.95)) {
            while (true) {
            while (false) {
            try {
            while (false) {
            if (false && true) {
            try {
            try {
            if (false && (413L >> ('f' + -16.45))) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 9; __cfwr_i99++) {
            if (false || (null / 94L)) {
            if (false && ((929L - null) | true)) {
            while (((-78.72f & null) >> null)) {
            return (-852 ^ 26.99f);
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e83) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        if (false && false) {
            if (true && false) {
            return ((-313 / 'Z') << -542);
        }
        }
        if (false || (-512L & -10.43f)) {
            try {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 6; __cfwr_i53++) {
            Long __cfwr_temp75 = null;
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
        return 'u';
    }
}
