/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        if (true || true) {
            for (int __cfwr_i71 = 0; __cfwr_i71 < 5; __cfwr_i71++) {
            Long __cfwr_obj66 = null;
        }
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
        return null;

    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      Object __cfwr_compute275(Object __cfwr_p0) {
        while ((-448L << -270)) {
            if (true && true) {
            if (true && false) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 10; __cfwr_i90++) {
            while (false) {
            try {
            int __cfwr_node4 = 159;
        } catch (Exception __cfwr_e7) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        try {
            long __cfwr_var68 = 688L;
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        for (int __cfwr_i93 = 0; __cfwr_i93 < 5; __cfwr_i93++) {
            byte __cfwr_entry59 = null;
        }
        return null;
    }
    protected static String __cfwr_aux916(double __cfwr_p0, char __cfwr_p1, Boolean __cfwr_p2) {
        while (true) {
            try {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 2; __cfwr_i20++) {
            return "hello2";
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return "temp86";
    }
    public static long __cfwr_compute653() {
        return null;
        for (int __cfwr_i10 = 0; __cfwr_i10 < 7; __cfwr_i10++) {
            if (false || true) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 10; __cfwr_i35++) {
            try {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 7; __cfwr_i8++) {
            Double __cfwr_temp58 = null;
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        }
        }
        }
        try {
            if (true && (null * null)) {
            if (false && true) {
            return 63.93;
        }
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        return ((true | 59.47) - null);
    }
}
