/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        for (int __cfwr_i74 = 0; __cfwr_i74 < 3; __cfwr_i74++) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(
        if (((-56.04f + null) & -962) && true) {
            try {
            String __cfwr_temp8 = "result55";
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        }
2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      public short __cfwr_process903(short __cfwr_p0, String __cfwr_p1, String __cfwr_p2) {
        for (int __cfwr_i23 = 0; __cfwr_i23 < 9; __cfwr_i23++) {
            return null;
        }
        double __cfwr_var41 = 4.69;
        return 652;
        char __cfwr_result1 = 'G';
        return (36.69f + 'f');
    }
}
