/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        if (('U' & -910) && true) {
            return 'k';
        }

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      static int __cfwr_func632(boolean __cfwr_p0, Double __cfwr_p1) {
        while (false) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 9; __cfwr_i64++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        short __cfwr_data74 = (null % 68.55);
        return 'm';
        try {
            while (false) {
            try {
            Long __cfwr_obj35 = null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        return (null | null);
    }
}
