/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        while ((false + 'C')) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 1; __cfwr_i69++) {
            try {
            while (false) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 4; __cfwr_i29++) {
         
        while (true) {
            while (false) {
            float __cfwr_var81 = 21.65f;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
   try {
            try {
            Float __cfwr_item32 = null;
        } catch (Exception __cfwr_e48) {
            // ignore
        }
        } catch (Exception __cfwr_e34) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      static Long __cfwr_util73(double __cfwr_p0, Float __cfwr_p1, long __cfwr_p2) {
        try {
            try {
            try {
            while ((29.52f & (true >> 'a'))) {
            int __cfwr_entry84 = -384;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        try {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 1; __cfwr_i23++) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 7; __cfwr_i49++) {
            return "result56";
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        return null;
    }
}
