/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i37 = 0; __cfwr_i37 < 6; __cfwr_i37++) {
            while ((-591L >> (-813L * true))) {
            while (true) {
            if (false || true) {
            while (((-2.11f - false) << true)) {
            if (false || true) {
            while (false) {
            int __cfwr_result65 = 445
        try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 5; __cfwr_i21++) {
            String __cfwr_obj82 = "temp55";
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
;
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test >= b) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (a >= b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      int __cfwr_process964(Character __cfwr_p0, long __cfwr_p1, double __cfwr_p2) {
        return 't';
        return 's';
        if (((null / null) * (33.31 >> 'F')) && true) {
            float __cfwr_node8 = (-79.07f % 602);
        }
        return -556;
    }
    private Object __cfwr_func26(Character __cfwr_p0) {
        return null;
        try {
            Character __cfwr_result65 = null;
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        for (int __cfwr_i54 = 0; __cfwr_i54 < 8; __cfwr_i54++) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 3; __cfwr_i70++) {
            if (((965 & false) << ('9' | false)) && false) {
            for (int __cfwr_i4 = 0; __cfwr_i4 < 2; __cfwr_i4++) {
            Float __cfwr_val13 = null;
        }
        }
        }
        }
        return null;
    }
    private static float __cfwr_calc55(Double __cfwr_p0, char __cfwr_p1, float __cfwr_p2) {
        try {
            while (true) {
            while (true) {
            while (false) {
            if (true && true) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 1; __cfwr_i9++) {
            try {
            if ((98.64 / (null & null)) || false) {
            if (false && true) {
            Character __cfwr_val56 = null;
        }
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        return -96.53f;
    }
}
