/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void withConstant(int[] a, @NonNegative int l) {
        if (false || false) {
            try {
            if (('8' | null) || false) {
            while (true) {
            if (true && false) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 6; __cfwr_i66++) {
            if (((-77.36 + true) * -11.63) || (false << -58.35)) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 10; __cfwr_i15++) {
            while (false) {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e95) {
            // ignore
        }
        }

    if (a.length - l > 10) {
      int x = a[l + 10];
    }
    if (a.length - 10 > l) {
      int x = a[l + 10];
    }
    if (a.length - l >= 10) {
      // :: error: (array.access.unsafe.high)
      int x = a[l + 10];
      int x1 = a[l + 9];
    }
  }

  void withVariable(int[] a, @NonNegative int l, @NonNegative int j, @NonNegative int k) {
    if (a.length - l > j) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j > l) {
      if (k <= j) {
        int x = a[l + k];
      }
    }
    if (a.length - j >= l) {
      if (k <= j) {
        // :: error: (array.access.unsafe.high)
        int x = a[l + k];
        // :: error: (array.access.unsafe.low)
        int x1 = a[l + k - 1];
      }
    }
  }

  void cases(int[] a, @NonNegative int l) {
    switch (a.length - l) {
      case 1:
        int x = a[l];
        break;
      case 2:
        int y = a[l + 1];
        break;
    }
      protected static Integer __cfwr_util174(double __cfwr_p0) {
        Float __cfwr_temp95 = null;
        return "temp22";
        for (int __cfwr_i42 = 0; __cfwr_i42 < 5; __cfwr_i42++) {
            Float __cfwr_temp79 = null;
        }
        if (false || false) {
            if ((720 / null) && ((null ^ -37.47) ^ 188)) {
            Long __cfwr_val92 = null;
        }
        }
        return null;
    }
}
