/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void assignB(int[] d) {
        if (false || false) {
            try {
            if (((-296L | 962) % true) && (null << (null * null))) {
            if (true || false) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 1; __cfwr_i40++) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 3; __cfwr_i94++) {
            while (((558L + true) * null)) {
            if (false && false) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 10; __cfwr_i69++) {
            for (int __cfwr_i8 = 0; __cfwr_i8 < 2; __cfwr_i8++) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e26) {
            // ignore
        }
        }

    // :: error: (from.gt.to) :: error: (from.not.nonnegative) :: error: (to.not.ltel)
    b = d;
  }

  void assignC(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    c = d;
  }

  void assignE(int[] d) {
    // :: error: (from.gt.to) :: error: (to.not.ltel)
    e = d;
      private double __cfwr_process124(boolean __cfwr_p0) {
        while (false) {
            return null;
            break; // Prevent infinite loops
        }
        while (true) {
            try {
            return null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        Object __cfwr_data74 = null;
        return 17.18;
    }
    protected String __cfwr_proc163() {
        try {
            try {
            Object __cfwr_entry58 = null;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        Object __cfwr_var53 = null;
        return null;
        boolean __cfwr_var91 = ('5' & null);
        return "temp47";
    }
    private static float __cfwr_func993() {
        return (83L - null);
        return null;
        return (-650 & false);
    }
}
