/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        return null;

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("thi
        try {
            while (false) {
            try {
            if ((-708 * (true + -92.01f)) && true) {
            try {
            while ((null << -12.37f)) {
            while (false) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 10; __cfwr_i70++) {
            if ((597 + -762) && (340 * false)) {
            while (true) {
            Double __cfwr_entry60 = null;
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
s.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      protected static Integer __cfwr_temp791(Double __cfwr_p0, Character __cfwr_p1, char __cfwr_p2) {
        Double __cfwr_temp36 = null;
        return null;
    }
    protected byte __cfwr_proc895(int __cfwr_p0, String __cfwr_p1) {
        short __cfwr_item76 = null;
        return (33.94f * (null ^ null));
    }
    protected static short __cfwr_func778() {
        for (int __cfwr_i26 = 0; __cfwr_i26 < 6; __cfwr_i26++) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 6; __cfwr_i84++) {
            try {
            while (true) {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 10; __cfwr_i38++) {
            while (false) {
            try {
            return ((null + null) & -75.22f);
        } catch (Exception __cfwr_e43) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        }
        }
        Float __cfwr_val42 = null;
        return "value51";
        return null;
    }
}
