/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        if (true || false) {
 
        for (int __cfwr_i55 = 0; __cfwr_i55 < 5; __cfwr_i55++) {
            String __cfwr_entry37 = "world85";
        }
           for (int __cfwr_i79 = 0; __cfwr_i79 < 8; __cfwr_i79++) {
            float __cfwr_entry51 = -73.36f;
        }
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      protected byte __cfwr_compute718(Object __cfwr_p0) {
        if ((78.29 & (-63.04 * true)) && false) {
            while (false) {
            for (int __cfwr_i37 = 0; __cfwr_i37 < 7; __cfwr_i37++) {
            return false;
        }
            break; // Prevent infinite loops
        }
        }
        return 637L;
        return null;
        return (-541L >> -96.45);
    }
    protected float __cfwr_helper350(Float __cfwr_p0, Boolean __cfwr_p1) {
        double __cfwr_data27 = ((-34.53f ^ '0') / 35.37);
        return (78.75f * 77.21);
    }
    protected static int __cfwr_helper293(char __cfwr_p0, float __cfwr_p1, Object __cfwr_p2) {
        try {
            if ((-481 - null) && ((null % 234L) % null)) {
            while (false) {
            while (((79.05 - 649) | false)) {
            return -94.17;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        return (-31.87f << -22.88);
    }
}
