/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class Polymorphic3_slice {
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        if (true || (-561L - 77.88)) {
            double __cfwr_node92 = -96.72;
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
  }

    protected double __cfwr_func392(double __cfwr_p0, Object __cfwr_p1) {
        if (false && false) {
            try {
            return null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        return 97.23;
    }
    public static Integer __cfwr_process58() {
        if (true || true) {
            if ((null - (23.62 - 62.20)) && (-31.66f + -98.52f)) {
            return null;
        }
        }
        return null;
    }
    boolean __cfwr_aux442() {
        return false;
        try {
            try {
            return null;
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        return true;
    }
}