/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        while (false) {
            for (int __cfwr_i74 = 0; __cfwr_i74 < 2; __cfwr_i74++) {
            Long __cfwr_var94 = null;
        }
            break; // Prevent infinite loops
        }

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLength
        return null;
Of("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      protected static Long __cfwr_helper571() {
        try {
            for (int __cfwr_i78 = 0; __cfwr_i78 < 1; __cfwr_i78++) {
            for (int __cfwr_i31 = 0; __cfwr_i31 < 2; __cfwr_i31++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 8; __cfwr_i3++) {
            Character __cfwr_result10 = null;
        }
        }
        }
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        for (int __cfwr_i62 = 0; __cfwr_i62 < 8; __cfwr_i62++) {
            return null;
        }
        long __cfwr_obj21 = -751L;
        return null;
    }
    long __cfwr_temp371(Double __cfwr_p0, float __cfwr_p1) {
        try {
            for (int __cfwr_i38 = 0; __cfwr_i38 < 5; __cfwr_i38++) {
            while (false) {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 9; __cfwr_i98++) {
            try {
            if (((295L - 88.35f) % -44.19f) || true) {
            try {
            while (false) {
            try {
            String __cfwr_result68 = "value3";
        } catch (Exception __cfwr_e1) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e85) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        while (false) {
            while ((68.60 >> true)) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        while (false) {
            while (false) {
            if (true || ('2' * -749L)) {
            while ((null << false)) {
            while (false) {
            double __cfwr_entry16 = 78.83;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if (false && true) {
            Boolean __cfwr_var75 = null;
        }
        return -969L;
    }
}
