/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class RefineGT_slice {
  void testLTL(@LTLengthOf("arr") int test) {
        while (false) {
            while (('0' ^ 471)) {
            for (int __cfwr_i48 = 0; __cfwr_i48 < 3; __cfwr_i48++) {
            if (false || ((true << -18.92) / (null >> -134L))) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 8; __cfwr_i39++) {
            byte __cfwr_item72 = null;
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

    protected Character __cfwr_aux165(Boolean __cfwr_p0, String __cfwr_p1, String __cfwr_p2) {
        while (((false - 897) % -918L)) {
            try {
            try {
            while (((-4.81 * -50.16) / -833L)) {
            while (false) {
            if (false || (null << null)) {
            while (true) {
            try {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 4; __cfwr_i86++) {
            if (true && true) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 1; __cfwr_i36++) {
            try {
            try {
            while (false) {
            float __cfwr_item22 = -74.63f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        if ((629L % null) && true) {
            boolean __cfwr_elem45 = (82.50 ^ (86.22 / -2.04));
        }
        return null;
    }
}