/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        if (true || ('N' >> (null % null))) {
            Float __cfwr_obj64 = null;
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      private short __cfwr_func625(Boolean __cfwr_p0) {
        for (int __cfwr_i61 = 0; __cfwr_i61 < 4; __cfwr_i61++) {
            int __cfwr_val79 = 516;
        }
        byte __cfwr_result79 = (792 * 'B');
        return ((74.81f >> -134L) | null);
    }
    protected static Double __cfwr_proc89(byte __cfwr_p0) {
        while (false) {
            if (false && true) {
            short __cfwr_data73 = (69.10 | (2.72f ^ 892L));
        }
            break; // Prevent infinite loops
        }
        while (false) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 3; __cfwr_i64++) {
            return (true - 98.28f);
        }
            break; // Prevent infinite loops
        }
        Double __cfwr_var1 = null;
        for (int __cfwr_i64 = 0; __cfwr_i64 < 1; __cfwr_i64++) {
            try {
            try {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 4; __cfwr_i14++) {
            return null;
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
        return null;
    }
}
