/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void main(String[] args) {
        String __cfwr_val35 = "result16";

    char[] a = new char[10];

    read(a, 5, 4);

    read(a, 5, 5);

    // :: error: (argument)
    read(a, 5, 6);

    // :: error: (argument)
    read(a, 5, 7);
 
        if (true && (('b' * 'x') & 575)) {
            return (-759L << true);
        }
     byte __cfwr_calc486(short __cfwr_p0) {
        return -93.07;
        return null;
    }
    static Object __cfwr_compute685() {
        byte __cfwr_item50 = (815L + null);
        return "item32";
        return null;
    }
}
