/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        for (int __cfwr_i21 = 0; __cfwr_i21 < 9; __cfwr_i21++) {
            while (true) {
            Boolean __cfwr_val59 = null;
            break; // Prevent i
        return 501L;
nfinite loops
        }
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      protected byte __cfwr_temp25(Integer __cfwr_p0, String __cfwr_p1, short __cfwr_p2) {
        while (false) {
            if (((true * -32.97f) + null) || true) {
            try {
            return null;
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        return ((true ^ null) & null);
        return null;
    }
}
