/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        while (true) {
            for (int __cfwr_i2 = 0; __cfwr_i2 < 9; __cfwr_i2++) {
            try {
            return "test57";
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      protected Integer __cfwr_func444(Float __cfwr_p0, long __cfwr_p1, Double __cfwr_p2) {
        return -49.64;
        if (true && true) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 6; __cfwr_i75++) {
            try {
            return null;
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
        }
        while ((true >> -747)) {
            try {
            while ((null & null)) {
            while ((false >> (-90.12f % true))) {
            for (int __cfwr_i90 = 0; __cfwr_i90 < 7; __cfwr_i90++) {
            return 'Y';
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e19) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        float __cfwr_node67 = -17.15f;
        return null;
    }
    protected static float __cfwr_calc82() {
        while (false) {
            byte __cfwr_node46 = ('l' / -542L);
            break; // Prevent infinite loops
        }
        return 84.03f;
    }
}

  void test4(@NonNegative int x1, int[] a) {
