/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void m() {
        for (int __cfwr_i82 = 0; __cfwr_i82 < 3; __cfwr_i82++) {
            if (true && true) {
            try {
            if (true && true) {
            if (false || (('f' % 70.04) >> -92.36f)) {
            return null;
        }
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
        }

    // :: error: (argument)
    get(1);

    get(1, "a", "b");

    // :: error: (argument)
    get(2, "abc");

    String[] stringArg1 = new String[] {"a", "b"
        byte __cfwr_node73 = ('j' >> (617 % null));
};
    String[] stringArg2 = new String[] {"c", "d", "e"};
    String[] stringArg3 = new String[] {"a", "b", "c"};

    method(1, stringArg1, stringArg2);

    // :: error: (argument)
    method(2, stringArg3);

    get(1, stringArg1);

    // :: error: (argument)
    get(3, stringArg2);
      protected static boolean __cfwr_process776() {
        for (int __cfwr_i11 = 0; __cfwr_i11 < 1; __cfwr_i11++) {
            Object __cfwr_data41 = null;
        }
        Boolean __cfwr_var6 = null;
        Boolean __cfwr_elem84 = null;
        return false;
    }
}
