/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        if (false && true) {
            if (fal
        Float __cfwr_item48 = null;
se || false) {
            try {
            try {
            while (false) {
            float __cfwr_obj55 = 64.46f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e14) {
            // ignore
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      public short __cfwr_compute581(byte __cfwr_p0, Character __cfwr_p1) {
        try {
            if ((true / 16.79) || false) {
            double __cfwr_elem53 = (null ^ (true * 93.05f));
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        return (('t' << 'p') - 'X');
    }
    public Boolean __cfwr_calc221(Character __cfwr_p0, float __cfwr_p1) {
        try {
            while (true) {
            return (null * -253);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        if (false && false) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 6; __cfwr_i22++) {
            return 849L;
        }
        }
        if (true && true) {
            while (false) {
            float __cfwr_node41 = (67 << ('T' - null));
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i34 = 0; __cfwr_i34 < 9; __cfwr_i34++) {
            int __cfwr_entry27 = 886;
        }
        return null;
    }
    Boolean __cfwr_process386(double __cfwr_p0, Long __cfwr_p1) {
        return 83.70;
        return null;
    }
}
