/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        for (int __cfwr_i30 = 0; __cfwr_i30 < 10; __cfwr_i30++) {
            while (false) {
            if (true || false) {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 5; __cfwr_i54++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 4; __cfwr_i47++) {
            try {
            if (true && false) {
            while (false) {
            for (int __cfwr_i95 = 0; __cfwr_i95 < 6; __cfwr_i95++) {
            return "value1";
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    private String __cfwr_proc432(double __cfwr_p0) {
        try {
            return 74.35;
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        for (int __cfwr_i87 = 0; __cfwr_i87 < 2; __cfwr_i87++) {
            while (((false % 'K') - null)) {
            if (false || (-444 >> 962L)) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 6; __cfwr_i21++) {
            if ((null | (null % 5.81f)) || (92.74f << 20.49)) {
            return null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        short __cfwr_obj29 = ((-56.12 / false) | true);
        return null;
        return "test98";
    }
}
