/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        while ((-91.92 >> null)) {
            if ((3.58f & -206) || true) {
            Integer __cfwr_data91 = null;
        }
            break; // Prevent infinite loops
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      static int __cfwr_func253(Float __cfwr_p0) {
        for (int __cfwr_i18 = 0; __cfwr_i18 < 6; __cfwr_i18++) {
            while (false) {
            for (int __cfwr_i63 = 0; __cfwr_i63 < 9; __cfwr_i63++) {
            char __cfwr_entry11 = '2';
        }
            break; // Prevent infinite loops
        }
        }
        if (true || true) {
            return 69.92;
        }
        return 185;
        return 208;
    }
    public static String __cfwr_process916(boolean __cfwr_p0, Long __cfwr_p1, Double __cfwr_p2) {
        try {
            return null;
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        try {
            Double __cfwr_temp65 = null;
        } catch (Exception __cfwr_e31) {
            // ignore
        }
        return "test97";
    }
}
