/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        if (((775L >> null) % (null % 94.90)) || true) {
            return -345;
        }

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      private static float __cfwr_proc165(Object __cfwr_p0, Character __cfwr_p1, Float __cfwr_p2) {
        return null;
        char __cfwr_result73 = (-32.03 * -429);
        return (-224L % 'G');
    }
    static Integer __cfwr_compute379(int __cfwr_p0, String __cfwr_p1) {
        long __cfwr_result44 = (-40.93 % ('s' * 'o'));
        return "item23";
        try {
            if (false || ('w' * true)) {
            if (('Y' & -576) && true) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 10; __cfwr_i94++) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 3; __cfwr_i94++) {
            Float __cfwr_item42 = null;
        }
        }
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        return null;
    }
    public char __cfwr_proc891() {
        return -26.60;
        try {
            while (false) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 10; __cfwr_i56++) {
            Float __cfwr_var26 = null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
        return (null << 32.26);
    }
}
