/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        for (int __cfwr_i78 = 0; __cfwr_i78 < 2; __cfwr_i78++) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 6; __cfwr_i40++) {
            if (false || true) {
            Long __cfwr_var79 = null;
        }
        }
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b",
        while (true) {
            while (false) {
            Long __cfwr_elem30 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
 "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2 = identity(abl);
      static Object __cfwr_proc468(Boolean __cfwr_p0) {
        return (-845L >> null);
        while (true) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 9; __cfwr_i67++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 9; __cfwr_i86++) {
            for (int __cfwr_i85 = 0; __cfwr_i85 < 4; __cfwr_i85++) {
            byte __cfwr_temp90 = null;
        }
        }
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    private Object __cfwr_compute281() {
        if (true || ('F' ^ null)) {
            Double __cfwr_data65 = null;
        }
        return null;
        for (int __cfwr_i16 = 0; __cfwr_i16 < 2; __cfwr_i16++) {
            try {
            Integer __cfwr_result43 = null;
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        }
        return ((false << -822) & null);
        return null;
    }
}
