/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testNewInstance(int i) {
        char __cfwr_result39 = (665 / false);

    // :: error: (argument)
    Array.
        if (false && ((58.52 << 240L) << 607)) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 9; __cfwr_i55++) {
            return -61;
        }
        }
newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
      Float __cfwr_aux30(String __cfwr_p0, byte __cfwr_p1) {
        Integer __cfwr_result87 = null;
        while (false) {
            while ((false + null)) {
            Object __cfwr_elem53 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if (true || (-816 << -318)) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 9; __cfwr_i99++) {
            long __cfwr_node27 = -594L;
        }
        }
        if (false && ('J' * '8')) {
            long __cfwr_val97 = -791L;
        }
        return null;
    }
}
