/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        try {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 8; __cfwr_i18++) {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 5; __cfwr_i13++) {
            try {
            Long __cfwr_result15 = null;
        } catch (Exception __cfwr_e40) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      static String __cfwr_process810() {
        if (true || false) {
            Float __cfwr_result43 = null;
        }
        double __cfwr_val20 = -2.45;
        return "world35";
    }
    public static Boolean __cfwr_util763(Float __cfwr_p0, long __cfwr_p1) {
        return null;
        char __cfwr_var60 = 'X';
        while (true) {
            if ((81.00 << (429L * null)) && false) {
            Long __cfwr_data98 = null;
        }
            break; // Prevent infinite loops
        }
        try {
            while (false) {
            while (true) {
            for (int __cfwr_i43 = 0; __cfwr_i43 < 1; __cfwr_i43++) {
            if (true && (null + true)) {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 1; __cfwr_i96++) {
            if (true && (null + null)) {
            try {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 6; __cfwr_i60++) {
            for (int __cfwr_i44 = 0; __cfwr_i44 < 10; __cfwr_i44++) {
            for (int __cfwr_i29 = 0; __cfwr_i29 < 5; __cfwr_i29++) {
            if (true || true) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 2; __cfwr_i64++) {
            try {
            for (int __cfwr_i97 = 0; __cfwr_i97 < 3; __cfwr_i97++) {
            try {
            if ((true % (false / false)) && ((-872L % -95.55) | 257)) {
            try {
            Long __cfwr_val39 = null;
        } catch (Exception __cfwr_e44) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        return null;
    }
    private static Double __cfwr_calc460() {
        try {
            short __cfwr_var8 = ('g' + -474);
        } catch (Exception __cfwr_e19) {
            // ignore
        }
        float __cfwr_elem96 = (null % (true >> 'v'));
        if (false || true) {
            for (int __cfwr_i15 = 0; __cfwr_i15 < 5; __cfwr_i15++) {
            while (true) {
            return false;
            break; // Prevent infinite loops
        }
        }
        }
        return null;
    }
}
