/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testNewInstance(int i) {
        int __cfwr_elem5 = (null >> (null % null));

    // :: error: (argument)
    
        while ((null / 32.08)) {
            Long __cfwr_val78 = null;
            break; // Prevent infinite loops
        }
Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
      public char __cfwr_proc45(Long __cfwr_p0, short __cfwr_p1) {
        if (false && true) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 4; __cfwr_i70++) {
            while (false) {
            if (false || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        for (int __cfwr_i41 = 0; __cfwr_i41 < 9; __cfwr_i41++) {
            try {
            long __cfwr_temp53 = 273L;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
        for (int __cfwr_i11 = 0; __cfwr_i11 < 4; __cfwr_i11++) {
            if (true || true) {
            Long __cfwr_item5 = null;
        }
        }
        return 'P';
    }
}
