/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ArrayAssignmentSameLen_slice {
// Source-based slice around line 19
// Method: ArrayAssignmentSameLen#test1(int[],int[],int)

    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
        char __cfwr_result39 = (665 / false);

    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  void tes
        if (false && ((58.52 << 240L) << 607)) {
            for (int __cfwr_i55 = 0; __cfwr_i55 < 9; __cfwr_i55++) {
            return -61;
        }
        }
t2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {

    Float __cfwr_aux30(String __cfwr_p0, byte __cfwr_p1) {
        Integer __cfwr_result87 = null;
        while (false) {
            while ((false + null)) {
            Object __cfwr_elem53 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        if (true || (-816 << -318)) {
            for (int __cfwr_i99 = 0; __cfwr_i99 < 9; __cfwr_i99++) {
            long __cfwr_node27 = -594L;
        }
        }
        if (false && ('J' * '8')) {
            long __cfwr_val97 = -791L;
        }
        return null;
    }
}