/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test_ubc_and(
      @IndexFor("#2") int i, int[] a, @LTLengthOf("#2") int j, int k, @NonNegative int m) {
        byte __cfwr_entry8 = (false ^ null);

    int x = a[i & k];
    int x1 = a[k & i];
    // :: error: (array.access.unsafe.low) :: error: (array.access.unsafe
        while (true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 6; __cfwr_i10++) {
            if ((true ^ 3.62f) && ((-490L << 642) << 29)) {
            if (false && false) {
            int __cfwr_entry60 = 442;
        }
        }
        }
            break; // Prevent infinite loops
        }
.high)
    int y = a[j & k];
    if (j > -1) {
      int z = a[j & k];
    }
    // :: error: (array.access.unsafe.high)
    int w = a[m & k];
    if (m < a.length) {
      int u = a[m & k];
    }
      int __cfwr_func363() {
        return 887L;
        return ((41.94 >> '1') >> (null & false));
    }
    private static String __cfwr_aux428() {
        Long __cfwr_val90 = null;
        return -812L;
        try {
            float __cfwr_result79 = -57.90f;
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        try {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 1; __cfwr_i79++) {
            if (((-319L - null) - null) && ('g' ^ -10.84f)) {
            return false;
        }
        }
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        return "data72";
    }
}
