/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @SuppressWarnings("lowerbound")
  public void subtraction4(String[] a, @IndexFor("#1") int i) {
        while ((61.83f | nu
        for (int __cfwr_i67 = 0; __cfwr_i67 < 10; __cfwr_i67++) {
            try {
            long __cfwr_item58 = 424L;
        } catch (Exception __cfwr_e22) {
            // ignore
        }
        }
ll)) {
            Long __cfwr_result89 = null;
            break; // Prevent infinite loops
        }

    if (1 - i < a.length) {
      // The error on this assignment is a false positive.
      // :: error: (assignment)
      @IndexFor("a") int j = 1 - i;

      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "1") int k = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction5(String[] a, int i) {
    if (1 - i < a.length) {
      // :: error: (assignment)
      @IndexFor("a") int j = i;
    }
  }

  @SuppressWarnings("lowerbound")
  public void subtraction6(String[] a, int i, int j) {
    if (i - j < a.length - 1) {
      @IndexFor("a") int k = i - j;
      // :: error: (assignment)
      @IndexFor("a") int k1 = i;
    }
  }

  public void multiplication1(String[] a, int i, @Positive int j) {
    if ((i * j) < (a.length + j)) {
      // :: error: (assignment)
      @IndexFor("a") int k = i;
      // :: error: (assignment)
      @IndexFor("a") int k1 = j;
    }
  }

  public void multiplication2(String @ArrayLen(5) [] a, @IntVal(-2) int i, @IntVal(20) int j) {
    if ((i * j) < (a.length - 20)) {
      @LTLengthOf("a") int k1 = i;
      // :: error: (assignment)
      @LTLengthOf(value = "a", offset = "20") int k2 = i;
      // :: error: (assignment)
      @LTLengthOf("a") int k3 = j;
    }
      public static Object __cfwr_temp43(Object __cfwr_p0) {
        while (false) {
            Character __cfwr_item29 = null;
            break; // Prevent infinite loops
        }
        while (true) {
            try {
            while (false) {
            try {
            char __cfwr_temp38 = 'x';
        } catch (Exception __cfwr_e96) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e29) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return null;
    }
    public long __cfwr_proc705(char __cfwr_p0, byte __cfwr_p1, Float __cfwr_p2) {
        int __cfwr_val77 = 25;
        for (int __cfwr_i81 = 0; __cfwr_i81 < 6; __cfwr_i81++) {
            while (false) {
            if ((20.68f % 'c') && true) {
            for (int __cfwr_i87 = 0; __cfwr_i87 < 7; __cfwr_i87++) {
            Object __cfwr_elem83 = null;
        }
        }
            break; // Prevent infinite loops
        }
        }
        return ((null + null) ^ false);
        return -475L;
    }
}
