/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @EnsuresLTLengthOf(value = "v1", targetValue = "value1", offset = "3")
  @EnsuresLTLengthOf(value = "v2", targetValue = "value2", offset = "2")
  @EnsuresLTLengthOf(value = "v3", targetValue = "value3", offset = "1")
  // :: error: (contracts.postcondition)
  public void withpostconditionsfunc1() {
        for (int __cfwr_i52 = 0; __cfwr_i52 < 10; __cfwr_i52++) {
            try {
            long __cfwr_elem14 = (null * true);
        } catch (Exception __cfwr_e97) {
            // ignore
        }
        }

    v1 = value1.length() - 3; // condition not satisfied here
    v2 = value2.length() - 3;
    v3 = value3.length() - 3;
  }

  @EnsuresLTLengthOfIf(expression = "v1", targetValue = "value1", offset = "3", result = true)
  @EnsuresLTLengthOfIf(expression = "v2", targetValue = "value2", offset = "2", result = true)
  @EnsuresLTLengthOfIf(expression = "v3", targetValue = "value3", offset = "1", result = true)
  public boolean withcondpostconditionsfunc2() {
    v1 = value1.length() - 3; // condition not satisfied here
    v2 = value2.length() - 3;
    v3 = value3.length() - 3;
    // :: error: (contracts.conditional.postcondition)
    return true;
  }

  @EnsuresLTLengthOf.List({
    @EnsuresLTLengthOf(value = "v1", targetValue = "value1", offset = "3"),
    @EnsuresLTLengthOf(value = "v2", targetValue = "value2", offset = "2")
  })
  @EnsuresLTLengthOf(value = "v3", targetValue = "value3", offset = "1")
  // :: error: (contracts.postcondition)
  public void withpostconditionfunc1() {
    v1 = value1.length() - 3; // condition not satisfied here
    v2 = value2.length() - 3;
    v3 = value3.length() - 3;
  }

  @EnsuresLTLengthOfIf.List({
    @EnsuresLTLengthOfIf(expression = "v1", targetValue = "value1", offset = "3", result = true),
    @EnsuresLTLengthOfIf(expression = "v2", targetValue = "value2", offset = "2", result = true)
  })
  @EnsuresLTLengthOfIf(expression = "v3", targetValue = "value3", offset = "1", result = true)
  public boolean withcondpostconditionfunc2() {
    v1 = value1.length() - 3; // condition not satisfied here
    v2 = value2.length() - 3;
    v3 = value3.length() - 3;
    // :: error: (contracts.conditional.postcondition)
    return true;
      public static Double __cfwr_handle964(double __cfwr_p0) {
        return false;
        for (int __cfwr_i40 = 0; __cfwr_i40 < 2; __cfwr_i40++) {
            while ((28.03f + ('U' << false))) {
            String __cfwr_node4 = "data42";
            break; // Prevent infinite loops
        }
        }
        try {
            try {
            for (int __cfwr_i34 = 0; __cfwr_i34 < 6; __cfwr_i34++) {
            for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            if (true || true) {
            while (((575L << 8.53f) + -959)) {
            return null;
            break; // Prevent infinite loops
        }
        }
        }
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }
        } catch (Exception __cfwr_e74) {
            // ignore
        }
        return null;
    }
    private static char __cfwr_func910(char __cfwr_p0, Float __cfwr_p1) {
        return "value62";
        while (false) {
            return -40.67;
            break; // Prevent infinite loops
        }
        for (int __cfwr_i68 = 0; __cfwr_i68 < 6; __cfwr_i68++) {
            while (true) {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 1; __cfwr_i59++) {
            try {
            while (true) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 8; __cfwr_i10++) {
            try {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 5; __cfwr_i23++) {
            for (int __cfwr_i6 = 0; __cfwr_i6 < 10; __cfwr_i6++) {
            if ((('j' * -62) + (null * null)) || (106 << (true >> null))) {
            try {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 7; __cfwr_i26++) {
            try {
            try {
            if ((null << 96.90f) || (null | 'n')) {
            for (int __cfwr_i23 = 0; __cfwr_i23 < 9; __cfwr_i23++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e62) {
            // ignore
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        }
        }
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e98) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        return 'w';
    }
    protected static long __cfwr_handle363(Object __cfwr_p0) {
        Object __cfwr_elem8 = null;
        return null;
        if (false && false) {
            for (int __cfwr_i69 = 0; __cfwr_i69 < 10; __cfwr_i69++) {
            if (true && true) {
            while (true) {
            try {
            while (true) {
            while (true) {
            return (49.47f - -668);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        for (int __cfwr_i51 = 0; __cfwr_i51 < 1; __cfwr_i51++) {
            for (int __cfwr_i49 = 0; __cfwr_i49 < 10; __cfwr_i49++) {
            try {
            while ((null * (null % 'g'))) {
            try {
            try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 8; __cfwr_i98++) {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 4; __cfwr_i56++) {
            return null;
        }
        }
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        }
        }
        return -626L;
    }
}
