/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void indexForOrHigh(String str, @IndexFor("#1") int i1, @IndexOrHigh("#1") int i2) {
        while ((null ^ (-33.01f << null))) {
            try {
            return -29.51;
        } catch (Exception __cfwr_e20) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    str.substring(Math.max(i1, i2));
    str.substring(Math.min(i1, i2));
    // :: error: (argument)
    str.charAt(Math.max(i1, i2));
    str.charAt(Math.min(i1, i2));
  }

  // max does not work with different sequences, min does
  void twoSequences(String str1, String str2, @IndexFor("#1") int i1, @IndexFor("#2") int i2) {
    // :: error: (argument)
    str1.charAt(Math.max(i1, i2));
    str1.charAt(Math.min(i1, i2));
      private static Object __cfwr_compute470(float __cfwr_p0) {
        for (int __cfwr_i4 = 0; __cfwr_i4 < 1; __cfwr_i4++) {
            while ((null & false)) {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    private Integer __cfwr_aux488(short __cfwr_p0) {
        try {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 1; __cfwr_i47++) {
            try {
            while (((null & 'a') >> (null + null))) {
            try {
            return -30L;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e89) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        try {
            for (int __cfwr_i62 = 0; __cfwr_i62 < 10; __cfwr_i62++) {
            boolean __cfwr_elem87 = false;
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        while ((-621L + (null << false))) {
            return (null + null);
            break; // Prevent infinite loops
        }
        return null;
    }
}
