/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        float __cfwr_item86 = 58.52f;

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      protected Integer __cfwr_calc335(double __cfwr_p0, Boolean __cfwr_p1, String __cfwr_p2) {
        if ((false & 994) || false) {
            while (false) {
            char __cfwr_val15 = 'P';
            break; // Prevent infinite loops
        }
        }
        while (true) {
            int __cfwr_data29 = 769;
            break; // Prevent infinite loops
        }
        return -93.35;
        return null;
    }
    private Character __cfwr_helper585() {
        for (int __cfwr_i56 = 0; __cfwr_i56 < 6; __cfwr_i56++) {
            for (int __cfwr_i67 = 0; __cfwr_i67 < 5; __cfwr_i67++) {
            long __cfwr_val6 = -924L;
        }
        }
        Object __cfwr_var33 = null;
        for (int __cfwr_i80 = 0; __cfwr_i80 < 8; __cfwr_i80++) {
            while (true) {
            for (int __cfwr_i94 = 0; __cfwr_i94 < 9; __cfwr_i94++) {
            while (false) {
            while ((-73.09 << true)) {
            byte __cfwr_node6 = null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
