/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void str(String argStr) {
        char __cfwr_obj69 =
        for (int __cfwr_i69 = 0; __cfwr_i69 < 2; __cfwr_i69++) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        }
 (748L >> '0');

    if (argStr.isEmpty()) {
      return;
    }
    if (argStr == "abc") {
      return;
    }
    // :: error: (argument)
    char c = "abc".charAt(argStr.length() - 1);
    // :: error: (argument)
    char c2 = "abc".charAt(argStr.length());
      private Long __cfwr_calc23() {
        return "world65";
        try {
            return "item81";
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        if (false && true) {
            try {
            return ('z' * 'q');
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        }
        while (true) {
            double __cfwr_var89 = (null & (null & -68.72f));
            break; // Prevent infinite loops
        }
        return null;
    }
    protected byte __cfwr_aux222() {
        long __cfwr_item8 = (-845L * null);
        while (true) {
            long __cfwr_item37 = (('x' ^ true) + 'Y');
            break; // Prevent infinite loops
        }
        for (int __cfwr_i93 = 0; __cfwr_i93 < 5; __cfwr_i93++) {
            return (953L - (null >> null));
        }
        return ('k' % 499);
    }
    private static int __cfwr_util893() {
        while ((('Y' & 9.37) | null)) {
            while (('Q' + false)) {
            try {
            return null;
        } catch (Exception __cfwr_e62) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i70 = 0; __cfwr_i70 < 7; __cfwr_i70++) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 7; __cfwr_i66++) {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 3; __cfwr_i21++) {
            return "result2";
        }
        }
        }
        return 980;
    }
}
