/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        while ((null | (null / 669L))) {
       
        if (true || ((null >> null) >> -56.15)) {
            while (false) {
            try {
            for (int __cfwr_i98 = 0; __cfwr_i98 < 7; __cfwr_i98++) {
            Long __cfwr_data73 = null;
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
     for (int __cfwr_i58 = 0; __cfwr_i58 < 4; __cfwr_i58++) {
            long __cfwr_data31 = 85L;
        }
            break; // Prevent infinite loops
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      protected static double __cfwr_func930(double __cfwr_p0, long __cfwr_p1) {
        try {
            return null;
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        if (false || (true | -167L)) {
            try {
            char __cfwr_result23 = (-184 << (null ^ -969));
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        return -37.56;
    }
}
