/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void badParam(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a", "b"},
              offset = {"c"})
          int x) {
        for (int __cfwr_i61 = 0; __cfwr_i61 < 9; __cfwr_i61++) {
            float __cfwr_val85 = 58.62f;
        }
}

  void badParam2(
      // :: error: (different.length.sequences.offsets)
      @LTLengthOf(
              value = {"a"},
              offset = {"c", "d"})
          int x) {    Object __cfwr_temp736(Boolean __cfwr_p0, long __cfwr_p1, Boolean __cfwr_p2) {
        if (true && false) {
            for (int __cfwr_i9 = 0; __cfwr_i9 < 4; __cfwr_i9++) {
            Boolean __cfwr_result33 = null;
        }
        }
        return null;
        Long __cfwr_result36 = null;
        return null;
    }
    public String __cfwr_helper180(Long __cfwr_p0, Integer __cfwr_p1) {
        boolean __cfwr_data57 = (null + (null & true));
        return "item88";
    }
    public String __cfwr_compute885(Character __cfwr_p0, Integer __cfwr_p1, short __cfwr_p2) {
        return null;
        return "temp25";
    }
}
