/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (true) {
            try {
            while (true) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e31) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so tha
        while (false) {
            double __cfwr_var82 = ((null | null) | (null / null));
            break; // Prevent infinite loops
        }
t the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private static short __cfwr_func716(double __cfwr_p0, Integer __cfwr_p1) {
        try {
            return -529L;
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        for (int __cfwr_i91 = 0; __cfwr_i91 < 4; __cfwr_i91++) {
            return null;
        }
        return (null | 2.37);
    }
    private static boolean __cfwr_calc533() {
        try {
            for (int __cfwr_i59 = 0; __cfwr_i59 < 5; __cfwr_i59++) {
            while (false) {
            return (939 - (-585 / null));
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        try {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 2; __cfwr_i18++) {
            try {
            try {
            Object __cfwr_item86 = null;
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e53) {
            // ignore
        }
        return null;
        return true;
    }
}
