/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class LTLDivide_slice {
  void test2(int[] array) {
        if (false && true) {
            while (false) {
            while (true) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 9; __cfwr_i57++) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 5; __cfwr_i47++) {
            int __cfwr_data85 = 110;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }

    int len = array.length;
    in
        long __cfwr_node72 = -336L;
t lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
  }

    private static Object __cfwr_compute822(Integer __cfwr_p0, byte __cfwr_p1, Integer __cfwr_p2) {
        for (int __cfwr_i90 = 0; __cfwr_i90 < 1; __cfwr_i90++) {
            return -544;
        }
        for (int __cfwr_i40 = 0; __cfwr_i40 < 7; __cfwr_i40++) {
            Integer __cfwr_var73 = null;
        }
        return null;
    }
}