/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        for (int __cfwr_i27 = 0; __cfwr_i27 < 1; __cfwr_i27++) {
            if (false || true) {
            byte __cfwr_result59 = null;
        }
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
      public static Integer __cfwr_aux667(byte __cfwr_p0) {
        if (true && false) {
            return 47.62f;
        }
        Float __cfwr_item80 = null;
        Integer __cfwr_data80 = null;
        return null;
    }
    protected byte __cfwr_func38(Double __cfwr_p0) {
        Double __cfwr_var85 = null;
        Boolean __cfwr_elem68 = null;
        return ('3' + (false % 8.28));
    }
}
