/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        try {
            return null;
        } catch (Exception __cfwr_e72) {
            // ignore
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      private static Boolean __cfwr_compute675(Character __cfwr_p0) {
        try {
            return -114L;
        } catch (Exception __cfwr_e75) {
            // ignore
        }
        return null;
    }
    public static boolean __cfwr_util397(Double __cfwr_p0, Float __cfwr_p1) {
        for (int __cfwr_i51 = 0; __cfwr_i51 < 6; __cfwr_i51++) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 3; __cfwr_i58++) {
            if (false || true) {
            if (('P' % -64.44) || false) {
            try {
            while (false) {
            return null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        }
        }
        }
        }
        float __cfwr_val49 = (-67.81f + (-40.97 % -934));
        return false;
    }
}
