/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest(int x) {
        String __cfwr_obj4 = "value14";

    test(0);
    test(1);
    // :: error: (argument)
    test(2);
    // :: error: (argument)
    test(3);
    test(arrayLen2.length - 1);
  }

  int @MinLen(4) [] arrayLen4 = {0, 1, 2, 4, 5};

  void test2(@IndexOrHigh("this.arrayLen4") int i) {
    if (i > 0) {
      int j = arrayLen4[i - 1];
    }
    int j2 = arrayLen4[1];
  }

  void callTest2(int x) {
    test2(0);
    test2(1);
    test2(2);
    test2(4);
    // :: error: (argument)
    test2(5);
    test2(arrayLen4.length);
      private Integer __cfwr_calc999(long __cfwr_p0, float __cfwr_p1, Character __cfwr_p2) {
        try {
            for (int __cfwr_i1 = 0; __cfwr_i1 < 8; __cfwr_i1++) {
            while (('2' - null)) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 7; __cfwr_i20++) {
            char __cfwr_obj4 = 'E';
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        return null;
        try {
            if (true || ((null % null) * 87.28)) {
            try {
            if (true && false) {
            return null;
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        for (int __cfwr_i92 = 0; __cfwr_i92 < 10; __cfwr_i92++) {
            return null;
        }
        return null;
    }
    protected static Boolean __cfwr_helper978(int __cfwr_p0, String __cfwr_p1) {
        return ('R' ^ 90.13);
        if ((33.74f & 462L) && false) {
            if (false || false) {
            Character __cfwr_elem77 = null;
        }
        }
        return null;
    }
}
