/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IntroSub_slice {
  void test(int[] arr, @LTLengthOf({"#1"}) int a) {
        for (int __cfwr_i50 = 0; __cfwr_i50 < 5; __cfwr_i50++) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 10; __cfwr_i22++) {
            for (int __cfwr_i3 = 0; __cfwr_i3 < 5; __cfwr_i3++) {
            byte __cfwr_item68 = null;
        }
        }
        }

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int c = a - (-1);
    @LTEqLengthOf({"arr"}) int c1 = a - (-1);
    @
        Character __cfwr_result56 = null;
LTLengthOf({"arr"}) int d = a - 0;
    @LTLengthOf({"arr"}) int e = a - 7;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int f = a - (-7);

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int j = 7;
  }

    protected static double __cfwr_compute712() {
        while (true) {
            while (true) {
            try {
            return -51;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        boolean __cfwr_node53 = (null + -12);
        String __cfwr_entry96 = "world72";
        return -59.83;
    }
}