/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test, @LTLengthOf("arr") int a, @LTLengthOf("arr") int a3) {
        if (true || (null & -78.79
        try {
            if (true || ((27.44f & 483) % (null - null))) {
            short __cfwr_node10 = null;
        }
        } catch (Exception __cfwr_e45) {
            // ignore
        }
)) {
            if (true || false) {
            if (true || true) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 10; __cfwr_i92++) {
            for (int __cfwr_i73 = 0; __cfwr_i73 < 2; __cfwr_i73++) {
            try {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 1; __cfwr_i86++) {
            try {
            for (int __cfwr_i21 = 0; __cfwr_i21 < 8; __cfwr_i21++) {
            Object __cfwr_temp94 = null;
        }
        } catch (Exception __cfwr_e65) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e33) {
            // ignore
        }
        }
        }
        }
        }
        }

    int b = 2;
    if (b < test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b < a3) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b < test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTEqLengthOf("arr") int c1 = b;

    if (b < a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTEqLengthOf("arr") int d = b;
    }
      private static Long __cfwr_temp370(byte __cfwr_p0, Integer __cfwr_p1) {
        float __cfwr_val93 = 89.78f;
        try {
            while (true) {
            try {
            return -426;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e46) {
            // ignore
        }
        Character __cfwr_item32 = null;
        return null;
        return null;
    }
    protected Integer __cfwr_temp358(char __cfwr_p0) {
        return null;
        return null;
        try {
            try {
            return "value26";
        } catch (Exception __cfwr_e69) {
            // ignore
        }
        } catch (Exception __cfwr_e20) {
            // ignore
        }
        return null;
    }
}
