/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    public int __cfwr_util506(int __cfwr_p0, Integer __cfwr_p1, boolean __cfwr_p2) {
        try {
            while ((-977L >> 978)) {
            while (false) {
            float __cfwr_entry71 = -45.37f;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }

        float __cfwr_data9 = -83.44f;
        if (false || true) {
            try {
            while ((null + null)) {
            while (true) {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 9; __cfwr_i64++) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 7; __cfwr_i92++) {
            return 937L;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e54) {
            // ignore
        }
        }
        Integer __cfwr_val1 = null;
        if ((true / -110L) && false) {
            Boolean __cfwr_result1 = null;
        }
        return 171;
    }
    float __cfwr_calc794(boolean __cfwr_p0, long __cfwr_p1, float __cfwr_p2) {
        return -120L;
        for (int __cfwr_i30 = 0; __cfwr_i30 < 6; __cfwr_i30++) {
            Integer __cfwr_var79 = null;
        }
        short __cfwr_val74 = ((68.49f - true) & 'H');
        return (('a' % 41.54) * ('w' << null));
    }
    static long __cfwr_compute57() {
        for (int __cfwr_i58 = 0; __cfwr_i58 < 8; __cfwr_i58++) {
            try {
            for (int __cfwr_i76 = 0; __cfwr_i76 < 2; __cfwr_i76++) {
            try {
            try {
            return "data40";
        } catch (Exception __cfwr_e10) {
            // ignore
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e42) {
            // ignore
        }
        }
        try {
            while ((534L / -3L)) {
            boolean __cfwr_temp67 = (null >> -366);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        return null;
        return 432L;
    }
