/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        Integer __cfwr_item73 = null;

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      static Long __cfwr_func435(String __cfwr_p0, Object __cfwr_p1, long __cfwr_p2) {
        if (((57.26f * 'v') & (-12.70f | -572)) || true) {
            try {
            while (false) {
            return 60.87f;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e39) {
            // ignore
        }
        }
        while (((72.98 / null) + ('w' >> 496))) {
            while ((-55.87f & (914L | 'c'))) {
            try {
            try {
            while (false) {
            while (false) {
            while (true) {
            if (false || false) {
            try {
            if (false || false) {
            try {
            return null;
        } catch (Exception __cfwr_e4) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        } catch (Exception __cfwr_e52) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i98 = 0; __cfwr_i98 < 2; __cfwr_i98++) {
            short __cfwr_temp95 = ((null + -678) & false);
        }
        byte __cfwr_val44 = null;
        return null;
    }
}
