/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while (((-79 % -10.95) % (true % null))) {
            return "world15";
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer
        while ((15.80f << null)) {
            return false;
            break; // Prevent infinite loops
        }
.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (test > b) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (a > b) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      public boolean __cfwr_helper968() {
        return null;
        return null;
        return ((112L << 'T') + (93.47f ^ 'f'));
    }
    Object __cfwr_aux269(Boolean __cfwr_p0, long __cfwr_p1, Character __cfwr_p2) {
        try {
            try {
            if (true && true) {
            try {
            float __cfwr_val15 = -93.58f;
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e30) {
            // ignore
        }
        } catch (Exception __cfwr_e87) {
            // ignore
        }
        try {
            return -709L;
        } catch (Exception __cfwr_e2) {
            // ignore
        }
        return null;
    }
}
