/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        while ((false >> ('k' * null))) {
            if (true && (51.91f >> (-80.13 << -392L))) {
            return (null << 15.60);
        }
            break; // Prevent infinite loops
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      private Double __cfwr_proc742(Long __cfwr_p0) {
        Float __cfwr_val2 = null;
        if (false || ((-43 & false) - null)) {
            if (false || (('X' % null) << false)) {
            while (false) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        for (int __cfwr_i54 = 0; __cfwr_i54 < 8; __cfwr_i54++) {
            byte __cfwr_temp82 = (8L & null);
        }
        return null;
    }
    public static long __cfwr_func681(Float __cfwr_p0, Object __cfwr_p1, Object __cfwr_p2) {
        return null;
        return null;
        return -730L;
    }
    public static double __cfwr_aux510(float __cfwr_p0, Double __cfwr_p1, double __cfwr_p2) {
        try {
            if (false && ((-59.37f | 'e') ^ (54L >> null))) {
            while (true) {
            while (false) {
            if (false || false) {
            float __cfwr_entry46 = -68.36f;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e21) {
            // ignore
        }
        return null;
        return null;
        return (null & 'l');
    }
}
