/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void m4(int @MinLen(1) [] shorter) {
        float __cfwr_val58 = 33.31f;

    int[] longer = new int[shorter.length * 1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m5(int[] shorter) {
    // :: error: (array.length.negative)
    int[] longer = new int[shorter.length * -1];
    // :: error: (assignment)
    @LTLengthOf("longer") int x = shorter.length;
    // :: error: (assignment)
    @LTEqLengthOf("longer") int y = shorter.length;
  }

  public static void m6(int @MinLen(1) [] shorter) {
    int[] longer = new int[4 * shorter.length];
    // TODO: enable when https://github.com/kelloggm/checker-framework/issues/211 is fixed
    // // :: error: (assignment)
    // @LTLengthOf("longer") int x = shorter.length;
    @LTEqLengthOf("longer") int y = shorter.length;
      public int __cfwr_util235(float __cfwr_p0, boolean __cfwr_p1, String __cfwr_p2) {
        while (false) {
            try {
            for (int __cfwr_i50 = 0; __cfwr_i50 < 5; __cfwr_i50++) {
            if ((true + (257 ^ null)) && false) {
            char __cfwr_entry80 = (null - (6.21 / -78.59));
        }
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        for (int __cfwr_i21 = 0; __cfwr_i21 < 1; __cfwr_i21++) {
            if (false && (-170 - 'K')) {
            return null;
        }
        }
        return -369;
    }
    private Boolean __cfwr_handle694(Long __cfwr_p0, Object __cfwr_p1) {
        if (false || true) {
            for (int __cfwr_i60 = 0; __cfwr_i60 < 7; __cfwr_i60++) {
            float __cfwr_val72 = ((-44.77f | 1L) - (663 << -250));
        }
        }
        while ((442 >> (-79.67f - 459))) {
            if (true || false) {
            while (false) {
            if ((-628 / 760) || false) {
            try {
            for (int __cfwr_i64 = 0; __cfwr_i64 < 1; __cfwr_i64++) {
            if (('Z' + -59.93f) || (false ^ null)) {
            Long __cfwr_entry9 = null;
        }
        }
        } catch (Exception __cfwr_e18) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return null;
        String __cfwr_var54 = "value43";
        return null;
    }
}
