/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        while ((-93.90 / (-1.7
        try {
            if (false || false) {
            try {
            return 140;
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e84) {
            // ignore
        }
6f * -63.21))) {
            try {
            try {
            for (int __cfwr_i26 = 0; __cfwr_i26 < 8; __cfwr_i26++) {
            try {
            Double __cfwr_node64 = null;
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e41) {
            // ignore
        }
        } catch (Exception __cfwr_e61) {
            // ignore
        }
            break; // Prevent infinite loops
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public short __cfwr_handle949(float __cfwr_p0, Boolean __cfwr_p1) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 2; __cfwr_i72++) {
            try {
            while (false) {
            for (int __cfwr_i39 = 0; __cfwr_i39 < 10; __cfwr_i39++) {
            long __cfwr_data31 = 466L;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e5) {
            // ignore
        }
        }
        float __cfwr_node68 = 32.62f;
        Float __cfwr_data82 = null;
        while ((-910 * true)) {
            boolean __cfwr_val34 = (true ^ (87.82 ^ -723));
            break; // Prevent infinite loops
        }
        return null;
    }
    String __cfwr_temp619(Character __cfwr_p0) {
        try {
            Float __cfwr_val5 = null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        try {
            try {
            while (((null % -17.90f) % (null | -40.24))) {
            try {
            byte __cfwr_node98 = null;
        } catch (Exception __cfwr_e27) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e99) {
            // ignore
        }
        } catch (Exception __cfwr_e77) {
            // ignore
        }
        return "hello73";
    }
    public Integer __cfwr_func886() {
        if (true || false) {
            boolean __cfwr_result8 = false;
        }
        Character __cfwr_var19 = null;
        return null;
        Object __cfwr_obj19 = null;
        return null;
    }
}
