/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        return null;

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;

    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      private char __cfwr_compute87() {
        try {
            if (false || true) {
            Character __cfwr_data18 = null;
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        int __cfwr_item13 = ((null * -28.07f) & null);
        for (int __cfwr_i65 = 0; __cfwr_i65 < 6; __cfwr_i65++) {
            while ((('W' + -39.34) | null)) {
            try {
            return null;
        } catch (Exception __cfwr_e83) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        return (-50.26 & -768);
    }
    static char __cfwr_temp442(Integer __cfwr_p0, byte __cfwr_p1) {
        for (int __cfwr_i3 = 0; __cfwr_i3 < 3; __cfwr_i3++) {
            try {
            Boolean __cfwr_var98 = null;
        } catch (Exception __cfwr_e17) {
            // ignore
        }
        }
        try {
            Integer __cfwr_result2 = null;
        } catch (Exception __cfwr_e1) {
            // ignore
        }
        return 'w';
    }
    private float __cfwr_func24() {
        while (true) {
            if (((-19.93 * 45.40) + (null >> null)) && true) {
            while (true) {
            try {
            return 364;
        } catch (Exception __cfwr_e74) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        return 585L;
        if (false || ((null & -98.33) & -262)) {
            Long __cfwr_data60 = null;
        }
        return 13.39f;
    }
}
