/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void addToNonNegative(@NonNegative int l, Object v) {
        for (int __cfwr_i96 = 0; __cfwr_i96 < 2; __cfwr_i96++) {
            return 96.49;
        }

    // :: error: (assignment)
    Object @MinLe
        return null;
n(100) [] o = new Object[l + 1];
    o[99] = v;
  }

  void addToPositive(@Positive int l, Object v) {
    // :: error: (assignment)
    Object @MinLen(100) [] o = new Object[l + 1];
    o[99] = v;
  }

  void addToUnboundedIntRange(@IntRange(from = 0) int l, Object v) {
    // :: error: (assignment)
    Object @MinLen(100) [] o = new Object[l + 1];
    o[99] = v;
  }

  // Similar code that correctly gives warnings
  void addToPositiveOK(@NonNegative int l, Object v) {
    Object[] o = new Object[l + 1];
    // :: error: (array.access.unsafe.high.constant)
    o[99] = v;
  }

  void addToBoundedIntRangeOK(@IntRange(from = 0, to = 1) int l, Object v) {
    // :: error: (assignment)
    Object @MinLen(100) [] o = new Object[l + 1];
    o[99] = v;
  }

  void subtractFromPositiveOK(@Positive int l, Object v) {
    // :: error: (assignment)
    Object @MinLen(100) [] o = new Object[l - 1];
    o[99] = v;
      public static long __cfwr_compute275(boolean __cfwr_p0, long __cfwr_p1, short __cfwr_p2) {
        for (int __cfwr_i63 = 0; __cfwr_i63 < 7; __cfwr_i63++) {
            if (true || (-85.49f | -15.02)) {
            if ((('8' | 'h') + (44.10f + null)) || (-54.43f % null)) {
            if (true && false) {
            try {
            try {
            if (true && true) {
            byte __cfwr_obj69 = ((null - 'F') << -31.47f);
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        } catch (Exception __cfwr_e60) {
            // ignore
        }
        }
        }
        }
        }
        return 277L;
    }
    public static float __cfwr_handle829() {
        for (int __cfwr_i15 = 0; __cfwr_i15 < 3; __cfwr_i15++) {
            return 64.48;
        }
        for (int __cfwr_i93 = 0; __cfwr_i93 < 8; __cfwr_i93++) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 2; __cfwr_i22++) {
            char __cfwr_result39 = 'I';
        }
        }
        try {
            while (((null ^ 623) | 941)) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 7; __cfwr_i89++) {
            if ((197L | false) && true) {
            while (false) {
            if (false && (false + -511L)) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 5; __cfwr_i66++) {
            if (true || true) {
            if (((781L / 903L) >> 3.97f) && (-57 % false)) {
            try {
            if (true || ((true + 'R') + null)) {
            try {
            try {
            if ((true * (-865L & -66.02)) || true) {
            if ((null << false) || false) {
            for (int __cfwr_i35 = 0; __cfwr_i35 < 8; __cfwr_i35++) {
            for (int __cfwr_i7 = 0; __cfwr_i7 < 7; __cfwr_i7++) {
            while ((-3.03 << null)) {
            if (false && false) {
            try {
            if (true || false) {
            while (false) {
            while (true) {
            if (true && (false | '6')) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 8; __cfwr_i53++) {
            Boolean __cfwr_val13 = null;
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e93) {
            // ignore
        }
        } catch (Exception __cfwr_e28) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e32) {
            // ignore
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e55) {
            // ignore
        }
        if (true || true) {
            return null;
        }
        return 75.75f;
    }
    protected String __cfwr_func496(long __cfwr_p0) {
        while (false) {
            if (true && false) {
            if (true && true) {
            String __cfwr_temp83 = "item72";
        }
        }
            break; // Prevent infinite loops
        }
        return false;
        Boolean __cfwr_val8 = null;
        return "item45";
    }
}
