/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class BasicSubsequence {
  // :: error: (not.final)
  @HasSubsequence(subsequence = "this", from = "this.x", to = "this.y")
  int[] b;

  int x;
  int y;

  void test2(@NonNegative @LessThan("y + 1") int x1, int[] a) {
        try {
            try {
            return "world71";
        } catch (Exception __cfwr_e20) {
            // 
        while (true) {
            short __cfwr_var44 = null;
            break; // Prevent infinite loops
        }
ignore
        }
        } catch (Exception __cfwr_e8) {
            // ignore
        }

    x = x1;
    // :: error: (to.not.ltel)
    b = a;
  }

  void test3(@NonNegative @LessThan("y") int x1, int[] a) {
    x = x1;
    // :: error: (to.not.ltel)
    b = a;
      private long __cfwr_compute353(Object __cfwr_p0) {
        while ((null - (-43.23f | true))) {
            try {
            Long __cfwr_obj41 = null;
        } catch (Exception __cfwr_e41) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        return ((996 % -73.31f) >> -346);
    }
    static Integer __cfwr_temp161(double __cfwr_p0) {
        return null;
        while ((null * (-806L * -86.94f))) {
            boolean __cfwr_entry39 = ((-837L / -845L) ^ ('m' % null));
            break; // Prevent infinite loops
        }
        if ((12.39f / (7.54 / '9')) && true) {
            while (true) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 1; __cfwr_i40++) {
            while (true) {
            if (true && ((true << null) + -7.44f)) {
            while ((-68.50f ^ (-436 + 4.05f))) {
            while (true) {
            try {
            byte __cfwr_var95 = null;
        } catch (Exception __cfwr_e52) {
            // ignore
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        }
        long __cfwr_entry7 = (82.76f ^ -27.97f);
        return null;
    }
}

  void test4(@NonNegative int x1, int[] a) {
