/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void pre1(int[] args) {
        if (true && true) {
            for (int
        return 6;
 __cfwr_i73 = 0; __cfwr_i73 < 5; __cfwr_i73++) {
            try {
            try {
            for (int __cfwr_i93 = 0; __cfwr_i93 < 1; __cfwr_i93++) {
            return null;
        }
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        }
        }

    int ii = 0;
    while ((ii < args.length)) {
      // :: error: (array.access.unsafe.high)
      int m = args[++ii];
    }
  }

  void pre2(int[] args) {
    int ii = 0;
    while ((ii < args.length)) {
      ii++;
      // :: error: (array.access.unsafe.high)
      int m = args[ii];
    }
      public static Character __cfwr_aux476(short __cfwr_p0) {
        long __cfwr_elem50 = -149L;
        for (int __cfwr_i56 = 0; __cfwr_i56 < 8; __cfwr_i56++) {
            try {
            byte __cfwr_data49 = null;
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        for (int __cfwr_i23 = 0; __cfwr_i23 < 3; __cfwr_i23++) {
            try {
            while (false) {
            try {
            return (null % 'w');
        } catch (Exception __cfwr_e88) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e67) {
            // ignore
        }
        }
        return null;
    }
}
