/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        return ('v' ^ null);

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1
        while (true) {
            try {
            while (true) {
            if (false && true) {
            if (false || (false >> 311)) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 9; __cfwr_i20++) {
            if (false || true) {
            for (int __cfwr_i24 = 0; __cfwr_i24 < 1; __cfwr_i24++) {
            try {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 10; __cfwr_i79++) {
            for (int __cfwr_i66 = 0; __cfwr_i66 < 7; __cfwr_i66++) {
            if (((true & 47.51f) % 'u') || ((904L * 'w') | null)) {
            if (false && false) {
            if (true && true) {
            return ('v' * null);
        }
        }
        }
        }
        }
        } catch (Exception __cfwr_e49) {
            // ignore
        }
        }
        }
        }
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e56) {
            // ignore
        }
            break; // Prevent infinite loops
        }
);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      private boolean __cfwr_temp782(short __cfwr_p0, Object __cfwr_p1) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 9; __cfwr_i72++) {
            if (false && false) {
            long __cfwr_item73 = 226L;
        }
        }
        if (true || true) {
            if ((null ^ (553 / 385)) && (null % -38.90)) {
            short __cfwr_temp95 = null;
        }
        }
        return true;
    }
}
