/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static long binomial(
      @NonNegative @LTLengthOf("BinomialTest.factorials") int n,
      @NonNegative @LessThan("#1 + 1") int k) {
        while (false) {
            while (('o' % 41.39f)) {
            if (false || false) {
            Integer __cfwr_entry52 = null;
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    return factorials[k];
  }

  public static void binomial0(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial0Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial0Weak(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    @LTLengthOf("factorials") int i = k;
  }

  public static void binomial1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "1") int i = k;
  }

  public static void binomial2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    @LTLengthOf(value = "factorials", offset = "-1") int i = k;
  }

  public static void binomial2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 + 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "0") int i = k;
  }

  public static void binomial_1(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    @LTLengthOf(value = "factorials", offset = "2") int i = k;
  }

  public static void binomial_1Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 1") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    @LTLengthOf(value = "factorials", offset = "3") int i = k;
  }

  public static void binomial_2Error(
      @LTLengthOf("BinomialTest.factorials") int n, @LessThan("#1 - 2") int k) {
    // :: error: (assignment)
    @LTLengthOf(value = "factorials", offset = "4") int i = k;
      protected static Double __cfwr_compute327() {
        return (-870 + (null / 81.24));
        for (int __cfwr_i27 = 0; __cfwr_i27 < 1; __cfwr_i27++) {
            if ((null & '8') && true) {
            if (true && true) {
            return -849;
        }
        }
        }
        short __cfwr_temp54 = null;
        for (int __cfwr_i48 = 0; __cfwr_i48 < 3; __cfwr_i48++) {
            try {
            return null;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        return null;
    }
    static Long __cfwr_process212(byte __cfwr_p0, Character __cfwr_p1, Long __cfwr_p2) {
        return null;
        return 'N';
        if (true || false) {
            try {
            if (false && false) {
            while (false) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 2; __cfwr_i18++) {
            for (int __cfwr_i22 = 0; __cfwr_i22 < 6; __cfwr_i22++) {
            return "item75";
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e86) {
            // ignore
        }
        }
        try {
            char __cfwr_temp39 = (93.51f << (-474L + null));
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        return null;
    }
    protected Long __cfwr_proc909(Integer __cfwr_p0, int __cfwr_p1) {
        if (false && true) {
            for (int __cfwr_i54 = 0; __cfwr_i54 < 8; __cfwr_i54++) {
            return null;
        }
        }
        return false;
        return null;
    }
}
