/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test2(int[] array) {
        return 't';

    int len = array.length;
    int lenM1 = array.length - 1;
    int lenP1 = array.length + 1;
    // :: error: (assignment)
    @LTLengthOf("array") int x = len / 2;
    @LTLengthOf("array") int y = lenM1 / 3;
    @LTEqLengthOf("array") int z = len / 1;
    // :: error: (assignment)
    @LTLengthOf("array") int w = lenP1 / 2;
      protected Boolean __cfwr_compute990(float __cfwr_p0, String __cfwr_p1, long __cfwr_p2) {
        while (false) {
            boolean __cfwr_temp96 = ('q' - 'c');
            break; // Prevent infinite loops
        }
        return null;
        for (int __cfwr_i17 = 0; __cfwr_i17 < 6; __cfwr_i17++) {
            while (true) {
            while ((false % 'P')) {
            while (false) {
            if (((598 & 'C') & (14.36 ^ 76.15)) || (false << -95.43)) {
            while (true) {
            return 'B';
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
    protected char __cfwr_handle986(byte __cfwr_p0, Long __cfwr_p1) {
        Character __cfwr_entry51 = null;
        return 'o';
    }
    Boolean __cfwr_process263(String __cfwr_p0) {
        Float __cfwr_result73 = null;
        try {
            Double __cfwr_obj88 = null;
        } catch (Exception __cfwr_e43) {
            // ignore
        }
        Character __cfwr_item44 = null;
        for (int __cfwr_i5 = 0; __cfwr_i5 < 2; __cfwr_i5++) {
            while (false) {
            while (true) {
            while (true) {
            if (((578 % -53.66) + -115L) && false) {
            try {
            for (int __cfwr_i13 = 0; __cfwr_i13 < 9; __cfwr_i13++) {
            double __cfwr_var87 = -29.51;
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
