/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        while (true) {
            while (((-327L & -37.32) >> 23.94)) {
            while (('W' | -11.55f)) {
            for (int __cfwr_i14 = 0; __cfwr_i14 < 4; __cfwr_i14++) {
            try {
            Long __cfwr_val13 = null;
        } catch (Exception __cfwr_e61) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      protected static long __cfwr_proc153(double __cfwr_p0) {
        try {
            Character __cfwr_item2 = null;
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        return -177L;
    }
    static Long __cfwr_helper729(boolean __cfwr_p0, Boolean __cfwr_p1) {
        short __cfwr_var18 = null;
        for (int __cfwr_i34 = 0; __cfwr_i34 < 1; __cfwr_i34++) {
            try {
            Float __cfwr_data35 = null;
        } catch (Exception __cfwr_e68) {
            // ignore
        }
        }
        try {
            for (int __cfwr_i25 = 0; __cfwr_i25 < 9; __cfwr_i25++) {
            while (true) {
            Integer __cfwr_elem95 = null;
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e70) {
            // ignore
        }
        return null;
    }
}
