/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class IndexForTestLBC_slice {
  void callTest1(int x) {
        Double __cfwr_data76 = null;

    test1(0);
    test1(1);
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
  }

    public static String __cfwr_helper534() {
        for (int __cfwr_i97 = 0; __cfwr_i97 < 6; __cfwr_i97++) {
            while (true) {
            Double __cfwr_entry28 = null;
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i34 = 0; __cfwr_i34 < 10; __cfwr_i34++) {
            for (int __cfwr_i46 = 0; __cfwr_i46 < 2; __cfwr_i46++) {
            try {
            if (false && true) {
            while (false) {
            if (true && true) {
            try {
            return null;
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        }
        } catch (Exception __cfwr_e12) {
            // ignore
        }
        }
        }
        return null;
        return "hello93";
    }
}