/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  static void fn1(int[] arr, @IndexFor("#1") int i) {
        Boolean __
        try {
            Long __cfwr_node71 = null;
        } catch (Exception __cfwr_e33) {
            // ignore
        }
cfwr_entry9 = null;

    if (i >= 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i > 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
    if (i != 33) {
      // :: error: (argument)
      fn2(arr, i);
    }
  }

  static void fn2(int[] arr, @NonNegative @LTOMLengthOf("#1") int i) {
    int c = arr[i + 1];
      private static Long __cfwr_util103() {
        for (int __cfwr_i36 = 0; __cfwr_i36 < 2; __cfwr_i36++) {
            if (true || false) {
            try {
            return null;
        } catch (Exception __cfwr_e24) {
            // ignore
        }
        }
        }
        Character __cfwr_node32 = null;
        return null;
    }
    public Float __cfwr_util103(float __cfwr_p0, Boolean __cfwr_p1, short __cfwr_p2) {
        return null;
        float __cfwr_node99 = ((null << null) ^ (428 | 80.23f));
        return null;
    }
    static Object __cfwr_compute478(boolean __cfwr_p0) {
        return null;
        return false;
        Boolean __cfwr_result76 = null;
        return -913;
        return null;
    }
}
