/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void testLTL(@LTLengthOf("arr") int test) {
        for (int __cfwr_i26 = 0; __cfwr_i26 < 1; __cfwr_i26++) {
            for (int __cfwr_i36 = 0; __cfwr_i36 < 6; __cfwr_i36++) {
            int __cfwr_item1 = (-93.16f | ('u' % -74));
        }
        }

    // The reason for the parsing is so that the Value Checker
    // can't figure it out but normal humans can.

    // :: error: (assignment)
    @LTLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
  }

  void testLTEL(@LTEqLengthOf("arr") int test) {
    // :: error: (assignment)
    @LTEqLengthOf("arr") int a = Integer.parseInt("1");

    // :: error: (assignment)
    @LTEqLengthOf("arr") int a3 = Integer.parseInt("3");

    int b = 2;
    if (b <= test) {
      @LTEqLengthOf("arr") int c = b;
    }
    // :: error: (assignment)
    @LTLengthOf("arr") int c1 = b;

    if (b <= a) {
      int potato = 7;
    } else {
      // :: error: (assignment)
      @LTLengthOf("arr") int d = b;
    }
      short __cfwr_util971(byte __cfwr_p0) {
        if (true || true) {
            return null;
        }
        boolean __cfwr_temp78 = false;
        for (int __cfwr_i47 = 0; __cfwr_i47 < 1; __cfwr_i47++) {
            return true;
        }
        return (-25.78 % null);
    }
    public static float __cfwr_func300() {
        return "world67";
        return null;
        if (false && true) {
            char __cfwr_entry62 = 'R';
        }
        return -35.86f;
    }
    static float __cfwr_compute620(Double __cfwr_p0) {
        Boolean __cfwr_obj21 = null;
        while ((1L >> (9.99 >> 79.98))) {
            for (int __cfwr_i57 = 0; __cfwr_i57 < 4; __cfwr_i57++) {
            return null;
        }
            break; // Prevent infinite loops
        }
        while (('R' + -99.80)) {
            return null;
            break; // Prevent infinite loops
        }
        return 23.75f;
    }
}
