/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        while (false) {
            for (int __cfwr_i84 = 0; __cfwr_i84 < 9; __cfwr_i84++) {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 4; __cfwr_i53++) {
            char __cfwr_item38 = (null ^ false);
        }
        }
            break; // Prevent infinite loops
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public Float __cfwr_func529() {
        if (('l' | -276L) && false) {
            if (true || true) {
            return null;
        }
        }
        if (('h' / 35.16) || true) {
            while (false) {
            while (false) {
            while (((-15.36 >> -934) << null)) {
            while (true) {
            while (false) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 1; __cfwr_i89++) {
            byte __cfwr_var75 = (true & ('B' >> 'S'));
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
        for (int __cfwr_i47 = 0; __cfwr_i47 < 7; __cfwr_i47++) {
            for (int __cfwr_i58 = 0; __cfwr_i58 < 3; __cfwr_i58++) {
            try {
            if (true && true) {
            try {
            while (true) {
            if (true || true) {
            while (false) {
            while (true) {
            while ((-25 * -80.74f)) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e47) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e72) {
            // ignore
        }
        }
        }
        return null;
    }
    protected int __cfwr_calc125(byte __cfwr_p0, char __cfwr_p1) {
        for (int __cfwr_i42 = 0; __cfwr_i42 < 9; __cfwr_i42++) {
            while (false) {
            Double __cfwr_entry37 = null;
            break; // Prevent infinite loops
        }
        }
        if (false || false) {
            double __cfwr_item44 = (false & ('l' ^ null));
        }
        if (false && ((73 ^ -830) | (77.09 >> -494L))) {
            return null;
        }
        return (('3' & null) << (-8.69 ^ 75.46));
    }
    Long __cfwr_compute553(double __cfwr_p0) {
        for (int __cfwr_i47 = 0; __cfwr_i47 < 7; __cfwr_i47++) {
            for (int __cfwr_i20 = 0; __cfwr_i20 < 5; __cfwr_i20++) {
            int __cfwr_node98 = -296;
        }
        }
        for (int __cfwr_i61 = 0; __cfwr_i61 < 9; __cfwr_i61++) {
            Character __cfwr_elem39 = null;
        }
        while (true) {
            Long __cfwr_obj53 = null;
            break; // Prevent infinite loops
        }
        while (true) {
            return null;
            break; // Prevent infinite loops
        }
        return null;
    }
}
