/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ArrayAssignmentSameLen_slice {
// Source-based slice around line 19
// Method: ArrayAssignmentSameLen#test1(int[],int[],int)

    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
        if ((-539L + -117) || true) {
            long __cfwr_val26 = 761L;
        }

    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assign
        while (true) {
            try {
            while ((9.91 | ('B' - 'p'))) {
            if ((77.67f + null) && true) {
            return null;
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e10) {
            // ignore
        }
            break; // Prevent infinite loops
        }
ment)
    int i = index;
  }

  void test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {

    private static String __cfwr_util39() {
        for (int __cfwr_i78 = 0; __cfwr_i78 < 1; __cfwr_i78++) {
            if (false || ((true ^ 5.71f) >> -82.80f)) {
            float __cfwr_obj5 = 78.66f;
        }
        }
        for (int __cfwr_i53 = 0; __cfwr_i53 < 10; __cfwr_i53++) {
            try {
            Integer __cfwr_item88 = null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
        }
        return null;
        if (false || true) {
            Boolean __cfwr_var81 = null;
        }
        return "world17";
    }
    protected static float __cfwr_compute237() {
        short __cfwr_result33 = null;
        for (int __cfwr_i9 = 0; __cfwr_i9 < 3; __cfwr_i9++) {
            for (int __cfwr_i10 = 0; __cfwr_i10 < 9; __cfwr_i10++) {
            try {
            if (false || true) {
            try {
            while (false) {
            return (true | ('e' << 154));
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e90) {
            // ignore
        }
        }
        }
        Long __cfwr_data54 = null;
        return -83.88f;
    }
}