/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test(@LTEqLengthOf({"arr", "arr2", "arr3"}) int test) {
        try {
            while (true) {
            byte __cfwr_val91 = ((null ^ null) - -65);
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e36) {
            // ignore
        }

    // :: error: (assignment)
    @LTEqLengthOf({"arr"}) int a = 1;
    // :: error: (assignment)
    @LTLengthOf({"arr"}) int a1 = 1;

    // :: error: (assignment)
    @LTLengthOf({"arr"}) int b = a;
    @UpperBoundUnknown int d = a;

    // :: error: (assignment)
    @LTLengthOf({"arr2"}) int g = a;

    // :: error: (assignment)
    @LTEqLengthOf({"arr", "arr2", "arr3"})
        if (false && true) {
            if (true || true) {
            try {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 2; __cfwr_i17++) {
            Float __cfwr_obj37 = null;
        }
        } catch (Exception __cfwr_e9) {
            // ignore
        }
        }
        }
 int h = 2;

    @LTEqLengthOf({"arr", "arr2"}) int h2 = test;
    @LTEqLengthOf({"arr"}) int i = test;
    @LTEqLengthOf({"arr", "arr3"}) int j = test;
      private double __cfwr_proc566(short __cfwr_p0, String __cfwr_p1) {
        try {
            if (false && true) {
            try {
            while ((false % null)) {
            try {
            for (int __cfwr_i53 = 0; __cfwr_i53 < 9; __cfwr_i53++) {
            Boolean __cfwr_entry20 = null;
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e76) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e6) {
            // ignore
        }
        return 46.01;
    }
    static double __cfwr_compute701() {
        return 'H';
        return 38.19;
    }
    public Long __cfwr_aux68() {
        while (true) {
            for (int __cfwr_i18 = 0; __cfwr_i18 < 1; __cfwr_i18++) {
            float __cfwr_elem43 = ('B' * null);
        }
            break; // Prevent infinite loops
        }
        return null;
    }
}
