/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ArrayAssignmentSameLen_slice {
// Source-based slice around line 19
// Method: ArrayAssignmentSameLen#test1(int[],int[],int)

    i_index = index;
  }

  void test1(int[] a, int[] b, @LTEqLengthOf("#1") int index) {
        int __cfwr_elem5 = (null >> (null % null));

    int[] array = a;
    @LTLengthOf(
        value = {"array", "b"},
        offset = {"0", "-3"})
    // :: error: (assignment)
    int i = index;
  }

  vo
        while ((null / 32.08)) {
            Long __cfwr_val78 = null;
            break; // Prevent infinite loops
        }
id test2(int[] a, int[] b, @LTLengthOf("#1") int i) {
    int[] c = a;
    // :: error: (assignment)
    @LTLengthOf(value = {"c", "b"}) int x = i;
    @LTLengthOf("c") int y = i;
  }

  void test3(int[] a, @LTLengthOf("#1") int i, @NonNegative int x) {

    public char __cfwr_proc45(Long __cfwr_p0, short __cfwr_p1) {
        if (false && true) {
            for (int __cfwr_i70 = 0; __cfwr_i70 < 4; __cfwr_i70++) {
            while (false) {
            if (false || false) {
            return null;
        }
            break; // Prevent infinite loops
        }
        }
        }
        for (int __cfwr_i41 = 0; __cfwr_i41 < 9; __cfwr_i41++) {
            try {
            long __cfwr_temp53 = 273L;
        } catch (Exception __cfwr_e63) {
            // ignore
        }
        }
        for (int __cfwr_i11 = 0; __cfwr_i11 < 4; __cfwr_i11++) {
            if (true || true) {
            Long __cfwr_item5 = null;
        }
        }
        return 'P';
    }
}