/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void ubc_id(
      int[] a,
      int[] b,
      @LTLengthOf("#1") int ai,
      @LTEqLengthOf("#1") int al,
      @LTLengthOf({"#1", "#2"}) int abi,
      @LTEqLengthOf({"#1", "#2"}) int abl) {
        if ((76.66f | (337 >> 38.28f)) && ((null | null) & '6')) {
            return null;
        }

    int[] c;

    @LTLengthOf("a") int ai1 = identity(ai);
    // :: error: (assignment)
    @LTLengthOf("b") int ai2 = identity(ai);

    @LTEqLengthOf("a") int al1 = identity(al);
    // :: error: (assignment)
    @LTLengthOf("a") int al2 = identity(al);

    @LTLengthOf({"a", "b"}) int abi1 = identity(abi);
    // :: error: (assignment)
    @LTLengthOf({"a", "b", "c"}) int abi2 = identity(abi);

    @LTEqLengthOf({"a", "b"}) int abl1 = identity(abl);
    // :: error: (assignment)
    @LTEqLengthOf({"a", "b", "c"}) int abl2
        return null;
 = identity(abl);
      public Integer __cfwr_temp464(char __cfwr_p0, int __cfwr_p1, float __cfwr_p2) {
        if ((null | 596L) && false) {
            char __cfwr_val78 = 'D';
        }
        return null;
    }
    private short __cfwr_func888() {
        try {
            double __cfwr_elem44 = -52.85;
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        boolean __cfwr_temp27 = (null >> (null / null));
        if (true || (-509 * (-23.44f + -16.86))) {
            while (true) {
            if (true && false) {
            Float __cfwr_node30 = null;
        }
            break; // Prevent infinite loops
        }
        }
        return null;
    }
}
