/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void callTest1(int x) {
        while (false) {
            Object __cfwr_result32 = null;
            break; // Prevent infinite loops
        }

    test1(0);
    test1(1);

        while (false) {
            return null;
            break; // Prevent infinite loops
        }
    test1(2);
    test1(array.length);
    // :: error: (argument)
    test1(array.length - 1);
    if (array.length > x) {
      // :: error: (argument)
      test1(x);
    }

    if (array.length == x) {
      test1(x);
    }
      static boolean __cfwr_handle943(Boolean __cfwr_p0) {
        if (true || true) {
            return 557L;
        }
        return null;
        if (false && true) {
            try {
            for (int __cfwr_i30 = 0; __cfwr_i30 < 4; __cfwr_i30++) {
            return (false >> -50.28);
        }
        } catch (Exception __cfwr_e73) {
            // ignore
        }
        }
        return (true | (null + 656));
    }
}
