/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        long __cfwr_item33 = 177L;

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      public long __cfwr_calc397(Integer __cfwr_p0, long __cfwr_p1, int __cfwr_p2) {
        if (false && false) {
            try {
            try {
            return null;
        } catch (Exception __cfwr_e25) {
            // ignore
        }
        } catch (Exception __cfwr_e38) {
            // ignore
        }
        }
        while ((-76.13 - (false % -6.73))) {
            if ((172 & true) || true) {
            try {
            char __cfwr_var70 = 'f';
        } catch (Exception __cfwr_e45) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        if (false || true) {
            try {
            try {
            while (true) {
            Long __cfwr_elem31 = null;
            break; // Prevent infinite loops
        }
        } catch (Exception __cfwr_e79) {
            // ignore
        }
        } catch (Exception __cfwr_e80) {
            // ignore
        }
        }
        return ('8' + (false % 41));
    }
    long __cfwr_proc920(Integer __cfwr_p0) {
        char __cfwr_node69 = (-15.16f - (null << -60L));
        return -923L;
    }
}
