/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    private static Double __cfwr_calc564(long __cfwr_p0, double __cfwr_p1) {
        while ((true * null)) {
            return null;
            break; // Prevent infinite loops
        }

        Boolean __cfwr_entry42 
        while (true) {
            for (int __cfwr_i75 = 0; __cfwr_i75 < 10; __cfwr_i75++) {
            return (null ^ (-30.75 % -12.29f));
        }
            break; // Prevent infinite loops
        }
= null;
        return null;
    }
    protected Character __cfwr_helper371(String __cfwr_p0, Object __cfwr_p1) {
        return "result34";
        return null;
        boolean __cfwr_val48 = ('L' << true);
        try {
            byte __cfwr_obj19 = (null >> (97.54f - null));
        } catch (Exception __cfwr_e51) {
            // ignore
        }
        return null;
    }
