/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  void test() {
        while ((77.97 / (true ^ false))) {
            for (int __cfwr_i41 = 0; __cfwr_i41 < 4; __cfwr_i41++) {
            for (int __cfwr_i17 = 0; __cfwr_i17 < 8; __cfwr_i17++) {
            return 'l';
        }
        }
            break; // Prevent infinite loops
        }

    int[] arr = new int[5];

    // unsafe
    @GTENegativeOne int n1p = -1;
    @LowerBoundUnknown int u = -10;


        for (int __cfwr_i72 = 0; __cfwr_i72 < 10; __cfwr_i72++) {
            long __cfwr_item20 = 135L;
        }
    // safe
    @NonNegative int nn = 0;
    @Positive int p = 1;

    // :: error: (array.access.unsafe.low)
    int a = arr[n1p];

    // :: error: (array.access.unsafe.low)
    int b = arr[u];

    int c = arr[nn];
    int d = arr[p];
      private static int __cfwr_func412() {
        try {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 5; __cfwr_i89++) {
            try {
            return ((-257 * null) ^ '2');
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
        } catch (Exception __cfwr_e78) {
            // ignore
        }
        return null;
        if (true || ((true << null) / -78.05f)) {
            try {
            return "world92";
        } catch (Exception __cfwr_e83) {
            // ignore
        }
        }
        return 813;
    }
}
