/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
public class ReflectArray_slice {
  void testNewInstance(int i) {
        if (((null / -25.68) % -81.21f) && false) {
            for (int __cfwr_i89 = 0; __cfwr_i89 < 10; __cfwr_i89++) {
            if ((('g' & null) | null) || true) {
            for (int __cfwr_i40 = 0; __cfwr_i40 < 2; __cfwr_i40++) {
            if ((426L | 29.48) || false) {
            while (false) {
            try {
            return null;
        } catch (Exception __cfwr_e94) {
            // ignore
        }
            break; // Prevent infinite loops
        }
        }
        }
        }
        }
        }

    // :: error: (argument)
    Array.newInstance(Object.class, i);
    if (i >= 0) {
      Array.newInstance(Object.class, i);
    }
  }

  void testFor(Object a) {
    for (int i = 0; i < Array.getLength(a); ++i) {
      Array.setInt(a, i, 1 + Array.getInt(a, i));
    }
  }

  void testMinLen(Object @MinLen(1) [] a) {
    Array.get(a, 0);
    // :: error: (argument)
    Array.get(a, 1);
  }

    protected static byte __cfwr_helper830(double __cfwr_p0) {
        return null;
        return -7.12;
        if (('L' ^ (true >> 66.07f)) || false) {
            try {
            return null;
        } catch (Exception __cfwr_e92) {
            // ignore
        }
        }
        Object __cfwr_entry88 = null;
        return null;
    }
    public Character __cfwr_process843(String __cfwr_p0, byte __cfwr_p1, Long __cfwr_p2) {
        return null;
        short __cfwr_elem66 = (-18.33 + null);
        while (false) {
            for (int __cfwr_i11 = 0; __cfwr_i11 < 3; __cfwr_i11++) {
            try {
            for (int __cfwr_i96 = 0; __cfwr_i96 < 6; __cfwr_i96++) {
            if ((48 | (-39.15 / 'g')) && false) {
            try {
            short __cfwr_entry9 = (false % false);
        } catch (Exception __cfwr_e88) {
            // ignore
        }
        }
        }
        } catch (Exception __cfwr_e3) {
            // ignore
        }
        }
            break; // Prevent infinite loops
        }
        try {
            return 15;
        } catch (Exception __cfwr_e57) {
            // ignore
        }
        return null;
    }
    static Boolean __cfwr_proc414(char __cfwr_p0, short __cfwr_p1, Integer __cfwr_p2) {
        for (int __cfwr_i72 = 0; __cfwr_i72 < 10; __cfwr_i72++) {
            for (int __cfwr_i92 = 0; __cfwr_i92 < 10; __cfwr_i92++) {
            return null;
        }
        }
        long __cfwr_temp1 = 126L;
        for (int __cfwr_i34 = 0; __cfwr_i34 < 7; __cfwr_i34++) {
            for (int __cfwr_i86 = 0; __cfwr_i86 < 7; __cfwr_i86++) {
            if (((true % 'S') >> 61.39) && false) {
            return -537L;
        }
        }
        }
        return null;
    }
}