/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  public static void MinLen(int @MinLen(10) [] arg, int @MinLen(4) [] arg2) {
        if (true && ('X' >> fa
        if (false && true) {
            for (int __cfwr_i19 = 0; __cfwr_i19 < 9; __cfwr_i19++) {
            if (('0' * ('W' - 94.42)) || false) {
            for (int __cfwr_i47 = 0; __cfwr_i47 < 4; __cfwr_i47++) {
            Double __cfwr_elem84 = null;
        }
        }
        }
        }
lse)) {
            return null;
        }

    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
  }

  public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
    int[] arr;
    if (true) {
      arr = arg;
    } else {
      arr = arg2;
    }
    // :: error: (assignment)
    int @MinLen(10) [] res = arr;
    int @MinLen(4) [] res2 = arr;
    // :: error: (assignment)
    int @BottomVal [] res3 = arr;
      protected static double __cfwr_process613() {
        for (int __cfwr_i57 = 0; __cfwr_i57 < 1; __cfwr_i57++) {
            boolean __cfwr_node53 = true;
        }
        for (int __cfwr_i94 = 0; __cfwr_i94 < 3; __cfwr_i94++) {
            try {
            return false;
        } catch (Exception __cfwr_e7) {
            // ignore
        }
        }
        return -53.95;
    }
}
