/*
 * CFWR augmentation: inserted irrelevant code for data augmentation.
 */
  @LessThan("bigger") double d;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") boolean bool;

  @LessThan("bigger") char c;

  @LessThan("bigger") Byte bBoxed;

  @LessThan("bigger") Short sBoxed;

  @LessThan("bigger") Integer iBoxed;

  @LessThan("bigger") Long lBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Float fBoxed;

  // :: error: (anno.on.irrelevant)
  @LessThan("bigger") Double dBoxed;

    int __cfwr_helper595() {
        while (false) 
        for (int __cfwr_i62 = 0; __cfwr_i62 < 7; __cfwr_i62++) {
            Long __cfwr_item2 = null;
        }
{
            return null;
            break; // Prevent infinite loops
        }

        while (true) {
            while (true) {
            return (-170L % -62.19f);
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        try {
            try {
            for (int __cfwr_i56 = 0; __cfwr_i56 < 9; __cfwr_i56++) {
            for (int __cfwr_i79 = 0; __cfwr_i79 < 10; __cfwr_i79++) {
            while (false) {
            Character __cfwr_val55 = null;
            break; // Prevent infinite loops
        }
        }
        }
        } catch (Exception __cfwr_e64) {
            // ignore
        }
        } catch (Exception __cfwr_e35) {
            // ignore
        }
        while ((-93.72f % (826L % null))) {
            if (false && false) {
            if (false || ('Z' - null)) {
            Integer __cfwr_result48 = null;
        }
        }
            break; // Prevent infinite loops
        }
        return -751;
    }
    public Character __cfwr_proc69(char __cfwr_p0, Character __cfwr_p1) {
        if (false && false) {
            if (false && (15.25 * 60.20f)) {
            if ((683 >> (941 * false)) || false) {
            Long __cfwr_result15 = null;
        }
        }
        }
        while (false) {
            while ((null & -19.63f)) {
            return null;
            break; // Prevent infinite loops
        }
            break; // Prevent infinite loops
        }
        return null;
        return null;
    }
